<!DOCTYPE html>
<html>
	<head>
		<META http-equiv="Content-Style-Type" content="text/css">
		<link rel="stylesheet" type="text/css" href="style.css">
	</head>
<body>
	<h1>About the Yeast Interactome Database </h1>
<br>format-version: 1.2
<br>date: 15:05:2009 14:41
<br>saved-by: luisa
<br>auto-generated-by: OBO-Edit 1.101
<br>subsetdef: Drugable "Drugable Genome Project"
<br>subsetdef: PSI-MI slim "Subset of PSI-MI"
<br>synonymtypedef: PSI-MI-alternate "Alternate label curated by PSI-MI" EXACT
<br>synonymtypedef: PSI-MI-short "Unique short label curated by PSI-MI" EXACT
<br>default-namespace: PSI-MI
<br>remark: Notes:
<br>remark: Each of the top level terms in this file is the root term of an independent controlled vocabulary
<br>remark: mapping an element of the PSI Molecular Interaction XML schema.
<br>remark: The correct use of these vocabularies in the PSI Molecular Interaction XML schema is
<br>remark: formalized in a mapping file available at http://www.psidev.info/files/validator/xml/MI-CVMapping.xml.
<br>remark: The PSI MI schema defines short labels for controlled vocabulary terms
<br>remark: short labels are reported as PSI-MI-short synonyms that are created when a term is more than 20 characteres long.
<br>remark: The last accession number used in this file is stored in a separate file,
<br>remark: psi-mi.lastac. It MUST be updated when this file is updated.
<br>remark: The maintenance of this file is ensured by Sandra Orchard orchard@ebi.ac.uk and Luisa Montecchi Palazzi luisa@ebi.ac.uk
<br>remark: coverage: This file collect controlled vocabularies describing different aspects of molecular interactions.
<br>remark: publisher: This file is published by the PSI MI working group see http://psidev.info/MI
<br>remark: CVversion: 2.5.4
<p><br>[Term]
<br>id: MI:0000
<br>name: molecular interaction
<br>def: "Controlled vocabularies originally created for protein protein interactions, extended to other molecules interactions." [PMID:14755292]
<br>subset: Drugable
<br>subset: PSI-MI slim
<br>synonym: "mi" EXACT PSI-MI-short []
<p><br>[Term]
<br>id: MI:0001
<br>name: interaction detection method
<br>def: "Method to determine the interaction." [PMID:14755292]
<br>subset: Drugable
<br>subset: PSI-MI slim
<br>synonym: "interaction detect" EXACT PSI-MI-short []
<br>relationship: part_of MI:0000 ! molecular interaction
<p><br>[Term]
<br>id: MI:0002
<br>name: participant identification method
<br>def: "Method to determine the molecules involved in the interaction." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "participant detection" EXACT PSI-MI-alternate []
<br>synonym: "participant ident" EXACT PSI-MI-short []
<br>relationship: part_of MI:0000 ! molecular interaction
<p><br>[Term]
<br>id: MI:0003
<br>name: feature detection method
<br>def: "Method to determine the features of the proteins involved in the interaction." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "feature detection" EXACT PSI-MI-short []
<br>relationship: part_of MI:0000 ! molecular interaction
<p><br>[Term]
<br>id: MI:0004
<br>name: affinity chromatography technology
<br>def: "This class of approaches is characterised by the use of affinity resins as tools to purify molecule of interest (baits) and their binding partners. The baits can be captured by a variety of high affinity ligands linked to a resin - for example, antibodies specific for the bait itself, antibodies for specific tags engineered to be expressed as part of the bait or other high affinity binders such as glutathione resins for GST fusion proteins, metal resins for histidine-tagged proteins." [PMID:7708014]
<br>subset: PSI-MI slim
<br>synonym: "affinity chrom" EXACT PSI-MI-short []
<br>synonym: "Affinity purification" EXACT PSI-MI-alternate []
<br>is_a: MI:0091 ! chromatography technology
<br>is_a: MI:0400 ! affinity technology
<p><br>[Term]
<br>id: MI:0005
<br>name: alanine scanning
<br>def: "This approach is used to identify the residues that are involved in an interaction. Several variants of the native protein are prepared by sequentially  mutating each residue of interest to an alanine. The mutated proteins are expressed and probed in the binding assay." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0810 ! substitution analysis
<p><br>[Term]
<br>id: MI:0006
<br>name: anti bait coimmunoprecipitation
<br>def: "A specific antibody for the molecule of interest (bait) is available, this is used to generate a high affinity resin to capture the endogenous bait present in a sample." [PMID:7708014]
<br>subset: PSI-MI slim
<br>synonym: "anti bait coip" EXACT PSI-MI-short []
<br>is_a: MI:0019 ! coimmunoprecipitation
<p><br>[Term]
<br>id: MI:0007
<br>name: anti tag coimmunoprecipitation
<br>def: "A specific antibody for the molecule of interest is not available, therefore the bait protein is expressed as a hybrid protein fused to a tag peptide/protein for which efficient and specific antibodies or a specific ligand are available." [PMID:7708014]
<br>subset: PSI-MI slim
<br>synonym: "anti tag coip" EXACT PSI-MI-short []
<br>is_a: MI:0019 ! coimmunoprecipitation
<p><br>[Term]
<br>id: MI:0008
<br>name: array technology
<br>def: "In this class of methodologies, the molecules to be tested are presented ordered in an array format (typically at high density) on planar supports. The characteristics and chemical nature of the planar support can vary. This format permits the simultaneous assay, in controlled conditions, of several thousand proteins/peptides/nucleic acids for different functions, for instance their ability to bind any given molecule." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0400 ! affinity technology
<p><br>[Term]
<br>id: MI:0009
<br>name: bacterial display
<br>def: "The protein of interest is presented on the outer membrane of Gram negative bacteria by expressing it as a fusion partner to peptide signals that direct heterologous proteins to the cell surface. For instance, a single chain Fv (scFv) antibody fragment, consisting of the variable heavy and variable light domains from two separate anti-digoxin monoclonal antibodies, was displayed on the outer membrane of Escherichia coli by fusing it to an Lpp-OmpA. Similar systems have also been developed for gram positive bacteria. Fluorescence-activated cell sorting (FACS), is used to specifically select clones displaying a protein binding to scFv-producing cells." [PMID:10436088, PMID:8248129]
<br>subset: PSI-MI slim
<br>is_a: MI:0034 ! display technology
<br>is_a: MI:0054 ! fluorescence-activated cell sorting
<p><br>[Term]
<br>id: MI:0010
<br>name: beta galactosidase complementation
<br>def: "Beta-galactosidase activity can be used to monitor the interaction of chimeric proteins. Pairs of inactive beta gal deletion mutants are capable of complementing to restore activity when fused to interacting protein partners. Critical to the success of this system is the choice of two poorly complementing mutant moieties, since strongly complementing mutants spontaneously assemble and produce functional beta-gal activity detectable in absence of any fused protein fragment." [PMID:12042868, PMID:9237989]
<br>subset: PSI-MI slim
<br>synonym: "beta galactosidase" EXACT PSI-MI-short []
<br>is_a: MI:0228 ! cytoplasmic complementation assay
<p><br>[Term]
<br>id: MI:0011
<br>name: beta lactamase complementation
<br>def: "This strategy is based on a protein fragment complementation assay (PCA) of the enzyme TEM-1 beta-lactamase. The approach includes a simple colorimetric in vitro assays using the cephalosporin nitrocefin and assays in intact cells using the fluorescent substrate CCF2/AM. The combination of in vitro colorimetric and in vivo fluorescence assays of beta-lactamase in mammalian cells permits a variety of sensitive and high-throughput large-scale applications." [PMID:12042868]
<br>subset: PSI-MI slim
<br>synonym: "beta lactamase" EXACT PSI-MI-short []
<br>is_a: MI:0228 ! cytoplasmic complementation assay
<p><br>[Term]
<br>id: MI:0012
<br>name: bioluminescence resonance energy transfer
<br>def: "In this variation of the FRET assay the donor fluorophore is replaced by a luciferase (typically Renilla luciferase). In the presence of its substrate, the luciferase catalyses a bioluminescent reaction that excites the acceptor fluorophore through a resonance energy transfer mechanism. As with FRET the energy transfer occurs only if the protein fused to the luciferase and the one fused to the acceptor fluorophore are in close proximity (10-100 Angstrom)." [PMID:9874787, PMID for application instance:10725388]
<br>subset: PSI-MI slim
<br>synonym: "BRET" EXACT PSI-MI-alternate []
<br>synonym: "bret" EXACT PSI-MI-short []
<br>synonym: "LRET" EXACT PSI-MI-alternate []
<br>is_a: MI:0051 ! fluorescence technology
<p><br>[Term]
<br>id: MI:0013
<br>name: biophysical
<br>def: "The application of physical principles and methods to biological experiments." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0045 ! experimental interaction detection
<p><br>[Term]
<br>id: MI:0014
<br>name: adenylate cyclase complementation
<br>def: "Adenylate cyclase is encoded by the cyaA gene and contains a catalytic domain which can be proteolytically cleaved into two complementary fragments, T25 and T18, which remain associated in the presence of calmodulin in a fully active ternary complex. In the absence of calmodulin, the mixture of the two fragments does not exhibit detectable activity, suggesting that the two fragments do not associate. When expressed in an adenylate cyclase-deficient E. coli strain (E. coli lacks calmodulin or calmodulin-related proteins), the T25 and T18 fragments fused to putative interacting proteins are brought into close association which result in cAMP synthesis. The level of reconstructed adenylate cyclase can be estimated by monitoring the expression of a cAMP dependent reporter gene." [PMID:9576956]
<br>subset: PSI-MI slim
<br>synonym: "adenylate cyclase" EXACT PSI-MI-short []
<br>synonym: "bacterial two-hybrid" EXACT PSI-MI-alternate []
<br>is_a: MI:0228 ! cytoplasmic complementation assay
<p><br>[Term]
<br>id: MI:0016
<br>name: circular dichroism
<br>def: "Circular dichroism (CD) is observed when optically active molecules absorb left and right hand circularly polarized light slightly differently. Linearly polarized light can be viewed as a superposition of two components of circularly polarized light of equal amplitude and phase but opposite handness. When this light passes through an optically active sample the two polarized components are absorbed differently. The difference in left and right handed absorbance A(l)- A(r) is the signal registered in CD spectra. This signal displays distinct features corresponding to different secondary structures present in peptides, proteins and nucleic acids. The analysis of CD spectra can therefore yield valuable information about the secondary structure of biological macromolecules and the interactions among molecules that influence their structure." [PMID:11578931]
<br>subset: PSI-MI slim
<br>synonym: "CD" EXACT PSI-MI-alternate []
<br>synonym: "cd" EXACT PSI-MI-short []
<br>is_a: MI:0013 ! biophysical
<p><br>[Term]
<br>id: MI:0017
<br>name: classical fluorescence spectroscopy
<br>def: "Proteins contain endogenous fluorophores such as tryptophan residue and heme or flavins groups. Protein folding and protein-protein interaction can be studied by monitoring changes in the tryptophan environment detected by changes in its intrinsic fluorescence. Changes in the fluorescence emission spectrum on complex formation can occur either due to a shift in the wavelength of maximum fluorescence emission or by a shift in fluorescence intensity caused by the mixing of two proteins. The interaction of two proteins causes a shift in the fluorescence emission spectrum relative to the sum of the individual fluorescence spectra, resulting in a difference spectrum [F (complex)-2 F (sum)], which is a measurable effect of the interaction. Loss of fluorescence signal from a substrate can be used to measure protein cleavage." [PMID:7708014]
<br>subset: PSI-MI slim
<br>synonym: "fluorescence spectr" EXACT PSI-MI-short []
<br>is_a: MI:0051 ! fluorescence technology
<p><br>[Term]
<br>id: MI:0018
<br>name: two hybrid
<br>def: "The classical two-hybrid system is a method that uses transcriptional activity as a measure of protein-protein interaction. It relies on the modular nature of many site-specific transcriptional activators (GAL 4) , which consist of a DNA-binding domain and a transcriptional activation domain. The DNA-binding domain serves to target the activator to the specific genes that will be expressed, and the activation domain contacts other proteins of the transcriptional machinery to enable transcription to occur. The two-hybrid system is based on the observation that the two domains of the activator need to be non-covalently brought together by the interaction of any two proteins. The application of this system requires the expression of two hybrid. Generally this assay is performed in yeast cell, but it can also be carried out in other organism." [PMID:10967325, PMID:12634794, PMID:1946372]
<br>subset: PSI-MI slim
<br>synonym: "2 hybrid" EXACT PSI-MI-short []
<br>synonym: "2-hybrid" EXACT PSI-MI-alternate []
<br>synonym: "2H" EXACT PSI-MI-alternate []
<br>synonym: "2h" EXACT PSI-MI-alternate []
<br>synonym: "classical two hybrid" EXACT PSI-MI-alternate []
<br>synonym: "Gal4 transcription regeneration" EXACT PSI-MI-alternate []
<br>synonym: "two-hybrid" EXACT PSI-MI-alternate []
<br>synonym: "yeast two hybrid" EXACT PSI-MI-alternate []
<br>is_a: MI:0232 ! transcriptional complementation assay
<p><br>[Term]
<br>id: MI:0019
<br>name: coimmunoprecipitation
<br>def: "In this approach an antibody, specific for the molecule of interest (bait) or any tag expressed within a fusion protein, is used to separate the bait from a protein mixture or a cell lysate and to capture its ligand simultaneously. The partners that bind to the bait molecule retained by the resin can then be eluted and identified. The antibody may be free or bound to a matrix during this process." [PMID:7708014]
<br>subset: PSI-MI slim
<br>synonym: "co-immunoprecipitation" EXACT PSI-MI-alternate []
<br>synonym: "Co-IP" EXACT PSI-MI-alternate []
<br>synonym: "CoIp" EXACT PSI-MI-alternate []
<br>synonym: "coip" EXACT PSI-MI-short []
<br>synonym: "immunoprecipitation" EXACT PSI-MI-alternate []
<br>is_a: MI:0004 ! affinity chromatography technology
<p><br>[Term]
<br>id: MI:0020
<br>name: transmission electron microscopy
<br>def: "During the treatment for microscope analysis a tissue section is incubated with high-specificity antibodies coupled to heavy metals (gold). Any tissue section can then be analysed by electron microscopy to localise the target proteins within the cell. This method supports very high resolution colocalisation of different molecules in a cell." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "tem" EXACT PSI-MI-short []
<br>is_a: MI:0040 ! electron microscopy
<p><br>[Term]
<br>id: MI:0021
<br>name: colocalization by fluorescent probes cloning
<br>def: "Two proteins can be localised to cell compartments, in the same experiment, if they are expressed as chimeric proteins fused to distinct proteins fluorescing at different wavelengths (Green Fluorescent Protein and Red Fluorescent Protein for example). Using a confocal microscope the two proteins can be visualized in living cells and it can be determined whether they have the same subcellular location. Fluorescence microscopy of cells expressing a GFP fusion protein can also demonstrate dynamic processes such as its translocation from one subcellular compartment to another.\nOBSOLETE: use imaging technique (MI:0428) and specific probe as feature of each interacting protein." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "coloc fluoresc probe" EXACT PSI-MI-short []
<br>is_obsolete: true
<p><br>[Term]
<br>id: MI:0022
<br>name: colocalization by immunostaining
<br>def: "The subcellular location of a protein can be demonstrated by treating cells fixed on a microscope slide with an antibody specific for the protein of interest. A secondary antibody conjugated with a reactive enzyme (e.g. horseradish peroxidase) is then added. Following a washing step to remove the unbound secondary ligand, a chromogenic substrate (e.g. 3,3', 5,5' tetramethyl benzidine chromogen [TMB]) is converted to a soluble coloured product by the conjugated enzyme and can then be visualised by standard microscopic techniques.\nOBSOLETE since combination of Interaction Detection Method and Interaction Type. Consider using the Interaction Detection Method imaging techniques (MI:0428) coupled with Interaction Type colocalisation (MI:0401) and Participant detection immunostaining (MI:0402) instead." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "coloc immunostaining" EXACT PSI-MI-short []
<br>synonym: "Immunofluorescence Staining" EXACT PSI-MI-alternate []
<br>synonym: "Immunostaining" EXACT PSI-MI-alternate []
<br>is_obsolete: true
<p><br>[Term]
<br>id: MI:0023
<br>name: colocalization/visualisation technologies
<br>def: "Techniques enabling the identification of the subcellular localisation of a protein or complex. Two different proteins show a similar distribution in the cell are said to co-localise. Obsolete since combination of Interaction Detection Method and Interaction Type.\nOBSOLETE. Consider using imaging techniques (MI:0428) as interaction detection method coupled with colocalisation (MI:0401) as interaction type and predetermined (MI:0396) as participant detection." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "coloc visual technol" EXACT PSI-MI-short []
<br>is_obsolete: true
<p><br>[Term]
<br>id: MI:0024
<br>name: confirmational text mining
<br>def: "Text mining is used to support interactions which have been determined by other methods." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "conformational tm" EXACT PSI-MI-short []
<br>is_a: MI:0110 ! text mining
<p><br>[Term]
<br>id: MI:0025
<br>name: copurification
<br>def: "Approaches designed to separate cell components on the basis of their physicochemical properties. The observation that two or more proteins copurify in one or several conditions is taken as an indication that they form a molecular complex.\nOBSOLETE since too non-specific. Consider use of cosedimentation (MI:0027) or comigration in non denaturing gel electrophoresis (MI:0404) or affinity chromatography technologies (MI:0004) or molecular sieving (MI:0071) or for unspecific cases biochemical (MI:0401)." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_obsolete: true
<p><br>[Term]
<br>id: MI:0026
<br>name: correlated mutations
<br>def: "Pairs of multiple alignments of orthologous sequences are used to identify potential interacting partners as proteins that show covariation of their residue identities between different species. Proteins displaying inter-protein correlated mutations during evolution are likely to be interacting proteins due to co-adapted evolution of their protein interacting interfaces." [PMID:11933068]
<br>subset: PSI-MI slim
<br>is_a: MI:0101 ! sequence based prediction
<br>is_a: MI:0660 ! feature prediction
<p><br>[Term]
<br>id: MI:0027
<br>name: cosedimentation
<br>def: "Separation of a protein mixture under the influence of artificial gravity." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0401 ! biochemical
<p><br>[Term]
<br>id: MI:0028
<br>name: cosedimentation in solution
<br>def: "The ultracentrifuge can be used to characterise and/or purify macromolecules in solution according to their mass and hydrodynamic properties. Sedimentation studies provide information about the molecular weight and shape of a molecule. It is also possible to measure the association state of the sample. Both the mass of a molecule and its shape, that influences the friction forces and diffusion that counterbalances gravity, determine the sedimentation speed." [PMID:10410796]
<br>subset: PSI-MI slim
<br>synonym: "solution sedimentati" EXACT PSI-MI-short []
<br>is_a: MI:0027 ! cosedimentation
<p><br>[Term]
<br>id: MI:0029
<br>name: cosedimentation through density gradient
<br>def: "Sedimentation through a density gradient measures the sedimentation rate of a mixture of proteins through either a glycerol or sucrose gradient. Two interacting proteins will sediment mostly as a complex at concentrations above the binding constant. By varying the concentration of one or both of the complex constituents and taking into account the dilution of the species during sedimentation, one can reasonably accurately estimate the binding constant." [PMID:10410796]
<br>subset: PSI-MI slim
<br>synonym: "density sedimentatio" EXACT PSI-MI-short []
<br>is_a: MI:0027 ! cosedimentation
<p><br>[Term]
<br>id: MI:0030
<br>name: cross-linking study
<br>def: "Analysis of complexes obtained by chemical treatments that promote the formation of covalent bonds among molecules in close proximity." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "crosslink" EXACT PSI-MI-short []
<br>is_a: MI:0401 ! biochemical
<p><br>[Term]
<br>id: MI:0031
<br>name: protein cross-linking with a bifunctional reagent
<br>def: "A cross-linker is a bifunctional molecule having two reactive ends linked by a spacer, often containing a disulfide bond. Cross-linkers induce the formation of covalent bonds among proteins that are neighbours. When a reducing agent is added the disulfide bridge is cleaved, the cross-linked pairs are released and can be identified. There are various classes of cross-linkers, the most common are those having photoreactive groups that become reactive fluorophores when activated by UV light thereby resulting in photolabeling the cross-linked moieties." [PMID:10679368, PMID:7708014]
<br>subset: PSI-MI slim
<br>synonym: "Label transfer techniques" EXACT PSI-MI-alternate []
<br>synonym: "Photoaffinity labelling" EXACT PSI-MI-alternate []
<br>synonym: "protein crosslink" EXACT PSI-MI-short []
<br>is_a: MI:0030 ! cross-linking study
<p><br>[Term]
<br>id: MI:0032
<br>name: de novo protein sequencing by mass spectrometry
<br>def: "The strategy to determine the complete amino acid sequence of a protein by mass spectrometry relies on the generation of a nested set of fragments differing by one amino acid. This reveals the identity of the residue that has been removed at each degradation step by measuring the mass difference of fragments differing of one residue. Peptide fragments can be obtained by protease treatment combined with the fragmentation promoted by collision (or other methods) within a tandem mass spectrometer. This approach can be carried out with LC MS/MS (Liquid Chromatography Tandem Mass Spectrometry), nanoESI MS/MS (nanoElectrospray Ionisation tandem mass spectrometry), or FTMS (Fourier Transform mass spectrometry) instruments." [PMID:10984529]
<br>subset: PSI-MI slim
<br>synonym: "ms protein sequence" EXACT PSI-MI-short []
<br>synonym: "MS/MS" RELATED []
<br>is_a: MI:0093 ! protein sequence identification
<br>is_a: MI:0427 ! Identification by mass spectrometry
<br>is_a: MI:0659 ! experimental feature detection
<p><br>[Term]
<br>id: MI:0033
<br>name: deletion analysis
<br>def: "In this approach, once a molecule is demonstrated to participate in an interaction, several deletion derivatives are produced and tested in the binding assay to identify the minimal fragment (domain) that can still support the interaction." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0074 ! mutation analysis
<p><br>[Term]
<br>id: MI:0034
<br>name: display technology
<br>def: "All the methods that permit the physical linking of a protein/peptide to its coding sequence. As a consequence affinity purification of the displayed peptide results in the genetic enrichment of its coding sequence. By these technologies genes encoding a peptide with desired binding properties can be selected over an excess of up to 1012 unrelated molecules." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0400 ! affinity technology
<p><br>[Term]
<br>id: MI:0035
<br>name: docking
<br>def: "Predicts the structure of a molecular complex from the unbound structures of its components. The initial approach in the majority of docking procedures is based largely on the 'rigid-body' assumption, whereby the proteins are treated as solid objects. Initial scoring of a complex is based on geometric fit or surface complementarity. This generally requires some knowledge of the binding site to limit the number of solutions." [PMID:9631301, PMID for application instance:11478868]
<br>subset: PSI-MI slim
<br>is_a: MI:0105 ! structure based prediction
<br>is_a: MI:0577 ! feature prediction from structure
<p><br>[Term]
<br>id: MI:0036
<br>name: domain fusion
<br>def: "The rosetta stone, or domain fusion procedure, is based on the assumption that proteins whose homologues in other organisms happen to be fused into a single protein chain are likely to interact or to be functionally related." [PMID:10573422]
<br>subset: PSI-MI slim
<br>synonym: "Rosetta Stone" EXACT PSI-MI-alternate []
<br>is_a: MI:0058 ! genome based prediction
<br>is_a: MI:0101 ! sequence based prediction
<p><br>[Term]
<br>id: MI:0037
<br>name: domain profile pairs
<br>def: "This approach uses a protein interaction network of a given organism to infer interaction in another organism using information about the interacting region. The regions or domains involved in interactions are clustered if they share sequence similarity and have common interacting partners. The resulting domain profiles are then used to screen the proteome of another organism and domain-domain interactions are inferred. Ultimately, an inferred protein interaction map is built in this second organism." [PMID:11473021]
<br>subset: PSI-MI slim
<br>is_a: MI:0046 ! experimental knowledge based
<br>is_a: MI:0101 ! sequence based prediction
<br>is_a: MI:0660 ! feature prediction
<p><br>[Term]
<br>id: MI:0038
<br>name: dynamic light scattering
<br>def: "In dynamic light scattering, particle diffusion in solution gives rise to fluctuations in the intensity of the scattered light on the microsecond scale. The hydrodynamic radius of the particles can be easily calculated." [PMID:9013660]
<br>subset: PSI-MI slim
<br>synonym: "dls" EXACT PSI-MI-short []
<br>is_a: MI:0067 ! light scattering
<p><br>[Term]
<br>id: MI:0039
<br>name: edman degradation
<br>def: "In this procedure the N-terminus amino acid is cleaved from a polypeptide and identified by high-pressure liquid chromatography. The cycle is repeated on the ever-shortening polypeptide until all the residues are identified. On average only 20-30 consecutive cycles can be performed and lead to amino acid identification. Longer polypeptides or full length proteins must be cleaved by specific protease before Edman degradation and their sequences built by fragment overlapping." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0433 ! partial identification of protein sequence
<p><br>[Term]
<br>id: MI:0040
<br>name: electron microscopy
<br>def: "Electron microscopy methods provide insights into the structure of biological macromolecules and their supramolecular assemblies. Resolution is on average around 10 Angstroms but can reach the atomic level when the samples analysed are 2D crystals. Different types of samples can be analysed by electron microscopy: crystals, single particles like viruses, macromolecular complexes or entire cells and tissue sections. Samples can be chemically fixed or vitrified by rapid freezing in liquid ethane, and then transferred into the electron microscope. Data collection consists of the recording of electron diffraction data (2D crystals) and images. Depending on the type of sample, different approaches are used to analyse and merge images and electron diffraction data." [PMID:11785754]
<br>subset: PSI-MI slim
<br>synonym: "Electron cryomicroscopy" EXACT PSI-MI-alternate []
<br>synonym: "Electron crystallography" EXACT PSI-MI-alternate []
<br>is_a: MI:0428 ! imaging technique
<p><br>[Term]
<br>id: MI:0041
<br>name: electron nuclear double resonance
<br>def: "A combination of NMR and EPR. The lines in the EPR spectrum that are caused by coupling of an unpaired electron nearby nuclei change in intensity when these nuclei are excited at their NMR frequency." [PMID:11817959, PMID:11988476, PMID for application instance:12186859]
<br>subset: PSI-MI slim
<br>synonym: "ENDOR" EXACT PSI-MI-alternate []
<br>synonym: "endor" EXACT PSI-MI-short []
<br>is_a: MI:0043 ! electron resonance
<p><br>[Term]
<br>id: MI:0042
<br>name: electron paramagnetic resonance
<br>def: "EPR (also called ESR, Electron Spin Resonance) spectroscopy is analogous to NMR, but is based on the excitation of unpaired electrons instead of nuclei. Unpaired (single) electrons are only found in radicals and some metal ions (paramagnetic species); the EPR spectrum provides information about the environment and mobility of the paramagnetic species. The magnetic interaction of two paramagnetic centres in a protein can be used to calculate the distance between them; this allows studies of the movements and interactions of protein segments. In proteins without any intrinsic unpaired electrons it is possible to attach a radical probe (spin label). Stable nitroxide radicals can be bound to amino acid residues, in analogy with fluorescent probes. In combination with site directed mutagenesis this method is used in particular to study structure and assembly of membrane proteins, by measuring with EPR whether an amino acid is in a polar or non polar environment." [PMID:11817959]
<br>subset: PSI-MI slim
<br>synonym: "EPR" EXACT PSI-MI-alternate []
<br>synonym: "epr" EXACT PSI-MI-short []
<br>synonym: "ESR" EXACT PSI-MI-alternate []
<br>is_a: MI:0043 ! electron resonance
<p><br>[Term]
<br>id: MI:0043
<br>name: electron resonance
<br>def: "A form of spectroscopy in which the absorption of microwave by a sample in a strong magnetic field is used to study atoms or molecules with unpaired electrons." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0013 ! biophysical
<br>is_a: MI:0659 ! experimental feature detection
<p><br>[Term]
<br>id: MI:0045
<br>name: experimental interaction detection
<br>def: "Methods based on laboratory experiments to determine an interaction." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "experimental interac" EXACT PSI-MI-short []
<br>is_a: MI:0001 ! interaction detection method
<p><br>[Term]
<br>id: MI:0046
<br>name: experimental knowledge based
<br>def: "Predictive algorithms that rely on the information obtained by experimental results." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "experimental info" EXACT PSI-MI-short []
<br>is_a: MI:0063 ! interaction prediction
<p><br>[Term]
<br>id: MI:0047
<br>name: far western blotting
<br>def: "Proteins are fractionated by PAGE (SDS-polyacrylamide gel electrophoresis), transferred to a nitrocellulose membrane and tested for the ability to bind to a protein, a peptide, or any other ligand. Cell lysates can also be fractionated before gel electrophoresis to increase the sensitivity of the method for detecting interactions with rare proteins. Denaturants are removed during the blotting procedure, which allows many proteins to recover (or partially recover) activity. However, if biological activity is not recoverable, the proteins can be fractionated by a non denaturing gel system. This variation of the method eliminates the problem of activity regeneration and allows the detection of binding when the presence of a protein complex is required for binding. The protein probe can be prepared by any one of several procedures, while fusion affinity tags greatly facilitate purification. Synthesis in E. coli with a GST fusion, epitope tag, or other affinity tag is most commonly used. The protein of interest can then be radioactively labelled, biotinylated, or used in the blotting procedure as an unlabeled probe that is detected by a specific antibody." [PMID:7708014]
<br>subset: PSI-MI slim
<br>synonym: "Affinity blotting" EXACT PSI-MI-alternate []
<br>is_a: MI:0892 ! solid phase assay
<p><br>[Term]
<br>id: MI:0048
<br>name: filamentous phage display
<br>def: "Filamentous phages (M13, f1, fd) have been extensively used to develop and implement the technology of phage display. Repertoires of relatively short peptides of random amino acid sequences or cDNA libraries have been constructed and searched successfully. Most experiments have taken advantage of the ability to assemble phages decorated with hybrid versions of the receptor protein pIII or of the major coat protein pVIII. Both systems allow the display of foreign peptides by fusion to the amino-terminus of the capsid protein but differ in the number of peptide copies that can be displayed on each phage particle. Display libraries of very diverse protein fragments have been constructed by fusing either genomic or cDNA fragments to gene III or gene VIII." [PMID:7682645]
<br>subset: PSI-MI slim
<br>synonym: "filamentous phage" EXACT PSI-MI-short []
<br>is_a: MI:0084 ! phage display
<p><br>[Term]
<br>id: MI:0049
<br>name: filter binding
<br>def: "A method in which separation depends upon the ability of one participant to bind to a filter or membrane which the other participants do not. Molecules interacting with the bound molecule will also be retain on the filter. For example, proteins expressed by different clones of an expression library are bound to a nitrocellulose membrane, by colony (bacterial library) or plaque (phage library) blotting. A labelled protein can then be used as a probe to identify clones expressing proteins that interact with the probe. Interactions occur on the nitrocellulose filters. The method is highly general and therefore widely applicable. A variety of approaches can be used to label the ligand, alternatively the ligand can be detected by a specific antibody." [PMID:7708014]
<br>subset: PSI-MI slim
<br>synonym: "Filter overlay assay" EXACT PSI-MI-alternate []
<br>is_a: MI:0892 ! solid phase assay
<p><br>[Term]
<br>id: MI:0050
<br>name: flag tag coimmunoprecipitation
<br>def: "The protein of interest is expressed as a fusion to the peptide DYKDDDDKV for which antibodies are commercially available. Sometimes multiple copies of the peptide are fused in tandem.\nOBSOLETE redundant term. Map to feature type: flag-tagged (MI:0518) and Interaction detection method: anti tag coimmunoprecipitation (MI:0007)." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "flag tag coip" EXACT PSI-MI-short []
<br>is_obsolete: true
<p><br>[Term]
<br>id: MI:0051
<br>name: fluorescence technology
<br>def: "Techniques based upon the measurement of the emission of one or more photons by a molecule activated by the absorption of a quantum of electro-magnetic radiation. Typically the emission, which is characterised by a wavelength that is longer than the one of excitatory radiation, occurs within 10-8 seconds." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "fluorescence" EXACT PSI-MI-short []
<br>is_a: MI:0013 ! biophysical
<p><br>[Term]
<br>id: MI:0052
<br>name: fluorescence correlation spectroscopy
<br>def: "FCS monitors the random motion of fluorescently labelled molecules inside a defined volume irradiated by a focused laser beam. These fluctuations provide information on the rate of diffusion or diffusion time of a particle and this is directly dependent on the particle mass. As a consequence, any increase in the mass of a biomolecule, e.g. as a result of an interaction with a second molecule, is readily detected as an increase in the diffusion time of the particle. From these results the concentration of the different molecules can be calculated as well as their binding constant." [PMID:10733953]
<br>subset: PSI-MI slim
<br>synonym: "FCS" EXACT PSI-MI-alternate []
<br>synonym: "fcs" EXACT PSI-MI-short []
<br>is_a: MI:0051 ! fluorescence technology
<p><br>[Term]
<br>id: MI:0053
<br>name: fluorescence polarization spectroscopy
<br>def: "Because of the long lifetimes of excited fluorescent molecules (nanoseconds), fluorescence can be used to monitor the rotational motion of molecules, which occurs on this timescale. This is accomplished experimentally by excitation with plane-polarized light, followed by measurement of the emission at parallel and perpendicular planes. Since rotational correlation times depend on the size of the molecule, this method can be used to measure the binding of two proteins because the observed polarization increase when a larger complex is formed. A fluorescence anisotropy experiment is normally carried out with a protein bearing a covalently added fluorescent group, which increases both the observed fluorescence lifetime of the excited state and the intensity of the fluorescent signal. Residue modification can be assessed by addition of an antibody which binds to the modified residue and alters the molecular weight of the complex. A variation of this technique has been used to show interaction of a DNA binding protein with another protein. In this case the DNA rather than protein is fluorescently labelled." [PMID:12805227, PMID:7708014]
<br>subset: PSI-MI slim
<br>synonym: "Fluorescence anisotropy" EXACT PSI-MI-alternate []
<br>synonym: "FPS" EXACT PSI-MI-alternate []
<br>synonym: "fps" EXACT PSI-MI-short []
<br>is_a: MI:0051 ! fluorescence technology
<p><br>[Term]
<br>id: MI:0054
<br>name: fluorescence-activated cell sorting
<br>def: "Cells in suspension flow through a laser beam, the scattered light or emitted fluorescence is measured, filtered and converted to digital values. Cells can be sorted according to their properties. Using flow cytometry, any fluorescent or light scattering experiment can be carried out on entire cells. With this instrument, interactions occurring either on cell surfaces or in any other subcellular location can be studied by using suitable fluorescent labels." [PMID:11988464]
<br>subset: PSI-MI slim
<br>synonym: "FACS" EXACT PSI-MI-alternate []
<br>synonym: "facs" EXACT PSI-MI-short []
<br>synonym: "Flow cytometry" EXACT PSI-MI-alternate []
<br>is_a: MI:0051 ! fluorescence technology
<p><br>[Term]
<br>id: MI:0055
<br>name: fluorescent resonance energy transfer
<br>def: "FRET is a quantum mechanical process involving the radiationless transfer of energy from a donor fluorophore to an appropriately positioned acceptor fluorophore. The fluorophores are genetically fused to the protein in analysis and cotransfected. Three basic conditions must be fulfilled for FRET to occur between a donor molecule and acceptor molecule. First, the donor emission spectrum must significantly overlap the absorption spectrum of the acceptor. Second, the distance between the donor and acceptor fluorophores must fall within the range 20 to 100 Angstrom. Third, the donor and acceptor fluorophores must be in favourable orientations." [PMID:11558993]
<br>subset: PSI-MI slim
<br>synonym: "FRET" EXACT PSI-MI-alternate []
<br>synonym: "fret" EXACT PSI-MI-short []
<br>synonym: "FRET analysis" EXACT PSI-MI-alternate []
<br>synonym: "RET" EXACT PSI-MI-alternate []
<br>is_a: MI:0051 ! fluorescence technology
<p><br>[Term]
<br>id: MI:0056
<br>name: full identification by DNA sequencing
<br>def: "Sequencing occurs during the course of the experiment. The DNA to be sequenced is used as template for the in vitro synthesis, by DNA polymerase, of a set of partial replicas, all beginning at the same place, but terminating at different points along the DNA chain. The key to this method is the use of dideoxyribonucleoside triphosphates in which the deoxyribose 3'-OH group present in normal nucleotides is missing; when such a modified nucleotide is incorporated into a DNA chain, it blocks the elongation of the chain. To determine the full sequence, the four different chain-terminating dideoxyribonucleosides are used in competition with an excess of deoxyribonucleosides in separate DNA synthesis reactions on the same DNA template. When the products of these four reactions are analysed by electrophoresis in four parallel lanes of a denaturing polyacrylamide gel, the DNA sequence can be derived. Every lane displays a family of DNA fragments of different lengths, reflecting the different sites at which a specific residue occurs in the original DNA." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "full dna sequence" EXACT PSI-MI-short []
<br>is_a: MI:0078 ! nucleotide sequence identification
<br>is_a: MI:0659 ! experimental feature detection
<p><br>[Term]
<br>id: MI:0057
<br>name: gene neighbourhood
<br>def: "Gene pairs that show a conserved topological neighbourhood in many prokaryotic genomes are considered by this approach to encode interacting or functionally related proteins. By measuring the physical distance of any given gene pair in different genomes, interacting partners are inferred." [PMID:9787636]
<br>subset: PSI-MI slim
<br>is_a: MI:0058 ! genome based prediction
<p><br>[Term]
<br>id: MI:0058
<br>name: genome based prediction
<br>def: "Methods that require fully sequenced genomes either because they are based on the comparison of genome topology or on the identification of orthologous sequences in different genomes." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "genome prediction" EXACT PSI-MI-short []
<br>is_a: MI:0063 ! interaction prediction
<p><br>[Term]
<br>id: MI:0059
<br>name: gst pull down
<br>def: "The bait protein is expressed and purified as a fusion to the glutathione S-tranferase protein. The bait protein is normally attached to a glutathione sepharose resin or alternatively to a support containing an anti-GST antibody.\nOBSOLETE redundant term. Map to feature type : gst-tagged (MI:0519) and Interaction detection method: pull down (MI:0096)." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_obsolete: true
<p><br>[Term]
<br>id: MI:0060
<br>name: ha tag coimmunoprecipitation
<br>def: "The protein of interest is expressed as a fusion to the peptide YPYDVPDYA (a fragment of the influenza hemaglutinin protein) for which antibodies are commercially available.\nOBSOLETE redundant term. Map to feature type : ha-tagged (MI:0520) and Interaction detection method: anti tag coimmunoprecipitation (MI:0007)." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "ha tag coip" EXACT PSI-MI-short []
<br>is_obsolete: true
<p><br>[Term]
<br>id: MI:0061
<br>name: his pull down
<br>def: "The bait protein is expressed and purified fused to an amino or carboxyterminal tail containing a variable number of histidines. The bait protein is normally attached to a metal (usually nickel) resin.\nOBSOLETE redundant term. Map to feature type : his-tagged (MI:0521) and Interaction detection method: pull down (MI:0096)." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_obsolete: true
<p><br>[Term]
<br>id: MI:0062
<br>name: his tag coimmunoprecipitation
<br>def: "The protein of interest is expressed as a fusion to a poly-His tail. This permits purification by chromatography over a metal column or by binding to commercially available anti poly-His antibodies.\nOBSOLETE redundant term. Map to feature type: his-tagged (MI:0521) and Interaction detection method: anti tag coimmunoprecipitation (MI:0007)." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "his tag coip" EXACT PSI-MI-short []
<br>is_obsolete: true
<p><br>[Term]
<br>id: MI:0063
<br>name: interaction prediction
<br>def: "Computational methods to predict an interaction." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "in silico methods" EXACT PSI-MI-alternate []
<br>synonym: "predicted interac" EXACT PSI-MI-short []
<br>is_a: MI:0001 ! interaction detection method
<p><br>[Term]
<br>id: MI:0064
<br>name: interologs mapping
<br>def: "Protein interactions, experimentally detected in an organism, are extended to a second organism assuming that homologue proteins, in different organisms, maintain their interaction properties." [PMID:11731503]
<br>subset: PSI-MI slim
<br>synonym: "Homology based interaction prediction" EXACT PSI-MI-alternate []
<br>is_a: MI:0046 ! experimental knowledge based
<br>is_a: MI:0101 ! sequence based prediction
<p><br>[Term]
<br>id: MI:0065
<br>name: isothermal titration calorimetry
<br>def: "Isothermal titration calorimetry (ITC) measures directly the energy associated with a chemical reaction triggered by the mixing of two components. A typical ITC experiment is carried out by the stepwise addition of one of the reactants (~10-6 L per injection) into the reaction cell (~1mL) containing the second reactant. The chemical reaction occurring after each injection either releases or absorbs heat (qi) proportional to the amount of ligand that binds to the protein with a characteristic binding enthalpy (DH). As modern ITC instruments operate on the heat compensation principle, the instrumental response (measured signal) is the amount of power (microcalories per second) necessary to maintain constant the temperature difference between the reaction and the reference cells. Because the amount of uncomplexed protein available progressively decreases after each successive injection, the magnitude of the peaks becomes progressively smaller until complete saturation is achieved. The difference between the concentration of bound ligand in the ith and (i-1)th injections depends on the binding constant Ka and the total ligand injected. The calculations depend on the binding model (number of substrates). Analysis of the data yields DH and DG = -RTlnKa. The entropy change is obtained by using the standard thermodynamic expression DG = DH-TDS." [PMID:11785756]
<br>subset: PSI-MI slim
<br>synonym: "ITC" EXACT PSI-MI-alternate []
<br>synonym: "itc" EXACT PSI-MI-short []
<br>is_a: MI:0013 ! biophysical
<p><br>[Term]
<br>id: MI:0066
<br>name: lambda phage display
<br>def: "Morphologically classified as one of the siphoviridae, lambda is a temperate bacteriophage of E.coli, with a double-stranded DNA genome. It has an icosahedral head attached to a flexible helical tail. Both the tail protein pV and the head protein pD have been used for displaying (C or N terminally) foreign peptides on the viral capsid." [PMID:7682645]
<br>subset: PSI-MI slim
<br>synonym: "lambda phage" EXACT PSI-MI-short []
<br>is_a: MI:0084 ! phage display
<p><br>[Term]
<br>id: MI:0067
<br>name: light scattering
<br>def: "Dynamic and static laser light scattering probes the size, shape, and structure of biological macromolecules or of their assemblies. A beam is focused on an optically clear cylindrical cell containing the sample. Most of the light passes directly through the sample. A small portion of the light is scattered; the scattered light intensity containing information about the scattering particle is detected at an angle (typically in the range 15-180degrees) from the direction of the incident beam." [PMID:9013660]
<br>subset: PSI-MI slim
<br>is_a: MI:0013 ! biophysical
<p><br>[Term]
<br>id: MI:0068
<br>name: mass detection of residue modification
<br>def: "Mass spectrometry can be used to characterise chemical modifications within peptides. One approach consists in the observation of a mass difference when a sample is treated with an enzyme that can specifically remove a peptide modification, for instance a phosphatase. The mass difference corresponds to the mass of the chemical group covalently linked to a residue. Such experiments carried out with a MALDI-TOF (Matrix-assisted laser desorption ionization time-of-flight ) do not allow the mapping of the modification site within the sequence, whereas any tandem mass spectrometer (LC MS/MS Liquid Chromatography Tandem Mass Spectrometry, nanoESI MS/MS nanoElectrospray Ionisation tandem mass spectrometry, FTMS Fourier Transform mass spectrometry) provide such information. A second approach consists of the direct mass measurement of the ionized chemical group dissociated from the residue within a tandem mass spectrometer. Both approaches need a prior enrichment of the modified peptide population in the samples with IMAC (Immobilized Metal Affinity Chromatography)or specific anti-modification antibodies." [PMID:11395414, PMID for application instance:11875433]
<br>subset: PSI-MI slim
<br>synonym: "modified residue ms" EXACT PSI-MI-short []
<br>is_a: MI:0659 ! experimental feature detection
<p><br>[Term]
<br>id: MI:0069
<br>name: mass spectrometry studies of complexes
<br>def: "Mass spectrometric approaches to the study of protein in complexes permits the identification of subunit stoichiometry and transient associations. By preserving complexes intact in the mass spectrometer, mass measurement can be used for monitoring changes in different experimental conditions, or to investigate how variations of collision energy affect their dissociation." [PMID:12504676, PMID for application instance:12057199]
<br>subset: PSI-MI slim
<br>synonym: "ms of complexes" EXACT PSI-MI-short []
<br>is_a: MI:0943 ! detection by mass spectrometry
<p><br>[Term]
<br>id: MI:0070
<br>name: mobility shift
<br>def: "Protein modifications can be identified by gel electrophoresis since any change in the mass and/or the charge of the protein can alter its mobility in PAGE. Although this method does not allow the unequivocal identification of the type of modification that has caused the shift, it is possible, by combining this approach with more direct methods, to correlate the extent of the shift to a specific modification." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0659 ! experimental feature detection
<p><br>[Term]
<br>id: MI:0071
<br>name: molecular sieving
<br>def: "In sizing columns (gel filtration), the elution position of a protein or of a complex depends on its Stokes radius. Molecules with a radius that is smaller than the bead size are retained and retarded by the interaction with the matrix. The observation that two proteins, loaded on a sieving column, elute in a fraction(s) corresponding to a MW that is larger than the MW of either protein may be taken as an indication that the two proteins interact. Furthermore this technique provides a conceptually simple method for evaluating the affinity of the interaction." [PMID:7708014]
<br>subset: PSI-MI slim
<br>synonym: "Gel Filtration" EXACT PSI-MI-alternate []
<br>synonym: "Siezing column" EXACT PSI-MI-alternate []
<br>synonym: "Size Exclusion Chromatography" EXACT PSI-MI-alternate []
<br>is_a: MI:0013 ! biophysical
<br>is_a: MI:0091 ! chromatography technology
<p><br>[Term]
<br>id: MI:0072
<br>name: monoclonal antibody western blot
<br>def: "Western blot assay carried out using monospecific antibodies produced in the supernatant of a cell line obtained by fusing a lymphocyte B to a myeloma cell line or selected by phage display technology." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "monoclonal western" EXACT PSI-MI-short []
<br>is_a: MI:0113 ! western blot
<p><br>[Term]
<br>id: MI:0073
<br>name: mrna display
<br>def: "This method relies on the covalent coupling of mRNA to the nascent polypeptide. The mRNA (natural or artificial) is first covalently linked to a short DNA linker carrying a puromycin moiety. The mRNA mixture is then translated in vitro. When the ribosome reaches the RNA-DNA junction the ribosome stalls and the puromycin moiety enters the peptidyltransferase site of the ribosome and forms a covalent linkage to the nascent polypeptide. As a result the protein and the mRNA are covalently joined and can be isolated from the ribosome and purified. In the current protocol, a cDNA strand is then synthesised to form a less sticky RNA-DNA hybrid and these complexes are finally used for affinity selection. As in most display approaches, several selections cycles (3-6) are sufficient to enrich for mRNAs encoding ligand proteins." [PMID:11551470]
<br>subset: PSI-MI slim
<br>is_a: MI:0034 ! display technology
<p><br>[Term]
<br>id: MI:0074
<br>name: mutation analysis
<br>def: "Mutant molecules are produced by random or directed techniques and assayed for their ability to support binding." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0659 ! experimental feature detection
<p><br>[Term]
<br>id: MI:0075
<br>name: myc tag coimmunoprecipitation
<br>def: "The protein of interest is expressed as a fusion to the peptide EUKLISEED (a fragment of the Myc oncogene protein) for which antibodies are commercially available. Sometimes multiple copies of the peptide are fused in tandem.\nOBSOLETE redundant term. Map to feature type: myc-tagged (MI:0522) and Interaction detection method: anti tag coimmunoprecipitation (MI:0007)." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "myc tag coip " EXACT PSI-MI-short []
<br>is_obsolete: true
<p><br>[Term]
<br>id: MI:0076
<br>name: neural network on interface properties
<br>def: "Neural networks are trained on the properties of residues belonging to a cluster of residues that are neighbours in space on protein surface. The predictor permits the inference of the residues that are likely to be on an interaction interface." [PMID:11455607, PMID:11874449]
<br>subset: PSI-MI slim
<br>synonym: "interface predictor" EXACT PSI-MI-short []
<br>is_a: MI:0577 ! feature prediction from structure
<p><br>[Term]
<br>id: MI:0077
<br>name: nuclear magnetic resonance
<br>def: "NMR requires a small volume of concentrated compound solution that is placed in a strong magnetic field. Certain atomic nuclei, and in particular hydrogen, have a magnetic moment or spin; that is, they have an intrinsic magnetisation, like a bar magnet. The spin aligns along the strong magnetic field, but can be changed to a misaligned excited state in response to applied radio frequency (RF) pulses of electromagnetic radiation. When the excited hydrogen nuclei relax to their aligned state, they emit RF radiation, which can be measured and displayed as a spectrum. The nature of the emitted radiation depends on the environment of each hydrogen nucleus, and if one nucleus is excited, it will influence the absorption and emission of radiation by other nuclei that lie close to it. It is consequently possible, by an ingenious elaboration of the basic NMR technique known as two-dimensional NMR, to distinguish the signals from hydrogen nuclei in different amino acid residues and to identify and measure the small shifts in these signals that occur when these hydrogen nuclei lie close enough to interact: the size of such a shift reveals the distance between the interacting pair of hydrogen atoms. In this way NMR can give information about the distances between the parts of the interacting molecule. NMR provides information about interacting atoms thereby permitting to obtain information about macromolecular structure and molecular interactions." [PMID:12120505, PMID for application instance:12062432]
<br>subset: PSI-MI slim
<br>synonym: "NMR" EXACT PSI-MI-alternate []
<br>synonym: "nmr" EXACT PSI-MI-short []
<br>is_a: MI:0013 ! biophysical
<br>is_a: MI:0659 ! experimental feature detection
<p><br>[Term]
<br>id: MI:0078
<br>name: nucleotide sequence identification
<br>def: "Identification of a nucleotide sequence. Depending on the experimental design, nucleotide sequence can be determined before the interaction detection while building a collection of clones or after the selection of randomly generated clones." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "nucleotide sequence" EXACT PSI-MI-short []
<br>synonym: "sequence cloning" EXACT PSI-MI-alternate []
<br>is_a: MI:0661 ! experimental participant identification
<p><br>[Term]
<br>id: MI:0079
<br>name: other biochemical technologies
<br>def: "Experimental methods that could not be assigned to the other large group of technologies.\nOBSOLETE use biochemical (MI:0401) instead." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "other biochemic tech" EXACT PSI-MI-short []
<br>is_obsolete: true
<p><br>[Term]
<br>id: MI:0080
<br>name: partial dna sequence identification by hybridization
<br>def: "Genes are recognised by hybridization of a probe with a fragment of the gene sequence." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "partial dna sequence" EXACT PSI-MI-short []
<br>is_a: MI:0078 ! nucleotide sequence identification
<p><br>[Term]
<br>id: MI:0081
<br>name: peptide array
<br>def: "The peptide synthesis methods offer numerous opportunities to synthesise and subsequently screen large arrays of synthetic peptides on planar cellulose supports. Discrete spots are arranged as arrays on membrane sheets where each spot is individually accessed by manual or automated delivery of the appropriate reagent solutions. Over the past few years protein-protein recognition, peptide-metal ion interactions, peptide-nucleic acid binding, enzymatic modification of peptides experiments, have been explored using synthetic peptide arrays on planar support." [PMID:11167074]
<br>subset: PSI-MI slim
<br>is_a: MI:0008 ! array technology
<p><br>[Term]
<br>id: MI:0082
<br>name: peptide massfingerprinting
<br>def: "This approach leads to protein identification by matching peptide masses, as measured by mass spectrometry, to the ones calculated from in silico fragmentation of a protein sequence database. A peptide mixture from a tryptic digest is analysed by MALDI-MS (Matrix-assisted laser desorption ionization mass spectrometry). The list of peptide masses obtained by MALDI MS is automatically compared to the calculated masses of the predicted peptide fragments for each entry in the database. High mass accuracy is very important in order to obtain a statistically significant and unambiguous match This method is best applied to completely sequenced genomes and well characterised proteomes. However, depending on the data quality, proteins that are highly homologous to already characterised proteins (greater than 80 to 90% sequence identity) can also be identified. The retrieved sequence are evaluated by mass accuracy of the peptides, matching of additional peptide masses in the MALDI spectrum after accounting for common modifications such as oxidation, acrylamidation of cysteine and missed cleavages and the use of secondary information (apparent isoelectric point and molecular weight). If any ambiguity about the identification by MALDI-MS still exists, the results must verified by an other identification method. Peptide mass fingerprint is generally carried out with a MALDI-TOF (Matrix-assisted laser desorption ionization time-of-flight ) instrument but can also be achieved ESI-TOF (Electrospray Ionisation time-of-flight) or LC-MS (Liquid Chromatography-Mass Spectrometry) mass spectrometer." [PMID:10967324, PMID:11752590, PMID for application instance:11805826]
<br>subset: PSI-MI slim
<br>synonym: "fingerprinting" EXACT PSI-MI-short []
<br>is_a: MI:0427 ! Identification by mass spectrometry
<br>is_a: MI:0433 ! partial identification of protein sequence
<br>is_a: MI:0815 ! confirmation by molecular weight
<p><br>[Term]
<br>id: MI:0083
<br>name: peptide synthesis
<br>def: "When one of the partners participates in the interaction with a relatively short peptide fragment, it is often convenient to precisely identify the minimal region that supports the interaction by synthesising a series of overlapping peptides and by testing them in the binding assay. Synthetic peptides that are identical with peptides synthesised in vivo are useful experimental tools for such studies. Peptides are routinely synthesised in a test tube from monomeric amino acids by condensation reactions that form peptide bonds. Peptides are constructed sequentially by coupling the C-terminus of a monomeric amino acid to the N-terminus of the growing peptide. To prevent unwanted reactions involving the amino groups and carboxyl groups of the side chains during the coupling steps, a protecting (blocking) group is attached to the side chains. Without these protecting groups, branched peptides would be generated. In the last steps of synthesis, the side chain-protecting groups are removed and the peptide is cleaved from the resin on which synthesis occurs." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0093 ! protein sequence identification
<p><br>[Term]
<br>id: MI:0084
<br>name: phage display
<br>def: "Peptide sequences or entire proteins can be displayed on phage capsids by fusion to coat proteins to generate a library of fusion phages each displaying a different peptide. Such a library can then be exploited to identify specific phages that display peptides that bind to any given bait molecule for instance an antibody. The selection is performed by a series of cycles of affinity purification known as panning. The bait protein, immobilized on a solid support (plastic, agarose, sepharose, magnetic beads and others) is soaked in the phage mixture and that phage that remains attached to the bait is amplified and carried through a further affinity purification step. Each cycle results in an approximately 1,000-fold enrichment of specific phage and after a few selection rounds (2-4), DNA sequencing of the tight-binding phage reveals only a small number of sequences. Phage display panning experiments can be carried out either on libraries of peptides of random amino acid sequence or on libraries of displaying natural peptides obtained by inserting cDNA fragments into the phage vector (cDNA libraries). Libraries have been assembled on several different phages (Fd, Lambda or T7)." [PMID:10975452, PMID:7708014]
<br>subset: PSI-MI slim
<br>is_a: MI:0034 ! display technology
<p><br>[Term]
<br>id: MI:0085
<br>name: phylogenetic profile
<br>def: "The phylogenetic profile of a protein stores information about the presence and the absence of that protein in a set of genomes. By clustering identical or similar profiles, proteins with similar functions and potentially interacting are identified." [PMID:10200254]
<br>subset: PSI-MI slim
<br>is_a: MI:0058 ! genome based prediction
<p><br>[Term]
<br>id: MI:0086
<br>name: polyclonal antibody western blot
<br>def: "Western blot assay carried out using a mixture of different antibodies that represent the immune response, normally in an experimental animal, to the protein of interest." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "polyclonal western" EXACT PSI-MI-short []
<br>is_a: MI:0113 ! western blot
<p><br>[Term]
<br>id: MI:0087
<br>name: predictive text mining
<br>def: "Methods based on natural language processing to detect possible interactions between proteins (direct physical interactions or indirect genetic interactions). This includes the detection of non ambiguous protein or gene names and analysis of the relation expressed in a sentence among them." [PMID:11791231]
<br>subset: PSI-MI slim
<br>synonym: "predictive tm" EXACT PSI-MI-short []
<br>is_a: MI:0110 ! text mining
<p><br>[Term]
<br>id: MI:0088
<br>name: primer specific pcr
<br>def: "Sequences can be identified in a DNA mixture by launching a PCR (Polymerase Chain Reaction) controlled by sequence specific primers. Such reaction starts only when the hybridization of the primer with a complementary sequence occurs." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0080 ! partial dna sequence identification by hybridization
<p><br>[Term]
<br>id: MI:0089
<br>name: protein array
<br>def: "The protein array technology allows the screening of biochemical activities or binding abilities of hundreds or thousands of protein samples in parallel. After synthesis and purification by high-throughput methodologies, the proteins are printed onto the chip by using an instrument (micro-arrayer) that is capable of spotting liquid samples in a reproducible manner onto a planar support. The ordered protein array can then be probed with labelled molecules to identify proteins that bind to the bait." [PMID:10976071, PMID:12067604]
<br>subset: PSI-MI slim
<br>is_a: MI:0008 ! array technology
<p><br>[Term]
<br>id: MI:0090
<br>name: protein complementation assay
<br>def: "The function of numerous proteins or ribonucleic particles (enzymes, transcription factors, and others) can be rationally dissected into two fragments that fold autonomously but cannot complement to reconstitute the complex function, unless they are located in close proximity. In a two hybrid experiment, restoration of the activity by complementation of the two fragments when expressed as fusion with two polypeptides is taken as an evidence that the two polypeptides interact together." [PMID:11495741]
<br>subset: PSI-MI slim
<br>synonym: "complementation" EXACT PSI-MI-short []
<br>synonym: "PCA" EXACT PSI-MI-alternate []
<br>is_a: MI:0045 ! experimental interaction detection
<p><br>[Term]
<br>id: MI:0091
<br>name: chromatography technology
<br>def: "Used to separate and/or analyse complex mixtures. The components to be separated are distributed between two phases: a stationary phase (bed) and a mobile phase which percolates through the stationary bed. The nature of the two phases determines the separation criteria exploited by the column such as affinity, ionic charges, size or hydrophobicity of the molecules under analysis. Each type of column can be implemented with the mobile phase under atmospheric or high pressure condition. In this later case columns are designated as High Pressure Liquid Chromatography (HPLC)." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "chromatography" EXACT PSI-MI-short []
<br>synonym: "column chromatography" EXACT PSI-MI-alternate []
<br>is_a: MI:0401 ! biochemical
<p><br>[Term]
<br>id: MI:0092
<br>name: protein in situ array
<br>def: "Protein In Situ Array is a method by which protein arrays are rapidly generated in one step directly from DNA, by cell-free protein expression and simultaneous in situ immobilisation at a surface. Individual genes or fragments are produce by PCR or RT-PCR depending on the source of genetic material using properly designed primers. The PISA is generated by cell-free protein synthesis using coupled transcription and translation to produce a double HexaHis-tagged protein, the reaction being carried out on a surface to which the protein adheres as soon as it is synthesised." [PMID:11470888]
<br>subset: PSI-MI slim
<br>synonym: "PISA" EXACT PSI-MI-alternate []
<br>synonym: "pisa" EXACT PSI-MI-short []
<br>is_a: MI:0089 ! protein array
<p><br>[Term]
<br>id: MI:0093
<br>name: protein sequence identification
<br>def: "Single amino acid identification along a protein sequence." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "protein sequence" EXACT PSI-MI-short []
<br>is_a: MI:0661 ! experimental participant identification
<p><br>[Term]
<br>id: MI:0094
<br>name: protein staining
<br>def: "A wide range of dyes have been used over the years to visualise proteins in polyacrylamide gels - Coomasie Blue and silver-staining being two classical methods. Fluorescent dyes such as Nile Red and SYPRO Orange are now increasingly used due to their superior dynamic range. Use of non-denaturing gels can allow visualisation of protein protein interactions. Several dyes can be used to specifically indicate residue modification, however this methodology will give no information as the number of residues modified or their position within the protein sequence. Examples include the use of acid fuscian or the fluorescent dansyl hydrazine to show protein glycosylation." [PMID:12015990]
<br>subset: PSI-MI slim
<br>is_a: MI:0659 ! experimental feature detection
<p><br>[Term]
<br>id: MI:0095
<br>name: proteinchip(r) on a surface-enhanced laser desorption/ionization
<br>def: "ProteinChip(r) Array technology is a surface-enhanced laser desorption/ionization (SELDI) approach (Ciphergen Biosystems Inc. Fremont, CA, USA) for sample fractionation accomplished by retentate chromatography. Retentate chromatography is performed on ProteinChip Arrays with varying chromatographic properties (e.g. anion exchange, cation exchange, metal affinity and reverse phase). By utilising arrays with differing surface chemistries in parallel and in series, a complex mixture of proteins, as from cells or body fluids, can be resolved into subsets of proteins with common properties. Specific analytes can also be examined by using preactivated arrays to which a bait molecule (such as an antibody or biotinylated DNA) is immobilized and a solution containing the binding partner(s) is presented to the array. This array-based immunoprecipitation or protein-binding experiment has been used with good success to study DNA-binding proteins, receptor-ligand interactions, and protein complexes. Any ligand retained on a SELDI chip can directly be identified by mass spectrometry." [PMID:11827829]
<br>subset: PSI-MI slim
<br>synonym: "seldi chip" EXACT PSI-MI-short []
<br>synonym: "SELDI ProteinChip" EXACT PSI-MI-alternate []
<br>is_a: MI:0089 ! protein array
<p><br>[Term]
<br>id: MI:0096
<br>name: pull down
<br>def: "A specific affinity chromatography method where a molecule of interest (bait) is expressed as a fusion to an affinity tag (GST, HIS tag and others) and binds to a support that has affinity for the tag. The protein may be expressed and purified first, often in an heterologous system, bound to the matrix at high concentration and then challenged with a solution or cellular extract containing the candidate partner molecules. Alternatively, a protein complex may be adsorbed to the resin and the retained binding proteins subsequently identified." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0004 ! affinity chromatography technology
<p><br>[Term]
<br>id: MI:0097
<br>name: reverse ras recruitment system
<br>def: "In this complementation approach the bait can be any membrane protein (for example a receptor or a channel protein), the prey is cloned as a fusion protein of any cDNA from a library and the coding sequence of cytoplasmic RAS (cdc25 in yeast). If the bait and the prey interact, RAS is recruited close to the membrane and can activate cell growth. This procedure must take place in cells having a mutated RAS (Cdc25-2 yeast strain having a temperature sensitive mutation of RAS) to avoid constitutive growth activation." [PMID:11160938]
<br>subset: PSI-MI slim
<br>synonym: "reverse RRS" EXACT PSI-MI-alternate []
<br>synonym: "reverse rrs" EXACT PSI-MI-short []
<br>is_a: MI:0228 ! cytoplasmic complementation assay
<p><br>[Term]
<br>id: MI:0098
<br>name: ribosome display
<br>def: "This method permits the coupling of phenotype to genotype via the formation of a non-covalent ternary complex between mRNAs and their encoded polypeptides while they are translated in an in vitro system. As a first step a cDNA library is constructed that encodes chimeric proteins in which the natural proteins or protein domains are fused to a C-terminal tether. As a consequence when the mRNA is translated in vitro the domain can fold while the tether is still in the ribosomal tunnel. Furthermore this chimeric mRNAs lack a stop codon, thus preventing release of the mRNA and the polypeptide from the ribosome. High concentrations of magnesium and low temperature further stabilise the ternary complex. Similarly to phage display, these complexes can be used directly to select for nucleic acids encoding proteins with desired properties." [PMID:11551470, PMID for application instance:12167034]
<br>subset: PSI-MI slim
<br>is_a: MI:0034 ! display technology
<p><br>[Term]
<br>id: MI:0099
<br>name: scintillation proximity assay
<br>def: "SPA relies upon the fact that a beta particle emitted from a radioisotope decay can excite a fluorophore only when it is at a very short distance in water solution (few micrometers). The ligand is labelled with a radioactive atom and its potential partner is fixed to fluorophore containing beads, the emitted fluorescence proving their interaction can be measured in a scintillation counter. The scintillator measures only the amount of bound radiolabelled ligand. Competition experiment with cold competitor can be done to estimate the binding affinities (50% inhibitory concentration [IC50], cold ligand versus labelled ligand). Loss of signal can also be used to measure substrate cleavage by an enzyme, and labelled antibodies used to titrate the degree of modified residue present." [PMID:3866247]
<br>subset: PSI-MI slim
<br>synonym: "RIA Radio Immuno Assay" EXACT PSI-MI-alternate []
<br>synonym: "SPA" EXACT PSI-MI-alternate []
<br>synonym: "spa" EXACT PSI-MI-short []
<br>is_a: MI:0013 ! biophysical
<p><br>[Term]
<br>id: MI:0100
<br>name: sequence based phylogenetic profile
<br>def: "Multiple alignments of orthologous sequences in the same species and their corresponding phylogenetic trees are built. Every phylogenetic tree is computed as a matrix of distances between all possible protein pairs. The covariation of the distance matrices reveals interacting protein pairs." [PMID:11707606]
<br>subset: PSI-MI slim
<br>synonym: "sequence phylogeny" EXACT PSI-MI-short []
<br>is_a: MI:0058 ! genome based prediction
<br>is_a: MI:0101 ! sequence based prediction
<p><br>[Term]
<br>id: MI:0101
<br>name: sequence based prediction
<br>def: "Computational methods based on evolutionary hypothesis, used as criteria to browse sequences and predict interacting pairs." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "predict from sequenc" EXACT PSI-MI-short []
<br>is_a: MI:0063 ! interaction prediction
<p><br>[Term]
<br>id: MI:0102
<br>name: sequence tag identification
<br>def: "This approach leads to protein identification by combining mass measurement and short amino acid sequence information obtained by tandem mass spectrometry. This information is then used to automatically find the best match in a sequence database. A mixture of peptides derived from a protease digestion is analysed by nanoelectrospray LC-MS/MS (Liquid Chromatography Tandem Mass Spectrometer or nanoESI MS/MS) mass spectrometry. Electrospray mass spectrometry cannot be applied to dilute samples and is affected by high salt. As a consequence peptides, normally extracted from acrylamide gels by in situ proteolysis, are desalted and concentrated on a microcolumn followed by elution into a capillary used for nanoelectrospray tandem mass spectrometry. A first mass spectrum (Normal mass spectrum or Q1 mass spectrum) gives information about the masses of all the peptides. Peptides observed in the normal mass spectrum are isolated in turn and dissociated into fragments by collision with gas molecules within the mass spectrometer. Some of the fragments obtained from a peptide constitute a nested set, differing by one amino acid, and the mass difference between them allows assignment of a partial sequence. The masses of the fragments define the position of the partial sequence in the peptide. Together with the cleavage specificity of the protease used to cleave the protein, and mass information such sequence tag provides much higher search specificity to match the a database entry. The procedure is repeated with several peptides from the digest, resulting in multiple identifications of the same protein or identification of several proteins from the peptide mixture. Unknown proteins can easily be identified by using the high specificity of the peptide sequence tag for searches in most sequence databases including EST or genome databases." [PMID:10967324, PMID:11752590, PMID for application instance:11805837]
<br>subset: PSI-MI slim
<br>synonym: "sequence tag" EXACT PSI-MI-short []
<br>is_a: MI:0427 ! Identification by mass spectrometry
<br>is_a: MI:0433 ! partial identification of protein sequence
<p><br>[Term]
<br>id: MI:0103
<br>name: southern blot
<br>def: "A standard procedure to identify DNA fragments containing specific gene sequences. In this procedure i) a genome is fragmented using a restriction enzyme ii) the generated fragments are separated by electrophoresis iii) the fragments are transferred to a membrane iv)the membrane is incubated with a radio labelled probe that hybridises any complementary subsequence." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0080 ! partial dna sequence identification by hybridization
<p><br>[Term]
<br>id: MI:0104
<br>name: static light scattering
<br>def: "In static light scattering, the average intensity of scattered light at multiple angles is measured. The data yield information on particle molecular weight, particle size and shape, and particle-particle interactions." [PMID:9013660]
<br>subset: PSI-MI slim
<br>synonym: "sls" EXACT PSI-MI-short []
<br>is_a: MI:0067 ! light scattering
<p><br>[Term]
<br>id: MI:0105
<br>name: structure based prediction
<br>def: "Methods based on 3D structure information." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "predict from struct" EXACT PSI-MI-short []
<br>is_a: MI:0063 ! interaction prediction
<p><br>[Term]
<br>id: MI:0106
<br>name: surface patches
<br>def: "Surface patches are built using 6 criteria: solvation potential, residue interface propensity, hydrophobicity, planarity, protrusion and accessible surface area. Protein structures having similar patches are likely to have the same interactions." [PMID:9299343]
<br>subset: PSI-MI slim
<br>is_a: MI:0577 ! feature prediction from structure
<p><br>[Term]
<br>id: MI:0107
<br>name: surface plasmon resonance
<br>def: "This method measures formation of complex by monitoring changes in the resonance angle of light impinging on a gold surface as a result of changes in the refractive index of the surface. A ligand of interest (small molecule, peptide, protein, sugar, lipid, nucleic acid) is immobilized on a gold surface, and the interacting partner is injected in buffer flow over it. Biomolecules that interact with the immobilized ligand are retained on the surface, and alter the resonance angle of impinging light as a result of the change in refractive index brought about by the increased biomolecule mass retained on the surface. Since all the biomolecules belonging to the same class have the same refractive index and since there is a linear correlation between resonance angle shift and biomolecule concentration near the surface, this allows one to measure changes in concentration at the surface as a consequence of interaction. Furthermore, this is done in real time, allowing direct measurement of both the on-rate and the off-rate and of the affinity constant of complex formation." [PMID:11896282, PMID:12120258, PMID:16338355]
<br>subset: PSI-MI slim
<br>synonym: "BIAcore(r)" EXACT PSI-MI-alternate []
<br>synonym: "Optical biosensor" EXACT PSI-MI-alternate []
<br>synonym: "spr" EXACT PSI-MI-short []
<br>is_a: MI:0013 ! biophysical
<p><br>[Term]
<br>id: MI:0108
<br>name: t7 phage display
<br>def: "T7 is a double stranded DNA bacteriophage with a thin-walled icosahedral capsid, ~550 Angstrom in diameter, which is decorated by 415 copies of the capsid protein, the product of gene 10. gp10 can tolerate insertions at the carboxyterminus without loosing its ability to be inserted into functional phage capsids. Both low density and high density display (albeit only with short peptides) can be achieved." [PMID:14755292]
<br>comment: Reference not index in medline: Rosenberg, A, Griffin, K, Studier, WS, McCormick, M, Berg, J, Novy, R, Mierendorf, R inNovations, 1996, 6, 1.
<br>subset: PSI-MI slim
<br>synonym: "t7 phage" EXACT PSI-MI-short []
<br>is_a: MI:0084 ! phage display
<p><br>[Term]
<br>id: MI:0109
<br>name: tap tag coimmunoprecipitation
<br>def: "The TAP method involves the fusion of the TAP tag (encoding a calmodulin binding peptide, a TEV cleavage site, and the Staphylococcus aureus Protein A) to the target protein and the introduction of the construct into the host cell or organism, maintaining the expression of the fusion protein at, or close to, its natural level. The fusion protein and associated components are recovered from cell extracts by affinity selection on an IgG matrix. After washing, the TEV protease is added to release the bound material. The eluate is incubated with calmodulin-coated beads in the presence of calcium. This second affinity step is required to remove the TEV protease as well as traces of contaminants remaining after the first affinity selection. After washing, the bound material is released with EGTA. This two steps purification steps ensures a highly selective complex purification of the tapped protein (first round of selection on the protein A, a high affinity tag) under mild condition (non denaturant pH or conditions required to remove the tag).\nOBSOLETE redundant term. Map to feature type: tap tagged (MI:0524) and  as interaction detection method  tandem affinity purification (MI:0676)." [PMID:10504710]
<br>subset: PSI-MI slim
<br>synonym: "tap tag coip" EXACT PSI-MI-short []
<br>is_obsolete: true
<p><br>[Term]
<br>id: MI:0110
<br>name: text mining
<br>def: "Text mining methods can be used to predict or confirm interactions by automated processing of scientific literature. Co-occurrence in the same sentence of an abstract of gene products labels are analysed to evaluate whether it represents a valid evidence of an interaction." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0046 ! experimental knowledge based
<p><br>[Term]
<br>id: MI:0111
<br>name: dihydrofolate reductase reconstruction
<br>def: "The gene for DHFR is rationally dissected into two fragments called F[1,2] and F[3]. Two proteins or protein domains that are thought to bind to each other can then be fused to either of the two DHFR fragments. Reconstitution of enzyme activity can be monitored in vivo by cell survival in DHFR-negative cells grown in the absence of nucleotides. A fluorescence assay can also be carried out taking advantage of fMTX binding to reconstituted DHFR. The basis of this assay is that complementary fragments of DHFR, when expressed and reassembled in cells, will bind with high affinity (Kd 5 540 pM) to fMTX in a 1:1 complex. fMTX is retained in cells by this complex, whereas the unbound fMTX is actively and rapidly transported out of the cells. Survival depends only on the number of molecules of DHFR reassembled." [PMID:10318894]
<br>subset: PSI-MI slim
<br>synonym: "dhfr reconstruction" EXACT PSI-MI-short []
<br>is_a: MI:0228 ! cytoplasmic complementation assay
<p><br>[Term]
<br>id: MI:0112
<br>name: ubiquitin reconstruction
<br>def: "In this method the two proteins, whose interaction is under investigation, (in this case mostly membrane proteins) are fused to an terminal fragment (Nub) and to a C-terminal fragment of ubiquitin (Cub). The two fragments do not associate to form a functional ubiquitin unless the two fused membrane proteins form a complex. The association is monitored by an ingenious trick. The C-term fragment of ubiquitin is expressed as a fusion to a transcription factor that being linked to a membrane protein cannot perform its function unless it is released form the ubiquitin fusion by a specific protease. This achieved through the activity of UBP (an ubiquitin specific protease) that cleaves off the reporter protein only when a functional ubiquitin is reconstituted." [PMID:9560251]
<br>subset: PSI-MI slim
<br>synonym: "ub reconstruction" EXACT PSI-MI-short []
<br>is_a: MI:0228 ! cytoplasmic complementation assay
<p><br>[Term]
<br>id: MI:0113
<br>name: western blot
<br>def: "Western blot is a procedure to identify and quantify proteins. A mixture of protein is first submitted to an electrophoresis in denaturing condition and then electro-transferred from the gel to a membrane. The membrane is then incubated with a primary antibody specific for a given protein or a specific residue modification in the sample under analysis. A secondary antibody, radiolabelled or fused to fluorophore or to a chromogenic enzyme, targets the first antibody and allows the visualisation of the protein band on the membrane." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "Immuno blot" EXACT PSI-MI-alternate []
<br>is_a: MI:0421 ! identification by antibody
<br>is_a: MI:0659 ! experimental feature detection
<p><br>[Term]
<br>id: MI:0114
<br>name: x-ray crystallography
<br>def: "Analysis of a diffraction pattern generated by a single crystal. X-rays have a wavelength, typically around 1 Angstrom (the diameter of a hydrogen atom). If a narrow parallel beam of X-rays is directed at a sample of a pure protein, most of the X-rays will pass straight through it. A small fraction, however, will be scattered by the atoms in the sample. If the sample is a well-ordered crystal, the scattered waves will reinforce one another at certain points and will appear as diffraction spots when the X-rays are recorded by a suitable detector. The position and intensity of each spot in the X-ray diffraction pattern contain information about the position and nature of the atoms in the crystal. The three-dimensional structure of a large molecule can be deduced from the electron-density map of its crystal. In recent years X-ray diffraction analysis has become increasingly automated, and now the slowest step is likely to be the production of suitable macromolecule crystals. This requires high concentration of very pure macromolecule and empirical searching for the proper crystallisation conditions." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "X-ray" EXACT PSI-MI-alternate []
<br>synonym: "x-ray diffraction" EXACT PSI-MI-short []
<br>is_a: MI:0013 ! biophysical
<br>is_a: MI:0659 ! experimental feature detection
<p><br>[Term]
<br>id: MI:0115
<br>name: yeast display
<br>def: "The proteins are displayed on the surface of the yeast S. cerevisiae by fusion to signal sequences for protein secretion. This method is limited by the low efficiency of the yeast display system but can take full advantage of exploiting cell sorting methods (FACS) to isolate cells that display molecules with desired binding properties." [PMID:9181578]
<br>subset: PSI-MI slim
<br>is_a: MI:0034 ! display technology
<br>is_a: MI:0054 ! fluorescence-activated cell sorting
<p><br>[Term]
<br>id: MI:0116
<br>name: feature type
<br>def: "Property of a subsequence that may interfere with the binding of a molecule." [PMID:14755292]
<br>subset: PSI-MI slim
<br>relationship: part_of MI:0000 ! molecular interaction
<p><br>[Term]
<br>id: MI:0117
<br>name: binding site
<br>def: "A region of a molecule identified as being involved in an interaction." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0252 ! biological feature
<p><br>[Term]
<br>id: MI:0118
<br>name: mutation
<br>def: "Sequence variation due to insertion, deletion or substitution event." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0252 ! biological feature
<p><br>[Term]
<br>id: MI:0119
<br>name: mutation decreasing interaction
<br>def: "Region of a molecule whose mutation or deletion decreases significantly interaction strength." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "hotspot" EXACT PSI-MI-alternate []
<br>synonym: "mutation decreasing" EXACT PSI-MI-short []
<br>is_a: MI:0118 ! mutation
<p><br>[Term]
<br>id: MI:0120
<br>name: post translation modification
<br>def: "Residue covalent modifications occurring in the specific protein form involved in an interaction." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "ptm" EXACT PSI-MI-short []
<br>is_a: MI:0252 ! biological feature
<p><br>[Term]
<br>id: MI:0121
<br>name: acetylated residue
<br>def: "Residue modification." [PMID:11125103]
<br>subset: PSI-MI slim
<br>is_a: MI:0120 ! post translation modification
<p><br>[Term]
<br>id: MI:0122
<br>name: n-acetyl-alanine
<br>def: "Residue modification." [PMID:11125103, RESID:AA0041]
<br>subset: PSI-MI slim
<br>synonym: "(S)-2-(acetylamino)propanoic acid" EXACT PSI-MI-alternate []
<br>synonym: "[A:ac]" EXACT PSI-MI-alternate []
<br>synonym: "AAC" EXACT PSI-MI-alternate []
<br>synonym: "acetylalanine" EXACT PSI-MI-alternate []
<br>synonym: "acetylalanine" EXACT PSI-MI-short []
<br>synonym: "N-acetyl-L-alanine" EXACT PSI-MI-alternate []
<br>is_a: MI:0121 ! acetylated residue
<p><br>[Term]
<br>id: MI:0123
<br>name: n2-acetyl-arginine
<br>def: "Residue modification." [PMID:11125103, RESID:AA0354]
<br>subset: PSI-MI slim
<br>synonym: "[R:ac]" EXACT PSI-MI-alternate []
<br>synonym: "acetylarginine" EXACT PSI-MI-alternate []
<br>synonym: "acetylarginine" EXACT PSI-MI-short []
<br>synonym: "alpha-acetylamino-delta-guanidinovaleric acid" EXACT PSI-MI-alternate []
<br>synonym: "N2-acetyl-L-arginine" EXACT PSI-MI-alternate []
<br>synonym: "RAC" EXACT PSI-MI-alternate []
<br>is_a: MI:0121 ! acetylated residue
<p><br>[Term]
<br>id: MI:0124
<br>name: n-acetyl-asparagine
<br>def: "Residue modification." [PMID:11125103]
<br>subset: PSI-MI slim
<br>synonym: "[N:ac]" EXACT PSI-MI-alternate []
<br>synonym: "acetylasparagine" EXACT PSI-MI-short []
<br>synonym: "N-acetyl-L-asparagine" EXACT PSI-MI-alternate []
<br>synonym: "NAC" EXACT PSI-MI-alternate []
<br>is_a: MI:0121 ! acetylated residue
<p><br>[Term]
<br>id: MI:0125
<br>name: n-acetyl-aspartic acid
<br>def: "Residue modification." [PMID:11125103, RESID:AA0042]
<br>subset: PSI-MI slim
<br>synonym: "(S)-2-(acetylamino)butanedioic acid" EXACT PSI-MI-alternate []
<br>synonym: "[D:ac]" EXACT PSI-MI-alternate []
<br>synonym: "acetylaspartate" EXACT PSI-MI-short []
<br>synonym: "acetylaspartic acid" EXACT PSI-MI-alternate []
<br>synonym: "DAC" EXACT PSI-MI-alternate []
<br>synonym: "N-acetyl-L-aspartic acid" EXACT PSI-MI-alternate []
<br>is_a: MI:0121 ! acetylated residue
<p><br>[Term]
<br>id: MI:0126
<br>name: n-acetyl-cysteine
<br>def: "Residue modification." [PMID:11125103, RESID:AA0043]
<br>subset: PSI-MI slim
<br>synonym: "(R)-2-acetylamino-3-sulfanylpropanoic acid" EXACT PSI-MI-alternate []
<br>synonym: "2-acetylamino-3-mercaptopropanoic acid" EXACT PSI-MI-alternate []
<br>synonym: "[C:ac]" EXACT PSI-MI-alternate []
<br>synonym: "acetylcysteine" EXACT PSI-MI-short []
<br>synonym: "CAC" EXACT PSI-MI-alternate []
<br>synonym: "N-acetyl-L-cysteine" EXACT PSI-MI-alternate []
<br>synonym: "N-acetylcysteine" EXACT PSI-MI-alternate []
<br>is_a: MI:0121 ! acetylated residue
<p><br>[Term]
<br>id: MI:0127
<br>name: n-acetyl-glutamine
<br>def: "Residue modification." [PMID:11125103, RESID:AA0045]
<br>subset: PSI-MI slim
<br>synonym: "(S)-2-acetylamino-5-pentanediamic acid" EXACT PSI-MI-alternate []
<br>synonym: "[Q:ac]" EXACT PSI-MI-alternate []
<br>synonym: "acetylglutamine" EXACT PSI-MI-alternate []
<br>synonym: "acetylglutamine" EXACT PSI-MI-short []
<br>synonym: "N-acetyl-L-glutamine" EXACT PSI-MI-alternate []
<br>synonym: "QAC" EXACT PSI-MI-alternate []
<br>is_a: MI:0121 ! acetylated residue
<p><br>[Term]
<br>id: MI:0128
<br>name: n-acetyl-glutamic acid
<br>def: "Residue modification." [PMID:11125103, RESID:AA0044]
<br>subset: PSI-MI slim
<br>synonym: "(S)-2-(acetylamino)pentanedioic acid" EXACT PSI-MI-alternate []
<br>synonym: "[E:ac]" EXACT PSI-MI-alternate []
<br>synonym: "acetylglutamate" EXACT PSI-MI-short []
<br>synonym: "acetylglutamic acid" EXACT PSI-MI-alternate []
<br>synonym: "EAC" EXACT PSI-MI-alternate []
<br>synonym: "N-acetyl-L-glutamic acid" EXACT PSI-MI-alternate []
<br>is_a: MI:0121 ! acetylated residue
<p><br>[Term]
<br>id: MI:0129
<br>name: n-acetylglycine
<br>def: "Residue modification." [PMID:11125103, RESID:AA0046]
<br>subset: PSI-MI slim
<br>synonym: "2-(acetylamino)ethanoic acid" EXACT PSI-MI-alternate []
<br>synonym: "[G:ac]" EXACT PSI-MI-alternate []
<br>synonym: "aceturic acid" EXACT PSI-MI-alternate []
<br>synonym: "acetylglycine" EXACT PSI-MI-short []
<br>synonym: "GAC" EXACT PSI-MI-alternate []
<br>synonym: "N-acetylglycine" EXACT PSI-MI-alternate []
<br>is_a: MI:0121 ! acetylated residue
<p><br>[Term]
<br>id: MI:0130
<br>name: n-acetyl-histidine
<br>def: "Residue modification." [PMID:11125103]
<br>subset: PSI-MI slim
<br>synonym: "[H:ac]" EXACT PSI-MI-alternate []
<br>synonym: "acetylhistidine" EXACT PSI-MI-short []
<br>synonym: "HAC" EXACT PSI-MI-alternate []
<br>synonym: "N-acetyl-L-histidine" EXACT PSI-MI-alternate []
<br>is_a: MI:0121 ! acetylated residue
<p><br>[Term]
<br>id: MI:0131
<br>name: n-acetyl-isoleucine
<br>def: "Residue modification." [PMID:11125103, RESID:AA0047]
<br>subset: PSI-MI slim
<br>synonym: "(2S,3S)-2-acetylamino-3-methylpentanoic acid" EXACT PSI-MI-alternate []
<br>synonym: "[I:ac]" EXACT PSI-MI-alternate []
<br>synonym: "acetylisoleucine" EXACT PSI-MI-alternate []
<br>synonym: "acetylisoleucine" EXACT PSI-MI-short []
<br>synonym: "IAC" EXACT PSI-MI-alternate []
<br>synonym: "N-acetyl-L-isoleucine" EXACT PSI-MI-alternate []
<br>is_a: MI:0121 ! acetylated residue
<p><br>[Term]
<br>id: MI:0132
<br>name: n-acetyl-leucine
<br>def: "Residue modification." [PMID:11125103]
<br>subset: PSI-MI slim
<br>synonym: "[L:ac]" EXACT PSI-MI-alternate []
<br>synonym: "acetylleucine" EXACT PSI-MI-short []
<br>synonym: "LAC" EXACT PSI-MI-alternate []
<br>synonym: "N-acetyl-L-leucine" EXACT PSI-MI-alternate []
<br>is_a: MI:0121 ! acetylated residue
<p><br>[Term]
<br>id: MI:0133
<br>name: n2-acetyl-lysine
<br>def: "Residue modification." [PMID:11125103, RESID:AA0048]
<br>subset: PSI-MI slim
<br>synonym: "(S)-2-acetylamino-6-aminohexanoic acid" EXACT PSI-MI-alternate []
<br>synonym: "[K:ac]" EXACT PSI-MI-alternate []
<br>synonym: "KAC" EXACT PSI-MI-alternate []
<br>synonym: "N2-acetyl-L-lysine" EXACT PSI-MI-alternate []
<br>synonym: "N2-acetyllysine" EXACT PSI-MI-alternate []
<br>synonym: "n2-acetyllysine" EXACT PSI-MI-short []
<br>is_a: MI:0526 ! n-acetyl-lysine
<p><br>[Term]
<br>id: MI:0134
<br>name: n6-acetyl-lysine
<br>def: "Residue modification." [PMID:11125103, RESID:AA0055]
<br>subset: PSI-MI slim
<br>synonym: "(S)-2-amino-6-(acetylamino)hexanoic acid" EXACT PSI-MI-alternate []
<br>synonym: "[K:N6ac]" EXACT PSI-MI-alternate []
<br>synonym: "epsilon-acetyllysine" EXACT PSI-MI-alternate []
<br>synonym: "KA6" EXACT PSI-MI-alternate []
<br>synonym: "N(zeta)-acetyllysine" EXACT PSI-MI-alternate []
<br>synonym: "N6-acetyl-L-lysine" EXACT PSI-MI-alternate []
<br>synonym: "n6-acetyllysine" EXACT PSI-MI-short []
<br>is_a: MI:0526 ! n-acetyl-lysine
<p><br>[Term]
<br>id: MI:0135
<br>name: n-acetyl-methionine
<br>def: "Residue modification." [PMID:11125103, RESID:AA0049]
<br>subset: PSI-MI slim
<br>synonym: "(S)-2-acetylamino-4-(methylsulfanyl)butanoic acid" EXACT PSI-MI-alternate []
<br>synonym: "2-acetylamino-4-(methylthio)butanoic acid" EXACT PSI-MI-alternate []
<br>synonym: "[M:ac]" EXACT PSI-MI-alternate []
<br>synonym: "acetylmethionine" EXACT PSI-MI-alternate []
<br>synonym: "acetylmethionine" EXACT PSI-MI-short []
<br>synonym: "MAC" EXACT PSI-MI-alternate []
<br>synonym: "methionamine" EXACT PSI-MI-alternate []
<br>synonym: "N-acetyl-L-methionine" EXACT PSI-MI-alternate []
<br>is_a: MI:0121 ! acetylated residue
<p><br>[Term]
<br>id: MI:0136
<br>name: n-acetyl-phenylalanine
<br>def: "Residue modification." [PMID:11125103]
<br>subset: PSI-MI slim
<br>synonym: "[F:ac]" EXACT PSI-MI-alternate []
<br>synonym: "acetylphenylalanine" EXACT PSI-MI-short []
<br>synonym: "FAC" EXACT PSI-MI-alternate []
<br>synonym: "N-acetyl-L-phenylalanine" EXACT PSI-MI-alternate []
<br>is_a: MI:0121 ! acetylated residue
<p><br>[Term]
<br>id: MI:0137
<br>name: n-acetyl-proline
<br>def: "Residue modification." [PMID:11125103, RESID:AA0050]
<br>subset: PSI-MI slim
<br>synonym: "(2S)-1-acetyl-2-pyrrolidinecarboxylic acid" EXACT PSI-MI-alternate []
<br>synonym: "[P:ac]" EXACT PSI-MI-alternate []
<br>synonym: "acetylproline" EXACT PSI-MI-alternate []
<br>synonym: "acetylproline" EXACT PSI-MI-short []
<br>synonym: "N-acetyl-L-proline" EXACT PSI-MI-alternate []
<br>synonym: "PAC" EXACT PSI-MI-alternate []
<br>is_a: MI:0121 ! acetylated residue
<p><br>[Term]
<br>id: MI:0138
<br>name: n-acetyl-serine
<br>def: "Residue modification." [PMID:11125103, RESID:AA0051]
<br>subset: PSI-MI slim
<br>synonym: "(S)-2-acetylamino-3-hydroxypropanoic acid" EXACT PSI-MI-alternate []
<br>synonym: "[S:ac]" EXACT PSI-MI-alternate []
<br>synonym: "acetylserine" EXACT PSI-MI-short []
<br>synonym: "N-acetyl-L-serine" EXACT PSI-MI-alternate []
<br>synonym: "N-acetylserine" EXACT PSI-MI-alternate []
<br>synonym: "SAC" EXACT PSI-MI-alternate []
<br>is_a: MI:0121 ! acetylated residue
<p><br>[Term]
<br>id: MI:0139
<br>name: n-acetyl-threonine
<br>def: "Residue modification." [PMID:11125103, RESID:AA0052]
<br>subset: PSI-MI slim
<br>synonym: "(2S,3R)-2-acetylamino-3-hydroxybutanoic acid" EXACT PSI-MI-alternate []
<br>synonym: "[T:ac]" EXACT PSI-MI-alternate []
<br>synonym: "acetylthreonine" EXACT PSI-MI-short []
<br>synonym: "N-acetyl-L-threonine" EXACT PSI-MI-alternate []
<br>synonym: "N-acetylthreonine" EXACT PSI-MI-alternate []
<br>synonym: "TAC" EXACT PSI-MI-alternate []
<br>is_a: MI:0121 ! acetylated residue
<p><br>[Term]
<br>id: MI:0140
<br>name: n-acetyl-tryptophan
<br>def: "Residue modification." [PMID:11125103]
<br>subset: PSI-MI slim
<br>synonym: "[W:ac]" EXACT PSI-MI-alternate []
<br>synonym: "acetyltryptophan" EXACT PSI-MI-short []
<br>synonym: "N-acetyl-L-tryptophan" EXACT PSI-MI-alternate []
<br>synonym: "WAC" EXACT PSI-MI-alternate []
<br>is_a: MI:0121 ! acetylated residue
<p><br>[Term]
<br>id: MI:0141
<br>name: n-acetyl-tyrosine
<br>def: "Residue modification." [PMID:11125103, RESID:AA0053]
<br>subset: PSI-MI slim
<br>synonym: "(S)-2-acetylamino-3-(4-hydoxyphenyl)propanoic acid" EXACT PSI-MI-alternate []
<br>synonym: "[Y:ac]" EXACT PSI-MI-alternate []
<br>synonym: "acetyltyrosine" EXACT PSI-MI-short []
<br>synonym: "N-acetyl-L-tyrosine" EXACT PSI-MI-alternate []
<br>synonym: "N-acetyltyrosine" EXACT PSI-MI-alternate []
<br>synonym: "YAC" EXACT PSI-MI-alternate []
<br>is_a: MI:0121 ! acetylated residue
<p><br>[Term]
<br>id: MI:0142
<br>name: n-acetyl-valine
<br>def: "Residue modification." [PMID:11125103, RESID:AA0054]
<br>subset: PSI-MI slim
<br>synonym: "(S)-2-acetylamino-3-methylbutanoic acid" EXACT PSI-MI-alternate []
<br>synonym: "[V:ac]" EXACT PSI-MI-alternate []
<br>synonym: "acetylvaline" EXACT PSI-MI-short []
<br>synonym: "N-acetyl-L-valine" EXACT PSI-MI-alternate []
<br>synonym: "N-acetylvaline" EXACT PSI-MI-alternate []
<br>synonym: "VAC" EXACT PSI-MI-alternate []
<br>is_a: MI:0121 ! acetylated residue
<p><br>[Term]
<br>id: MI:0143
<br>name: amidated residue
<br>def: "Residue modification." [PMID:11125103]
<br>subset: PSI-MI slim
<br>is_a: MI:0120 ! post translation modification
<p><br>[Term]
<br>id: MI:0144
<br>name: alanine amide
<br>def: "Residue modification." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "(S)-2-aminopropanamide" EXACT PSI-MI-alternate []
<br>synonym: "[A:am]" EXACT PSI-MI-alternate []
<br>synonym: "AAM" EXACT PSI-MI-alternate []
<br>synonym: "alaninamide" EXACT PSI-MI-alternate []
<br>synonym: "alanineamide" EXACT PSI-MI-short []
<br>synonym: "L-alanine amide" EXACT PSI-MI-alternate []
<br>is_a: MI:0143 ! amidated residue
<p><br>[Term]
<br>id: MI:0145
<br>name: arginine amide
<br>def: "Residue modification." [PMID:11125103, RESID:AA0082]
<br>subset: PSI-MI slim
<br>synonym: "(S)-2-amino-5-guanidinopentanamide" EXACT PSI-MI-alternate []
<br>synonym: "[R:am]" EXACT PSI-MI-alternate []
<br>synonym: "argininamide" EXACT PSI-MI-alternate []
<br>synonym: "arginineamide" EXACT PSI-MI-short []
<br>synonym: "L-arginine amide" EXACT PSI-MI-alternate []
<br>synonym: "RAM" EXACT PSI-MI-alternate []
<br>is_a: MI:0143 ! amidated residue
<p><br>[Term]
<br>id: MI:0146
<br>name: formylated residue
<br>def: "Residue modification." [PMID:11125103]
<br>subset: PSI-MI slim
<br>is_a: MI:0120 ! post translation modification
<p><br>[Term]
<br>id: MI:0147
<br>name: n-formyl-methionine
<br>def: "Residue modification." [PMID:11125103, RESID:AA0021]
<br>subset: PSI-MI slim
<br>synonym: "(S)-2-formylamino-4-(methylsulfanyl)butanoic acid" EXACT PSI-MI-alternate []
<br>synonym: "2-formylamino-4-(methylthio)butanoic acid" EXACT PSI-MI-alternate []
<br>synonym: "[M:form]" EXACT PSI-MI-alternate []
<br>synonym: "formylmethionine" EXACT PSI-MI-short []
<br>synonym: "MFM" EXACT PSI-MI-alternate []
<br>synonym: "N-formyl-L-methionine" EXACT PSI-MI-alternate []
<br>is_a: MI:0146 ! formylated residue
<p><br>[Term]
<br>id: MI:0148
<br>name: hydroxylated residue
<br>def: "Residue modification." [PMID:11125103]
<br>subset: PSI-MI slim
<br>is_a: MI:0120 ! post translation modification
<p><br>[Term]
<br>id: MI:0149
<br>name: 4-hydroxy-proline
<br>def: "Residue modification." [PMID:11125103, RESID:AA0030]
<br>subset: PSI-MI slim
<br>synonym: "4-hydroxy-L-proline" EXACT PSI-MI-alternate []
<br>synonym: "4hydroxyproline" EXACT PSI-MI-short []
<br>synonym: "[P:hy_g]" EXACT PSI-MI-alternate []
<br>synonym: "HYP" EXACT PSI-MI-alternate []
<br>is_a: MI:0148 ! hydroxylated residue
<p><br>[Term]
<br>id: MI:0150
<br>name: lipid modification
<br>def: "Residue modification." [PMID:11125103]
<br>subset: PSI-MI slim
<br>is_a: MI:0120 ! post translation modification
<p><br>[Term]
<br>id: MI:0151
<br>name: s-farnesyl-cysteine
<br>def: "Residue modification." [PMID:11125103, RESID:AA0102]
<br>subset: PSI-MI slim
<br>synonym: "(R,E,E)-2-amino-3-(3,7,11-trimethyl-2,6,10-dodecatrienylsulfanyl)propanoic acid" EXACT PSI-MI-alternate []
<br>synonym: "2-amino-3-(3,7,11-trimethyl-2,6,10-dodecatrienylthio)propanoic acid" EXACT PSI-MI-alternate []
<br>synonym: "[C:farn]" EXACT PSI-MI-alternate []
<br>synonym: "CFN" EXACT PSI-MI-alternate []
<br>synonym: "farnesylcysteine" EXACT PSI-MI-short []
<br>synonym: "S-farnesyl-L-cysteine" EXACT PSI-MI-alternate []
<br>is_a: MI:0547 ! s-prenyl-cysteine
<p><br>[Term]
<br>id: MI:0152
<br>name: s-geranylgeranyl-cysteine
<br>def: "Residue modification." [PMID:11125103, RESID:AA0104]
<br>subset: PSI-MI slim
<br>synonym: "(R,E,E,E)-2-amino-3-(3,7,11,15-tetramethyl-2,6,10,14-hexadecatetraenylsulfanyl)propanoic acid" EXACT PSI-MI-alternate []
<br>synonym: "2-amino-3-(3,7,11,15-tetramethyl-2,6,10,14-hexadecatetraenylthio)propanoic acid" EXACT PSI-MI-alternate []
<br>synonym: "[C:ger]" EXACT PSI-MI-alternate []
<br>synonym: "CGR" EXACT PSI-MI-alternate []
<br>synonym: "geranylgeranylcys" EXACT PSI-MI-short []
<br>synonym: "S-geranylgeranyl-L-cysteine" EXACT PSI-MI-alternate []
<br>is_a: MI:0547 ! s-prenyl-cysteine
<p><br>[Term]
<br>id: MI:0153
<br>name: n-palmitoyl-cysteine
<br>def: "Residue modification." [PMID:11125103, RESID:AA0060]
<br>subset: PSI-MI slim
<br>synonym: "(R)-2-hexadecanoylamino-3-sulfanylpropanoic acid" EXACT PSI-MI-alternate []
<br>synonym: "2-hexadecanoylamino-3-mercaptopropanoic acid" EXACT PSI-MI-alternate []
<br>synonym: "[C:palm_n]" EXACT PSI-MI-alternate []
<br>synonym: "CPN" EXACT PSI-MI-alternate []
<br>synonym: "N-(1-oxahexadecyl)-L-cysteine" EXACT PSI-MI-alternate []
<br>synonym: "N-palmitoyl-L-cysteine" EXACT PSI-MI-alternate []
<br>synonym: "n-palmitoylcysteine" EXACT PSI-MI-short []
<br>is_a: MI:0561 ! palmitoylated residue
<p><br>[Term]
<br>id: MI:0154
<br>name: s-palmitoyl-cysteine
<br>def: "Residue modification." [PMID:11125103, RESID:AA0106]
<br>subset: PSI-MI slim
<br>synonym: "(R)-2-amino-3-(hexadecanoylsulfanyl)propanoic acid" EXACT PSI-MI-alternate []
<br>synonym: "2-amino-3-(hexadecanoylthio)propanoic acid" EXACT PSI-MI-alternate []
<br>synonym: "[C:palm_s]" EXACT PSI-MI-alternate []
<br>synonym: "CPS" EXACT PSI-MI-alternate []
<br>synonym: "cysteine hexadecanoate thioester" EXACT PSI-MI-alternate []
<br>synonym: "cysteine palmitate thioester" EXACT PSI-MI-alternate []
<br>synonym: "S-palmitoyl-L-cysteine" EXACT PSI-MI-alternate []
<br>synonym: "s-palmitoylcysteine" EXACT PSI-MI-short []
<br>is_a: MI:0561 ! palmitoylated residue
<p><br>[Term]
<br>id: MI:0155
<br>name: n-myristoyl-glycine
<br>def: "Residue modification." [PMID:11125103, RESID:AA0059]
<br>subset: PSI-MI slim
<br>synonym: "[G:myr]" EXACT PSI-MI-alternate []
<br>synonym: "GMY" EXACT PSI-MI-alternate []
<br>synonym: "myristoylglycine" EXACT PSI-MI-short []
<br>synonym: "N-myristoyl-glycine" EXACT PSI-MI-alternate []
<br>is_a: MI:0560 ! myristoylated residue
<p><br>[Term]
<br>id: MI:0156
<br>name: n6-myristoyl-lysine
<br>def: "Residue modification." [PMID:11125103, RESID:AA0078]
<br>subset: PSI-MI slim
<br>synonym: "(S)-2-amino-6-(tetradecanoylamino)hexanoic acid" EXACT PSI-MI-alternate []
<br>synonym: "[K:myr]" EXACT PSI-MI-alternate []
<br>synonym: "epsilon-myristoyllysine" EXACT PSI-MI-alternate []
<br>synonym: "KMY" EXACT PSI-MI-alternate []
<br>synonym: "myristoyllysine" EXACT PSI-MI-short []
<br>synonym: "N(zeta)-myristoyllysine" EXACT PSI-MI-alternate []
<br>synonym: "N6-(1-oxotetradecyl)-L-lysine" EXACT PSI-MI-alternate []
<br>synonym: "N6-myristoyl-L-lysine" EXACT PSI-MI-alternate []
<br>is_a: MI:0560 ! myristoylated residue
<p><br>[Term]
<br>id: MI:0157
<br>name: methylated residue
<br>def: "Residue modification." [PMID:11125103]
<br>subset: PSI-MI slim
<br>is_a: MI:0120 ! post translation modification
<p><br>[Term]
<br>id: MI:0158
<br>name: n-methyl-alanine
<br>def: "Residue modification." [PMID:11125103, RESID:AA0061]
<br>subset: PSI-MI slim
<br>synonym: "(S)-2-methylaminopropanoic acid" EXACT PSI-MI-alternate []
<br>synonym: "[A:meth_n]" EXACT PSI-MI-alternate []
<br>synonym: "AMT" EXACT PSI-MI-alternate []
<br>synonym: "methylalanine" EXACT PSI-MI-short []
<br>synonym: "N-methyl-L-alanine" EXACT PSI-MI-alternate []
<br>synonym: "N-methylalanine" EXACT PSI-MI-alternate []
<br>is_a: MI:0562 ! methylated alanine
<p><br>[Term]
<br>id: MI:0159
<br>name: n,n,n-trimethyl-alanine
<br>def: "Residue modification." [PMID:11125103, RESID:AA0062]
<br>subset: PSI-MI slim
<br>synonym: "(S)-1-carboxy-N,N,N-trimethylethanaminium" EXACT PSI-MI-alternate []
<br>synonym: "(S)-2-(trimethylammonio)propanoic acid" EXACT PSI-MI-alternate []
<br>synonym: "[A:meth_n3]" EXACT PSI-MI-alternate []
<br>synonym: "AM3" EXACT PSI-MI-alternate []
<br>synonym: "N,N,N-trimethyl-L-alanine" EXACT PSI-MI-alternate []
<br>synonym: "trimethylalanine" EXACT PSI-MI-short []
<br>is_a: MI:0562 ! methylated alanine
<p><br>[Term]
<br>id: MI:0160
<br>name: omega-n,omega-n-dimethyl-arginine
<br>def: "Residue modification." [PMID:11125103, RESID:AA0068]
<br>subset: PSI-MI slim
<br>synonym: "(S)-2-amino-5-[((dimethylamino)iminomethyl)amino]pentanoic acid" EXACT PSI-MI-alternate []
<br>synonym: "[R:meth_n7]" EXACT PSI-MI-alternate []
<br>synonym: "asymmetric dimethylarginine" EXACT PSI-MI-alternate []
<br>synonym: "dimethylarginine" EXACT PSI-MI-short []
<br>synonym: "NG,NG-dimethylarginine" EXACT PSI-MI-alternate []
<br>synonym: "omega-N,omega-N-dimethyl-L-arginine" EXACT PSI-MI-alternate []
<br>synonym: "RM2" EXACT PSI-MI-alternate []
<br>is_a: MI:0563 ! methylated arginine
<p><br>[Term]
<br>id: MI:0161
<br>name: beta-methylthioaspartic acid
<br>def: "Residue modification." [PMID:11125103, RESID:AA0232]
<br>subset: PSI-MI slim
<br>synonym: "(2R,3Xi)-2-amino-3-methylsulfanylbutanedioic acid" EXACT PSI-MI-alternate []
<br>synonym: "3-carboxy-S-methyl-cysteine" EXACT PSI-MI-alternate []
<br>synonym: "3-methylthio-aspartic acid" EXACT PSI-MI-alternate []
<br>synonym: "[D:meth_b]" EXACT PSI-MI-alternate []
<br>synonym: "beta-methylthio-aspartic acid" EXACT PSI-MI-alternate []
<br>synonym: "DM2" EXACT PSI-MI-alternate []
<br>synonym: "L-beta-methylthioaspartic acid" EXACT PSI-MI-alternate []
<br>synonym: "methylthioaspartate" EXACT PSI-MI-short []
<br>is_a: MI:0157 ! methylated residue
<p><br>[Term]
<br>id: MI:0162
<br>name: n5-methyl-glutamine
<br>def: "Residue modification." [PMID:11125103, RESID:AA0071]
<br>subset: PSI-MI slim
<br>synonym: "(S)-2-amino-N5-methylpentanediamic acid" EXACT PSI-MI-alternate []
<br>synonym: "[Q:meth_n5]" EXACT PSI-MI-alternate []
<br>synonym: "gamma-methylglutamine" EXACT PSI-MI-alternate []
<br>synonym: "methylglutamine" EXACT PSI-MI-short []
<br>synonym: "N(delta)-methylglutamine" EXACT PSI-MI-alternate []
<br>synonym: "N-methylglutamine" EXACT PSI-MI-alternate []
<br>synonym: "N5-methyl-L-glutamine" EXACT PSI-MI-alternate []
<br>synonym: "QM5" EXACT PSI-MI-alternate []
<br>is_a: MI:0157 ! methylated residue
<p><br>[Term]
<br>id: MI:0163
<br>name: glutamic acid 5-methyl ester
<br>def: "Residue modification." [PMID:11125103, RESID:AA0072]
<br>subset: PSI-MI slim
<br>synonym: "(S)-2-aminopentanedioic acid 5-methyl ester" EXACT PSI-MI-alternate []
<br>synonym: "5-methyl-L-glutamate" EXACT PSI-MI-alternate []
<br>synonym: "[E:meth_o5]" EXACT PSI-MI-alternate []
<br>synonym: "EM5" EXACT PSI-MI-alternate []
<br>synonym: "glutamatemethylester" EXACT PSI-MI-short []
<br>synonym: "glutamic acid gamma-methyl ester" EXACT PSI-MI-alternate []
<br>synonym: "L-glutamic acid 5-methyl ester" EXACT PSI-MI-alternate []
<br>is_a: MI:0157 ! methylated residue
<p><br>[Term]
<br>id: MI:0164
<br>name: 3'-methyl-histidine
<br>def: "Residue modification." [PMID:11125103, RESID:AA0073]
<br>subset: PSI-MI slim
<br>synonym: "(S)-2-amino-3-(3-methyl-3H-imidazol-4-yl)propanoic acid" EXACT PSI-MI-alternate []
<br>synonym: "1-methylhistidine [misnomer]" EXACT PSI-MI-alternate []
<br>synonym: "3'-methyl-L-histidine" EXACT PSI-MI-alternate []
<br>synonym: "[H:meth_n4]" EXACT PSI-MI-alternate []
<br>synonym: "methylhistidine" EXACT PSI-MI-short []
<br>synonym: "MHS" EXACT PSI-MI-alternate []
<br>synonym: "N(delta)-methylhistidine" EXACT PSI-MI-alternate []
<br>synonym: "pi-methylhistidine" EXACT PSI-MI-alternate []
<br>synonym: "pros-methylhistidine" EXACT PSI-MI-alternate []
<br>is_a: MI:0157 ! methylated residue
<p><br>[Term]
<br>id: MI:0165
<br>name: n6-methyl-lysine
<br>def: "Residue modification." [PMID:11125103]
<br>subset: PSI-MI slim
<br>synonym: "(S)-2-amino-6-methylaminohexanoic acid" EXACT PSI-MI-alternate []
<br>synonym: "[K:meth_1]" EXACT PSI-MI-alternate []
<br>synonym: "epsilon-methyllysine" EXACT PSI-MI-alternate []
<br>synonym: "methyllysine" EXACT PSI-MI-short []
<br>synonym: "MLZ" EXACT PSI-MI-alternate []
<br>synonym: "N(zeta)-methyllysine" EXACT PSI-MI-alternate []
<br>synonym: "N6-methyl-L-lysine" EXACT PSI-MI-alternate []
<br>is_a: MI:0548 ! methylated-lysine
<p><br>[Term]
<br>id: MI:0166
<br>name: n6,n6-dimethyl-lysine
<br>def: "Residue modification." [PMID:11125103, RESID:AA0075]
<br>subset: PSI-MI slim
<br>synonym: "(S)-2-amino-6-dimethylaminohexanoic acid" EXACT PSI-MI-alternate []
<br>synonym: "[K:meth_2]" EXACT PSI-MI-alternate []
<br>synonym: "dimethyllysine" EXACT PSI-MI-short []
<br>synonym: "epsilon-dimethyllysine" EXACT PSI-MI-alternate []
<br>synonym: "lysine derivative Lys(y)" EXACT PSI-MI-alternate []
<br>synonym: "MLY" EXACT PSI-MI-alternate []
<br>synonym: "N(zeta)-dimethyllysine" EXACT PSI-MI-alternate []
<br>synonym: "N6,N6-dimethyl-L-lysine" EXACT PSI-MI-alternate []
<br>is_a: MI:0548 ! methylated-lysine
<p><br>[Term]
<br>id: MI:0167
<br>name: n6,n6,n6-trimethyl-lysine
<br>def: "Residue modification." [PMID:11125103, RESID:AA0074]
<br>subset: PSI-MI slim
<br>synonym: "(S)-2-amino-6-(trimethylammonio)hexanoic acid" EXACT PSI-MI-alternate []
<br>synonym: "(S)-5-amino-5-carboxy-N,N,N-trimethylpentanaminium" EXACT PSI-MI-alternate []
<br>synonym: "[K:meth_3]" EXACT PSI-MI-alternate []
<br>synonym: "epsilon-trimethyllysine" EXACT PSI-MI-alternate []
<br>synonym: "M3L" EXACT PSI-MI-alternate []
<br>synonym: "N(zeta)-trimethyllysine" EXACT PSI-MI-alternate []
<br>synonym: "N6,N6,N6-trimethyl-L-lysine" EXACT PSI-MI-alternate []
<br>synonym: "trimethyllysine" EXACT PSI-MI-short []
<br>is_a: MI:0548 ! methylated-lysine
<p><br>[Term]
<br>id: MI:0168
<br>name: n-methyl-methionine
<br>def: "Residue modification." [PMID:11125103, RESID:AA0064]
<br>subset: PSI-MI slim
<br>synonym: "(S)-2-methylamino-4-(methylsulfanyl)butanoic acid" EXACT PSI-MI-alternate []
<br>synonym: "2-methylamino-4-(methylthio)butanoic acid" EXACT PSI-MI-alternate []
<br>synonym: "[M:meth]" EXACT PSI-MI-alternate []
<br>synonym: "methylmethionine" EXACT PSI-MI-short []
<br>synonym: "MMT" EXACT PSI-MI-alternate []
<br>synonym: "N-methyl-L-methionine" EXACT PSI-MI-alternate []
<br>synonym: "N-methylmethionine" EXACT PSI-MI-alternate []
<br>is_a: MI:0157 ! methylated residue
<p><br>[Term]
<br>id: MI:0169
<br>name: n-methyl-phenylalanine
<br>def: "Residue modification." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "(S)-2-methylamino-3-phenylpropanoic acid" EXACT PSI-MI-alternate []
<br>synonym: "[F:meth]" EXACT PSI-MI-alternate []
<br>synonym: "FMT" EXACT PSI-MI-alternate []
<br>synonym: "methylphenylalanine" EXACT PSI-MI-short []
<br>synonym: "N-methyl-L-phenylalanine" EXACT PSI-MI-alternate []
<br>synonym: "N-methylphenylalanine" EXACT PSI-MI-alternate []
<br>is_a: MI:0157 ! methylated residue
<p><br>[Term]
<br>id: MI:0170
<br>name: phosphorylated residue
<br>def: "Residue modification." [PMID:11125103]
<br>subset: PSI-MI slim
<br>synonym: "phosphorylated" EXACT PSI-MI-short []
<br>is_a: MI:0120 ! post translation modification
<p><br>[Term]
<br>id: MI:0171
<br>name: omega-n-phospho-arginine
<br>def: "Residue modification." [PMID:11125103, RESID:AA0222]
<br>subset: PSI-MI slim
<br>synonym: "(S)-2-amino-5-[imino(phosphonoamino)methyl]aminopentanoic acid" EXACT PSI-MI-alternate []
<br>synonym: "[R:po]" EXACT PSI-MI-alternate []
<br>synonym: "alpha-amino-delta-phosphonoguanidinovaleric acid" EXACT PSI-MI-alternate []
<br>synonym: "N5-[imino(phosphonoamino)methyl]-L-ornithine" EXACT PSI-MI-alternate []
<br>synonym: "omega-N-phospho-L-arginine" EXACT PSI-MI-alternate []
<br>synonym: "phosphoarginine" EXACT PSI-MI-short []
<br>synonym: "RPO" EXACT PSI-MI-alternate []
<br>is_a: MI:0170 ! phosphorylated residue
<p><br>[Term]
<br>id: MI:0172
<br>name: aspartic 4-phosphoric anhydride
<br>def: "Residue modification." [PMID:11125103, RESID:AA0033]
<br>subset: PSI-MI slim
<br>synonym: "(S)-2-aminobutanedioic 4-phosphoric anhydride" EXACT PSI-MI-alternate []
<br>synonym: "[D:po]" EXACT PSI-MI-alternate []
<br>synonym: "beta-aspartyl phosphate" EXACT PSI-MI-alternate []
<br>synonym: "DPO" EXACT PSI-MI-alternate []
<br>synonym: "L-aspartic 4-phosphoric anhydride" EXACT PSI-MI-alternate []
<br>synonym: "phosphoaspartic acid" EXACT PSI-MI-alternate []
<br>synonym: "phosphoaspartic acid" EXACT PSI-MI-short []
<br>is_a: MI:0170 ! phosphorylated residue
<p><br>[Term]
<br>id: MI:0173
<br>name: s-phospho-cysteine
<br>def: "Residue modification." [PMID:11125103, RESID:AA0034]
<br>subset: PSI-MI slim
<br>synonym: "(R)-2-amino-3-(phosphonosulfanyl)propanoic acid" EXACT PSI-MI-alternate []
<br>synonym: "[C:po]" EXACT PSI-MI-alternate []
<br>synonym: "CPO" EXACT PSI-MI-alternate []
<br>synonym: "cysteine phosphate thioester" EXACT PSI-MI-alternate []
<br>synonym: "phosphocysteine" EXACT PSI-MI-short []
<br>synonym: "S-phospho-L-cysteine" EXACT PSI-MI-alternate []
<br>synonym: "S-phosphonocysteine" EXACT PSI-MI-alternate []
<br>synonym: "S3-phosphocysteine" EXACT PSI-MI-alternate []
<br>is_a: MI:0170 ! phosphorylated residue
<p><br>[Term]
<br>id: MI:0174
<br>name: 1'-phospho-histidine
<br>def: "Residue modification." [PMID:11125103, RESID:AA0035]
<br>subset: PSI-MI slim
<br>synonym: "(S)-2-amino-3-(1-phosphono-1H-imidazol-4-yl)propanoic acid" EXACT PSI-MI-alternate []
<br>synonym: "1'-phospho-L-histidine" EXACT PSI-MI-alternate []
<br>synonym: "1-phosphohistidine" EXACT PSI-MI-short []
<br>synonym: "[H:po_e]" EXACT PSI-MI-alternate []
<br>synonym: "histidine-3-phosphate [misnomer]" EXACT PSI-MI-alternate []
<br>synonym: "histidine-N(epsilon)-phosphate" EXACT PSI-MI-alternate []
<br>synonym: "histidine-N1'-phosphate" EXACT PSI-MI-alternate []
<br>synonym: "HPE" EXACT PSI-MI-alternate []
<br>synonym: "tau-phosphohistidine" EXACT PSI-MI-alternate []
<br>synonym: "tele-phosphohistidine" EXACT PSI-MI-alternate []
<br>is_a: MI:0555 ! phospho-histidine
<p><br>[Term]
<br>id: MI:0175
<br>name: 3'-phospho-histidine
<br>def: "Residue modification." [PMID:11125103, RESID:AA0036]
<br>subset: PSI-MI slim
<br>synonym: "(S)-2-amino-3-(3-phosphono-3H-imidazol-4-yl)propanoic acid" EXACT PSI-MI-alternate []
<br>synonym: "3-phosphohistidine" EXACT PSI-MI-short []
<br>synonym: "[H:po_d]" EXACT PSI-MI-alternate []
<br>synonym: "histidine-1-phosphate [misnomer]" EXACT PSI-MI-alternate []
<br>synonym: "histidine-N(delta)-phosphate" EXACT PSI-MI-alternate []
<br>synonym: "histidine-N3'-phosphate" EXACT PSI-MI-alternate []
<br>synonym: "HPD" EXACT PSI-MI-alternate []
<br>synonym: "pi-phosphohistidine" EXACT PSI-MI-alternate []
<br>synonym: "pros-phosphohistidine" EXACT PSI-MI-alternate []
<br>is_a: MI:0555 ! phospho-histidine
<p><br>[Term]
<br>id: MI:0176
<br>name: o-phospho-serine
<br>def: "Residue modification." [PMID:11125103, RESID:AA0037]
<br>subset: PSI-MI slim
<br>synonym: "(S)-2-amino-3-(phosphonooxy)propanoic acid" EXACT PSI-MI-alternate []
<br>synonym: "2-amino-3-hydroxypropanoic acid 3-phosphate" EXACT PSI-MI-alternate []
<br>synonym: "2-amino-3-hydroxypropanoic acid 3-phosphate;O-phosphonoserine;O3-phosphoserine" EXACT PSI-MI-alternate []
<br>synonym: "[S:po]" EXACT PSI-MI-alternate []
<br>synonym: "O-phospho-L-serine" EXACT PSI-MI-alternate []
<br>synonym: "O-phosphonoserine" EXACT PSI-MI-alternate []
<br>synonym: "O3-phosphoserine" EXACT PSI-MI-alternate []
<br>synonym: "phosphoserine" EXACT PSI-MI-short []
<br>synonym: "serine phosphate ester" EXACT PSI-MI-alternate []
<br>synonym: "SPO" EXACT PSI-MI-alternate []
<br>is_a: MI:0170 ! phosphorylated residue
<p><br>[Term]
<br>id: MI:0177
<br>name: o-phospho-threonine
<br>def: "Residue modification." [PMID:11125103, RESID:AA0038]
<br>subset: PSI-MI slim
<br>synonym: "(2S,3R)-2-amino-3-phosphonooxybutanoic acid" EXACT PSI-MI-alternate []
<br>synonym: "2-amino-3-hydroxybutanoic acid 3-phosphate" EXACT PSI-MI-alternate []
<br>synonym: "[T:po]" EXACT PSI-MI-alternate []
<br>synonym: "O-phospho-L-threonine" EXACT PSI-MI-alternate []
<br>synonym: "O3-phosphothreonine" EXACT PSI-MI-alternate []
<br>synonym: "phosphothreonine" EXACT PSI-MI-short []
<br>synonym: "threonine phosphate ester" EXACT PSI-MI-alternate []
<br>synonym: "TPO" EXACT PSI-MI-alternate []
<br>is_a: MI:0170 ! phosphorylated residue
<p><br>[Term]
<br>id: MI:0178
<br>name: o4'-phospho-tyrosine
<br>def: "Residue modification." [PMID:11125103, RESID:AA0039]
<br>subset: PSI-MI slim
<br>synonym: "(S)-2-amino-3-(4-phosphonooxyphenyl)propanoic acid" EXACT PSI-MI-alternate []
<br>synonym: "2-amino-3-(4-hydroxyphenyl)propanoic acid 4'-phosphate" EXACT PSI-MI-alternate []
<br>synonym: "[Y:po]" EXACT PSI-MI-alternate []
<br>synonym: "O4'-phospho-L-tyrosine" EXACT PSI-MI-alternate []
<br>synonym: "O4-phosphotyrosine" EXACT PSI-MI-alternate []
<br>synonym: "phosphotyrosine" EXACT PSI-MI-short []
<br>synonym: "tyrosine phosphate" EXACT PSI-MI-alternate []
<br>synonym: "YPO" EXACT PSI-MI-alternate []
<br>is_a: MI:0170 ! phosphorylated residue
<p><br>[Term]
<br>id: MI:0179
<br>name: other modification
<br>def: "Residue modification." [PMID:11125103]
<br>subset: PSI-MI slim
<br>is_a: MI:0120 ! post translation modification
<p><br>[Term]
<br>id: MI:0180
<br>name: selenocysteine
<br>def: "Residue modification." [PMID:11125103, RESID:AA0022]
<br>subset: PSI-MI slim
<br>synonym: "3-selenylalanine" EXACT PSI-MI-alternate []
<br>synonym: "[C:sel]" EXACT PSI-MI-alternate []
<br>synonym: "CSE" EXACT PSI-MI-alternate []
<br>synonym: "L-selenocysteine" EXACT PSI-MI-alternate []
<br>synonym: "selenium cysteine" EXACT PSI-MI-alternate []
<br>is_a: MI:0179 ! other modification
<p><br>[Term]
<br>id: MI:0181
<br>name: selenomethionine
<br>def: "Residue modification." [PMID:11125103]
<br>subset: PSI-MI slim
<br>synonym: "[M:sel]" EXACT PSI-MI-alternate []
<br>synonym: "L-selenomethionine" EXACT PSI-MI-alternate []
<br>synonym: "MSE" EXACT PSI-MI-alternate []
<br>is_a: MI:0179 ! other modification
<p><br>[Term]
<br>id: MI:0182
<br>name: 3-oxoalanine
<br>def: "Residue modification." [PMID:11125103, RESID:AA0185]
<br>subset: PSI-MI slim
<br>synonym: "(S)-2-amino-3-oxopropanoic acid" EXACT PSI-MI-alternate []
<br>synonym: "2-amino-3-oxopropionic acid" EXACT PSI-MI-alternate []
<br>synonym: "[S:oxal]" EXACT PSI-MI-alternate []
<br>synonym: "L-3-oxoalanine" EXACT PSI-MI-alternate []
<br>synonym: "L-alpha-formylglycine" EXACT PSI-MI-alternate []
<br>synonym: "L-amino-malonic acid semialdehyde" EXACT PSI-MI-alternate []
<br>synonym: "L-aminomalonaldehydic acid" EXACT PSI-MI-alternate []
<br>synonym: "L-serinesemialdehyde [misnomer]" EXACT PSI-MI-alternate []
<br>synonym: "oxoalanine" EXACT PSI-MI-short []
<br>synonym: "SOX" EXACT PSI-MI-alternate []
<br>is_a: MI:0179 ! other modification
<p><br>[Term]
<br>id: MI:0183
<br>name: 2-pyrrolidone-5-carboxylic acid
<br>def: "Residue modification." [PMID:11125103, RESID:AA0031]
<br>subset: PSI-MI slim
<br>synonym: "(S)-5-oxo-2-pyrrolidinecarboxylic acid" EXACT PSI-MI-alternate []
<br>synonym: "2-oxopyrrolidine-5-carboxylic acid" EXACT PSI-MI-alternate []
<br>synonym: "5-oxoproline" EXACT PSI-MI-alternate []
<br>synonym: "5-oxopyrrolidine-2-carboxylic acid" EXACT PSI-MI-alternate []
<br>synonym: "[E:pyro]" EXACT PSI-MI-alternate []
<br>synonym: "pyroglutamic acid" EXACT PSI-MI-alternate []
<br>synonym: "pyroglutamic acid" EXACT PSI-MI-short []
<br>is_a: MI:0179 ! other modification
<p><br>[Term]
<br>id: MI:0184
<br>name: glutamyl 5-glycerylphosphorylethanolamine
<br>def: "Residue modification." [PMID:11125103, RESID:AA0170]
<br>subset: PSI-MI slim
<br>synonym: "(S)-2-amino-5-[2-([([2,3-dihydroxypropyl]oxy)(hydroxy)phosphoryl]oxy)ethyl]amino-5-oxopentanoic acid" EXACT PSI-MI-alternate []
<br>synonym: "[E:gpe]" EXACT PSI-MI-alternate []
<br>synonym: "EGE" EXACT PSI-MI-alternate []
<br>synonym: "glycerylpo4etohamine" EXACT PSI-MI-short []
<br>synonym: "L-glutamyl 5-glycerophosphoethanolamine" EXACT PSI-MI-alternate []
<br>synonym: "L-glutamyl 5-glycerophosphorylethanolamine" EXACT PSI-MI-alternate []
<br>synonym: "L-glutamyl 5-glycerylphosphorylethanolamine" EXACT PSI-MI-alternate []
<br>is_a: MI:0179 ! other modification
<p><br>[Term]
<br>id: MI:0185
<br>name: 2'-[3-carboxamido-3-(trimethylammonio)propyl]-histidine
<br>def: "Residue modification." [PMID:11125103, RESID:AA0040]
<br>subset: PSI-MI slim
<br>synonym: "2'-[3-carboxamido-3-(trimethylammonio)propyl]-L-histidine" EXACT PSI-MI-alternate []
<br>synonym: "2-[(Xi)-3-carboxamido-3-(trimethylammonio)propyl]-4-((S)-2-amino-2-carboxyethyl)-1H-imidazole" EXACT PSI-MI-alternate []
<br>synonym: "2-[3-carboxamido-3-(trimethylammonio)propyl]histidine" EXACT PSI-MI-alternate []
<br>synonym: "2-amino-3-[[2-(3-amino-3-carbamoyl-prop-1-enyl)-1,1,3-trimethyl-2,3-dihydroimidazol-5-yl]]propanoic acid" EXACT PSI-MI-alternate []
<br>synonym: "2-amino-4-[[5-(2-amino-2-carboxylato-ethyl)-1,1,3-trimethyl-2,3-dihydroimidazol-2-yl]]but-3-enamide" EXACT PSI-MI-alternate []
<br>synonym: "[H:diph]" EXACT PSI-MI-alternate []
<br>synonym: "alpha-(aminocarbonyl)-4-(2-amino-2-carboxyethyl)-N,N,N-trimethyl-1H-imidazole-2-propanaminium" EXACT PSI-MI-alternate []
<br>synonym: "diphthamide" EXACT PSI-MI-alternate []
<br>synonym: "diphthamide" EXACT PSI-MI-short []
<br>synonym: "HDP" EXACT PSI-MI-alternate []
<br>is_a: MI:0179 ! other modification
<p><br>[Term]
<br>id: MI:0186
<br>name: n6-biotinyl-lysine
<br>def: "Residue modification." [PMID:11125103, RESID:AA0117]
<br>subset: PSI-MI slim
<br>synonym: "(3aS-(3aalpha,4beta,6aalpha))-N6-(5-(hexahydro-2-oxo-1H-thieno(3,4-d)imidazol-4-yl)-1-oxopentyl)-L-lysine" EXACT PSI-MI-alternate []
<br>synonym: "(S)-2-amino-6-[5-((3aS,4S,6aR)-hexahydro-2-oxo-1H-thieno[3,4-d]imidazol-4-yl)-1-oxopentyl]aminohexanoic acid" EXACT PSI-MI-alternate []
<br>synonym: "[K:biotin]" EXACT PSI-MI-alternate []
<br>synonym: "biocytin" EXACT PSI-MI-alternate []
<br>synonym: "biotinyllysine" EXACT PSI-MI-short []
<br>synonym: "epsilon-N-biotinyllysine" EXACT PSI-MI-alternate []
<br>synonym: "KBT" EXACT PSI-MI-alternate []
<br>synonym: "N6-[5-((3aS,4S,6aR)-hexahydro-2-oxo-1H-thieno[3,4-d]imidazol-4-yl)-1-oxopentyl]-L-lysine" EXACT PSI-MI-alternate []
<br>synonym: "N6-biotinyl-L-lysine" EXACT PSI-MI-alternate []
<br>synonym: "N6-biotinyllysine" EXACT PSI-MI-alternate []
<br>is_a: MI:0179 ! other modification
<p><br>[Term]
<br>id: MI:0187
<br>name: n6-(4-amino-2-hydroxybutyl)-lysine
<br>def: "Residue modification." [PMID:11125103, RESID:AA0116]
<br>subset: PSI-MI slim
<br>synonym: "(S,R)-2-amino-6-(4-amino-2-hydroxybutylamino)hexanoic acid" EXACT PSI-MI-alternate []
<br>synonym: "[K:hypu]" EXACT PSI-MI-alternate []
<br>synonym: "hypusine" EXACT PSI-MI-alternate []
<br>synonym: "hypusine" EXACT PSI-MI-short []
<br>synonym: "KHY" EXACT PSI-MI-alternate []
<br>synonym: "N6-(4-amino-2-hydroxybutyl)-L-lysine" EXACT PSI-MI-alternate []
<br>is_a: MI:0179 ! other modification
<p><br>[Term]
<br>id: MI:0188
<br>name: n6-retinal-lysine
<br>def: "Residue modification." [PMID:11125103, RESID:AA0120]
<br>subset: PSI-MI slim
<br>synonym: "(S)-2-amino-6-[(2E,4E,6E,8E)-3,7-dimethyl-9-(2,6,6-trimethylcyclohex-1-en-1-yl)-2,4,6,8-nonatetraenylidene]aminohexanoic acid" EXACT PSI-MI-alternate []
<br>synonym: "[K:retin]" EXACT PSI-MI-alternate []
<br>synonym: "KRT" EXACT PSI-MI-alternate []
<br>synonym: "N6-retinal-L-lysine" EXACT PSI-MI-alternate []
<br>synonym: "N6-retinyl-lysine" EXACT PSI-MI-alternate []
<br>synonym: "N6-retinylidene-L-lysine" EXACT PSI-MI-alternate []
<br>synonym: "retinallysine" EXACT PSI-MI-short []
<br>is_a: MI:0179 ! other modification
<p><br>[Term]
<br>id: MI:0189
<br>name: ubiquitinated lysine
<br>def: "Residue modification due to a cross-link between a lysine and a glycine from the ubiquitine protein." [PMID:11125103, RESID:AA0125]
<br>subset: PSI-MI slim
<br>synonym: "[K:ub]" EXACT PSI-MI-alternate []
<br>synonym: "KUB" EXACT PSI-MI-alternate []
<br>synonym: "N6-glycyl-L-lysine" EXACT PSI-MI-alternate []
<br>synonym: "N6-glycyllysine" EXACT PSI-MI-alternate []
<br>is_a: MI:0179 ! other modification
<p><br>[Term]
<br>id: MI:0190
<br>name: interaction type
<br>def: "Connection between molecule." [PMID:14755292]
<br>subset: PSI-MI slim
<br>relationship: part_of MI:0000 ! molecular interaction
<p><br>[Term]
<br>id: MI:0191
<br>name: aggregation
<br>def: "Physical association among molecules.\nOBSOLETE since too non-specific. Consider using physical interaction (MI:0218) instead." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_obsolete: true
<p><br>[Term]
<br>id: MI:0192
<br>name: acetylation reaction
<br>def: "Reaction, that can affect K,C,A,D,E,Q,G,I,K,M,P,S,T,Y,V residues." [GO:0006473, PMID:14755292, RESID:AA0041, RESID:AA0042, RESID:AA0043, RESID:AA0044, RESID:AA0045, RESID:AA0046, RESID:AA0047, RESID:AA0048, RESID:AA0049, RESID:AA0050, RESID:AA0051, RESID:AA0052, RESID:AA0053, RESID:AA0054, RESID:AA0055, RESID:AA0056]
<br>subset: PSI-MI slim
<br>synonym: "acetylation" EXACT PSI-MI-short []
<br>is_a: MI:0414 ! enzymatic reaction
<p><br>[Term]
<br>id: MI:0193
<br>name: amidation reaction
<br>def: "Irreversible reaction that can affect A,R,N,D,C,Q,E,G,H,I,L,K,M,F,P,S,T,W,Y or V residues." [GO:0001519, PMID:14755292, RESID:AA0081, RESID:AA0082, RESID:AA0083, RESID:AA0084, RESID:AA0085, RESID:AA0086, RESID:AA0087, RESID:AA0088, RESID:AA0089, RESID:AA0090, RESID:AA0091, RESID:AA0092, RESID:AA0093, RESID:AA0094, RESID:AA0095, RESID:AA0096, RESID:AA0097, RESID:AA0098, RESID:AA0099, RESID:AA0100]
<br>subset: PSI-MI slim
<br>synonym: "amidation" EXACT PSI-MI-short []
<br>is_a: MI:0414 ! enzymatic reaction
<p><br>[Term]
<br>id: MI:0194
<br>name: cleavage reaction
<br>def: "Covalent bond breakage in a molecule leading to the formation of smaller molecules." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "cleavage" EXACT PSI-MI-short []
<br>is_a: MI:0414 ! enzymatic reaction
<p><br>[Term]
<br>id: MI:0195
<br>name: covalent binding
<br>def: "Interaction leading to the formation of covalent bond within an autocatalytic molecule or between partners." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0407 ! direct interaction
<p><br>[Term]
<br>id: MI:0196
<br>name: covalent interaction
<br>def: "Physical interaction mediated by covalent bond rearrangement.\nOBSOLETE use covalent binding (MI:0195) instead." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_obsolete: true
<p><br>[Term]
<br>id: MI:0197
<br>name: deacetylation reaction
<br>def: "N6-acetyl-L-lysine or S-acetyl-L-cysteine are cleaved and return K or C residues." [GO:0006476, PMID:14755292, RESID:AA0055, RESID:AA0056]
<br>subset: PSI-MI slim
<br>synonym: "deacetylation" EXACT PSI-MI-short []
<br>is_a: MI:0414 ! enzymatic reaction
<p><br>[Term]
<br>id: MI:0198
<br>name: defarnesylation reaction
<br>def: "S-farnesyl-L-cysteined is cleaved and returns a C residue." [PMID:14755292, RESID:AA0102]
<br>subset: PSI-MI slim
<br>synonym: "defarnesylation" EXACT PSI-MI-short []
<br>is_a: MI:0212 ! lipid cleavage
<p><br>[Term]
<br>id: MI:0199
<br>name: deformylation reaction
<br>def: "N6-formyl-L-lysine is cleaved and returns a K residue." [PMID:14755292, RESID:AA0211]
<br>subset: PSI-MI slim
<br>synonym: "deformylation" EXACT PSI-MI-short []
<br>is_a: MI:0414 ! enzymatic reaction
<p><br>[Term]
<br>id: MI:0200
<br>name: degeranylation reaction
<br>def: "S-geranylgeranyl-L-cysteine is cleaved and returns a C residue." [PMID:14755292, RESID:AA0104]
<br>subset: PSI-MI slim
<br>synonym: "degeranylation" EXACT PSI-MI-short []
<br>is_a: MI:0212 ! lipid cleavage
<p><br>[Term]
<br>id: MI:0201
<br>name: demyristoylation reaction
<br>def: "N6-myristoyl-L-lysine is cleaved and returns a K residue." [PMID:14755292, RESID:AA0078]
<br>subset: PSI-MI slim
<br>synonym: "demyristoylation" EXACT PSI-MI-short []
<br>is_a: MI:0212 ! lipid cleavage
<p><br>[Term]
<br>id: MI:0202
<br>name: depalmitoylation reaction
<br>def: "S-palmitoyl-L-cysteine, N6-palmitoyl-L-lysine, O-palmitoyl-L-threonine or O-palmitoyl-L-serine are cleaved and return C,K,T or S residues." [PMID:14755292, RESID:AA0060, RESID:AA0077, RESID:AA0106]
<br>subset: PSI-MI slim
<br>synonym: "depalmitoylation" EXACT PSI-MI-short []
<br>is_a: MI:0212 ! lipid cleavage
<p><br>[Term]
<br>id: MI:0203
<br>name: dephosphorylation reaction
<br>def: "Phosphoresidues are cleaved and return D,C,H,S,T,Y or R residues." [GO:0016311, PMID:14755292, RESID:AA0033, RESID:AA0034, RESID:AA0035, RESID:AA0036, RESID:AA0037, RESID:AA0038, RESID:AA0039, RESID:AA0222]
<br>subset: PSI-MI slim
<br>synonym: "dephosphorylation" EXACT PSI-MI-short []
<br>is_a: MI:0414 ! enzymatic reaction
<p><br>[Term]
<br>id: MI:0204
<br>name: deubiquitination reaction
<br>def: "Cleavage of the G-K bond and release of ubiquitin or ubiquitin like proteins." [GO:0016579, PMID:11583613, RESID:AA0125]
<br>subset: PSI-MI slim
<br>synonym: "deubiquitination" EXACT PSI-MI-short []
<br>is_a: MI:0414 ! enzymatic reaction
<p><br>[Term]
<br>id: MI:0205
<br>name: disaggregation
<br>def: "Dissociation of partners interacting via non-covalent bond.\nOBSOLETE because considered misleading." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_obsolete: true
<p><br>[Term]
<br>id: MI:0206
<br>name: farnesylation reaction
<br>def: "Reversible reaction that can affect C residue." [GO:0018347, PMID:14755292, RESID:AA0102]
<br>subset: PSI-MI slim
<br>synonym: "farnesylation" EXACT PSI-MI-short []
<br>is_a: MI:0211 ! lipid addition
<p><br>[Term]
<br>id: MI:0207
<br>name: formylation reaction
<br>def: "Reaction that can affect K or G residues." [GO:0018256, PMID:14755292, RESID:AA0057, RESID:AA0211]
<br>subset: PSI-MI slim
<br>synonym: "formylation" EXACT PSI-MI-short []
<br>is_a: MI:0414 ! enzymatic reaction
<p><br>[Term]
<br>id: MI:0208
<br>name: genetic interaction
<br>def: "Two genes A and B \"genetically interact\" when the phenotype generated as the result of mutations in both genes (double mutant ab) is unexpectedly not just a combination of the phenotypes of the two single mutants a and b." [PMID:16527956]
<br>subset: PSI-MI slim
<br>is_a: MI:0190 ! interaction type
<p><br>[Term]
<br>id: MI:0209
<br>name: geranylgeranylation reaction
<br>def: "Reversible reaction that can affect C residue." [GO:0018348, PMID:14755292, RESID:AA0104]
<br>subset: PSI-MI slim
<br>synonym: "geranylgeranylation" EXACT PSI-MI-short []
<br>is_a: MI:0211 ! lipid addition
<p><br>[Term]
<br>id: MI:0210
<br>name: hydroxylation reaction
<br>def: "Irreversible reaction that can affect K,P,Y or R residues." [GO:0018126, PMID:14755292, RESID:AA0028, RESID:AA0029, RESID:AA0030, RESID:AA0146, RESID:AA0215]
<br>subset: PSI-MI slim
<br>synonym: "hydroxylation" EXACT PSI-MI-short []
<br>is_a: MI:0414 ! enzymatic reaction
<p><br>[Term]
<br>id: MI:0211
<br>name: lipid addition
<br>def: "Covalent or non covalent binding of lipid group on a protein residue." [GO:0006497, PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0414 ! enzymatic reaction
<p><br>[Term]
<br>id: MI:0212
<br>name: lipid cleavage
<br>def: "Cleavage of a lipid group covalently bound to a protein residue." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0194 ! cleavage reaction
<p><br>[Term]
<br>id: MI:0213
<br>name: methylation reaction
<br>def: "The covalent attachment of a methyl residue to one or more monomeric units in a polypeptide, polynucleotide, polysaccharide, or other biological polymer. Irreversible reaction that can affect A,G,M,F,P,C,R,N,Q,E,H,or K residues." [GO:0043414, PMID:14755292, RESID:AA0061, RESID:AA0062, RESID:AA0063, RESID:AA0064, RESID:AA0065, RESID:AA0066, RESID:AA0067, RESID:AA0068, RESID:AA0069, RESID:AA0070, RESID:AA0071, RESID:AA0072, RESID:AA0073, RESID:AA0074, RESID:AA0075, RESID:AA0076, RESID:AA0234, RESID:AA0272]
<br>subset: PSI-MI slim
<br>synonym: "methylation" EXACT PSI-MI-short []
<br>is_a: MI:0414 ! enzymatic reaction
<p><br>[Term]
<br>id: MI:0214
<br>name: myristoylation reaction
<br>def: "Reaction that can affect K or G residues." [GO:0018319, PMID:14755292, RESID:AA0059, RESID:AA0078]
<br>subset: PSI-MI slim
<br>synonym: "myristoylation" EXACT PSI-MI-short []
<br>is_a: MI:0211 ! lipid addition
<p><br>[Term]
<br>id: MI:0215
<br>name: non covalent interaction
<br>def: "Interaction mediated by non-covalent, weak forces rearrangement.\nOBSOLETE use physical interaction (MI:0218) instead." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "non covalent inter" EXACT PSI-MI-short []
<br>is_obsolete: true
<p><br>[Term]
<br>id: MI:0216
<br>name: palmitoylation reaction
<br>def: "Reversible reaction that can affect C,K,T or S residues." [GO:0018318, PMID:14755292, RESID:AA0060, RESID:AA0077, RESID:AA0079, RESID:AA0080, RESID:AA0106]
<br>subset: PSI-MI slim
<br>synonym: "palmitoylation" EXACT PSI-MI-short []
<br>is_a: MI:0211 ! lipid addition
<p><br>[Term]
<br>id: MI:0217
<br>name: phosphorylation reaction
<br>def: "Reversible reaction that can affect D,C,H,S,T,Y,R residues." [GO:0016310, PMID:14755292, RESID:AA0033, RESID:AA0034, RESID:AA0035, RESID:AA0036, RESID:AA0037, RESID:AA0038, RESID:AA0039, RESID:AA0222]
<br>subset: PSI-MI slim
<br>synonym: "phosphorylation" EXACT PSI-MI-short []
<br>is_a: MI:0414 ! enzymatic reaction
<p><br>[Term]
<br>id: MI:0218
<br>name: physical interaction
<br>def: "Interaction among molecules that can be direct or indirect.\nOBSOLETE: splitted to 'association' MI:0914 and 'physical association' MI:0915. For remapping consider the experimental setting of an interaction. For bulk remapping a possible criteria is to whatever physical interaction that has among its participant a bait should become 'association' MI:0914 the others can become 'physical association' MI:0915. Two hybrid interactions are an expection and can be 'physical association' MI:0915." [PMID:14755292]
<br>is_obsolete: true
<p><br>[Term]
<br>id: MI:0219
<br>name: synthetic lethal
<br>def: "Death phenotype observed on cells carrying combination of two independently silent mutations.\nOBSOLETE: remap to CV intraction type 'synthetic interaction' MI:0794 and external CV for phenotype description (lethal FBcv:0000351)" [PMID:15608217]
<br>subset: PSI-MI slim
<br>is_obsolete: true
<p><br>[Term]
<br>id: MI:0220
<br>name: ubiquitination reaction
<br>def: "Reversible reaction that create a covalent bond between a C-terminus G of ubiquitin and a K residue of the target." [GO:0016567, PMID:11583613, RESID:AA0125]
<br>subset: PSI-MI slim
<br>synonym: "ubiquitination" EXACT PSI-MI-short []
<br>is_a: MI:0414 ! enzymatic reaction
<p><br>[Term]
<br>id: MI:0221
<br>name: expression level
<br>def: "Synthesis rate of a molecule under investigation described in comparison with its naturally occurring expression level in a cell." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0346 ! experimental preparation
<p><br>[Term]
<br>id: MI:0222
<br>name: physiological level
<br>def: "A molecule whose synthesis is under control of its natural gene promoter or estimated to be expressed at a similar rate." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "endogenous" EXACT PSI-MI-alternate []
<br>synonym: "endogenous level" EXACT PSI-MI-alternate []
<br>is_a: MI:0221 ! expression level
<p><br>[Term]
<br>id: MI:0223
<br>name: under expressed level
<br>def: "A molecule is estimated to be expressed at lower levels than in physiological condition." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "under-expressed" EXACT PSI-MI-short []
<br>is_a: MI:0221 ! expression level
<br>is_a: MI:0803 ! expression level alteration
<p><br>[Term]
<br>id: MI:0225
<br>name: chromatin immunoprecipitation array
<br>def: "The method combines a modified chromatin immunoprecipitation (ChIP) procedure, with DNA microarray analysis. Cells are fixed with formaldehyde, harvested, and disrupted by sonication. The DNA fragments cross-linked to a protein of interest are enriched by immunoprecipitation with a specific antibody. After reversal of the cross-links, the enriched DNA is amplified and labeled with a fluorescent dye (Cy5) by using a ligation-mediatedpolymerase chain reaction (LM-PCR). In parallel a sample of DNA that is not enriched by immunoprecipitation is subjected to LM-PCR in the presence of a different fluorophore (Cy3), and both immunoprecipitation (IP)-enriched and unenriched pools of labeled DNA were hybridized to a single DNA microarray containing a set of intergenic sequences. The ratio of the Cy5 to Cy3 fluorescence intensities measured at each DNA element in the microarray provided a measure of the extent of binding of the transcription factor to the corresponding genomic locus." [PMID:11125145, PMID:11206552]
<br>subset: PSI-MI slim
<br>synonym: "chip-chip" EXACT PSI-MI-short []
<br>is_a: MI:0008 ! array technology
<br>is_a: MI:0402 ! chromatin immunoprecipitation assay
<p><br>[Term]
<br>id: MI:0226
<br>name: ion exchange chromatography
<br>def: "Stable complexes and their component proteins can be separated on the basis of their net charge by ion-exchange chromatography. If a protein has a net positive charge at pH 7, it will usually bind to a column of beads containing carboxylate groups, and can then be eluted by increasing the concentration of sodium chloride or another salt in the eluting buffer by competition of sodium ions with positively charged groups on the protein for binding to the column. Protein that have a low density of net positive charge will tend to emerge first, followed by those having a higher charge density. Positively charged complexes or proteins (cationic proteins) can be separated on negatively charged carboxymethyl-cellulose (CM-cellulose) columns. Conversely, negatively charged complexes or proteins (anionic proteins) can be separated by chromatography on positively charged diethylaminoethyl-cellulose (DEAE-cellulose) columns." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "IEC" EXACT PSI-MI-alternate []
<br>synonym: "ion exchange chrom" EXACT PSI-MI-short []
<br>is_a: MI:0091 ! chromatography technology
<p><br>[Term]
<br>id: MI:0227
<br>name: reverse phase chromatography
<br>def: "Reverse phase chromatography operates on the basis of hydrophilicity and lipophilicity. The stationary phase consists of silica based packings with n-alkyl chains covalently bound. For example, C-8 signifies an octyl chain and C-18 an octadecyl ligand in the matrix. The more hydrophobic the matrix on each ligand, the greater is the tendency of the column to retain hydrophobic moieties. Thus hydrophilic compounds elute more quickly than do hydrophobic compounds." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "reverse phase chrom" EXACT PSI-MI-short []
<br>is_a: MI:0091 ! chromatography technology
<p><br>[Term]
<br>id: MI:0228
<br>name: cytoplasmic complementation assay
<br>def: "Protein complementation assay performed by dissecting a cytoplasmic protein activity and restoring it through the two hybrid proteins interaction." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "cytoplasmic compl" EXACT PSI-MI-short []
<br>is_a: MI:0090 ! protein complementation assay
<p><br>[Term]
<br>id: MI:0229
<br>name: green fluorescence protein complementation assay
<br>def: "Protein-protein interaction trap based on fusions of bait and prey protein to two dissected fragment of GFP. The system implemented in E. coli, allow screening of partners, including membrane proteins and also detect transient interaction that are stabilized by the complemented GFP." [PMID:15631464]
<br>subset: PSI-MI slim
<br>synonym: "gfp complementation" EXACT PSI-MI-short []
<br>is_a: MI:0228 ! cytoplasmic complementation assay
<p><br>[Term]
<br>id: MI:0230
<br>name: membrane bound complementation assay
<br>def: "Protein complementation assay based on dissection of a membrane protein." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "membrane compl" EXACT PSI-MI-short []
<br>is_a: MI:0090 ! protein complementation assay
<p><br>[Term]
<br>id: MI:0231
<br>name: mammalian protein protein interaction trap
<br>def: "The MAPPIT(mammalian protein-protein interaction Trap) is a screening method for protein-protein interaction in mammalian cells, based on the reconstitution of a membrane STAT (signal transducers and activators of transcription) receptor. The bait protein is fused to a STAT recruitment-deficient receptor and the prey protein to a functional STAT recruitment sites. In such a configuration, a given baitprey interaction restores a STAT-dependent responses leading to the expression of a reporter gene. This system, enable to demonstrate not only protein interaction but also modification-independent and tyrosine phosphorylation- dependent interactions." [PMID:12853652]
<br>subset: PSI-MI slim
<br>synonym: "mappit" EXACT PSI-MI-short []
<br>is_a: MI:0230 ! membrane bound complementation assay
<p><br>[Term]
<br>id: MI:0232
<br>name: transcriptional complementation assay
<br>def: "Protein complementation assay performed by dissecting a transcription factor activity (DNA binding domain and transcription activation domain) its restoration through the two hybrid proteins interaction that lead to a reporter gene expression." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "transcription compl" EXACT PSI-MI-short []
<br>is_a: MI:0090 ! protein complementation assay
<p><br>[Term]
<br>id: MI:0233
<br>name: protein dna complex
<br>def: "A stable set of interacting protein and DNA that can be copurified and operate as a functional unit." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0314 ! complex
<p><br>[Term]
<br>id: MI:0234
<br>name: 131i radiolabel
<br>def: "Molecule labelled with 131 radio isotope of iodine atoms." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "131I" EXACT PSI-MI-alternate []
<br>synonym: "I131" EXACT PSI-MI-alternate []
<br>is_a: MI:0517 ! radiolabel
<p><br>[Term]
<br>id: MI:0235
<br>name: 14c radiolabel
<br>def: "Molecule labelled with the radio isotope 14 of carbon atoms." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "14C" EXACT PSI-MI-alternate []
<br>synonym: "C14" EXACT PSI-MI-alternate []
<br>is_a: MI:0517 ! radiolabel
<p><br>[Term]
<br>id: MI:0236
<br>name: 32p radiolabel
<br>def: "Molecule labelled with the radio isotope 32 of phosphorus atoms." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "32P" EXACT PSI-MI-alternate []
<br>synonym: "P32" EXACT PSI-MI-alternate []
<br>is_a: MI:0517 ! radiolabel
<p><br>[Term]
<br>id: MI:0237
<br>name: 33p radiolabel
<br>def: "Molecule labelled with the radio isotope 33 of phosphorus atoms." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "33P" EXACT PSI-MI-alternate []
<br>synonym: "P33" EXACT PSI-MI-alternate []
<br>is_a: MI:0517 ! radiolabel
<p><br>[Term]
<br>id: MI:0238
<br>name: 3h radiolabel
<br>def: "Molecules labelled with isotope 3 of hydrogen atoms." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "3H" EXACT PSI-MI-alternate []
<br>synonym: "H3" EXACT PSI-MI-alternate []
<br>synonym: "tritium" EXACT PSI-MI-alternate []
<br>is_a: MI:0517 ! radiolabel
<p><br>[Term]
<br>id: MI:0239
<br>name: biotin tag
<br>def: "Biotin, a 244 Dalton vitamin found in tissue and blood, binds with high affinity to avidin and streptavidin protein. Since biotin is a relatively small molecule, it can be conjugated to many proteins or nucleic acids without significantly altering their biological activity. Biotinylation reagents are available for targeting a variety of specific functional groups, including primary amines, sulfhydryls, carboxyls and carbohydrates that lead to nucleotides or amino acid biotinilation." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0507 ! tag
<p><br>[Term]
<br>id: MI:0240
<br>name: fusion protein
<br>def: "The protein under study is expressed as a fusion with a labelling protein, having either fluorescence properties or an enzymatic activity that facilitates its purification, identification, localisation or quantification." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0507 ! tag
<p><br>[Term]
<br>id: MI:0241
<br>name: horseradish peroxidase tag
<br>def: "Protein is fused to horseradish peroxidase, and the measure of this enzyme activity can be taken as indicative of presence of protein." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "hrp tag" EXACT PSI-MI-short []
<br>is_a: MI:0365 ! enzyme tag
<p><br>[Term]
<br>id: MI:0242
<br>name: gene ontology definition reference
<br>def: "This qualifier is used when the crossreference is imported from the Gene Ontology tag definition_reference." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "go-definition-ref" EXACT PSI-MI-short []
<br>is_a: MI:0353 ! cross-reference type
<p><br>[Term]
<br>id: MI:0243
<br>name: isoform parent sequence reference
<br>def: "Reference to the master sequence from which this isoform has been derived." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "isoform-parent" EXACT PSI-MI-short []
<br>is_a: MI:0353 ! cross-reference type
<p><br>[Term]
<br>id: MI:0244
<br>name: reactome complex
<br>def: "Collection of functional complexes within Reactome - a knowledgebase of biological processes.\nhttp://www.reactome.org/" [PMID:14755292]
<br>subset: PSI-MI slim
<br>xref: id-validation-regexp:\"REACT_[0-9\]\{1\,4}\\.[0-9\]\{1\,3}|[0-9\]+\"
<br>xref: search-url: "http://www.reactome.org/cgi-bin/eventbrowser?ID=${ac}"
<br>is_a: MI:0467 ! reactome
<p><br>[Term]
<br>id: MI:0245
<br>name: reactome protein
<br>def: "Collection of protein within the Reactome database - a knowledgebase of biological processes.\nhttp://www.reactome.org/" [PMID:14755292]
<br>subset: PSI-MI slim
<br>xref: id-validation-regexp:\"REACT_[0-9\]\{1\,4}\\.[0-9\]\{1\,3}|[OPQ\][0-9\][A-Z0-9\]\{3}[0-9\]|[OPQ\][0-9\][A-Z0-9\]\{3}[0-9\]-[0-9\]+|[A-Z\]\{3}[0-9\]\{5}|[OPQ\][0-9\][A-Z0-9\]\{3}[0-9\]-PRO_[0-9\]\{10}\"
<br>xref: search-url: "http://www.reactome.org/cgi-bin/search?SUBMIT=1&QUERY_CLASS=DatabaseIdentifier&QUERY=SWALL:${ac}"
<br>is_a: MI:0467 ! reactome
<p><br>[Term]
<br>id: MI:0246
<br>name: cabri
<br>def: "CABRI cell lines catalogue available at.\nhttp://www.cabri.org/" [PMID:14755292]
<br>subset: PSI-MI slim
<br>xref: id-validation-regexp:\"[0-9\]+|ACC\\s[A-Z0-9\]+|ECACC\\s[A-Z0-9\]+|LMBP\\s[A-Z0-9\]+|ICLC\\s[A-Z0-9\]+|CIP-[0-9\]+|DSMZ_MUTZ\\\:ACC\\s[0-9\]+ \"
<br>xref: search-url: "http://srs.ebi.ac.uk/srsbin/cgi-bin/wgetz?[libs%3d{dsmz_mutz%20ecacc_cell%20iclc%20CABI_BACT%20CIP_BACT%20CABI_YEAST%20DSMZ_PLANT_CELL}-id:${ac}]+-e"
<br>is_a: MI:0473 ! participant database
<p><br>[Term]
<br>id: MI:0247
<br>name: newt
<br>def: "New EBI Web Taxonomy.\nhttp://www.ebi.ac.uk/newt OBSOLETE: Consider remapping to uniprot taxonomy MI:0942" [PMID:14755292]
<br>subset: PSI-MI slim
<br>xref: id-validation-regexp:\"[0-9\]+\"
<br>xref: search-url: "http://www.ebi.ac.uk/newt/display?search=${ac}"
<br>is_obsolete: true
<p><br>[Term]
<br>id: MI:0248
<br>name: resid
<br>def: "The RESID Database of Protein Modifications is a comprehensive collection of annotations and structures for protein modifications including amino-terminal, carboxyl-terminal and peptide chain cross-link post-translational modifications.\nhttp://www.ebi.ac.uk/RESID/index.html" [PMID:14755292]
<br>subset: PSI-MI slim
<br>xref: id-validation-regexp:\"AA[0-9\]\{4}\"
<br>xref: search-url: "http://srs.ebi.ac.uk/cgi-bin/wgetz?[resid-id:${ac}]+-e"
<br>is_a: MI:0447 ! feature database
<br>is_a: MI:0473 ! participant database
<p><br>[Term]
<br>id: MI:0249
<br>name: huge
<br>def: "A Database of Human Unidentified Gene-Encoded Large Proteins Analyzed by Kazusa Human cDNA Project.\nhttp://www.kazusa.or.jp/huge/" [PMID:14755292]
<br>subset: PSI-MI slim
<br>xref: id-validation-regexp:\"KIAA[0-9\]\{4}[A-Z\]\{0\,1}\"
<br>xref: search-url: "http://www.kazusa.or.jp/huge/gfpage/${ac}"
<br>is_a: MI:0683 ! sequence database
<p><br>[Term]
<br>id: MI:0250
<br>name: gene
<br>def: "interactor for genetic interaction." [PMID:14755292, SO:0000704]
<br>subset: PSI-MI slim
<br>is_a: MI:0313 ! interactor type
<p><br>[Term]
<br>id: MI:0251
<br>name: gene product
<br>def: "Reference of a protein object pointing to its genomic or nucleic acid sequence." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0353 ! cross-reference type
<p><br>[Term]
<br>id: MI:0252
<br>name: biological feature
<br>def: "Property of a subsequence that may be involved with or interfere with the binding of a molecule and are supported by experimental evidences." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0116 ! feature type
<p><br>[Term]
<br>id: MI:0253
<br>name: isotope label
<br>def: "One of several nuclides having the same number of protons in their nuclei and hence having the same atomic number, but differing in the number of neutrons and therefore, in the mass number." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0505 ! experimental feature
<p><br>[Term]
<br>id: MI:0254
<br>name: genetic interference
<br>def: "This term refers to methods that aim at interfering with the activity of a specific gene by altering the gene regulatory or coding sequences. This goal can be achieved either by a classical genetic approach (random mutagenesis followed by phenotype characterization and genetic mapping) or by a reverse genetics approach where a gene of interest is modified by directed mutagenesis." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0045 ! experimental interaction detection
<p><br>[Term]
<br>id: MI:0255
<br>name: post transcriptional interference
<br>def: "This term refers to methods designed to interfere with gene expression at post-transcriptional level rather than with the gene itself." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "expression interfer" EXACT PSI-MI-short []
<br>is_a: MI:0045 ! experimental interaction detection
<p><br>[Term]
<br>id: MI:0256
<br>name: rna interference
<br>def: "RNA interference (RNAi) is a post-transcriptional gene silencing method reproducing a naturally occurring phenomena. RNAi is the process whereby double-stranded RNA (dsRNA) induces the sequence-specific degradation of homologous mRNA. RNAi or dsRNA-induced silencing phenomena are present in evolutionarily diverse organisms, e.g., nematodes, plants, fungi, and trypanosomes. The mechanisms by which RNAi works is initiated by a progressive cleavage of dsRNA into 21 to 23 nucleotide (nt) short interfering RNAs (siRNAs). These native siRNA duplexes are then incorporated into a protein complex called RNA-induced silencing complex (RISC). ATP-dependent unwinding of the siRNA duplex generates an active RISC complex. Guided by the antisense strand of siRNA, the active RISC complex recognizes and cleaves the corresponding mRNA." [PMID:12110901, PMID:12408823]
<br>subset: PSI-MI slim
<br>synonym: "rnai" EXACT PSI-MI-short []
<br>is_a: MI:0255 ! post transcriptional interference
<p><br>[Term]
<br>id: MI:0257
<br>name: antisense rna
<br>def: "This approach is based on the observation that expression of RNA that is complementary to a specific mRNA can decrease the synthesis of its gene product either by increasing the degradation of the targeted mRNA or by interfering with its translation." [PMID:1340158]
<br>subset: PSI-MI slim
<br>is_a: MI:0255 ! post transcriptional interference
<p><br>[Term]
<br>id: MI:0258
<br>name: inhibitor antibodies
<br>def: "Intracellular or extracellular expression of antibodies is used to target specific gene products in order to inactivate them. Most of the times the antibody scaffold need s to reengineered for efficient expression and solubility in the cytoplasm.\nOBSOLETE as such method can be described using the biological role inhibitor (MI:0586)." [PMID:10189716]
<br>subset: PSI-MI slim
<br>is_obsolete: true
<p><br>[Term]
<br>id: MI:0259
<br>name: perturbagens peptides
<br>def: "This approach is based on the expression of peptides that bind to specific target proteins thereby interfering with their activity. In a standard approach the interfering peptide is expressed by genetic fusion to a stable protein scaffold. Interfering peptides can also be introduced into cells by fusing them to proteins or peptides (homeodomains, Tat protein.) having the property of penetrating the cell membrane. The peptide-carrier fusion protein is either synthesized chemically or produced in vivo, normally in bacteria. When the purified fusion protein is added to a cell culture, it penetrates the cell membrane thereby permitting the fused peptide to interfere with its target protein.\nOBSOLETE as such method can be described using the biological role inhibitor (MI:0586)." [PMID:11731788, PMID:8606778]
<br>subset: PSI-MI slim
<br>synonym: "perturbagens pep" EXACT PSI-MI-short []
<br>is_obsolete: true
<p><br>[Term]
<br>id: MI:0260
<br>name: inhibitor small molecules
<br>def: "Protein activity is inhibited by small inorganic molecules (drugs) that specifically bind to their targets. Recently this classical pharmacological approach is sometime referred to as 'chemical genetics'.\nOBSOLETE as such method can be described using the biological role inhibitor (MI:0586)." [PMID:10542155, PMID:10780927]
<br>subset: PSI-MI slim
<br>synonym: "inhibitor small mol" EXACT PSI-MI-short []
<br>is_obsolete: true
<p><br>[Term]
<br>id: MI:0261
<br>name: suppression
<br>def: "A supressed gene mutation cause of an altered phenotype that is reverted to wild type phenotype when cell also carry a suppressor gene with a specific mutation or altered expression level.\nOBSOLETE: remap to CV intraction type 'suppressive interaction' MI:0793." [PMID:15608217]
<br>subset: PSI-MI slim
<br>is_obsolete: true
<p><br>[Term]
<br>id: MI:0262
<br>name: suppression mutation
<br>def: "A given (suppressed) mutation phenotype is reverted by a supressor gene mutation.\nOBSOLETE: remap to CV intraction type 'suppressive interaction' MI:0793 and genetic experimental form 'mutated gene' MI:0804. " [PMID:15608217]
<br>subset: PSI-MI slim
<br>is_obsolete: true
<p><br>[Term]
<br>id: MI:0263
<br>name: suppression knockout
<br>def: "Knocked out gene is the suppressor of a phenotype.\nOBSOLETE: remap to CV intraction type 'suppressive interaction' MI:0793 and genetic experimental form 'knock out' MI:0788. " [PMID:15608217]
<br>subset: PSI-MI slim
<br>synonym: "suppress knockout" EXACT PSI-MI-short []
<br>is_obsolete: true
<p><br>[Term]
<br>id: MI:0264
<br>name: suppression partial alteration
<br>def: "A mutation is the partial suppressor of a mutant phenotype.\nOBSOLETE: remap to CV intraction type 'suppressive interaction' MI:0793 and genetic experimental form 'knock down' MI:0789. " [PMID:15608217]
<br>subset: PSI-MI slim
<br>synonym: "suppression partial" EXACT PSI-MI-short []
<br>is_obsolete: true
<p><br>[Term]
<br>id: MI:0265
<br>name: suppression expression alteration
<br>def: "An altered expression is the suppressor of a phenotype.\nOBSOLETE: remap to CV intraction type 'suppressive interaction' MI:0793 and genetic experimental form 'expression level alteration' MI:0803. " [PMID:15608217]
<br>subset: PSI-MI slim
<br>synonym: "suppress expression" EXACT PSI-MI-short []
<br>is_obsolete: true
<p><br>[Term]
<br>id: MI:0266
<br>name: suppression overexpression
<br>def: "Overexpression is the suppressor of a phenotype.\nOBSOLETE: remap to CV intraction type 'suppressive interaction' MI:0793 and genetic experimental form 'over expressed' MI:0506. " [PMID:15608217]
<br>subset: PSI-MI slim
<br>synonym: "suppress overexpress" EXACT PSI-MI-short []
<br>is_obsolete: true
<p><br>[Term]
<br>id: MI:0267
<br>name: suppression scalable
<br>def: "Level of over/underexpression scales the 'extend' of a phenotype.\nOBSOLETE: remap to CV intraction type 'suppressive interaction' MI:0793 and genetic experimental form 'expression level alteration' MI:0803. " [PMID:15608217]
<br>subset: PSI-MI slim
<br>is_obsolete: true
<p><br>[Term]
<br>id: MI:0268
<br>name: suppression underexpression
<br>def: "Underexpression is the suppressor of a phenotype.\nOBSOLETE: remap to CV intraction type 'suppressive interaction' MI:0793 and genetic experimental form 'under expressed' MI:0223. " [PMID:15608217]
<br>subset: PSI-MI slim
<br>synonym: "suppress underexpres" EXACT PSI-MI-short []
<br>is_obsolete: true
<p><br>[Term]
<br>id: MI:0269
<br>name: synthetic phenotype
<br>def: "Two silent mutations show an altered phenotype when they co-occur on the same cell.\nOBSOLETE: remap to CV intraction type 'synthetic interaction' MI:0794." [PMID:15608217]
<br>subset: PSI-MI slim
<br>is_obsolete: true
<p><br>[Term]
<br>id: MI:0270
<br>name: conditional synthetic lethal
<br>def: "Two silent mutations show a conditional synthetic lethal phenotype when they co-occur on the same cell.\nOBSOLETE: remap to CV intraction type 'synthetic interaction' MI:0794 and external CV for phenotype description (lethal FBcv:0000351)" [PMID:15608217]
<br>subset: PSI-MI slim
<br>synonym: "cond syntetic lethal" EXACT PSI-MI-short []
<br>is_obsolete: true
<p><br>[Term]
<br>id: MI:0271
<br>name: conditional synthetic lethal temperature-sensitivity
<br>def: "Two silent mutations show a temperature sensitive lethal phenotype when they co-occur on the same cell.\nOBSOLETE: remap to CV intraction type 'synthetic interaction' MI:0794 and external CV for phenotype description (FBcv:0000310 'temperature conditional')" [PMID:15608217]
<br>subset: PSI-MI slim
<br>synonym: "temprtr synt lethal" EXACT PSI-MI-short []
<br>is_obsolete: true
<p><br>[Term]
<br>id: MI:0273
<br>name: synthetic growth effect
<br>def: "Two silent mutations show altered growth effect when they co-occur on the same cell.\nOBSOLETE: remap to CV intraction type 'synthetic interaction' MI:0794 and external CV for phenotype description ( FBcv:0000427 'cell growth defective')" [PMID:15608217]
<br>subset: PSI-MI slim
<br>synonym: "synt growth effect" EXACT PSI-MI-short []
<br>is_obsolete: true
<p><br>[Term]
<br>id: MI:0274
<br>name: synthetic growth defect
<br>def: "Two silent mutations show growth defect when they co-occur on the same cell.\nOBSOLETE: remap to CV intraction type 'synthetic interaction' MI:0794 and external CV for phenotype description ( FBcv:0000427 'cell growth defective')" [PMID:15608217]
<br>subset: PSI-MI slim
<br>synonym: "synt growth defect" EXACT PSI-MI-short []
<br>is_obsolete: true
<p><br>[Term]
<br>id: MI:0275
<br>name: synthetic growth increase
<br>def: "Two silent mutations show growth increase when they co-occur on the same cell.\nOBSOLETE: remap to CV intraction type 'synthetic interaction' MI:0794 and external CV for phenotype description ( FBcv:0000427 'cell growth defective')" [PMID:15608217]
<br>subset: PSI-MI slim
<br>synonym: "synt growth increase" EXACT PSI-MI-short []
<br>is_obsolete: true
<p><br>[Term]
<br>id: MI:0276
<br>name: blue native page
<br>def: "Blue native PAGE (BN-PAGE) permits a high-resolution separation of multi-protein complexes under native conditions. Blue native (BN)-PAGE is a charge shift method, in which the electrophoretic mobility of a complex is determined by the negative charge of the bound Coomassie dye and the size and shape of the complex. Coomassie does not act as a detergent and preserves the structure of complexes. Importantly, the resolution of BN-PAGE is much higher than that of other methods such as gel filtration or sucrose-gradient ultracentrifugation. Combined with other pre-purifications or dialysis steps this method permits the analysis of multi-protein complexes of whole cellular lysates by BN-PAGE." [PMID:14665681]
<br>subset: PSI-MI slim
<br>synonym: "bn-page" EXACT PSI-MI-short []
<br>is_a: MI:0404 ! comigration in non denaturing gel electrophoresis
<p><br>[Term]
<br>id: MI:0300
<br>name: alias type
<br>def: "Descriptor of type of nomenclature used to describe interactor." [PMID:14755292]
<br>subset: Drugable
<br>subset: PSI-MI slim
<br>synonym: "CvAliasType" EXACT PSI-MI-alternate []
<br>relationship: part_of MI:0000 ! molecular interaction
<p><br>[Term]
<br>id: MI:0301
<br>name: gene name
<br>def: "Gene name." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "gene" EXACT PSI-MI-alternate []
<br>is_a: MI:0300 ! alias type
<p><br>[Term]
<br>id: MI:0302
<br>name: gene name synonym
<br>def: "Gene name synonym." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0300 ! alias type
<p><br>[Term]
<br>id: MI:0303
<br>name: gene ontology synonym
<br>def: "Synonym as used in Gene Ontology." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "go synonym" EXACT PSI-MI-short []
<br>is_a: MI:0300 ! alias type
<p><br>[Term]
<br>id: MI:0304
<br>name: isoform synonym
<br>def: "Isoform synonym." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0300 ! alias type
<p><br>[Term]
<br>id: MI:0305
<br>name: ordered locus name
<br>def: "A name used to represent an ORF in a completely sequenced genome or chromosome. It is generally based on a prefix representing the organism and a number which usually represents the sequential ordering of genes on the chromosome. Depending on the genome sequencing center, numbers are attributed only to protein-coding genes, or also to pseudogenes, or also to tRNAs and other features." [PMID:14755292]
<br>comment: For instance HI0934, Rv3245c, At5g34500, YER456W.
<br>subset: PSI-MI slim
<br>synonym: "CDS number" EXACT PSI-MI-alternate []
<br>synonym: "locus name" EXACT PSI-MI-short []
<br>synonym: "ONL" EXACT PSI-MI-alternate []
<br>synonym: "Ordered locus name" EXACT PSI-MI-alternate []
<br>synonym: "ORF number" EXACT PSI-MI-alternate []
<br>synonym: "systematic gene number" EXACT PSI-MI-alternate []
<br>is_a: MI:0300 ! alias type
<p><br>[Term]
<br>id: MI:0306
<br>name: open reading frame name
<br>def: "A name temporarily attributed by a sequencing project to an open reading frame. This name is generally based on a cosmid numbering system." [PMID:14755292]
<br>comment: For instance MtCY277-28c, SYGP-ORF50, SpBC2F12-04, C06E1, CG10954. Also called Sequencing names or Contig names or Temporary ORFNames.
<br>subset: PSI-MI slim
<br>synonym: "orf name" EXACT PSI-MI-short []
<br>is_a: MI:0300 ! alias type
<p><br>[Term]
<br>id: MI:0307
<br>name: delivery method
<br>def: "Method by which molecule is delivered or engineered into a cell." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0346 ! experimental preparation
<p><br>[Term]
<br>id: MI:0308
<br>name: electroporation
<br>def: "Method for temporarily permeabilising cell membranes so as to facilitate the entry of large or hydrophilic molecules (as in transfection). A brief (ca 1 msec) electric pulse is given with potential gradients of about 700V/cm." [PMID:6329708]
<br>subset: PSI-MI slim
<br>is_a: MI:0307 ! delivery method
<p><br>[Term]
<br>id: MI:0309
<br>name: genomic tagging
<br>def: "A cassette coding for a protein tag is inserted by homologous recombination onto the genomic copy of an open reading frame. The advantage of this delivery method is that the resulting engineered protein is expressed under its natural promoter control.\nOBSOLETE redundant term. Map to feature type : tag (MI:0507)." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "genetic tag insertion" EXACT PSI-MI-alternate []
<br>is_obsolete: true
<p><br>[Term]
<br>id: MI:0310
<br>name: infection
<br>def: "Molecule introduced into a cell via an external organism, usually a virus or bacteria." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0307 ! delivery method
<p><br>[Term]
<br>id: MI:0311
<br>name: microinjection
<br>def: "The insertion of a substance into a cell through a microneedle. To extrude the substances through the very fine needle tip, either hydrostatic pressure (pressure injection) or electric currents (ionophoresis) is employed." [PMID:3016916]
<br>subset: PSI-MI slim
<br>synonym: "micro-injection" EXACT PSI-MI-alternate []
<br>is_a: MI:0307 ! delivery method
<p><br>[Term]
<br>id: MI:0312
<br>name: nucleic acid transfection
<br>def: "Introducing DNA into eukaryotic cells, such as animal cells, is called transfection. Transfection typically involves opening transient \"holes\" or gates in cells to allow the entry of extracellular molecules, typically supercoiled plasmid DNA, but also siRNA, among others. Transfection differs from transformation since the DNA is not generally incorporated into the cell's genome, it is only transiently expressed." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "nucl transfection" EXACT PSI-MI-short []
<br>is_a: MI:0704 ! nucleic acid delivery
<p><br>[Term]
<br>id: MI:0313
<br>name: interactor type
<br>def: "Molecular species involved in the interaction." [PMID:14755292]
<br>subset: Drugable
<br>subset: PSI-MI slim
<br>synonym: "participant type" EXACT PSI-MI-alternate []
<br>relationship: part_of MI:0000 ! molecular interaction
<p><br>[Term]
<br>id: MI:0314
<br>name: complex
<br>def: "Set of interacting molecules that can be copurified. This term and its children should be use only at PARTICIPANT level." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0313 ! interactor type
<p><br>[Term]
<br>id: MI:0315
<br>name: protein complex
<br>def: "A stable set of interacting proteins that can be copurified and operate as a functional unit." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0314 ! complex
<p><br>[Term]
<br>id: MI:0316
<br>name: ribonucleoprotein complex
<br>def: "A macromolecular complex containing both protein and RNA molecules." [GO:0030529, PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "protein rna complex" EXACT PSI-MI-alternate []
<br>synonym: "ribonucleoprot compl" EXACT PSI-MI-short []
<br>is_a: MI:0314 ! complex
<p><br>[Term]
<br>id: MI:0317
<br>name: interaction
<br>def: "Previously described interaction now being used as an interactor to describe hierarchical build-up of complexes." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0313 ! interactor type
<p><br>[Term]
<br>id: MI:0318
<br>name: nucleic acid
<br>def: "Linear polymers of nucleotides, linked by 3',5' phosphodiester linkages." [PMID:14755292, SO:0000348]
<br>subset: PSI-MI slim
<br>is_a: MI:0383 ! biopolymer
<p><br>[Term]
<br>id: MI:0319
<br>name: deoxyribonucleic acid
<br>def: "Polymer formed by the deoxyribose sugar group, and the nucleotides bases adenine, guanine, thymine and cytosine." [PMID:14755292, SO:0000352]
<br>subset: PSI-MI slim
<br>synonym: "deoxyribonucleic acid" EXACT PSI-MI-alternate []
<br>synonym: "DNA" EXACT PSI-MI-alternate []
<br>synonym: "dna" EXACT PSI-MI-short []
<br>is_a: MI:0318 ! nucleic acid
<p><br>[Term]
<br>id: MI:0320
<br>name: ribonucleic acid
<br>def: "Polymer formed by ribose sugar group, and the bases of the nucleotides adenine, guanine, uracil and cytosine." [PMID:14755292, SO:0000356]
<br>subset: PSI-MI slim
<br>synonym: "RNA" EXACT PSI-MI-alternate []
<br>synonym: "rna" EXACT PSI-MI-short []
<br>is_a: MI:0318 ! nucleic acid
<p><br>[Term]
<br>id: MI:0321
<br>name: catalytic rna
<br>def: "Species of RNA that catalyses cleavage or trans-esterification of the phosphodiester link." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "catalytic ribonucleic acid" EXACT PSI-MI-alternate []
<br>synonym: "catalytic RNA" EXACT PSI-MI-alternate []
<br>synonym: "crna" EXACT PSI-MI-short []
<br>is_a: MI:0320 ! ribonucleic acid
<p><br>[Term]
<br>id: MI:0322
<br>name: guide rna
<br>def: "Small RNA molecules that hybridize to specific mRNAs and direct their RNA editing." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "grna" EXACT PSI-MI-short []
<br>synonym: "guide RNA" EXACT PSI-MI-alternate []
<br>is_a: MI:0320 ! ribonucleic acid
<p><br>[Term]
<br>id: MI:0323
<br>name: heterogeneous nuclear rna
<br>def: "A heterogeneous mixture of RNA molecules with a rapid turnover rate that occurs in cell nuclei during protein synthesis; it is the form of RNA synthesized in eukaryotes by RNA polymerase II, which is translated into protein." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "heterogeneous nuclear ribonucleic acid" EXACT PSI-MI-alternate []
<br>synonym: "heterogeneous nuclear RNA" EXACT PSI-MI-alternate []
<br>synonym: "hnrna" EXACT PSI-MI-short []
<br>is_a: MI:0320 ! ribonucleic acid
<p><br>[Term]
<br>id: MI:0324
<br>name: messenger rna
<br>def: "Single-stranded RNA molecule that specifies the amino acid sequence of one or more polypeptide chains." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "mRNA" EXACT PSI-MI-alternate []
<br>synonym: "mrna" EXACT PSI-MI-short []
<br>is_a: MI:0320 ! ribonucleic acid
<p><br>[Term]
<br>id: MI:0325
<br>name: transfer rna
<br>def: "The low molecular weight RNAs that specifically bind amino acids by aminoacetylation to form aminoacyl tRNA and which possess a special nucleotide triplet, the anticodon." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "transfer ribonucleic acid" EXACT PSI-MI-alternate []
<br>synonym: "transfer RNA" EXACT PSI-MI-alternate []
<br>synonym: "tRNA" EXACT PSI-MI-alternate []
<br>synonym: "trna" EXACT PSI-MI-short []
<br>is_a: MI:0320 ! ribonucleic acid
<p><br>[Term]
<br>id: MI:0326
<br>name: protein
<br>def: "A linear polymer of amino acids joined by peptide bonds in a specific sequence." [PMID:14755292, SO:0000358]
<br>subset: PSI-MI slim
<br>is_a: MI:0383 ! biopolymer
<p><br>[Term]
<br>id: MI:0327
<br>name: peptide
<br>def: "Chains of amino acids joined by peptide bonds. Distinction between peptides, oligopeptides and polypeptides is arbitrarily by length; a polypeptide is perhaps more than 15 residues." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "oligopeptide" EXACT PSI-MI-alternate []
<br>synonym: "polypeptide" EXACT PSI-MI-alternate []
<br>is_a: MI:0383 ! biopolymer
<p><br>[Term]
<br>id: MI:0328
<br>name: small molecule
<br>def: "Molecule not part of or directly encoded by the genome, encompasses any constitutionally or isotopically distinct atom, molecule, ion, ion pair, radical, radical ion, complex, conformer, etc., identifiable as a separately distinguishable entity." [PMID:14755292]
<br>subset: Drugable
<br>subset: PSI-MI slim
<br>is_a: MI:0313 ! interactor type
<p><br>[Term]
<br>id: MI:0329
<br>name: unknown participant
<br>def: "Any type of molecule, including complexes, that may be observed but not identified. This term should be use only at PARTICIPANT level." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0313 ! interactor type
<p><br>[Term]
<br>id: MI:0330
<br>name: molecular source
<br>def: "Defines whether molecule is endogenously expressed or has in any way been altered, in sequence or expression level, from its native state. For a complete description of the experimental molecule form use the orthogonal CVs expression level, delivery method, and sample process." [PMID:14755272]
<br>subset: PSI-MI slim
<br>is_a: MI:0346 ! experimental preparation
<p><br>[Term]
<br>id: MI:0331
<br>name: engineered
<br>def: "Molecule has been added into system from an external source or altered within the cell." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0330 ! molecular source
<p><br>[Term]
<br>id: MI:0332
<br>name: naturally occurring
<br>def: "Unaltered endogenous molecule in its naturally occurring state." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "endogenous" EXACT PSI-MI-alternate []
<br>is_a: MI:0330 ! molecular source
<p><br>[Term]
<br>id: MI:0333
<br>name: feature range status
<br>def: "Describes sequence positions resolution of a given participant feature. In PSI schema this CV is associated with the start and end position of a feature range." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "CvFuzzyType" EXACT PSI-MI-alternate []
<br>synonym: "endStatus" EXACT PSI-MI-alternate []
<br>synonym: "startStatus" EXACT PSI-MI-alternate []
<br>relationship: part_of MI:0000 ! molecular interaction
<p><br>[Term]
<br>id: MI:0334
<br>name: c-terminal position
<br>def: "Term describing the last amino acid of a peptide chain." [PMID:14755292]
<br>comment: Displayed as 'c'.
<br>subset: PSI-MI slim
<br>synonym: "c-term" EXACT PSI-MI-alternate []
<br>synonym: "c-terminal" EXACT PSI-MI-short []
<br>synonym: "c-terminus" EXACT PSI-MI-alternate []
<br>synonym: "carboxy-terminus" EXACT PSI-MI-alternate []
<br>is_a: MI:0333 ! feature range status
<p><br>[Term]
<br>id: MI:0335
<br>name: certain sequence position
<br>def: "Position within the sequence clearly defined." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "certain" EXACT PSI-MI-short []
<br>is_a: MI:0333 ! feature range status
<p><br>[Term]
<br>id: MI:0336
<br>name: greater-than
<br>def: "Partially determined sequence position known to be in a location higher than a given position." [PMID:14755292]
<br>comment: Displayed as '>'.
<br>subset: PSI-MI slim
<br>is_a: MI:0333 ! feature range status
<p><br>[Term]
<br>id: MI:0337
<br>name: less-than
<br>def: "Partially determined sequence position known to be in a position lower than a given position." [PMID:14755292]
<br>comment: Displayed as '<'.
<br>subset: PSI-MI slim
<br>is_a: MI:0333 ! feature range status
<p><br>[Term]
<br>id: MI:0338
<br>name: range
<br>def: "Describes a sequence position known to be in a certain range, where the exact position is unclear." [PMID:14755292]
<br>comment: For instance when an amino acid modification is known to be in the region from 5 to 7. Displayed as '..'.
<br>subset: PSI-MI slim
<br>is_a: MI:0333 ! feature range status
<p><br>[Term]
<br>id: MI:0339
<br>name: undetermined sequence position
<br>def: "Term describing a completely unknown or unspecified sequence position." [PMID:14755292]
<br>comment: Displayed as '?'.
<br>subset: PSI-MI slim
<br>synonym: "undetermined" EXACT PSI-MI-short []
<br>is_a: MI:0333 ! feature range status
<p><br>[Term]
<br>id: MI:0340
<br>name: n-terminal position
<br>def: "Term describing the first amino acid of a peptide chain." [PMID:14755292]
<br>comment: Displayed as 'n'.
<br>subset: PSI-MI slim
<br>synonym: "amino-terminus" EXACT PSI-MI-alternate []
<br>synonym: "n-term" EXACT PSI-MI-alternate []
<br>synonym: "n-terminal " EXACT PSI-MI-short []
<br>synonym: "n-terminus" EXACT PSI-MI-alternate []
<br>is_a: MI:0333 ! feature range status
<p><br>[Term]
<br>id: MI:0341
<br>name: ragged n-terminus
<br>def: "Mixture of protein forms where N-terminus has been progressively truncated." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0340 ! n-terminal position
<p><br>[Term]
<br>id: MI:0342
<br>name: sample process
<br>def: "Indicates the sample context in which each interacting molecule is presented to its partner." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0346 ! experimental preparation
<p><br>[Term]
<br>id: MI:0343
<br>name: cdna library
<br>def: "Mixed population of cDNAs (complementaryDNA) made from mRNA from a defined source, usually a specific cell type. This term should be associated only to nucleic acid interactors not to their proteins product. For instance in 2h screening use living cells (MI:0349) as sample process." [PMID:6110205]
<br>subset: PSI-MI slim
<br>is_a: MI:0342 ! sample process
<p><br>[Term]
<br>id: MI:0344
<br>name: cell lysate
<br>def: "Cell has been physically or chemically broken open and molecule present in resulting mixture of cellular components." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0342 ! sample process
<p><br>[Term]
<br>id: MI:0345
<br>name: author assigned name
<br>def: "Name assigned to a molecule by the authors within a paper that may differ from the reference database." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0300 ! alias type
<p><br>[Term]
<br>id: MI:0346
<br>name: experimental preparation
<br>def: "Set of terms to describe the participant experimental treatment and status. This term groups a number of orthologous short controlled vocabularies delivery method, expression level, molecular source, and sample process. Each participant can then be annotated with a maximum of 4 terms selected from each short list." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "experimental prep" EXACT PSI-MI-short []
<br>relationship: part_of MI:0000 ! molecular interaction
<p><br>[Term]
<br>id: MI:0348
<br>name: fixed cell
<br>def: "Cells has been fixed by treatment with organic solvent, staining and inclusion in a resin for microscopic analysis." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0342 ! sample process
<p><br>[Term]
<br>id: MI:0349
<br>name: living cell
<br>def: "Molecule is observed within in a living cell." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0342 ! sample process
<p><br>[Term]
<br>id: MI:0350
<br>name: purified
<br>def: "Molecule has undergone one or more purification steps to isolate it from the cellular environment." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0342 ! sample process
<p><br>[Term]
<br>id: MI:0351
<br>name: homogeneous
<br>def: "The author states a molecule is completely pure, i.e. no other molecular species are present." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "pure" EXACT PSI-MI-alternate []
<br>is_a: MI:0350 ! purified
<p><br>[Term]
<br>id: MI:0352
<br>name: partially purified
<br>def: "The author states a molecule is only partially purified, i.e. other molecular species also known to be present." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0350 ! purified
<p><br>[Term]
<br>id: MI:0353
<br>name: cross-reference type
<br>def: "Qualifier to describe the type of information a cross-reference is pointing to." [PMID:14755292]
<br>subset: Drugable
<br>subset: PSI-MI slim
<br>synonym: "CvXrefQualifier" EXACT PSI-MI-alternate []
<br>synonym: "refType" EXACT PSI-MI-alternate []
<br>synonym: "xref type" EXACT PSI-MI-short []
<br>relationship: part_of MI:0000 ! molecular interaction
<p><br>[Term]
<br>id: MI:0354
<br>name: gene ontology term for cellular component
<br>def: "Cross reference pointing to a Gene Ontology -'cellular component' term." [PMID:14681407]
<br>subset: PSI-MI slim
<br>synonym: "component" EXACT PSI-MI-short []
<br>synonym: "go component term" EXACT PSI-MI-alternate []
<br>is_a: MI:0353 ! cross-reference type
<p><br>[Term]
<br>id: MI:0355
<br>name: gene ontology term for cellular function
<br>def: "Cross reference pointing to a Gene Ontology -'cellular function' term." [PMID:14681407]
<br>subset: PSI-MI slim
<br>synonym: "function" EXACT PSI-MI-short []
<br>synonym: "go function term" EXACT PSI-MI-alternate []
<br>is_a: MI:0353 ! cross-reference type
<p><br>[Term]
<br>id: MI:0356
<br>name: identical object
<br>def: "Reference to the corresponding object in another database. Correspondence may be complete or partial." [PMID:14755292]
<br>comment: For instance this qualifier, in an interaction entry, can be associated to a cross reference to the same interaction in an other database.
<br>subset: Drugable
<br>subset: PSI-MI slim
<br>synonym: "identity" EXACT PSI-MI-short []
<br>is_a: MI:0353 ! cross-reference type
<p><br>[Term]
<br>id: MI:0357
<br>name: method reference
<br>def: "Reference to a related paper which more fully describes either the experimental method or one or more of the interactors used within the experiment." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0353 ! cross-reference type
<p><br>[Term]
<br>id: MI:0358
<br>name: primary-reference
<br>def: "Used to indicate the PMID from which the experimental data is extracted." [PMID:14755292]
<br>subset: Drugable
<br>subset: PSI-MI slim
<br>is_a: MI:0353 ! cross-reference type
<p><br>[Term]
<br>id: MI:0359
<br>name: gene ontology term for cellular process
<br>def: "Cross reference pointing to a Gene Ontology -'cellular process' term." [PMID:14681407]
<br>subset: PSI-MI slim
<br>synonym: "go process term" EXACT PSI-MI-alternate []
<br>synonym: "process" EXACT PSI-MI-short []
<br>is_a: MI:0353 ! cross-reference type
<p><br>[Term]
<br>id: MI:0360
<br>name: secondary accession number
<br>def: "Reference to the corresponding object in another database (like identity xref qualifier) but the identifier used in the external database is a secondary identifier or former accession number." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "secondary-ac" EXACT PSI-MI-short []
<br>is_a: MI:0353 ! cross-reference type
<p><br>[Term]
<br>id: MI:0361
<br>name: see-also
<br>def: "Related object within the same database or pointing to an external database." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0353 ! cross-reference type
<p><br>[Term]
<br>id: MI:0362
<br>name: inference
<br>def: "Evidence based on human assumption, either when the complete experimental support is not available or when the results are extended by homology to closely related orthologues sequences." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "modeled" EXACT PSI-MI-alternate []
<br>synonym: "modelled" EXACT PSI-MI-alternate []
<br>is_a: MI:0001 ! interaction detection method
<br>is_a: MI:0002 ! participant identification method
<br>is_a: MI:0003 ! feature detection method
<p><br>[Term]
<br>id: MI:0363
<br>name: inferred by author
<br>def: "Evidence based on the author of a paper assumption, either when the complete experimental support is not available or when the results are extended by homology to closely related orthologues sequences." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "modeled by author" EXACT PSI-MI-alternate []
<br>synonym: "modelled by author" EXACT PSI-MI-alternate []
<br>is_a: MI:0362 ! inference
<p><br>[Term]
<br>id: MI:0364
<br>name: inferred by curator
<br>def: "Evidence based on a curator assumption, either when the complete experimental support is not available or when the results are extended by homology to closely related orthologues sequences." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "modeled by curator" EXACT PSI-MI-alternate []
<br>synonym: "modelled by curator" EXACT PSI-MI-alternate []
<br>is_a: MI:0362 ! inference
<p><br>[Term]
<br>id: MI:0365
<br>name: enzyme tag
<br>def: "Molecule under study is fused to an enzyme, for example alkaline phosphatase, and measure of enzyme activity can be taken as indicative of presence of protein." [PMID:10935637]
<br>subset: PSI-MI slim
<br>is_a: MI:0240 ! fusion protein
<p><br>[Term]
<br>id: MI:0366
<br>name: alkaline phosphatase tag
<br>def: "Protein is fused to alkaline phosphatase, and the measure of this enzyme activity can be taken as indicative of presence of protein." [PMID:10935637]
<br>subset: PSI-MI slim
<br>synonym: "alk phosphatase tag" EXACT PSI-MI-short []
<br>is_a: MI:0365 ! enzyme tag
<p><br>[Term]
<br>id: MI:0367
<br>name: green fluorescent protein tag
<br>def: "The green fluorescent protein of the bioluminescent jellyfish Aequorea victoria can be fused to individual proteins which then acquire fluorescence excitation and emission spectra virtually identical to those of the native." [PMID:7491768]
<br>subset: PSI-MI slim
<br>synonym: "GFP" EXACT PSI-MI-alternate []
<br>synonym: "gfp tag" EXACT PSI-MI-short []
<br>synonym: "green fluorescent protein" EXACT PSI-MI-alternate []
<br>is_a: MI:0687 ! fluorescent protein tag
<p><br>[Term]
<br>id: MI:0368
<br>name: yellow fluorescent protein tag
<br>def: "Yellow fluorescent protein from Vibrio fischeri can be fused to individual proteins which then acquire fluorescence excitation and emission spectra virtually identical to those of the native." [PMID:10929120]
<br>subset: PSI-MI slim
<br>synonym: "yellow fluorescent protein" EXACT PSI-MI-alternate []
<br>synonym: "YFP" EXACT PSI-MI-alternate []
<br>synonym: "yfp tag" EXACT PSI-MI-short []
<br>is_a: MI:0687 ! fluorescent protein tag
<p><br>[Term]
<br>id: MI:0369
<br>name: lex-a dimerization assay
<br>def: "The method is based on the repression of a reporter gene activity by two LexA DNA binding domains with different binding specificities. LexA is a transcription factor with an N-terminal DNA binding/activation domain (DBAct) and a C-terminal dimerization domain. LexA dimerization is required to repress transcription efficiently. The discovery of LexA DNA binding domains that bind to different DNA sequence enabled the development of this system." [PMID:12446730]
<br>subset: PSI-MI slim
<br>synonym: "gallex" EXACT PSI-MI-alternate []
<br>synonym: "gallex" EXACT PSI-MI-short []
<br>is_a: MI:0232 ! transcriptional complementation assay
<p><br>[Term]
<br>id: MI:0370
<br>name: tox-r dimerization assay
<br>def: "This assay allow identification of interactions in the inner membrane of E. coli. by using a chimeric construct ToxR-TM-MBP composed of the N-terminal DNA binding/transcriptional activation domain of ToxR (a dimerization dependant transcription factor) fused to a transmembrane domain of interest (TM) and a monomeric periplasmic anchor (the maltose binding protein). Association of the two TM results in the ToxR-mediated activation of a reporter gene such as CAT (chloroamphenicol acetyltransferase activity). The level of CAT expression indicates the strength of TM association. CAT expression can then be tested and quantify by measuring CAM resistance with disk diffusion assay or CAT activity assays on cell-free extracts." [PMID:9927659]
<br>subset: PSI-MI slim
<br>synonym: "toxcat" EXACT PSI-MI-alternate []
<br>synonym: "toxcat" EXACT PSI-MI-short []
<br>is_a: MI:0230 ! membrane bound complementation assay
<p><br>[Term]
<br>id: MI:0371
<br>name: 35s radiolabel
<br>def: "Molecule labelled with 35 radio isotope of sulfur. Proteins are often metabolically labelled, usually be growth in 35S labelled culture medium." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "35S" EXACT PSI-MI-alternate []
<br>synonym: "S35" EXACT PSI-MI-alternate []
<br>synonym: "s35 radiolabelled" EXACT PSI-MI-alternate []
<br>is_a: MI:0517 ! radiolabel
<p><br>[Term]
<br>id: MI:0372
<br>name: subcellular preparation
<br>def: "Cell lysates are partially fractionated to isolate a specific subcellular fraction." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "subcellular prep" EXACT PSI-MI-short []
<br>is_a: MI:0344 ! cell lysate
<p><br>[Term]
<br>id: MI:0373
<br>name: dye label
<br>def: "Dye coupled to a molecule allowing its identification isolation and monitoring." [PMID:14577292]
<br>subset: PSI-MI slim
<br>synonym: "dye labelled" EXACT PSI-MI-short []
<br>is_a: MI:0505 ! experimental feature
<p><br>[Term]
<br>id: MI:0374
<br>name: cyanine label
<br>def: "The organic cyanine dyes emit in the red range (>550 nm), offer a number of advantages. Their emission range is such that background fluorescence is often reduced. In addition these molecules can be linked directly to specific locations in synthetically produced nucleic acids." [PMID:14577292]
<br>subset: PSI-MI slim
<br>is_a: MI:0373 ! dye label
<br>is_a: MI:0857 ! fluorescent dye label
<p><br>[Term]
<br>id: MI:0375
<br>name: cy3 label
<br>def: "The organic cyanine Cy3 emits maximally at 570 nm." [PMID:14577292]
<br>subset: PSI-MI slim
<br>is_a: MI:0374 ! cyanine label
<p><br>[Term]
<br>id: MI:0376
<br>name: cy5 label
<br>def: "The organic cyanine Cy5 emits maximally at 670 nm." [PMID:14577292]
<br>subset: PSI-MI slim
<br>is_a: MI:0374 ! cyanine label
<p><br>[Term]
<br>id: MI:0377
<br>name: fluorescein isothiocyanate label
<br>def: "Fluorescein isothiocyanate is a yellow-green coloured low molecular weight dye which couples to proteins via reaction with primary amine groups at high pH. FITC is excitable at 488nm, close to its absorption maximum at 494nm, and produces maximum fluorescence emission around 520nm." [PMID:14577292]
<br>subset: PSI-MI slim
<br>synonym: "FITC labelled" EXACT PSI-MI-alternate []
<br>synonym: "fitc labelled" EXACT PSI-MI-short []
<br>synonym: "fluorescein isothiocyanate labbeled" EXACT PSI-MI-alternate []
<br>is_a: MI:0373 ! dye label
<br>is_a: MI:0939 ! fluorescein label
<p><br>[Term]
<br>id: MI:0378
<br>name: rare isotope label
<br>def: "Molecule can be labelled including rare isotopes among its constituent atoms that can be used to identify, localize or quantify the full molecule." [PMID:14577292]
<br>subset: PSI-MI slim
<br>synonym: "rare isotope label" EXACT PSI-MI-short []
<br>is_a: MI:0253 ! isotope label
<p><br>[Term]
<br>id: MI:0379
<br>name: 13c label
<br>def: "Molecules labelled with isotope 13 of carbon atoms." [PMID:14577292]
<br>subset: PSI-MI slim
<br>synonym: "13C" EXACT PSI-MI-alternate []
<br>synonym: "C13" EXACT PSI-MI-alternate []
<br>is_a: MI:0378 ! rare isotope label
<p><br>[Term]
<br>id: MI:0380
<br>name: 15n label
<br>def: "Molecules labelled with isotope 15 of nytrogen atoms." [PMID:14577292]
<br>subset: PSI-MI slim
<br>synonym: "15N" EXACT PSI-MI-alternate []
<br>synonym: "N15" EXACT PSI-MI-alternate []
<br>is_a: MI:0378 ! rare isotope label
<p><br>[Term]
<br>id: MI:0381
<br>name: 2h label
<br>def: "Molecules labelled with isotope 2 of hydrogen atoms." [PMID:14577292]
<br>subset: PSI-MI slim
<br>synonym: "2H2" EXACT PSI-MI-alternate []
<br>synonym: "D2" EXACT PSI-MI-alternate []
<br>synonym: "deuterium" EXACT PSI-MI-alternate []
<br>is_a: MI:0378 ! rare isotope label
<p><br>[Term]
<br>id: MI:0382
<br>name: mutation increasing interaction
<br>def: "Region of a molecule whose mutation or deletion increases significantly interaction strength." [PMID:14577292]
<br>subset: PSI-MI slim
<br>synonym: "mutation increasing" EXACT PSI-MI-short []
<br>is_a: MI:0118 ! mutation
<p><br>[Term]
<br>id: MI:0383
<br>name: biopolymer
<br>def: "Molecule consisting of a specific sequence of amino acidic or nucleotidic monomers strung together through chemical bonds." [PMID:14577292]
<br>subset: PSI-MI slim
<br>is_a: MI:0313 ! interactor type
<p><br>[Term]
<br>id: MI:0384
<br>name: alexa label
<br>def: "All Alexa dyes are fluorescent iodoacetamide dyes that can be conjugated with the primary amines of biomolecules. All Alexa dyes and their conjugates are more fluorescent and more photostable than the commonly used dyes. The numbers in the Alexa names indicate the approximate excitation wavelength maximum in nm)." [PMID:10449539]
<br>subset: PSI-MI slim
<br>is_a: MI:0373 ! dye label
<br>is_a: MI:0857 ! fluorescent dye label
<p><br>[Term]
<br>id: MI:0385
<br>name: alexa 350 label
<br>def: "Alexa fluorescent dye analogue to AMCA (7-amino-4-methylcoumarin-3-acetic acid) with an approximate excitation wavelength maximum of 350 nm." [PMID:10449539]
<br>subset: PSI-MI slim
<br>is_a: MI:0384 ! alexa label
<p><br>[Term]
<br>id: MI:0386
<br>name: alexa 430 label
<br>def: "Alexa fluorescent dye analogue to Lucifer Yellow with an approximate excitation wavelength maximum of 430 nm." [PMID:10449539]
<br>subset: PSI-MI slim
<br>is_a: MI:0384 ! alexa label
<p><br>[Term]
<br>id: MI:0387
<br>name: alexa 488 label
<br>def: "Alexa fluorescent dye analogue to Oregon Green 488 with an approximate excitation wavelength maximum of 488 nm." [PMID:10449539]
<br>subset: PSI-MI slim
<br>is_a: MI:0384 ! alexa label
<p><br>[Term]
<br>id: MI:0388
<br>name: alexa 532 label
<br>def: "Alexa fluorescent dye analogue to Rhodamine 6G with an approximate excitation wavelength maximum of 532nm." [PMID:10449539]
<br>subset: PSI-MI slim
<br>is_a: MI:0384 ! alexa label
<p><br>[Term]
<br>id: MI:0389
<br>name: alexa 546 label
<br>def: "Alexa fluorescent dye analogue to Cy3 with an approximate excitation wavelength maximum of 546nm." [PMID:10449539]
<br>subset: PSI-MI slim
<br>is_a: MI:0384 ! alexa label
<p><br>[Term]
<br>id: MI:0390
<br>name: alexa 568 label
<br>def: "Alexa fluorescent dye analogue to Rhodamine Red-X with an approximate excitation wavelength maximum of 568nm." [PMID:10449539]
<br>subset: PSI-MI slim
<br>is_a: MI:0384 ! alexa label
<p><br>[Term]
<br>id: MI:0391
<br>name: alexa 594 label
<br>def: "Alexa fluorescent dye analogue to Texas Red-X with an approximate excitation wavelength maximum of 594nm." [PMID:10449539]
<br>subset: PSI-MI slim
<br>is_a: MI:0384 ! alexa label
<p><br>[Term]
<br>id: MI:0396
<br>name: predetermined participant
<br>def: "Molecule whose sequence identity is not checked and has been assumed from external or previous experimental evidence(s)." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "predetermined" EXACT PSI-MI-short []
<br>is_a: MI:0661 ! experimental participant identification
<p><br>[Term]
<br>id: MI:0397
<br>name: two hybrid array
<br>def: "Two-hybrid screening can be done in a colony array format, in which each colony expresses a defined pair of proteins. Because the particular protein pair expressed in each colony is defined by its position in the array, positive signals identify interacting proteins without further characterization, thus obviating the need for DNA purification and sequencing. The interrogation of a two-hybrid colony array usually involves a mating strategy in which every DNA binding domain hybrid (the bait) is tested against all activation domain hybrids (the preys) in a grid pattern. Arrays usually use full-length open reading frames." [PMID:11827624, PMID for application instance:10688190]
<br>subset: PSI-MI slim
<br>is_a: MI:0018 ! two hybrid
<p><br>[Term]
<br>id: MI:0398
<br>name: two hybrid pooling approach
<br>def: "In the pooling strategy the sets of bait and prey hybrid vectors are randomly mated. The positives double hybrid clones are the interacting partners." [PMID:12634794, PMID for application instance:11283351]
<br>subset: PSI-MI slim
<br>synonym: "two hybrid pooling" EXACT PSI-MI-short []
<br>is_a: MI:0018 ! two hybrid
<p><br>[Term]
<br>id: MI:0399
<br>name: two hybrid fragment pooling approach
<br>def: "This two hybrid approach involves the screening of a large number of individual proteins against a comprehensive library of randomly generated fragment as prey. The usage of degenerated fragment allows identification of the minimal protein region required for the interaction. Since multiple clones that encode overlapping regions of protein are often identified, the minimal domain for interaction may be readily apparent from the initial screen." [PMID:12634794, PMID for application instance:11196647]
<br>subset: PSI-MI slim
<br>synonym: "2h fragment pooling" EXACT PSI-MI-short []
<br>is_a: MI:0398 ! two hybrid pooling approach
<p><br>[Term]
<br>id: MI:0400
<br>name: affinity technology
<br>def: "Techniques which depend upon the strength of the interaction between two entities." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "affinity techniques" EXACT PSI-MI-short []
<br>is_a: MI:0401 ! biochemical
<p><br>[Term]
<br>id: MI:0401
<br>name: biochemical
<br>def: "The application of chemical principles and methods to biological experiments to demonstrate an interaction." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0045 ! experimental interaction detection
<p><br>[Term]
<br>id: MI:0402
<br>name: chromatin immunoprecipitation assay
<br>def: "Chromatin immunoprecipitation (ChIP) is a powerful approach that allows one to define the interaction of factors with specific chromosomal sites in living cells. ChIP involves treating cells or tissue briefly with formaldehyde to crosslink proteins to DNA. An antibody against a protein suspected of binding a given cis-element is then used to immunoprecipitate chromatin fragments. Polymerase chain reaction analysis of the immunoprecipitate with primers flanking the cis-element reveals whether a specific DNA sequence is recovered in an immune-specific manner and therefore whether the protein contacted the site in living cells." [PMID:12054902]
<br>subset: PSI-MI slim
<br>synonym: "ch-ip" EXACT PSI-MI-short []
<br>is_a: MI:0400 ! affinity technology
<p><br>[Term]
<br>id: MI:0403
<br>name: colocalization
<br>def: "Coincident occurrence of molecules in a given subcellular fraction observed with a low resolution methodology from which a physical interaction among those molecules cannot be inferred." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0190 ! interaction type
<p><br>[Term]
<br>id: MI:0404
<br>name: comigration in non denaturing gel electrophoresis
<br>def: "comigration by non denaturing gel electrophoresis." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "comig non denat gel" EXACT PSI-MI-short []
<br>is_a: MI:0807 ! comigration in gel electrophoresis
<p><br>[Term]
<br>id: MI:0405
<br>name: competition binding
<br>def: "Competitive binding experiments measure equilibrium binding of a single concentration of ligand at various concentrations of an unlabeled competitor. Analysis of these data gives the affinity of the receptor for the competitor." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0400 ! affinity technology
<p><br>[Term]
<br>id: MI:0406
<br>name: deacetylase assay
<br>def: "Measures the catalysis of the hydrolysis of an acetyl group or groups from a substrate molecule." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0415 ! enzymatic study
<p><br>[Term]
<br>id: MI:0407
<br>name: direct interaction
<br>def: "Interaction that is proven to involve only its interactors." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0915 ! physical association
<p><br>[Term]
<br>id: MI:0408
<br>name: disulfide bond
<br>def: "Covalent bond mediated by 2 sulfur atoms." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0195 ! covalent binding
<p><br>[Term]
<br>id: MI:0409
<br>name: dna footprinting
<br>def: "Experimental method used to identify the region of a nucleic acid involved in an interaction with a protein. One sample of a radiolabeled nucleic acid of known sequence is submitted to partial digestion. A second sample is incubated with its interacting partner and then is submitted to the same partial digestion. The two samples are then analyzed in parallel by electrophoresis on a denaturing acrylamide gel. After autoradiography the identification of the bands that correspond to fragments missing from the lane loaded with the second sample reveals the region of the nucleic acid that is protected from nuclease digestion upon binding.\nOBSOLETE because redundant with MI:0417 'footprinting' combined with interactor type MI:0319 'DNA' \nreplace by:MI:0417" [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_obsolete: true
<p><br>[Term]
<br>id: MI:0410
<br>name: electron tomography
<br>def: "Electron tomography is a general method for three-dimensional (3D) reconstruction of single, transparent objects from a series of projection images (i.e. from a tilt series) recorded with a transmission electron microscope. It offers the opportunity to obtain 3D information on structural cellular arrangements with a high resolution." [PMID:12160704]
<br>subset: PSI-MI slim
<br>synonym: "sem" EXACT PSI-MI-short []
<br>is_a: MI:0040 ! electron microscopy
<p><br>[Term]
<br>id: MI:0411
<br>name: enzyme linked immunosorbent assay
<br>def: "Following non-covalent binding of a purified primary ligand to a solid phase, a blocking reagent is added to prevent any non-specific binding. A specific antigen is then allowed to bind to the primary ligand. Unbound antigen is removed by washing and a secondary antibody conjugated to an enzyme (e.g. horseradish peroxidase) is added. Following a washing step to remove unbound secondary ligand, the extent to which a chromogenic substrate (e.g. 3,3', 5,5' tetramethyl benzidine chromogen [TMB]) is converted to a soluble coloured product by the conjugated enzyme in a given time is determined by spectrophotometry using a standard microplate absorbance reader. A similar type of approach can be utilized to detect enzymatic activities. The substrate, attached to a solid phase is incubated in the presence of the enzyme and the enzymatic modification is monitored by an antibody that is specific for the modified substrate (for instance a phosphorylated protein)." [PMID:11906746]
<br>subset: PSI-MI slim
<br>synonym: "ELISA" EXACT PSI-MI-alternate []
<br>synonym: "elisa" EXACT PSI-MI-short []
<br>is_a: MI:0421 ! identification by antibody
<br>is_a: MI:0892 ! solid phase assay
<p><br>[Term]
<br>id: MI:0412
<br>name: electrophoretic mobility supershift assay
<br>def: "The EMSA supershift is a EMSA experiment carried out using a third lane loaded with the radiolabeled nucleic acid, a protein mixture and an antibody for a specific protein. If an extra retardation is observed, this is due to the formation of a larger complex including the antibody. By this approach, at least one protein of the complex is directly identified." [PMID:12169687]
<br>subset: PSI-MI slim
<br>synonym: "emsa supershift" EXACT PSI-MI-short []
<br>is_a: MI:0413 ! electrophoretic mobility shift assay
<p><br>[Term]
<br>id: MI:0413
<br>name: electrophoretic mobility shift assay
<br>def: "This method proves the interaction between a nucleic acid and a protein partner. On the same electrophoresis gel 1 lane is loaded with a radiolabeled nucleic acid of known sequence, a second lane is loaded with the same nucleic acid together with a purified protein (or a protein mixture). After the electrophoresis run and autoradiography by comparing the nucleic acid migration in the two lanes the retardation of the nucleic acid due to its interaction with a protein can be easily observed." [PMID:12169687]
<br>subset: PSI-MI slim
<br>synonym: "band shift" EXACT PSI-MI-alternate []
<br>synonym: "emsa" EXACT PSI-MI-short []
<br>synonym: "Gel retardation assay" EXACT PSI-MI-alternate []
<br>is_a: MI:0892 ! solid phase assay
<p><br>[Term]
<br>id: MI:0414
<br>name: enzymatic reaction
<br>def: "terms aiming to represent biochemical reactions referring to their resulting product modifications." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "Biochemical reaction" EXACT PSI-MI-alternate []
<br>is_a: MI:0407 ! direct interaction
<p><br>[Term]
<br>id: MI:0415
<br>name: enzymatic study
<br>def: "Participants are enzyme or substrate in a biochemical reaction." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0401 ! biochemical
<p><br>[Term]
<br>id: MI:0416
<br>name: fluorescence microscopy
<br>def: "Fluorescent microscopy uses a high intensity light to illuminate the sample. This light excites fluorescence species in the sample, which then emit light of a longer wavelength. A fluorescent microscope also produces a magnified image of the sample, but the image is based on the second light source -- the light emanating from the fluorescent species -- rather than from the light originally used to illuminate, and excite, the sample." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "fluorescence imaging" EXACT PSI-MI-short []
<br>is_a: MI:0428 ! imaging technique
<p><br>[Term]
<br>id: MI:0417
<br>name: footprinting
<br>def: "footprinting analysis is used to identify regions of molecules directly involved in binding other macromolecules and therefore protected from the effects of degradative enzymes." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0401 ! biochemical
<p><br>[Term]
<br>id: MI:0418
<br>name: genetic
<br>def: "methods supporting genetic interactions.\nOBSOLETE as too unspecific use Genetic interference instead MI:0254." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_obsolete: true
<p><br>[Term]
<br>id: MI:0419
<br>name: gtpase assay
<br>def: "Measures the catalysis of the reaction: GTP + H2O = GDP + phosphate." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "gtp hydrolisis" EXACT PSI-MI-alternate []
<br>synonym: "GTPase" EXACT PSI-MI-alternate []
<br>is_a: MI:0879 ! nucleoside triphosphatase assay
<p><br>[Term]
<br>id: MI:0420
<br>name: kinase homogeneous time resolved fluorescence
<br>def: "Measures quenching of the nonradiative energy transfer between fluorescent long-lifetime lanthanide chelates and different acceptors. Relies on a fluorescence energy donor and acceptor being added from close proximity on the phosphorylated substrate due to the action of the kinase." [PMID:14987100]
<br>subset: PSI-MI slim
<br>synonym: "homogeneous time-resolved fluorescence" EXACT PSI-MI-alternate []
<br>synonym: "kinase HTRF" EXACT PSI-MI-alternate []
<br>synonym: "kinase htrf" EXACT PSI-MI-short []
<br>is_a: MI:0424 ! protein kinase assay
<br>is_a: MI:0510 ! homogeneous time resolved fluorescence
<p><br>[Term]
<br>id: MI:0421
<br>name: identification by antibody
<br>def: "Antibody mediated participant identification." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "antibody detection" EXACT PSI-MI-short []
<br>is_a: MI:0661 ! experimental participant identification
<p><br>[Term]
<br>id: MI:0422
<br>name: immunostaining
<br>def: "Method using an antibody coupled with some colouring agent to detect a specific protein within a cell or tissue sample. In some cases the primary antibody is directly linked to a colouring agent, more often the primary antibody is targeted by a secondary antibody, targeting the primary antibody." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0421 ! identification by antibody
<p><br>[Term]
<br>id: MI:0423
<br>name: in-gel kinase assay
<br>def: "Substrate protein radio-labelled by kinase transferring an isotope of phosphate from the nucleotide. Substrate isolated by gel electrophoresis and radio-labelling confirmed by autoradiography." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "in gel kinase assay" EXACT PSI-MI-short []
<br>is_a: MI:0424 ! protein kinase assay
<p><br>[Term]
<br>id: MI:0424
<br>name: protein kinase assay
<br>def: "Catalysis of the transfer of a phosphate group, usually from ATP, to a protein substrate." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0415 ! enzymatic study
<p><br>[Term]
<br>id: MI:0425
<br>name: kinase scintillation proximity assay
<br>def: "Relies on the radiolabelling of a peptide substrate immobilized on a scintillant coated SPA-bead. The kinase transfers a phosphate isotope from the nucleotide to the substrate." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "kinase spa" EXACT PSI-MI-short []
<br>is_a: MI:0099 ! scintillation proximity assay
<br>is_a: MI:0424 ! protein kinase assay
<p><br>[Term]
<br>id: MI:0426
<br>name: light microscopy
<br>def: "Light visible microscopy uses environmental light to illuminate the sample and produce a magnified image of the sample." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0428 ! imaging technique
<p><br>[Term]
<br>id: MI:0427
<br>name: Identification by mass spectrometry
<br>def: "identification by mass spectrometry." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "ms participant" EXACT PSI-MI-short []
<br>is_a: MI:0661 ! experimental participant identification
<p><br>[Term]
<br>id: MI:0428
<br>name: imaging technique
<br>def: "Methods that provide images of molecules at various resolution depending on the technology used." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "microscopy" EXACT PSI-MI-alternate []
<br>is_a: MI:0045 ! experimental interaction detection
<p><br>[Term]
<br>id: MI:0429
<br>name: necessary binding site
<br>def: "A sequence range within a molecule identified as being absolutely required for an interaction." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "required to bind" EXACT PSI-MI-short []
<br>is_a: MI:0117 ! binding site
<p><br>[Term]
<br>id: MI:0430
<br>name: nucleic acid uv cross-linking assay
<br>def: "Radiolabelled nucleic acids with a known sequence are incubated with purified proteins or reproducible protein mixture (HPLC eluate fractions) and then irradiated with UV. The complex are treated with an enzyme to remove the unbound nucleic acid (Rnase A or Dnase I for example), washed and then analyzed by electrophoresis. The eventual complexes are identified by autoradiography. The proteins involved in the complex can be recognized by specific antibodies or by retrieving the original protein mixture and carrying further analysis on it. The affinity of the binding can be estimated by adding a non-labelled nucleic acid as competitor before the UV irradiation." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "nucl ac uv crosslink" EXACT PSI-MI-short []
<br>is_a: MI:0030 ! cross-linking study
<p><br>[Term]
<br>id: MI:0431
<br>name: obsolete
<br>def: "Deprecated terms.\nOBSOLETE term replaced by the default OBO class 'Obsolete'." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_obsolete: true
<p><br>[Term]
<br>id: MI:0432
<br>name: one hybrid
<br>def: "Protein-DNA complementation assay where a single promoter act as bait and is screened against a library of prey transcription factors." [PMID:10589421]
<br>subset: PSI-MI slim
<br>synonym: "one-hybrid" EXACT PSI-MI-alternate []
<br>synonym: "yeast one hybrid" EXACT PSI-MI-alternate []
<br>synonym: "yeast one-hybrid" EXACT PSI-MI-alternate []
<br>is_a: MI:0232 ! transcriptional complementation assay
<p><br>[Term]
<br>id: MI:0433
<br>name: partial identification of protein sequence
<br>def: "partial protein sequence identification." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "partial id prot seq" EXACT PSI-MI-short []
<br>is_a: MI:0093 ! protein sequence identification
<p><br>[Term]
<br>id: MI:0434
<br>name: phosphatase assay
<br>def: "Measures the catalysis of the reaction: a phosphoprotein + H2O = a protein + phosphate." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0415 ! enzymatic study
<p><br>[Term]
<br>id: MI:0435
<br>name: protease assay
<br>def: "Measures the enzymatic hydrolysis of a peptide bond within a peptide or protein substrate." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0415 ! enzymatic study
<p><br>[Term]
<br>id: MI:0436
<br>name: protein footprinting
<br>def: "Protein footprinting is a technique for identifying structural changes modulated by protein-ligand binding, and mapping protein-ligand interfaces This technique involves attaching cutting reagents randomly to amino acid residue (e.g. lysine or cysteine) on the proteins surface and then using this lysine-labelled protein to cleave polypeptide backbone of the other protein at exposed residues adjacent to its binding site." [PMID:14600024, PMID:14967031, PMID:14987073]
<br>subset: PSI-MI slim
<br>is_a: MI:0659 ! experimental feature detection
<p><br>[Term]
<br>id: MI:0437
<br>name: protein tri hybrid
<br>def: "Two hybrid assay performed with a third protein component co-transfected into a recombinant yeast strain together with a bait and a prey construct. Negative control shows that the interaction between the bait and the prey do not occur when the third protein is not co-transfected." [PMID:12052864, PMID:12761205, PMID:12935900]
<br>subset: PSI-MI slim
<br>synonym: "bridge assay" EXACT PSI-MI-alternate []
<br>synonym: "trihybrid" EXACT PSI-MI-alternate []
<br>is_a: MI:0018 ! two hybrid
<br>is_a: MI:0588 ! 3 hybrid method
<p><br>[Term]
<br>id: MI:0438
<br>name: rna tri hybrid
<br>def: "In vivo reconstruction of specific RNA-proteins interactions. The DNA binding and transcription activator domains of GAL4 are brought together via the interaction of recombinant RNA. The first hybrid protein contains the DNA binding domain of GAL4 fused to RevM10 (a mutated RNA binding protein of HIV-1 that binds specifically to the Rev responsive element RRE of the env gene). A recombinant RNA contains the RRE sequence and a target RNA sequence X. The second hybrid protein contains the activation domain of GAL4 fused to protein Y tested for its ability to bind the target RNA X. If this interaction occurs the three hybrid reconstructs GAL4 and the transcription of a reporter gene is activated." [PMID:12162957, PMID:8972875]
<br>subset: PSI-MI slim
<br>synonym: "Three hybrid system" EXACT PSI-MI-alternate []
<br>is_a: MI:0232 ! transcriptional complementation assay
<br>is_a: MI:0588 ! 3 hybrid method
<p><br>[Term]
<br>id: MI:0439
<br>name: random spore analysis
<br>def: "A technique used to detect genetic interactions between 2 (or more) genes in a sporulating organism by scoring a large population of haploid spores for a phenotype and correlating the phenotype with the presence of single vs double (multiple) mutations. A diploid heterozygous organism harbouring mutations in two (or more) genes is induced to sporulate. Resulting spores are meiotic segregants that are haploid and are either wild type or mutant at each locus. Spores are scored for a phenotype, such as loss of viability." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "random-spore analysis" EXACT PSI-MI-alternate []
<br>synonym: "RSA" EXACT PSI-MI-alternate []
<br>synonym: "rsa" EXACT PSI-MI-short []
<br>synonym: "spore germination" EXACT PSI-MI-alternate []
<br>is_a: MI:0254 ! genetic interference
<p><br>[Term]
<br>id: MI:0440
<br>name: saturation binding
<br>def: "Saturation binding experiments measure specific ligand binding at equilibrium at various concentrations of the ligand. Analysis of these data can determine receptor number and affinity." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0400 ! affinity technology
<p><br>[Term]
<br>id: MI:0441
<br>name: synthetic genetic analysis
<br>def: "Identification of genetic interactions by generation of an organism harbouring mutations in 2 or more genes and scoring for a phenotype, such as loss of viability, that is not observed for any of the mutations in isolation." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "SGA" EXACT PSI-MI-alternate []
<br>synonym: "sga" EXACT PSI-MI-short []
<br>is_a: MI:0254 ! genetic interference
<p><br>[Term]
<br>id: MI:0442
<br>name: sufficient binding site
<br>def: "Binding will occur when this sequence range is present within a molecule of part of a molecule." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "sufficient to bind" EXACT PSI-MI-short []
<br>is_a: MI:0117 ! binding site
<p><br>[Term]
<br>id: MI:0443
<br>name: ubiquitin binding
<br>def: "Interaction concerning ubiquitin that is covalently attached to any Lys residue of its interaction partner.\nOBSOLETE remap to ubiquitination reaction (MI:0220) or describe ubiquitine as a participant on the interaction using physical interaction (MI:0218) or covalent binding (MI:0195) as interaction type." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_obsolete: true
<p><br>[Term]
<br>id: MI:0444
<br>name: database citation
<br>def: "Database citation list names of databases commonly used to cross reference interaction data." [PMID:14755292]
<br>subset: Drugable
<br>subset: PSI-MI slim
<br>relationship: part_of MI:0000 ! molecular interaction
<p><br>[Term]
<br>id: MI:0445
<br>name: literature database
<br>def: "Databases acting as a source of literature information." [PMID:14755292]
<br>subset: Drugable
<br>subset: PSI-MI slim
<br>synonym: "experiment xref" EXACT PSI-MI-short []
<br>is_a: MI:0444 ! database citation
<p><br>[Term]
<br>id: MI:0446
<br>name: pubmed
<br>def: "PubMed is designed to provide access to citations from biomedical literature.\nhttp://www.ncbi.nlm.nih.gov/entrez/query.fcgi" [PMID:14755292]
<br>subset: Drugable
<br>subset: PSI-MI slim
<br>xref: id-validation-regexp:\"[0-9\]+\"
<br>xref: search-url: "http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&list_uids=${ac}&dopt=Abstract"
<br>is_a: MI:0445 ! literature database
<p><br>[Term]
<br>id: MI:0447
<br>name: feature database
<br>def: "A database describing a feature on a molecule." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "feature xref" EXACT PSI-MI-short []
<br>is_a: MI:0444 ! database citation
<p><br>[Term]
<br>id: MI:0448
<br>name: gene ontology
<br>def: "The objective of Gene Ontology (GO) is to provide controlled vocabularies for the description of the molecular function, biological process and cellular component of gene products.\nhttp://www.ebi.ac.uk/GO" [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "go" EXACT PSI-MI-short []
<br>xref: id-validation-regexp:\"GO\:[0-9\]\{7}\"
<br>is_a: MI:0447 ! feature database
<br>is_a: MI:0461 ! interaction database
<p><br>[Term]
<br>id: MI:0449
<br>name: interpro
<br>def: "InterPro combines a number of databases (referred to as member databases) that use different methodologies and a varying degree of biological information on well-characterised proteins to derive protein signatures that predict family membership and domain composition of naive protein sequences.\nhttp://www.ebi.ac.uk/interpro/" [PMID:1252001]
<br>subset: PSI-MI slim
<br>synonym: "InterPro" EXACT PSI-MI-alternate []
<br>xref: id-validation-regexp:\"IPR[0-9\]\{6}\"
<br>xref: search-url: "http://www.ebi.ac.uk/interpro/DisplayIproEntry?ac=${ac}"
<br>is_a: MI:0447 ! feature database
<p><br>[Term]
<br>id: MI:0450
<br>name: cdd
<br>def: "The Conserved Domain Database may be used to identify the conserved domains present in a protein sequence.\nhttp://www.ncbi.nlm.nih.gov/Structure/cdd/cdd.shtml" [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "CDD" EXACT PSI-MI-alternate []
<br>xref: id-validation-regexp:\"[0-9\]+\"
<br>is_a: MI:0447 ! feature database
<p><br>[Term]
<br>id: MI:0451
<br>name: pfam
<br>def: "Pfam is a large collection of multiple sequence alignments and hidden Markov models covering many common protein domains.\nhttp://www.sanger.ac.uk/Software/Pfam" [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "Pfam" EXACT PSI-MI-alternate []
<br>xref: id-validation-regexp:\"PF[0-9\]\{5}\"
<br>is_a: MI:0449 ! interpro
<p><br>[Term]
<br>id: MI:0452
<br>name: pirsf
<br>def: "PIRSF is a classification system based on evolutionary relationship of whole proteins.\nhttp://pir.georgetown.edu/pirwww/dbinfo/pirsf.shtml" [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "PIRSF" EXACT PSI-MI-alternate []
<br>xref: id-validation-regexp:\"PIRSF[0-9\]\{5}\"
<br>is_a: MI:0449 ! interpro
<p><br>[Term]
<br>id: MI:0453
<br>name: prints
<br>def: "PRINTS is a compendium of protein fingerprints. A fingerprint is a group of conserved motifs used to characterise a protein family.\nhttp://umber.sbs.man.ac.uk/dbbrowser/PRINTS/" [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "PRINTS" EXACT PSI-MI-alternate []
<br>xref: id-validation-regexp:\"PR[0-9\]\{6}\"
<br>is_a: MI:0449 ! interpro
<p><br>[Term]
<br>id: MI:0454
<br>name: prodom
<br>def: "The ProDom protein domain database consists of an automatic compilation of homologous domains.\nhttp://protein.toulouse.inra.fr/prodom.html" [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "ProDom" EXACT PSI-MI-alternate []
<br>xref: id-validation-regexp:\"PD[0-9\]\{6}\"
<br>is_a: MI:0449 ! interpro
<p><br>[Term]
<br>id: MI:0455
<br>name: prosite
<br>def: "PROSITE is a database of protein families and domains. It consists of biologically significant sites, patterns and profiles.\nhttp://us.expasy.org/prosite/" [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "Prosite" EXACT PSI-MI-alternate []
<br>xref: id-validation-regexp:\"PS[0-9\]\{5}\"
<br>is_a: MI:0449 ! interpro
<p><br>[Term]
<br>id: MI:0456
<br>name: scop superfamily
<br>def: "SUPERFAMILY is a library of profile hidden Markov models that represent all proteins of known structure. The library is based on the SCOP classification of proteins: each model corresponds to a SCOP domain.\nhttp://supfam.mrc-lmb.cam.ac.uk/SUPERFAMILY/" [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "SCOP superfamily" EXACT PSI-MI-alternate []
<br>xref: id-validation-regexp:\"[0-9\]+\"
<br>is_a: MI:0449 ! interpro
<p><br>[Term]
<br>id: MI:0457
<br>name: smart
<br>def: "SMART (a Simple Modular Architecture Research Tool) allows the identification and annotation of genetically mobile domains and the analysis of domain architectures.\nhttp://smart.embl-heidelberg.de/" [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "SMART" EXACT PSI-MI-alternate []
<br>xref: id-validation-regexp:\"SM[0-9\]\{5}\"
<br>is_a: MI:0449 ! interpro
<p><br>[Term]
<br>id: MI:0458
<br>name: tigrfams
<br>def: "TIGRFAMs is a collection of protein families, featuring curated multiple sequence alignments, Hidden Markov Models (HMMs) and annotation.\nhttp://www.tigr.org/TIGRFAMs" [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "TIGRFAMs" EXACT PSI-MI-alternate []
<br>xref: id-validation-regexp:\"TIGR[0-9\]+\"
<br>is_a: MI:0449 ! interpro
<p><br>[Term]
<br>id: MI:0459
<br>name: mmdb
<br>def: "MMDB (Molecular Modeling DataBase), is a subset of three-dimensional structures obtained from the Protein Data Bank.\nhttp://www.ncbi.nlm.nih.gov/Structure" [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "MMDB" EXACT PSI-MI-alternate []
<br>xref: id-validation-regexp:\"[0-9\]+\"
<br>is_a: MI:0447 ! feature database
<br>is_a: MI:0461 ! interaction database
<p><br>[Term]
<br>id: MI:0460
<br>name: rcsb pdb
<br>def: "The RCSB PDB provides a variety of tools and resources for studying the structures of biological macromolecules and their relationships to sequence, function, and disease. \nhttp://www.pdb.org/" [PMID:14634627]
<br>subset: Drugable
<br>subset: PSI-MI slim
<br>synonym: "PDB" EXACT PSI-MI-alternate []
<br>xref: id-validation-regexp:\"[0-9\][a-zA-Z0-9\]\{3}\"
<br>xref: search-url: "http://www.pdb.org/pdb/explore/explore.do?structureId=${ac}"
<br>is_a: MI:0805 ! wwpdb
<p><br>[Term]
<br>id: MI:0461
<br>name: interaction database
<br>def: "Databases that contain experimental or predictive molecular interaction data." [PMID:14755292]
<br>subset: Drugable
<br>subset: PSI-MI slim
<br>synonym: "interaction xref" EXACT PSI-MI-short []
<br>is_a: MI:0444 ! database citation
<p><br>[Term]
<br>id: MI:0462
<br>name: bind
<br>def: "The Biomolecular Interaction Network Database (BIND) is a collection of records documenting molecular interactions.\nhttp://www.blueprint.org/bind" [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "BIND" EXACT PSI-MI-alternate []
<br>xref: id-validation-regexp:\"[0-9\]+\"
<br>is_a: MI:0461 ! interaction database
<br>is_a: MI:0489 ! source database
<p><br>[Term]
<br>id: MI:0463
<br>name: grid
<br>def: "The General Repository for Interaction Datasets (GRID) is a database of genetic and physical interactions.\nhttp://biodata.mshri.on.ca/grid" [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "GRID" EXACT PSI-MI-alternate []
<br>is_a: MI:0461 ! interaction database
<br>is_a: MI:0489 ! source database
<p><br>[Term]
<br>id: MI:0464
<br>name: cygd
<br>def: "The MIPS Comprehensive Yeast Genome Database (CYGD) aims to present information on the molecular structure and functional network of the entirely sequenced, well-studied model eukaryote, the budding yeast Saccharomyces cerevisiae. In addition the data of various projects on related yeasts are used for comparative analysis.\nhttp://mips.gsf.de/proj/yeast/CYGD.\nhttp://mips.gsf.de/genre/proj/mpact" [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "CYGD" EXACT PSI-MI-alternate []
<br>synonym: "CYGD (MIPS)" EXACT PSI-MI-alternate []
<br>synonym: "MIPS" EXACT PSI-MI-alternate []
<br>synonym: "MPact" EXACT PSI-MI-alternate []
<br>xref: id-validation-regexp:\"[0-9\]+|[A-Z\]\{3}[0-9\]\{3}[A-Za-z\](-[A-Za-z\])?|[A-Z0-9\]+\\.[0-9\]+|YM[A-Z\][0-9\]\{3}[a-z\][0-9\]\"
<br>is_a: MI:0461 ! interaction database
<br>is_a: MI:0489 ! source database
<br>is_a: MI:0683 ! sequence database
<p><br>[Term]
<br>id: MI:0465
<br>name: dip
<br>def: "The database of interacting protein (DIP) database stores experimentally determined interactions between proteins. It combines information from a variety of sources to create a single, consistent set of protein-protein interactions.\nhttp://dip.doe-mbi.ucla.edu/" [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "DIP" EXACT PSI-MI-alternate []
<br>xref: id-validation-regexp:\"DIP[0-9\]+[NE\]\"
<br>is_a: MI:0461 ! interaction database
<br>is_a: MI:0489 ! source database
<p><br>[Term]
<br>id: MI:0466
<br>name: ecocyc
<br>def: "EcoCyc is a bioinformatics database that describes the genome and the biochemical machinery of E. coli K12 MG1655.\nhttp://ecocyc.org/" [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "EcoCyc" EXACT PSI-MI-alternate []
<br>is_a: MI:0461 ! interaction database
<p><br>[Term]
<br>id: MI:0467
<br>name: reactome
<br>def: "The Reactome project is a collaboration among Cold Spring Harbor Laboratory, The European Bioinformatics Institute, and The Gene Ontology Consortium to develop a curated resource of core pathways and reactions in human biology.\nhttp://www.reactome.org/" [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "Genome Knowledge Base" EXACT PSI-MI-alternate []
<br>synonym: "GKB" EXACT PSI-MI-alternate []
<br>synonym: "Reactome" EXACT PSI-MI-alternate []
<br>is_a: MI:0461 ! interaction database
<p><br>[Term]
<br>id: MI:0468
<br>name: hprd
<br>def: "The Human Protein Reference Database represents a centralized platform to visually depict and integrate information pertaining to domain architecture, post-translational modifications, interaction networks and disease association for each protein in the human proteome.\nhttp://www.hprd.org/" [PMID:14755292]
<br>subset: Drugable
<br>subset: PSI-MI slim
<br>synonym: "HPRD" EXACT PSI-MI-alternate []
<br>is_a: MI:0461 ! interaction database
<p><br>[Term]
<br>id: MI:0469
<br>name: intact
<br>def: "INTerAction datanase (IntAct) provides an open source database and toolkit for the storage, presentation and analysis of protein interactions.\nhttp://www.ebi.ac.uk/intact" [PMID:14681455]
<br>subset: PSI-MI slim
<br>synonym: "IntAct" EXACT PSI-MI-alternate []
<br>xref: id-validation-regexp:\"EBI-[0-9\]+|IA\:[0-9\]+\"
<br>xref: search-url: "http://www.ebi.ac.uk/intact/search/do/hvWelcome?searchString=${ac}"
<br>is_a: MI:0461 ! interaction database
<br>is_a: MI:0489 ! source database
<p><br>[Term]
<br>id: MI:0470
<br>name: kegg
<br>def: "KEGG (Kyoto Encyclopedia of Genes and Genomes) is a knowledge base for systematic analysis of gene functions, linking genomic information with higher order functional information and also supplies information about chemical compounds, enzyme molecules and enzymatic reactions.\nhttp://www.genome.ad.jp/kegg/" [PMID:14755292]
<br>subset: Drugable
<br>subset: PSI-MI slim
<br>synonym: "KEGG" EXACT PSI-MI-alternate []
<br>xref: id-validation-regexp:\"[a-zA-Z\]+\:[a-zA-Z\]+[0-9\]+\"
<br>is_a: MI:0461 ! interaction database
<br>is_a: MI:0473 ! participant database
<p><br>[Term]
<br>id: MI:0471
<br>name: mint
<br>def: "The Moleculer INTeraction database (MINT) is a relational database designed to store interactions between biological molecules.\nhttp://mint.bio.uniroma2.it/mint/" [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "MINT" EXACT PSI-MI-alternate []
<br>xref: id-validation-regexp:\"MINT_[0-9\]+\"
<br>is_a: MI:0461 ! interaction database
<br>is_a: MI:0489 ! source database
<p><br>[Term]
<br>id: MI:0472
<br>name: pdbe
<br>def: "The Protein Data Bank in Europe - the European project for the collection, management and distribution of data about macromolecular structures, derived in part from the Protein Data Bank (PDB).\nhttp://www.ebi.ac.uk/pdbe/" [PMID:16381867]
<br>subset: PSI-MI slim
<br>synonym: "<new synonym>" RELATED []
<br>synonym: "e-MSD" EXACT []
<br>synonym: "MSD" RELATED []
<br>synonym: "PQS" EXACT PSI-MI-alternate []
<br>xref: id-validation-regexp:\"[0-9\][a-zA-Z0-9\]\{3}\"
<br>xref: search-url: "http://www.ebi.ac.uk/pdbe-srv/atlas?id=${ac}"
<br>is_a: MI:0805 ! wwpdb
<p><br>[Term]
<br>id: MI:0473
<br>name: participant database
<br>def: "Database of molecules participating in molecular interactions." [PMID:14755292]
<br>subset: Drugable
<br>subset: PSI-MI slim
<br>synonym: "participant xref" EXACT PSI-MI-short []
<br>is_a: MI:0444 ! database citation
<p><br>[Term]
<br>id: MI:0474
<br>name: chebi
<br>def: "A definitive, freely available database of Chemical compounds of Biological Interest (ChEBI).\nhttp://www.ebi.ac.uk/chebi/" [PMID:14755292]
<br>subset: Drugable
<br>subset: PSI-MI slim
<br>synonym: "ChEBI" EXACT PSI-MI-alternate []
<br>xref: id-validation-regexp:\"CHEBI\:[0-9\]+\"
<br>xref: search-url: "http://www.ebi.ac.uk/chebi/searchId.do?chebiId=${ac}"
<br>is_a: MI:2054 ! bioactive entity reference
<p><br>[Term]
<br>id: MI:0475
<br>name: ddbj/embl/genbank
<br>def: "DDBJ EMBL GenBank Nucleotide Sequence Database Collaboration exchange new and updated data on a daily basis to achieve optimal synchronisation.\nhttp://www.ebi.ac.uk/embl/Contact/collaboration" [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "DDBJ" EXACT PSI-MI-alternate []
<br>synonym: "DDBJ/EMBL/GenBank" EXACT PSI-MI-alternate []
<br>synonym: "EMBL" EXACT PSI-MI-alternate []
<br>synonym: "GenBank" EXACT PSI-MI-alternate []
<br>xref: id-validation-regexp:\"[A-Z\][0-9\]\{5}|[A-Z\][0-9\]\{5}\\.[0-9\]+|[A-Z\]\{2}[0-9\]\{6}|[A-Z\]\{2}[0-9\]\{6}\\.[0-9\]+|[A-Z\]\{4}[0-9\]\{8}|[A-Z\]\{4}[0-9\]\{8}\\.[0-9\]+\"
<br>xref: search-url: "http://www.ebi.ac.uk/cgi-bin/dbfetch?db=EMBLSVA&id=${ac}"
<br>is_a: MI:0683 ! sequence database
<p><br>[Term]
<br>id: MI:0476
<br>name: ensembl
<br>def: "Ensembl is a joint project between the EMBL-EBI and the Wellcome Trust Sanger Institute that aims at developing a system that maintains automatic annotation of large eukaryotic genomes.\nhttp://www.ebi.ac.uk/ensembl" [PMID:15078858]
<br>subset: PSI-MI slim
<br>synonym: "Ensembl" EXACT PSI-MI-alternate []
<br>xref: id-validation-regexp:\"ENS[A-Z\]+[0-9\]\{11}|[A-Z\]\{3}[0-9\]\{3}[A-Za-z\](-[A-Za-z\])?|CG[0-9\]+|[A-Z0-9\]+\\.[0-9\]+|YM[A-Z\][0-9\]\{3}[a-z\][0-9\]\"
<br>is_a: MI:0683 ! sequence database
<p><br>[Term]
<br>id: MI:0477
<br>name: entrez gene/locuslink
<br>def: "LocusLink provides a single query interface to curated sequence and descriptive information about genetic loci.\nhttp://www.ncbi.nlm.nih.gov/LocusLink/" [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "Entrez gene/locuslink" EXACT PSI-MI-alternate []
<br>synonym: "entrezgene/locuslink" EXACT PSI-MI-short []
<br>xref: id-validation-regexp:\"[0-9\]+|[A-Z\]\{1\,2}_[0-9\]+|[A-Z\]\{1\,2}_[A-Z\]\{1\,4}[0-9\]+\"
<br>is_a: MI:0683 ! sequence database
<p><br>[Term]
<br>id: MI:0478
<br>name: flybase
<br>def: "FlyBase is a comprehensive database for information on the genetics and molecular biology of Drosophila.\nhttp://fbserver.gen.cam.ac.uk:7081/" [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "FlyBase" EXACT PSI-MI-alternate []
<br>xref: id-validation-regexp:\"FBgn[0-9\]\{7}\"
<br>xref: search-url: "http://flybase.net/cgi-bin/fbannq.html?acc=${ac}"
<br>is_a: MI:0683 ! sequence database
<p><br>[Term]
<br>id: MI:0479
<br>name: mgd/mgi
<br>def: "Mouse Genome Informatics (MGI) provides integrated access to data on the genetics, genomics, and biology of the laboratory mouse.\nhttp://www.informatics.jax.org/" [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "MGD/MGI" EXACT PSI-MI-alternate []
<br>xref: id-validation-regexp:\"MGI\:[0-9\]+\"
<br>is_a: MI:0683 ! sequence database
<p><br>[Term]
<br>id: MI:0480
<br>name: omim
<br>def: "Online Mendelian Inheritance in Man (OMIM) is a catalogue of human genes and genetic disorders, with links to literature references, sequence records, maps, and related databases.\nhttp://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=OMIM" [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "OMIM" EXACT PSI-MI-alternate []
<br>xref: id-validation-regexp:\"[0-9\]+\"
<br>xref: search-url: "http://www.ncbi.nlm.nih.gov/entrez/dispomim.cgi?id=${ac}"
<br>is_a: MI:0683 ! sequence database
<p><br>[Term]
<br>id: MI:0481
<br>name: refseq
<br>def: "The Reference Sequence (RefSeq) collection aims to provide a comprehensive, integrated, non-redundant set of sequences, including genomic DNA, transcript (RNA), and protein products, for a number of organisms.\nhttp://www.ncbi.nlm.nih.gov/RefSeq/" [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "Refseq" EXACT PSI-MI-alternate []
<br>xref: id-validation-regexp:\"[XNZ\][A-Z\]_[0-9\]+|[0-9\]+|[XNZ\][A-Z\]_[0-9\]+\\.[0-9\]+\"
<br>is_a: MI:0683 ! sequence database
<p><br>[Term]
<br>id: MI:0482
<br>name: rfam
<br>def: "Rfam is a large collection of multiple sequence alignments and covariance models covering many common non-coding RNA families.\nhttp://www.sanger.ac.uk/Software/Rfam/" [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "rfam" EXACT PSI-MI-alternate []
<br>xref: id-validation-regexp:\"RF[0-9\]\{5}\"
<br>is_a: MI:0683 ! sequence database
<p><br>[Term]
<br>id: MI:0483
<br>name: rgd
<br>def: "The Rat Genome Database (RGD) curates and integrates rat genetic and genomic data.\nhttp://rgd.mcw.edu/" [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "RGD" EXACT PSI-MI-alternate []
<br>xref: id-validation-regexp:\"[0-9\]+\"
<br>is_a: MI:0683 ! sequence database
<p><br>[Term]
<br>id: MI:0484
<br>name: sgd
<br>def: "SGD is a scientific database of the molecular biology and genetics of the yeast Saccharomyces cerevisiae.\nhttp://www.yeastgenome.org/" [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "SGD" EXACT PSI-MI-alternate []
<br>xref: id-validation-regexp:\"S[0-9\]\{9}\"
<br>xref: search-url: "http://genome-www4.stanford.edu/cgi-bin/SGD/locus.pl?locus=${ac}"
<br>is_a: MI:0683 ! sequence database
<p><br>[Term]
<br>id: MI:0485
<br>name: uniparc
<br>def: "UniProt Archive (UniParc) is part of UniProt project. It is a non-redundant archive of protein sequences derived from many sources.\nhttp://www.ebi.ac.uk/uniparc/" [PMID:14681372]
<br>subset: PSI-MI slim
<br>synonym: "UniParc" EXACT PSI-MI-alternate []
<br>xref: id-validation-regexp:\"UPI[A-F0-9\]\{10}\"
<br>xref: search-url: "http://www.ebi.uniprot.org/entry/${ac}"
<br>is_a: MI:0683 ! sequence database
<p><br>[Term]
<br>id: MI:0486
<br>name: uniprot knowledge base
<br>def: "UniProt (Universal Protein Resource) is the world's most comprehensive catalogue of information on proteins. It is a central repository of protein sequence and function created by joining the information contained in Swiss-Prot, TrEMBL, and PIR.\nhttp://www.expasy.uniprot.org/" [PMID:14681372]
<br>subset: PSI-MI slim
<br>synonym: "UniProt" EXACT PSI-MI-alternate []
<br>synonym: "uniprotkb" EXACT PSI-MI-short []
<br>xref: id-validation-regexp:\"[A-Z\][0-9\][A-Z0-9\]\{3}[0-9\]|[A-Z\][0-9\][A-Z0-9\]\{3}[0-9\]-[0-9\]+|[A-Z\][0-9\][A-Z0-9\]\{3}[0-9\]-PRO_[0-9\]\{10}\"
<br>xref: search-url: "http://www.ebi.uniprot.org/entry/${ac}"
<br>is_a: MI:0683 ! sequence database
<p><br>[Term]
<br>id: MI:0487
<br>name: wormbase
<br>def: "WormBase is the central worm database that houses the gene reports, locus reports, translation reports, expression pattern data and genome browser.\nhttp://www.wormbase.org/" [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "WormBase" EXACT PSI-MI-alternate []
<br>xref: id-validation-regexp:\"WBGene[0-9\]\{8}\"
<br>is_a: MI:0683 ! sequence database
<p><br>[Term]
<br>id: MI:0488
<br>name: psi-mi
<br>def: "PSI-MI." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "PSI-MI" EXACT PSI-MI-alternate []
<br>xref: id-validation-regexp:\"MI\:[0-9\]\{4}\"
<br>xref: search-url: "http://www.ebi.ac.uk/ontology-lookup/?termId=${ac}"
<br>is_a: MI:0444 ! database citation
<p><br>[Term]
<br>id: MI:0489
<br>name: source database
<br>def: "Database that originally provided the interaction record for exchange purposes." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0444 ! database citation
<p><br>[Term]
<br>id: MI:0490
<br>name: experiment condition
<br>def: "Describes the location of the experiment.\nOBSOLETE as a full host organisms description is recommended using tax id == -1 as convention to refer to 'in vitro' interaction." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_obsolete: true
<p><br>[Term]
<br>id: MI:0491
<br>name: in silico
<br>def: "Results generated by predictive bioinformatics approaches rather than experimental data.\nOBSOLETE as a full host organisms description is recommended using tax id == -1 as convention to refer to 'in vitro' interaction." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "Predictive" EXACT PSI-MI-alternate []
<br>is_obsolete: true
<p><br>[Term]
<br>id: MI:0492
<br>name: in vitro
<br>def: "Experiments performed with participants removed from the cellular environment e.g. cell extracts, isolated proteins.\nOBSOLETE as a full host organisms description is recommended using tax id == -1 as convention to refer to 'in vitro' interaction." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_obsolete: true
<p><br>[Term]
<br>id: MI:0493
<br>name: in vivo
<br>def: "Experiment undertaken within a cellular environment, although this may not be the natural host of the proteins in the study.\nOBSOLETE as a full host organisms description is recommended using tax id == -1 as convention to refer to 'in vitro' interaction." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_obsolete: true
<p><br>[Term]
<br>id: MI:0494
<br>name: in situ
<br>def: "Literally, in place i.e. the protein is in its natural environment during the experiment.\nOBSOLETE as a full host organisms is recommended using tax id == -1 as convention to refer to 'in vitro' interaction." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_obsolete: true
<p><br>[Term]
<br>id: MI:0495
<br>name: experimental role
<br>def: "Role played by the participant within the experiment." [PMID:14755292]
<br>subset: PSI-MI slim
<br>relationship: part_of MI:0000 ! molecular interaction
<p><br>[Term]
<br>id: MI:0496
<br>name: bait
<br>def: "Molecule experimentally treated to capture its interacting partners." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0495 ! experimental role
<p><br>[Term]
<br>id: MI:0497
<br>name: neutral component
<br>def: "Molecule role in an experimental setting that does not have an embedded asymmetry." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0495 ! experimental role
<p><br>[Term]
<br>id: MI:0498
<br>name: prey
<br>def: "Molecule experimentally identified as being captured by a given bait." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0495 ! experimental role
<p><br>[Term]
<br>id: MI:0499
<br>name: unspecified role
<br>def: "Role not specified or not applicable to the data." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0495 ! experimental role
<br>is_a: MI:0500 ! biological role
<p><br>[Term]
<br>id: MI:0500
<br>name: biological role
<br>def: "Physiological role of an interactor in a cell or in vivo environment, which is reproduced in the current experiment." [PMID:14755292]
<br>subset: PSI-MI slim
<br>relationship: part_of MI:0000 ! molecular interaction
<p><br>[Term]
<br>id: MI:0501
<br>name: enzyme
<br>def: "Molecule catalyzing a modification on its interacting partner." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0500 ! biological role
<p><br>[Term]
<br>id: MI:0502
<br>name: enzyme target
<br>def: "Molecule that is the target of its binding partner catalytic activity." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "substrate" EXACT PSI-MI-alternate []
<br>is_a: MI:0500 ! biological role
<p><br>[Term]
<br>id: MI:0503
<br>name: self
<br>def: "Molecule that makes intramolecular interactions." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0495 ! experimental role
<br>is_a: MI:0500 ! biological role
<p><br>[Term]
<br>id: MI:0505
<br>name: experimental feature
<br>def: "The form of a molecule that was actually used to experimentally demonstrate the interaction, that may differ from the sequence described by the identifying accession number." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0116 ! feature type
<p><br>[Term]
<br>id: MI:0506
<br>name: over expressed level
<br>def: "A molecule is estimated to be expressed at higher levels than in physiological condition." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "over-expressed" EXACT PSI-MI-short []
<br>is_a: MI:0221 ! expression level
<br>is_a: MI:0803 ! expression level alteration
<p><br>[Term]
<br>id: MI:0507
<br>name: tag
<br>def: "Small molecules, peptides or full proteins that can be used as label as they confer some property that facilitates identification purification and monitoring to the labelled molecule." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0505 ! experimental feature
<p><br>[Term]
<br>id: MI:0508
<br>name: deacetylase radiometric assay
<br>def: "Measures the release of radiolabelled acetic acid from pre-labeled histone." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "radiolabeled acetate" EXACT PSI-MI-short []
<br>is_a: MI:0406 ! deacetylase assay
<p><br>[Term]
<br>id: MI:0509
<br>name: phosphatase homogeneous time resolved fluorescence
<br>def: "Measures quenching of the nonradiative energy transfer between fluorescent long-lifetime lanthanide chelates and different acceptors. Relies on a fluorescence energy donor and acceptor being removed from close proximity on the phosphorylated substrate due to the action of the phosphatase." [PMID:14987100]
<br>subset: PSI-MI slim
<br>synonym: "homogeneous time-resolved fluorescence" EXACT PSI-MI-alternate []
<br>synonym: "phosphatase HTRF" EXACT PSI-MI-alternate []
<br>synonym: "phosphatase htrf" EXACT PSI-MI-short []
<br>is_a: MI:0434 ! phosphatase assay
<br>is_a: MI:0510 ! homogeneous time resolved fluorescence
<p><br>[Term]
<br>id: MI:0510
<br>name: homogeneous time resolved fluorescence
<br>def: "Methods based on the exceptionally long fluorescence lifetime characteristics of certain fluorophores, which allows the elimination of the effects of background fluorescence. Uses nonradiative energy transfer or quenching between fluorescent lanthanide chelates and different acceptors to measure reaction rates." [PMID:14987100]
<br>subset: PSI-MI slim
<br>synonym: "homogeneous time-resolved fluorescence" EXACT PSI-MI-alternate []
<br>synonym: "htrf" EXACT PSI-MI-short []
<br>is_a: MI:0051 ! fluorescence technology
<p><br>[Term]
<br>id: MI:0511
<br>name: protease homogeneous time resolved fluorescence
<br>def: "Measures quenching of the nonradiative energy transfer between fluorescent long-lifetime lanthanide chelates and different acceptors. Fluorescence donor and acceptor are on the same peptide molecule and separated by the action of the protease." [PMID:14987100]
<br>subset: PSI-MI slim
<br>synonym: "Protease HTRF" EXACT PSI-MI-alternate []
<br>synonym: "protease htrf" EXACT PSI-MI-short []
<br>is_a: MI:0435 ! protease assay
<br>is_a: MI:0510 ! homogeneous time resolved fluorescence
<p><br>[Term]
<br>id: MI:0512
<br>name: zymography
<br>def: "Samples run on a gelatine containing gels under non-reducing condition, gels then incubated under conditions in which the enzyme is active. Gels are stained with coomasie and gelatine-free regions of the gel taken as a measure of enzyme activity." [PMID:2071592]
<br>subset: PSI-MI slim
<br>is_a: MI:0435 ! protease assay
<p><br>[Term]
<br>id: MI:0513
<br>name: collagen film assay
<br>def: "Measures the amount of radiolabel released into the medium when enzyme is added onto a film of isotope-labelled collagen." [PMID:6247938]
<br>subset: PSI-MI slim
<br>is_a: MI:0435 ! protease assay
<p><br>[Term]
<br>id: MI:0514
<br>name: in gel phosphatase assay
<br>def: "Substrate protein pre-radiolabelled either synthetically or through the action of a kinase transferring an isotope of phosphate from a nucleotide. Substrate then exposed to phosphate under assay conditions. Substrate isolated by gel electrophoresis and loss of radiolabelling confirmed by autoradiography." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "in gel phosphatase" EXACT PSI-MI-short []
<br>is_a: MI:0434 ! phosphatase assay
<p><br>[Term]
<br>id: MI:0515
<br>name: methyltransferase assay
<br>def: "Measures the catalysis of the transfer of a methyl group to an acceptor molecule." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "methyltransferase as" EXACT PSI-MI-short []
<br>is_a: MI:0415 ! enzymatic study
<p><br>[Term]
<br>id: MI:0516
<br>name: methyltransferase radiometric assay
<br>def: "Measures the transfer of a radiolabelled methyl group of a donor, for example S-adenosyl-L-methionine (SAM) to a carboxyl group of an acceptor." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "radiolabeled methyl" EXACT PSI-MI-short []
<br>is_a: MI:0515 ! methyltransferase assay
<p><br>[Term]
<br>id: MI:0517
<br>name: radiolabel
<br>def: "A radiolabelled molecule has radio isotopes among its constituent atoms that can be used to identify, localize or quantify the full molecule." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "radiolabeled" EXACT PSI-MI-alternate []
<br>synonym: "radiolabelled" EXACT PSI-MI-short []
<br>is_a: MI:0253 ! isotope label
<p><br>[Term]
<br>id: MI:0518
<br>name: flag tag
<br>def: "The protein of interest is expressed as a fusion to the peptide DYKDDDDKV for which antibodies are commercially available. Sometimes multiple copies of the peptide are fused in tandem." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "DYKDDDDKV epitope tag" EXACT PSI-MI-alternate []
<br>synonym: "FLAG" EXACT PSI-MI-alternate []
<br>synonym: "FLAG-tagged" EXACT PSI-MI-alternate []
<br>is_a: MI:0507 ! tag
<p><br>[Term]
<br>id: MI:0519
<br>name: glutathione s tranferase tag
<br>def: "The protein is expressed and purified as a fusion to the glutathione S-tranferase protein." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "glutathione S-tranferase tag" EXACT PSI-MI-alternate []
<br>synonym: "gst tag" EXACT PSI-MI-short []
<br>is_a: MI:0365 ! enzyme tag
<p><br>[Term]
<br>id: MI:0520
<br>name: ha tag
<br>def: "The protein of interest is expressed as a fusion to the peptide YPYDVPDYA (a fragment of the influenza hemagglutinin protein) for which antibodies are commercially available." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "YPYDVPDYA epitope tag" EXACT PSI-MI-alternate []
<br>is_a: MI:0507 ! tag
<p><br>[Term]
<br>id: MI:0521
<br>name: his tag
<br>def: "The protein of interest is expressed as a fusion to a poly-His tail. This permits purification by chromatography over a metal column or by binding to commercially available anti poly-His antibodies." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "6-His-tag" EXACT PSI-MI-alternate []
<br>synonym: "Hexa-His-tag" EXACT PSI-MI-alternate []
<br>synonym: "Histidine-tag" EXACT PSI-MI-alternate []
<br>is_a: MI:0507 ! tag
<p><br>[Term]
<br>id: MI:0522
<br>name: myc tag
<br>def: "The protein of interest is expressed as a fusion to the peptide EUKLISEED (a fragment of the Myc oncogene protein) for which antibodies are commercially available. Sometimes multiple copies of the peptide are fused in tandem." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "EUKLISEED epitope tag" EXACT PSI-MI-alternate []
<br>is_a: MI:0507 ! tag
<p><br>[Term]
<br>id: MI:0523
<br>name: t7 tag
<br>def: "The protein of interest is expressed as a fusion to the peptide MASMTGGQQMG for which antibodies are commercially available." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "MASMTGGQQMG epitope tag" EXACT PSI-MI-alternate []
<br>is_a: MI:0507 ! tag
<p><br>[Term]
<br>id: MI:0524
<br>name: calmodulin binding peptide plus protein a tag
<br>def: "Tag encoding a calmodulin binding peptide, a TEV cleavage site, and the Staphylococcus aureus Protein A fused to a target protein and the introduction of the construct into the host cell or organism, maintaining the expression of the fusion protein at, or close to, its natural level. The fusion protein and associated components are recovered from cell extracts by affinity selection on an IgG matrix. After washing, the TEV protease is added to release the bound material. The eluate is incubated with calmodulin-coated beads in the presence of calcium. This second affinity step is required to remove the TEV protease as well as traces of contaminants remaining after the first affinity selection. After washing, the bound material is released with EGTA. This tag allows two steps purification steps ensuring a highly selective purification of the tapped protein (first round of selection on the protein A, a high affinity tag) under mild condition (non denaturant pH or conditions required to remove the tag)." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "CBP-ProtA tagged" EXACT PSI-MI-alternate []
<br>synonym: "tap tagged" EXACT PSI-MI-short []
<br>is_a: MI:0677 ! tandem tag
<p><br>[Term]
<br>id: MI:0525
<br>name: v5 tag
<br>def: "The protein of interest is expressed as a fusion to the peptide GKPIPNPLLGLDST for which antibodies are commercially available." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "GKPIPNPLLGLDST epitope tag" EXACT PSI-MI-alternate []
<br>is_a: MI:0507 ! tag
<p><br>[Term]
<br>id: MI:0526
<br>name: n-acetyl-lysine
<br>def: "Residue modification." [PMID:14755292, RESID:AA0048, RESID:AA0055]
<br>subset: PSI-MI slim
<br>synonym: "acetyllysine" EXACT PSI-MI-short []
<br>is_a: MI:0121 ! acetylated residue
<p><br>[Term]
<br>id: MI:0527
<br>name: adp ribosylated residue
<br>def: "Residue modification." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "adp-ribosylated" EXACT PSI-MI-short []
<br>is_a: MI:0120 ! post translation modification
<p><br>[Term]
<br>id: MI:0528
<br>name: omega-n-(adp-ribosyl)-arginine
<br>def: "Residue modification." [PMID:14755292, RESID:AA0168]
<br>subset: PSI-MI slim
<br>synonym: "(S)-2-amino-5-([imino([adenosine 5'-(trihydrogen diphosphate) 5'->5'-ester with alpha-D-ribofuranosyl]amino)methyl]amino)pentanoic acid" EXACT PSI-MI-alternate []
<br>synonym: "adp-ribosylarginine" EXACT PSI-MI-short []
<br>synonym: "N(omega)-[alpha-D-ribofuranoside 5'->5'-ester with adenosine 5'-(trihydrogen diphosphate)]-L-arginine" EXACT PSI-MI-alternate []
<br>synonym: "N(omega)-alpha-D-ribofuranosyl-L-arginine 5'->5'-ester with adenosine 5'-(trihydrogen diphosphate)" EXACT PSI-MI-alternate []
<br>synonym: "omega-N-(ADP-ribosyl)-L-arginine" EXACT PSI-MI-alternate []
<br>is_a: MI:0527 ! adp ribosylated residue
<p><br>[Term]
<br>id: MI:0529
<br>name: s-(adp-ribosyl)-cysteine
<br>def: "Residue modification." [PMID:14755292, RESID:AA0169]
<br>subset: PSI-MI slim
<br>synonym: "(R)-2-amino-3-([adenosine 5'-(trihydrogen diphosphate) 5'->5'-ester with alpha-D-ribofuranosyl]sulfanyl)propanoic acid" EXACT PSI-MI-alternate []
<br>synonym: "adp-ribosylcysteine" EXACT PSI-MI-short []
<br>synonym: "S-(ADP-ribosyl)-L-cysteine" EXACT PSI-MI-alternate []
<br>synonym: "S-alpha-D-ribofuranosyl-L-cysteine 5'->5'-ester with adenosine 5'-(trihydrogen diphosphate)" EXACT PSI-MI-alternate []
<br>synonym: "S-L-cysteine alpha-D-ribofuranoside 5'->5'-ester with adenosine 5'-(trihydrogen diphosphate)" EXACT PSI-MI-alternate []
<br>is_a: MI:0527 ! adp ribosylated residue
<p><br>[Term]
<br>id: MI:0530
<br>name: glutamyl-5-poly(adp-ribose)
<br>def: "Residue modification." [PMID:14755292, RESID:AA0295]
<br>subset: PSI-MI slim
<br>synonym: "(S)-2-amino-5-poly[2'-adenosine 5'-(trihydrogen diphosphate) 5'->5'-ester with 1alpha-D-ribofuranosyl]oxy-5-oxopentanoic acid" EXACT PSI-MI-alternate []
<br>synonym: "adp-ribosylglutamate" EXACT PSI-MI-short []
<br>synonym: "L-glutamyl-5-poly(ADP-ribose)" EXACT PSI-MI-alternate []
<br>synonym: "L-isoglutamyl-poly(ADP-ribose)" EXACT PSI-MI-alternate []
<br>is_a: MI:0527 ! adp ribosylated residue
<p><br>[Term]
<br>id: MI:0531
<br>name: o-(adp-ribosyl)-serine
<br>def: "Residue modification." [PMID:14755292, RESID:AA0237]
<br>subset: PSI-MI slim
<br>synonym: "(S)-2-amino-3-([adenosine 5'-(trihydrogen diphosphate) 5'->5'-ester with alpha-D-ribofuranosyl]oxy)-propanoic acid Formula" EXACT PSI-MI-alternate []
<br>synonym: "adp-ribosylserine" EXACT PSI-MI-short []
<br>synonym: "O-(ADP-ribosyl)-L-serine" EXACT PSI-MI-alternate []
<br>synonym: "O3-(ADP-ribosyl)-L-serine" EXACT PSI-MI-alternate []
<br>synonym: "O3-[alpha-D-ribofuranoside 5'->5'-ester with adenosine 5'-(trihydrogen diphosphate)]-L-serine" EXACT PSI-MI-alternate []
<br>synonym: "O3-alpha-D-ribofuranosyl-L-serine 5'->5'-ester with adenosine 5'-(trihydrogen diphosphate)" EXACT PSI-MI-alternate []
<br>is_a: MI:0527 ! adp ribosylated residue
<p><br>[Term]
<br>id: MI:0532
<br>name: n4-(adp-ribosyl)-asparagine
<br>def: "Residue modification." [PMID:14755292, RESID:AA0231]
<br>subset: PSI-MI slim
<br>synonym: "(S)-2-amino-4-([adenosine 5'-(trihydrogen diphosphate) 5'->5'-ester with alpha-D-ribofuranosyl]amino)-4-oxobutanoic acid" EXACT PSI-MI-alternate []
<br>synonym: "adpribosylasparagine" EXACT PSI-MI-short []
<br>synonym: "N4-(ADP-ribosyl)-L-asparagine" EXACT PSI-MI-alternate []
<br>synonym: "N4-[alpha-D-ribofuranoside 5'->5'-ester with adenosine 5'-(trihydrogen diphosphate)]-L-asparagine" EXACT PSI-MI-alternate []
<br>synonym: "N4-alpha-D-ribofuranosyl-L-asparagine 5'->5'-ester with adenosine 5'-(trihydrogen diphosphate)" EXACT PSI-MI-alternate []
<br>is_a: MI:0527 ! adp ribosylated residue
<p><br>[Term]
<br>id: MI:0533
<br>name: glycosylated residue
<br>def: "Residue modification." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0120 ! post translation modification
<p><br>[Term]
<br>id: MI:0534
<br>name: glycosyl-cysteine
<br>def: "Residue modification." [PMID:14755292, RESID:AA0122]
<br>subset: PSI-MI slim
<br>synonym: "S-glycosyl-L-cysteine" EXACT PSI-MI-alternate []
<br>is_a: MI:0533 ! glycosylated residue
<p><br>[Term]
<br>id: MI:0535
<br>name: glycosyl-serine
<br>def: "Residue modification." [PMID:14755292, RESID:AA0154]
<br>subset: PSI-MI slim
<br>synonym: "O-glycosyl-L-serine" EXACT PSI-MI-alternate []
<br>synonym: "O3-glycosyl-L-serine" EXACT PSI-MI-alternate []
<br>is_a: MI:0533 ! glycosylated residue
<p><br>[Term]
<br>id: MI:0536
<br>name: glycosyl-threonine
<br>def: "Residue modification." [PMID:14755292, RESID:AA0155]
<br>subset: PSI-MI slim
<br>synonym: "O-glycosyl-L-threonine" EXACT PSI-MI-alternate []
<br>synonym: "O3-glycosyl-L-threonine" EXACT PSI-MI-alternate []
<br>is_a: MI:0533 ! glycosylated residue
<p><br>[Term]
<br>id: MI:0537
<br>name: omega-n-glycosyl-arginine
<br>def: "Residue modification." [PMID:14755292, RESID:AA0327]
<br>subset: PSI-MI slim
<br>synonym: "glycosylarginine" EXACT PSI-MI-short []
<br>synonym: "omega-N-glycosyl-L-arginine" EXACT PSI-MI-alternate []
<br>is_a: MI:0533 ! glycosylated residue
<p><br>[Term]
<br>id: MI:0538
<br>name: n4-glycosyl-asparagine
<br>def: "Residue modification." [PMID:14755292, RESID:AA0151]
<br>subset: PSI-MI slim
<br>synonym: "glycosylasparagine" EXACT PSI-MI-short []
<br>synonym: "N4-glycosyl-L-asparagine" EXACT PSI-MI-alternate []
<br>is_a: MI:0533 ! glycosylated residue
<p><br>[Term]
<br>id: MI:0539
<br>name: gpi anchor residue
<br>def: "Residue modification." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0150 ! lipid modification
<p><br>[Term]
<br>id: MI:0540
<br>name: gpi-anchor amidated alanine
<br>def: "Residue modification." [PMID:14755292, RESID:AA0163]
<br>subset: PSI-MI slim
<br>synonym: "gpi-alanine" EXACT PSI-MI-short []
<br>synonym: "N-alanyl-glycosylphosphatidylinositolethanolamine" EXACT PSI-MI-alternate []
<br>is_a: MI:0539 ! gpi anchor residue
<p><br>[Term]
<br>id: MI:0541
<br>name: gpi-anchor amidated asparagine
<br>def: "Residue modification." [PMID:14755292, RESID:AA0158]
<br>subset: PSI-MI slim
<br>synonym: "gpi-asparagine" EXACT PSI-MI-short []
<br>synonym: "N-asparaginyl-glycosylphosphatidylinositolethanolamine" EXACT PSI-MI-alternate []
<br>is_a: MI:0539 ! gpi anchor residue
<p><br>[Term]
<br>id: MI:0542
<br>name: gpi-anchor amidated aspartate
<br>def: "Residue modification." [PMID:14755292, RESID:AA0159]
<br>subset: PSI-MI slim
<br>synonym: "gpi-aspartate" EXACT PSI-MI-short []
<br>synonym: "N-aspartyl-glycosylphosphatidylinositolethanolamine" EXACT PSI-MI-alternate []
<br>is_a: MI:0539 ! gpi anchor residue
<p><br>[Term]
<br>id: MI:0543
<br>name: gpi-anchor amidated cysteine
<br>def: "Residue modification." [PMID:14755292, RESID:AA0160]
<br>subset: PSI-MI slim
<br>synonym: "gpi-cysteine" EXACT PSI-MI-short []
<br>synonym: "N-cysteinyl-glycosylphosphatidylinositolethanolamine" EXACT PSI-MI-alternate []
<br>is_a: MI:0539 ! gpi anchor residue
<p><br>[Term]
<br>id: MI:0544
<br>name: gpi-anchor amidated glycine
<br>def: "Residue modification." [PMID:14755292, RESID:AA0161]
<br>subset: PSI-MI slim
<br>synonym: "gpi-glycine" EXACT PSI-MI-short []
<br>synonym: "N-glycyl-glycosylphosphatidylinositolethanolamine" EXACT PSI-MI-alternate []
<br>is_a: MI:0539 ! gpi anchor residue
<p><br>[Term]
<br>id: MI:0545
<br>name: gpi-anchor amidated serine
<br>def: "Residue modification." [PMID:14755292, RESID:AA0162]
<br>subset: PSI-MI slim
<br>synonym: "gpi-serine" EXACT PSI-MI-short []
<br>synonym: "N-seryl-glycosylphosphatidylinositolethanolamine" EXACT PSI-MI-alternate []
<br>is_a: MI:0539 ! gpi anchor residue
<p><br>[Term]
<br>id: MI:0546
<br>name: gpi-anchor amidated threonine
<br>def: "Residue modification." [PMID:14755292, RESID:AA0164]
<br>subset: PSI-MI slim
<br>synonym: "gpi-threonine" EXACT PSI-MI-short []
<br>synonym: "N-threonyl-glycosylphosphatidylinositolethanolamine" EXACT PSI-MI-alternate []
<br>is_a: MI:0539 ! gpi anchor residue
<p><br>[Term]
<br>id: MI:0547
<br>name: s-prenyl-cysteine
<br>def: "Residue modification." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "prenylcysteine" EXACT PSI-MI-short []
<br>is_a: MI:0150 ! lipid modification
<p><br>[Term]
<br>id: MI:0548
<br>name: methylated-lysine
<br>def: "Residue modification." [PMID:14755292, RESID:AA0074, RESID:AA0075, RESID:AA0076]
<br>subset: PSI-MI slim
<br>synonym: "methylatedlysine" EXACT PSI-MI-short []
<br>is_a: MI:0157 ! methylated residue
<p><br>[Term]
<br>id: MI:0549
<br>name: alkylated cysteine
<br>def: "Artificial residue modification enabling studies of cysteine binding status." [PMID:15325307]
<br>subset: PSI-MI slim
<br>is_a: MI:0179 ! other modification
<p><br>[Term]
<br>id: MI:0550
<br>name: gamma-carboxyglutamic acid
<br>def: "Residue modification." [PMID:14755292, RESID:AA0032]
<br>subset: PSI-MI slim
<br>synonym: "(S)-3-amino-1,1,3-propanetricarboxylic acid" EXACT PSI-MI-alternate []
<br>synonym: "1-carboxyglutamic acid [misnomer]" EXACT PSI-MI-alternate []
<br>synonym: "4-carboxyglutamic acid" EXACT PSI-MI-alternate []
<br>synonym: "carboxyglutamic acid" EXACT PSI-MI-short []
<br>synonym: "L-gamma-carboxyglutamic acid" EXACT PSI-MI-alternate []
<br>is_a: MI:0179 ! other modification
<p><br>[Term]
<br>id: MI:0551
<br>name: nitro-tyrosine
<br>def: "Residue modification." [PMID:15657065, PMID:9636206]
<br>subset: PSI-MI slim
<br>synonym: "nitrated tyrosine" EXACT PSI-MI-alternate []
<br>is_a: MI:0179 ! other modification
<p><br>[Term]
<br>id: MI:0552
<br>name: s-nitrosyl-cysteine
<br>def: "Residue modification." [PMID:14755292, RESID:AA0230]
<br>subset: PSI-MI slim
<br>synonym: "(R)-2-amino-3-nitrososulfanyl-propanoic acid" EXACT PSI-MI-alternate []
<br>synonym: "L-cysteine nitrite ester" EXACT PSI-MI-alternate []
<br>synonym: "nitrosylcysteine" EXACT PSI-MI-short []
<br>synonym: "S-nitrosocysteine" EXACT PSI-MI-alternate []
<br>synonym: "S-nitrosyl-L-cysteine" EXACT PSI-MI-alternate []
<br>is_a: MI:0179 ! other modification
<p><br>[Term]
<br>id: MI:0553
<br>name: o4'-sulfo-tyrosine
<br>def: "Residue modification." [PMID:14755292, RESID:AA0172]
<br>subset: PSI-MI slim
<br>synonym: "(S)-2-amino-3-(4-sulfooxyphenyl)propanoic acid" EXACT PSI-MI-alternate []
<br>synonym: "2-amino-3-(4-hydroxyphenyl)propanoic acid 4'-sulfate" EXACT PSI-MI-alternate []
<br>synonym: "O4'-sulfo-L-tyrosine" EXACT PSI-MI-alternate []
<br>synonym: "O4-sulfotyrosine" EXACT PSI-MI-alternate []
<br>synonym: "sulfotyrosine" EXACT PSI-MI-short []
<br>synonym: "tyrosine sulfate" EXACT PSI-MI-alternate []
<br>is_a: MI:0179 ! other modification
<p><br>[Term]
<br>id: MI:0554
<br>name: sumoylated lysine
<br>def: "Residue modification due to a cross-link between a lysine and a glycine from the sumo (Small Ubiquitin-related MOdifier) protein." [PMID:12612601, RESID:AA0125]
<br>subset: PSI-MI slim
<br>synonym: "(S)-2-amino-6-[(aminoacetyl)amino]hexanoic acid" EXACT PSI-MI-alternate []
<br>synonym: "N6-glycyl-L-lysine" EXACT PSI-MI-alternate []
<br>synonym: "N6-glycyllysine" EXACT PSI-MI-alternate []
<br>is_a: MI:0179 ! other modification
<p><br>[Term]
<br>id: MI:0555
<br>name: phospho-histidine
<br>def: "Residue modification." [PMID:14755292, RESID:AA0035, RESID:AA0036]
<br>subset: PSI-MI slim
<br>synonym: "phosphoshistidine" EXACT PSI-MI-short []
<br>is_a: MI:0170 ! phosphorylated residue
<p><br>[Term]
<br>id: MI:0556
<br>name: transglutamination reaction
<br>def: "Gln-Lys cross-link catalyzed by a transglutaminase." [PMID:14755292, RESID:AA0124]
<br>subset: PSI-MI slim
<br>synonym: "transglutamination" EXACT PSI-MI-short []
<br>is_a: MI:0195 ! covalent binding
<p><br>[Term]
<br>id: MI:0557
<br>name: adp ribosylation reaction
<br>def: "Reaction that can affect Arg, Cys, Glu, Arg and Asn residues." [GO:0006471, PMID:14755292, RESID:AA0168, RESID:AA0169, RESID:AA0231, RESID:AA0237, RESID:AA0295]
<br>subset: PSI-MI slim
<br>synonym: "adp ribosylation" EXACT PSI-MI-short []
<br>is_a: MI:0414 ! enzymatic reaction
<p><br>[Term]
<br>id: MI:0558
<br>name: deglycosylation reaction
<br>def: "Reaction catalyzed by PNGase, a deglycosylating enzyme that promotes the hydrolysis of the beta-aspartylglycosylamine bond of aspargine-linked glycopeptides and glycoproteins." [GO:0006517, PMID:15670854]
<br>subset: PSI-MI slim
<br>synonym: "deglycosylation" EXACT PSI-MI-short []
<br>is_a: MI:0414 ! enzymatic reaction
<p><br>[Term]
<br>id: MI:0559
<br>name: glycosylation reaction
<br>def: "The covalent attachment of a glycosyl residue to one or more monomeric units in a polypeptide, polynucleotide, polysaccharide, or other biological polymer. Reaction that can affect Ser, Thr, Cys, Arg, and Asn residues. This reaction is known to be reversible in the case of Asn substrate." [GO:0043413, PMID:14755292, RESID:AA0122, RESID:AA0151, RESID:AA0154, RESID:AA0155, RESID:AA0327]
<br>subset: PSI-MI slim
<br>synonym: "glycosylation" EXACT PSI-MI-short []
<br>is_a: MI:0414 ! enzymatic reaction
<p><br>[Term]
<br>id: MI:0560
<br>name: myristoylated residue
<br>def: "Residue modification." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "myristoylated aa" EXACT PSI-MI-short []
<br>is_a: MI:0150 ! lipid modification
<p><br>[Term]
<br>id: MI:0561
<br>name: palmitoylated residue
<br>def: "Residue modification." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "palmitoylated aa" EXACT PSI-MI-short []
<br>is_a: MI:0150 ! lipid modification
<p><br>[Term]
<br>id: MI:0562
<br>name: methylated alanine
<br>def: "Residue modification." [PMID:14755292, RESID:AA0061, RESID:AA0062]
<br>subset: PSI-MI slim
<br>is_a: MI:0157 ! methylated residue
<p><br>[Term]
<br>id: MI:0563
<br>name: methylated arginine
<br>def: "Residue modification." [PMID:14755292, RESID:AA0068, RESID:AA0069]
<br>subset: PSI-MI slim
<br>is_a: MI:0157 ! methylated residue
<p><br>[Term]
<br>id: MI:0564
<br>name: omega-n-methyl-arginine
<br>def: "Residue modification." [PMID:14755292, RESID:AA0069]
<br>subset: PSI-MI slim
<br>synonym: "(S)-2-amino-5-[(imino(methylamino)methyl)amino]pentanoic acid" EXACT PSI-MI-alternate []
<br>synonym: "methylarginine" EXACT PSI-MI-short []
<br>synonym: "NG-methylarginine;" EXACT PSI-MI-alternate []
<br>synonym: "omega-N-methyl-L-arginine" EXACT PSI-MI-alternate []
<br>is_a: MI:0563 ! methylated arginine
<p><br>[Term]
<br>id: MI:0565
<br>name: neddylated lysine
<br>def: "Residue modification due to a cross-link between a lysine and a glycine from the Nedd8 protein family." [PMID:111]
<br>subset: PSI-MI slim
<br>is_a: MI:0179 ! other modification
<p><br>[Term]
<br>id: MI:0566
<br>name: sumoylation reaction
<br>def: "Reversible reaction that create a covalent bond between a C-terminus G of an ubiquitine like sumo protein and a K residue of the target." [GO:0016925, PMID:15985640]
<br>subset: PSI-MI slim
<br>synonym: "sumoylation" EXACT PSI-MI-short []
<br>is_a: MI:0414 ! enzymatic reaction
<p><br>[Term]
<br>id: MI:0567
<br>name: neddylation reaction
<br>def: "Reversible reaction that create a covalent bond between a Glycine residue of an ubiquitine like NEDD8 protein and a lysine residue of the target." [GO:0045116, PMID:16127432]
<br>subset: PSI-MI slim
<br>synonym: "neddylation" EXACT PSI-MI-short []
<br>is_a: MI:0414 ! enzymatic reaction
<p><br>[Term]
<br>id: MI:0568
<br>name: desumoylation reaction
<br>def: "Cleavage of the G-K bond and release of the SUMO ubiquitin like proteins." [GO:0016926, PMID:15985640]
<br>subset: PSI-MI slim
<br>synonym: "desumoylation" EXACT PSI-MI-short []
<br>is_a: MI:0414 ! enzymatic reaction
<p><br>[Term]
<br>id: MI:0569
<br>name: deneddylation reaction
<br>def: "Cleavage of the G-K bond and release of the NEDD8 ubiquitin like proteins. Deneddylation, which removes the NEDD8 moiety, requires the isopeptidase activity of the COP9 signalosome." [GO:0000338, PMID:16127432]
<br>subset: PSI-MI slim
<br>synonym: "deneddylation" EXACT PSI-MI-short []
<br>is_a: MI:0414 ! enzymatic reaction
<p><br>[Term]
<br>id: MI:0570
<br>name: protein cleavage
<br>def: "Covalent modification of a polypeptide occuring during its maturation or its proteolytic degradation." [PMID:14744292]
<br>subset: PSI-MI slim
<br>is_a: MI:0194 ! cleavage reaction
<p><br>[Term]
<br>id: MI:0571
<br>name: mrna cleavage
<br>def: "Any process by which a pre-mRNA or mRNA molecule is cleaved at specific sites or in a regulated manner." [GO:0006379, PMID:14681407]
<br>subset: PSI-MI slim
<br>is_a: MI:0902 ! rna cleavage
<p><br>[Term]
<br>id: MI:0572
<br>name: dna cleavage
<br>def: "Covalent bond breakage of a DNA molecule leading to the formation of smaller fragments." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0910 ! nucleic acid cleavage
<p><br>[Term]
<br>id: MI:0573
<br>name: mutation disrupting interaction
<br>def: "Region of a molecule whose mutation or deletion totally disrupts an interaction." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "mutation disrupting" EXACT PSI-MI-short []
<br>is_a: MI:0119 ! mutation decreasing interaction
<p><br>[Term]
<br>id: MI:0574
<br>name: digital object identifier
<br>def: "Identifier of a publication prior to pubmed indexing." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "doi" EXACT PSI-MI-short []
<br>xref: id-validation-regexp:\"\\d+.\\d+/[a-zA-Z0-9\\.\\\:\]+\"
<br>xref: search-url: "http://dx.doi.org/${ac}"
<br>is_a: MI:0445 ! literature database
<p><br>[Term]
<br>id: MI:0575
<br>name: alliance for cellular signaling
<br>def: "Alliance for Cellular Signaling (AfCS -Nature) store yeast 2-hybrid Interaction data and expression data. Information and data are freely available to all.\nhttp://www.signaling-gateway.org" [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "AfCS" EXACT PSI-MI-alternate []
<br>synonym: "afcs" EXACT PSI-MI-short []
<br>xref: id-validation-regexp:\"[0-9\]+\"
<br>xref: search-url: "http://www.signaling-gateway.org/data/Y2H/cgi-bin/y2h_int.cgi?id=${ac}"
<br>is_a: MI:0461 ! interaction database
<br>is_a: MI:0489 ! source database
<p><br>[Term]
<br>id: MI:0576
<br>name: structural proximity
<br>def: "Method to identify domain-domain interactions within the same resolved structure. Domains are first projected onto a pdb structure and then the distance between all pairs of residues in different domains are calculated. When the distance between 2 residues is below the non covalent bond threshold, the corresponding pair of domains is predicted to interact." [PMID:15353450]
<br>subset: PSI-MI slim
<br>is_a: MI:0577 ! feature prediction from structure
<p><br>[Term]
<br>id: MI:0577
<br>name: feature prediction from structure
<br>def: "Group of method taking advantage of 3D structure to calculate and infer the feature of interacting molecules." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "feature struct pred" EXACT PSI-MI-short []
<br>is_a: MI:0660 ! feature prediction
<p><br>[Term]
<br>id: MI:0578
<br>name: maltose binding protein tag
<br>def: "The protein is expressed and purified as a fusion to the glutathione maltose-binding protein (MBP). The MBP-fusion protein can be purified by affinity chromatography using an amylose resin." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "maltose binding protein tag" EXACT PSI-MI-alternate []
<br>synonym: "mbp tag" EXACT PSI-MI-short []
<br>is_a: MI:0240 ! fusion protein
<p><br>[Term]
<br>id: MI:0579
<br>name: electron donor
<br>def: "Any molecule that is able to transfer an electron to another chemical species." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0918 ! donor
<p><br>[Term]
<br>id: MI:0580
<br>name: electron acceptor
<br>def: "Molecule to which and electron may be transferred from an electron donor." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0919 ! acceptor
<p><br>[Term]
<br>id: MI:0581
<br>name: suppressor gene
<br>def: "Gene whose mutation suppress the phenotype associated to a suppressed mutation." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0495 ! experimental role
<p><br>[Term]
<br>id: MI:0582
<br>name: suppressed gene
<br>def: "Gene whose mutation phenotype is suppressed by a given suppressor mutation." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0495 ! experimental role
<p><br>[Term]
<br>id: MI:0583
<br>name: fluorescence donor
<br>def: "Fluorophore which emits electromagnetic radiation of given wavelength." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0495 ! experimental role
<p><br>[Term]
<br>id: MI:0584
<br>name: fluorescence acceptor
<br>def: "Fluorophore able to absorb the electromagnetic radiation at given wavelength from a specific donor fluorophore, the re-emission of its own characteristic fluorescence." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "fluorescence accept" EXACT PSI-MI-short []
<br>is_a: MI:0495 ! experimental role
<p><br>[Term]
<br>id: MI:0585
<br>name: intenz
<br>def: "IntEnz is the name for the Integrated relational Enzyme database and is the official version of the Enzyme Nomenclature. The Enzyme Nomenclature comprises recommendations of the Nomenclature Committee of the International Union of Bio chemistry and Molecular Biology (NC-IUBMB) on the nomenclature and classification of enzyme-catalysed reactions. IntEnz is supported by NC-IUBMB and contains enzyme data curated and approved by this committee. The database IntEnz is available at.\nhttp://www.ebi.ac.uk/intenz" [PMID:14681451]
<br>subset: PSI-MI slim
<br>xref: id-validation-regexp:\"[0-6\]\{1}\\.(\\d+|-)\\.(\\d+|-)\\.(\\d+|-)\"
<br>xref: search-url: "http://www.ebi.ac.uk/intenz/query?cmd=Search&q=${ac}&t=exact&fields=ec"
<br>is_a: MI:0461 ! interaction database
<p><br>[Term]
<br>id: MI:0586
<br>name: inhibitor
<br>def: "Molecule inhibiting an interaction by interacting with one or more of its participants." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0500 ! biological role
<p><br>[Term]
<br>id: MI:0587
<br>name: inhibited
<br>def: "Molecule being identified as target of an inhibitor.\nOBSOLETE as term is deprecated to describe the target of an inhibitor that can have any other biological role." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_obsolete: true
<p><br>[Term]
<br>id: MI:0588
<br>name: 3 hybrid method
<br>def: "Group of method based on complementation assay where a third participant is shown to be necessary for the binding of a given bait prey pair." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0090 ! protein complementation assay
<p><br>[Term]
<br>id: MI:0589
<br>name: in vitro translated protein
<br>def: "Protein sample collected by in vitro translation of its mRNA taking advantage of purified translation machinery." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "in vitro translated" EXACT PSI-MI-short []
<br>is_a: MI:0342 ! sample process
<p><br>[Term]
<br>id: MI:0590
<br>name: attribute name
<br>def: "Collection of topics describing the free text stored as an attribute value." [PMID:14755292]
<br>subset: Drugable
<br>subset: PSI-MI slim
<br>synonym: "CvTopic" EXACT PSI-MI-alternate []
<br>relationship: part_of MI:0000 ! molecular interaction
<p><br>[Term]
<br>id: MI:0591
<br>name: experiment description
<br>def: "The experimental condition text description, should contain information about the organisms hosting the interaction." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "experiment descripti" EXACT PSI-MI-short []
<br>is_a: MI:0665 ! experiment attibute name
<p><br>[Term]
<br>id: MI:0592
<br>name: ipfam
<br>def: "Web resource that allows the investigation of protein interactions in the Protein Data Bank structures at the level of Pfam domains and amino acid residues. iPfam is available on the Web for browsing at.\nhttp://www.sanger.ac.uk/Software/Pfam/iPfam/" [PMID:15353450]
<br>subset: PSI-MI slim
<br>is_a: MI:0447 ! feature database
<p><br>[Term]
<br>id: MI:0593
<br>name: translocation
<br>def: "Xref pointing to a GO process term describing the start and end location of a migrating molecule, for instance see GO:0006611, 'protein-nucleus export'." [PMID:14681407]
<br>subset: PSI-MI slim
<br>is_a: MI:0353 ! cross-reference type
<p><br>[Term]
<br>id: MI:0594
<br>name: translocation start
<br>def: "Xref pointing to a GO compartment term describing the start location of a migrating molecule." [PMID:14681407]
<br>subset: PSI-MI slim
<br>is_a: MI:0593 ! translocation
<p><br>[Term]
<br>id: MI:0595
<br>name: translocation end
<br>def: "Xref pointing to a GO compartment term describing the end location of a migrating molecule." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0593 ! translocation
<p><br>[Term]
<br>id: MI:0596
<br>name: experimental form description
<br>def: "Free text description of all the tags and artificial process undergone by a molecule during an experiment." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "experimental form de" EXACT PSI-MI-short []
<br>is_a: MI:0666 ! participant attribute name
<p><br>[Term]
<br>id: MI:0597
<br>name: feature description
<br>def: "The feature text description may include information about the feature detection method." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0668 ! feature attribute name
<p><br>[Term]
<br>id: MI:0598
<br>name: feature constraint
<br>def: "The feature constraint free text will specificity whether a biological feature is shown to be possible (just observed) or required (experimentally demonstrated to be necessary for an interaction)." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0668 ! feature attribute name
<p><br>[Term]
<br>id: MI:0599
<br>name: figure legend
<br>def: "Text pointing to a specific paper figure legend where the experimental evidences for an interaction are to be found." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0664 ! interaction attribute name
<br>is_a: MI:0665 ! experiment attibute name
<p><br>[Term]
<br>id: MI:0600
<br>name: conditional synthetic lethal nutrition-sensitivity
<br>def: "Two silent mutations show a nutrition sensitive lethal phenotype when they co-occur on the same cell.\nOBSOLETE: remap to CV intraction type 'synthetic interaction' MI:0794 and external CV for phenotype description." [PMID:15608217]
<br>subset: PSI-MI slim
<br>synonym: "nutrition synt letal" EXACT PSI-MI-short []
<br>is_obsolete: true
<p><br>[Term]
<br>id: MI:0601
<br>name: sequence ontology
<br>def: "The Sequence Ontology (SO) is a structured controlled vocabulary for the parts of a genomic annotation. SO provides a common set of terms and definitions that will facilitate the exchange, analysis and management of genomic data." [PMID:15892872]
<br>subset: PSI-MI slim
<br>synonym: "so" EXACT PSI-MI-short []
<br>xref: id-validation-regexp:\"SO\:[0-9\]\{7}\"
<br>is_a: MI:0447 ! feature database
<br>is_a: MI:0473 ! participant database
<p><br>[Term]
<br>id: MI:0602
<br>name: chemical footprinting
<br>def: "Binding sites are identified by altered reactivity of a complex to a chemical treatment compared to the unbound molecules. Residues in close contact with the binding partner are protected from cleavage by the enzyme. When these chemicals are administrated to intact cells, the pattern of protection from the probes identifies the location of DNA-protein or protein-protein interactions in vivo." [PMID:8238889]
<br>subset: PSI-MI slim
<br>synonym: "chemical footprint" EXACT PSI-MI-short []
<br>is_a: MI:0417 ! footprinting
<p><br>[Term]
<br>id: MI:0603
<br>name: dimethylsulphate footprinting
<br>def: "Dimethylsulphate (DMS) is the most commonly used chemical to study DNA-protein interactions. DMS induces methylation of guanine residues so DNA interaction with protein binding to AT rich sequences or to the phosphate backbone may be not detected by DMS footprinting. However as DMS diffuses across membrane it can also be used for in vivo footprinting. The experiment involves the treatment with DMS of two DNA samples with identical sequence, one protein bound and the other naked. The two samples are treated with piperidine to induce chemical cleavage of the DMS modified guanine residues followed by digestion with restriction enzymes. Once labelled the samples are run in parallel on a gel to visualize the pattern of nested fragments sharing a common end generated by restriction enzyme(or PCR primer extension) and a variable end guanine dependent. The missing bands of the protein bound sample correspond to the guanine residues protected from modification by an interaction." [PMID:8238889]
<br>subset: PSI-MI slim
<br>synonym: "dms footprinting" EXACT PSI-MI-short []
<br>is_a: MI:0602 ! chemical footprinting
<p><br>[Term]
<br>id: MI:0604
<br>name: potassium permanganate footprinting
<br>def: "Potassium permanganate bind to single-stranded pyrimidine residues, it is commonly used to detect promoters opening regions in vivo. KMnO4 treatment of cells, followed by treatment with piperidine, followed by either PCR and/or acrylamide gel electrophoresis allows detection of interaction between transcription factor and the DNA sequence under their control." [PMID:8238889]
<br>subset: PSI-MI slim
<br>synonym: "k-mn-04 footprinting" EXACT PSI-MI-short []
<br>is_a: MI:0602 ! chemical footprinting
<p><br>[Term]
<br>id: MI:0605
<br>name: enzymatic footprinting
<br>def: "Binding sites are identified by altered reactivity of a complex to an enzymatic probe compared to the unbound molecules. Residues in close contact with the binding partner are protected from cleavage by the enzyme. When these enzymes are administrated to intact cells, the pattern of protection from the probes identifies the location of DNA-protein or protein-protein interactions in vivo." [PMID:8238889]
<br>subset: PSI-MI slim
<br>synonym: "enzymatic footprint" EXACT PSI-MI-short []
<br>is_a: MI:0417 ! footprinting
<p><br>[Term]
<br>id: MI:0606
<br>name: DNase I footprinting
<br>def: "Deoxyribonuclease I (DNase I) do not have high specificity for given sequences or residues, thus footprinting with DNase I permits the exact delineation of the protein-DNA binding site. Moreover DNase I, can be used for in vivo footprinting by treating intact cells with permeabilising drugs. In this latter case DNase I in vivo footprinting allow studies of the chromatin structure in genomic DNA." [PMID:8238889]
<br>subset: PSI-MI slim
<br>synonym: "dnase 1 footprinting" EXACT PSI-MI-short []
<br>is_a: MI:0605 ! enzymatic footprinting
<p><br>[Term]
<br>id: MI:0607
<br>name: small nuclear rna
<br>def: "These RNA molecules are relatively short (less than 200 nucleotides each), and there are five of them (U1, U2, U4, U5, and U6) involved in the major form of pre-mRNA splicing. Known as snRNAs (small nuclear RNAs), each is complexed with at least seven protein subunits to form a snRNP (small nuclear ribonucleoprotein). These snRNPs form the core of the spliceosome." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "snRNA" EXACT PSI-MI-alternate []
<br>synonym: "snrna" EXACT PSI-MI-short []
<br>is_a: MI:0320 ! ribonucleic acid
<p><br>[Term]
<br>id: MI:0608
<br>name: ribosomal rna
<br>def: "RNA that is transcribed from the DNA of the nucleolus and is found, together with characteristic proteins, in the ribosomes." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "rRNA" EXACT PSI-MI-alternate []
<br>synonym: "rrna" EXACT PSI-MI-short []
<br>is_a: MI:0320 ! ribonucleic acid
<p><br>[Term]
<br>id: MI:0609
<br>name: small nucleolar rna
<br>def: "The small nucleolar RNAs, are stable RNA contained in the nucleoli and these RNAs exist as snoRNA: protein complexes called snoRNPs (also called 'snorps'). The snoRNPs function is in the maturation of ribosomal RNA and other RNAs, by: creating two types of modified nucleotides, (2'-O-methylated nucleotides and pseudouridine), and mediating endonucleolytic cleavages of pre-rRNA." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "snoRNA" EXACT PSI-MI-alternate []
<br>synonym: "snorna" EXACT PSI-MI-short []
<br>is_a: MI:0320 ! ribonucleic acid
<p><br>[Term]
<br>id: MI:0610
<br>name: small interfering rna
<br>def: "Ribonucleic acid used in RNAi study. These RNA have the reverse complementary sequence of a target gene's mRNA transcript and inhibit its expression." [PMID:10542148]
<br>subset: PSI-MI slim
<br>synonym: "dicer RNA" EXACT PSI-MI-alternate []
<br>synonym: "micro RNA" EXACT PSI-MI-alternate []
<br>synonym: "siRNA" EXACT PSI-MI-alternate []
<br>synonym: "sirna" EXACT PSI-MI-short []
<br>is_a: MI:0320 ! ribonucleic acid
<p><br>[Term]
<br>id: MI:0611
<br>name: signal recognition particle rna
<br>def: "Small (300 nucleotides) stable RNAs transcribed by RNA pol III that are part of the signal-recognition particle (SRP). This particle comprises one RNA and six proteins bound to the RNA. SRP function is to assist secretory proteins sorting in the endoplasmic reticulum (ER). SRP is a cytosolic particle that transiently binds to the ER signal sequence of a nascent protein, to the large ribosomal unit, and to the SRP receptor in the ER membrane." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "srpRNA" EXACT PSI-MI-alternate []
<br>synonym: "srprna" EXACT PSI-MI-short []
<br>is_a: MI:0320 ! ribonucleic acid
<p><br>[Term]
<br>id: MI:0612
<br>name: comment
<br>def: "Comment for public view. This attribute can be associated to interaction, experiment, CV term, an organism and any participant." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0664 ! interaction attribute name
<br>is_a: MI:0665 ! experiment attibute name
<br>is_a: MI:0666 ! participant attribute name
<br>is_a: MI:0667 ! controlled vocabulary attribute name
<br>is_a: MI:0668 ! feature attribute name
<br>is_a: MI:0669 ! organism attribute name
<p><br>[Term]
<br>id: MI:0613
<br>name: function
<br>def: "Biological function of a participant or of an interaction." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0664 ! interaction attribute name
<br>is_a: MI:0666 ! participant attribute name
<p><br>[Term]
<br>id: MI:0614
<br>name: url
<br>def: "URL/Web address describing an experiment, an interaction, a Cv term or an organism." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0664 ! interaction attribute name
<br>is_a: MI:0665 ! experiment attibute name
<br>is_a: MI:0667 ! controlled vocabulary attribute name
<br>is_a: MI:0669 ! organism attribute name
<p><br>[Term]
<br>id: MI:0615
<br>name: search-url
<br>def: "Search engine URL associated to Cv Database terms." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0667 ! controlled vocabulary attribute name
<p><br>[Term]
<br>id: MI:0616
<br>name: example
<br>def: "Example generally associated to Cv terms. Test." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0300 ! alias type
<br>is_a: MI:0667 ! controlled vocabulary attribute name
<p><br>[Term]
<br>id: MI:0617
<br>name: disease
<br>def: "The interaction has a known or demonstrated disease association." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0664 ! interaction attribute name
<p><br>[Term]
<br>id: MI:0618
<br>name: caution
<br>def: "Warning about errors or grounds for confusion. Can be associated to an interaction, experiment, CV term or any participant." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0664 ! interaction attribute name
<br>is_a: MI:0665 ! experiment attibute name
<br>is_a: MI:0666 ! participant attribute name
<br>is_a: MI:0667 ! controlled vocabulary attribute name
<br>is_a: MI:0668 ! feature attribute name
<br>is_a: MI:0669 ! organism attribute name
<p><br>[Term]
<br>id: MI:0619
<br>name: pathway
<br>def: "Refers to the metabolic or signalling pathway involving an interaction or a complex." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0664 ! interaction attribute name
<p><br>[Term]
<br>id: MI:0620
<br>name: search-url-ascii
<br>def: "Search URL to retrieve an external entry in ASCII format. Generally associated to Cv Database terms." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0667 ! controlled vocabulary attribute name
<p><br>[Term]
<br>id: MI:0621
<br>name: author-confidence
<br>def: "Confidence classification assigned by the author of the publication to a specific interaction." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0664 ! interaction attribute name
<p><br>[Term]
<br>id: MI:0622
<br>name: confidence-mapping
<br>def: "Description of confidence assessment method generally associated to the experiment." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0665 ! experiment attibute name
<p><br>[Term]
<br>id: MI:0623
<br>name: inhibition
<br>def: "The interaction between the proteins or the formation of a complex is disrupted by a biological molecule or by a modification of the interactors." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0664 ! interaction attribute name
<p><br>[Term]
<br>id: MI:0624
<br>name: stimulation
<br>def: "Reaction occurs at a faster rate in the presence of this compound or molecule i.e. the molecule directly physically co-operates with the interaction. Reaction may not occur at all in the absence of this molecule." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0664 ! interaction attribute name
<p><br>[Term]
<br>id: MI:0625
<br>name: agonist
<br>def: "Any chemical applied externally to cells or any type of environmental condition, such as hypoxia, that stimulates an interaction, potentially by causing modification of one or more of the interactors." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0666 ! participant attribute name
<p><br>[Term]
<br>id: MI:0626
<br>name: antagonist
<br>def: "Any chemical applied externally to cells or any type of environmental condition, such as hypoxia, that inhibits an interaction, potentially by alteration of amount or binding affinity of one or more of the interactors." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0666 ! participant attribute name
<p><br>[Term]
<br>id: MI:0627
<br>name: experiment modification
<br>def: "Modifications of the standard experimental method described in the CV." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "exp-modification" EXACT PSI-MI-short []
<br>is_a: MI:0665 ! experiment attibute name
<p><br>[Term]
<br>id: MI:0628
<br>name: validation regular expression
<br>def: "Regular Expression used to check the validity of cross references' identifier. Attribute generally associated to terms in Cv Database." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "id-validation-regexp" EXACT PSI-MI-short []
<br>is_a: MI:0667 ! controlled vocabulary attribute name
<p><br>[Term]
<br>id: MI:0629
<br>name: complex-properties
<br>def: "Information on the complex being annotated. Attribute generally associated to an interaction." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0664 ! interaction attribute name
<br>is_a: MI:0665 ! experiment attibute name
<p><br>[Term]
<br>id: MI:0630
<br>name: 3d-structure
<br>def: "Comments on the 3D structure. This attribute is generally associated to an interaction." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0664 ! interaction attribute name
<p><br>[Term]
<br>id: MI:0631
<br>name: 3d-r-factors
<br>def: "Free R-Factor and working R-Factor for the quality of the crystallographic model. This attribute is generally associated to an interaction." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0664 ! interaction attribute name
<p><br>[Term]
<br>id: MI:0632
<br>name: 3d-resolution
<br>def: "Resolution of the 3D structure. This attribute is generally associated to an interaction." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0664 ! interaction attribute name
<p><br>[Term]
<br>id: MI:0633
<br>name: data-processing
<br>def: "Information about how the data was processed. This attribute is used mainly for large scale experiment." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0665 ! experiment attibute name
<p><br>[Term]
<br>id: MI:0634
<br>name: contact-email
<br>def: "E-mail address to contact the author or organisation which has produced the data. This attribute is generally associated to an experiment." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0665 ! experiment attibute name
<p><br>[Term]
<br>id: MI:0635
<br>name: contact-comment
<br>def: "Free text notes on how to contact the author or organisation which has produced the data This attribute is generally associated to an experiment." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0665 ! experiment attibute name
<p><br>[Term]
<br>id: MI:0636
<br>name: author-list
<br>def: "List of authors associated to a publication. This attribute is generally associated to an experiment." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0665 ! experiment attibute name
<p><br>[Term]
<br>id: MI:0637
<br>name: isoform-comment
<br>def: "Participant isoform's comment. This attribute can be attached to interactions or participants." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0664 ! interaction attribute name
<br>is_a: MI:0666 ! participant attribute name
<p><br>[Term]
<br>id: MI:0638
<br>name: prerequisite-ptm
<br>def: "Post translational modification required for an interaction to occur." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0925 ! observed ptm
<p><br>[Term]
<br>id: MI:0639
<br>name: resulting-ptm
<br>def: "Post translational modification occurs subsequently to an interaction." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0925 ! observed ptm
<p><br>[Term]
<br>id: MI:0640
<br>name: parameter type
<br>def: "Parameter for enzymatic or binding kinetic studies." [PMID:14755292]
<br>subset: Drugable
<br>subset: PSI-MI slim
<br>relationship: part_of MI:0000 ! molecular interaction
<p><br>[Term]
<br>id: MI:0641
<br>name: ic50
<br>def: "Molar concentration of an antagonist which produces 50% of the maximum possible inhibitory response for that antagonist. Note this measure depends on the specific antagonist used and upon experimental conditions, notably temperature, pH and solution composition (e.g., salts, chelating agents and others). Thus the ic50 is a relative measure and its values can be compared only when sharing the same experimental setting. Unit Molar." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "IC50" EXACT PSI-MI-alternate []
<br>is_a: MI:0640 ! parameter type
<p><br>[Term]
<br>id: MI:0642
<br>name: ec50
<br>def: "Molar concentration of an agonist which produces 50% of the maximum possible response for that agonist. Note this measure depends on the specific agonist used and upon experimental conditions, notably temperature, pH and solution composition (e.g., salts, chelating agents and others). Thus the ec50 is a relative measure and its values can be compared only when sharing the same experimental setting. Unit Molar." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "EC50" EXACT PSI-MI-alternate []
<br>is_a: MI:0640 ! parameter type
<p><br>[Term]
<br>id: MI:0643
<br>name: ki
<br>def: "Equilibrium constant for dissociation of an inhibitor. Unit Molar." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "Ki" EXACT PSI-MI-alternate []
<br>is_a: MI:0640 ! parameter type
<p><br>[Term]
<br>id: MI:0644
<br>name: km
<br>def: "Michaelis-Menten constant: concentration of substrate at which the reaction rate is equal to half the maximal rate (i.e. Km={s} when Vo=1/2Vmax). Unit Molar." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "Km" EXACT PSI-MI-alternate []
<br>is_a: MI:0640 ! parameter type
<p><br>[Term]
<br>id: MI:0645
<br>name: kcat
<br>def: "The number of substrate molecules converted to product in a given unit of time, on a single enzyme molecule when the enzyme is saturated with substrate. Unit per second, or s-1." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "turnover number" EXACT PSI-MI-alternate []
<br>is_a: MI:0640 ! parameter type
<p><br>[Term]
<br>id: MI:0646
<br>name: Kd
<br>def: "The equilibrium dissociation constant of a receptor/ligand or proteinA/proteinB complex. Unit Molar (generally M-1)." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "Kd" EXACT PSI-MI-alternate []
<br>is_a: MI:0640 ! parameter type
<p><br>[Term]
<br>id: MI:0647
<br>name: parameter unit
<br>def: "Controlled vocabulary for kinetic constant units." [PMID:14755292]
<br>subset: PSI-MI slim
<br>relationship: part_of MI:0000 ! molecular interaction
<p><br>[Term]
<br>id: MI:0648
<br>name: molar
<br>def: "Molarity is the number of moles of solute dissolved in one liter of solution. The units, therefore are moles per liter, specifically it's moles of solute per liter of solution. These units are abbreviated as M and it means moles per liter (not just moles)." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "M" EXACT PSI-MI-alternate []
<br>is_a: MI:0647 ! parameter unit
<p><br>[Term]
<br>id: MI:0649
<br>name: second
<br>def: "The second is the duration of 9 192 631 770 periods of the radiation corresponding to the transition between the two hyperfine levels of the ground state of the cesium-133 atom. The second was originally defined as 1/86 400 mean solar day until astronomers discovered that the mean solar day is actually not constant." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "s" EXACT PSI-MI-alternate []
<br>synonym: "sec" EXACT PSI-MI-alternate []
<br>is_a: MI:0647 ! parameter unit
<p><br>[Term]
<br>id: MI:0650
<br>name: millimolar
<br>def: "10E-3 moles per liter of solution.\nOBSOLETE: term redundant with the schema exponent attribute of the parameter." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "mM" EXACT PSI-MI-alternate []
<br>is_obsolete: true
<p><br>[Term]
<br>id: MI:0651
<br>name: micromolar
<br>def: "10E-6 moles per liter of solution.\nOBSOLETE: term redundant with the schema exponent attribute of the parameter." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "uM" EXACT PSI-MI-alternate []
<br>is_obsolete: true
<p><br>[Term]
<br>id: MI:0652
<br>name: nanomolar
<br>def: "10E-9 moles per liter of solution.\nOBSOLETE: term redundant with the schema exponent attribute of the parameter." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "nM" EXACT PSI-MI-alternate []
<br>is_obsolete: true
<p><br>[Term]
<br>id: MI:0653
<br>name: picomolar
<br>def: "10E-12 moles per liter of solution.\nOBSOLETE: term redundant with the schema exponent attribute of the parameter." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "pM" EXACT PSI-MI-alternate []
<br>is_obsolete: true
<p><br>[Term]
<br>id: MI:0654
<br>name: fentomolar
<br>def: "10E-15 moles per liter of solution.\nOBSOLETE: term redundant with the schema exponent attribute of the parameter." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "fM" EXACT PSI-MI-alternate []
<br>is_obsolete: true
<p><br>[Term]
<br>id: MI:0655
<br>name: lambda repressor two hybrid
<br>def: "A protein of interest (the bait) is fused to the full-length bacteriophage lambda repressor protein (lambdacI, 237 amino acids), containing the amino terminal DNA-binding domain and the carboxylterminal dimerization domain. The corresponding target (prey) protein is fused to the N-terminal domain of the alfa-subunit of RNA polymerase (248 amino acids). The bait is tethered to the lambda operator sequence upstream of the reporter promoter through the DNA-binding domain of lambdacI. When the bait and prey interact, they recruit and stabilize the binding of RNA polymerase at the promoter and activate the transcription of the HIS3 reporter gene. Due to the tendency of both the lambda repressor protein and the N-terminal domain of the alfa-subunit of RNA polymerase to dimerize, this system might not be optimal for the analysis of proteins that self-associate unless their interaction with other protein partners depends on the oligomerization." [PMID:15792953]
<br>subset: PSI-MI slim
<br>synonym: "BacterioMatch" EXACT PSI-MI-alternate []
<br>synonym: "lambda two hybrid" EXACT PSI-MI-short []
<br>is_a: MI:0232 ! transcriptional complementation assay
<p><br>[Term]
<br>id: MI:0656
<br>name: identified peptide
<br>def: "Peptide whose sequence is experimentally identified and can lead to a full protein identification." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0505 ! experimental feature
<p><br>[Term]
<br>id: MI:0657
<br>name: systematic evolution of ligands by exponential enrichment
<br>def: "RNA and cDNA constructs with variable central sequences and a constant flanking region are collected in a complex library. The library is then screened to select either specific binding partners of a bait molecule (generally a protein) or particular enzymatic activities of the nucleic acid molecules themselves. The selected nucleic acids are amplified using the constant flanking regions to increase their abundance. Cycles of selection-amplification can be repeated to increase the specificity of the targets that, at the end, are individually identified by sequencing." [PMID:11539574]
<br>subset: PSI-MI slim
<br>synonym: "in vitro evolution of nucleic acids" EXACT PSI-MI-alternate []
<br>synonym: "selex" EXACT PSI-MI-short []
<br>is_a: MI:0400 ! affinity technology
<p><br>[Term]
<br>id: MI:0658
<br>name: multidimensional protein identification technology
<br>def: "MudPIT is a method for rapid and large-scale protein identification by multidimensional liquid chromatography associated with tandem mass spectrometry. The chromatography step consists of strong cation exchange material back-to-back with reversed phase material inside fused silica capillaries. The peptides bound to the cation-exchange resin are freed by the gradually increasing salt concentration of the buffer and are subsequently retained by the reversed phase resin. Increasing buffers hydrophobicity progressively elute peptides from the reversed phase packing directly into the mass spectrometer. Typically this mass spectrometer will be a tandem electrospray, so peptides undergo ionization in the liquid phase, are separated in a primary mass spectrometer, analysed in the second mass spectrometer and identified." [PMID:11231557]
<br>subset: PSI-MI slim
<br>synonym: "mudpit" EXACT PSI-MI-short []
<br>is_a: MI:0093 ! protein sequence identification
<br>is_a: MI:0427 ! Identification by mass spectrometry
<br>is_a: MI:0815 ! confirmation by molecular weight
<p><br>[Term]
<br>id: MI:0659
<br>name: experimental feature detection
<br>def: "Experimental method by which a feature is detected or identified." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "exp feature detect" EXACT PSI-MI-short []
<br>is_a: MI:0003 ! feature detection method
<p><br>[Term]
<br>id: MI:0660
<br>name: feature prediction
<br>def: "Feature detection based on computational analysis." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0003 ! feature detection method
<p><br>[Term]
<br>id: MI:0661
<br>name: experimental participant identification
<br>def: "experimental participant identification." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "experimental particp" EXACT PSI-MI-short []
<br>is_a: MI:0002 ! participant identification method
<p><br>[Term]
<br>id: MI:0662
<br>name: imex-primary
<br>def: "IMEx primary identifier that is assigned to an experiment record by the database that created the record in the context of IMEx consortium. The identifiers are unique across all member database as they are all generated by a centralized key-assigner.\nhttp://imex.sourceforge.net/" [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0353 ! cross-reference type
<p><br>[Term]
<br>id: MI:0663
<br>name: confocal microscopy
<br>def: "A confocal is a standard epifluorescence microscope with improvement essentially coming from the rejection of out-of-focus light interference. Confocal imaging system achieves this by two strategies: a) by illuminating a single point of the specimen at any one time with a focused beam, so that illumination intensity drops off rapidly and b) by the use of blocking a pinhole aperture in a conjugate focal plane to the specimen so that light emitted away from the point in the specimen being illuminated is blocked from reaching the detector. Only the light from the single point illuminated of the specimen passing through the image pinhole is detected by a photodetector. Usually a computer is used to control the sequential scanning of the sample and to assemble the image for display onto a video screen." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0428 ! imaging technique
<p><br>[Term]
<br>id: MI:0664
<br>name: interaction attribute name
<br>def: "Attribute name of annotation associated to an interaction element." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "interaction att name" EXACT PSI-MI-short []
<br>is_a: MI:0590 ! attribute name
<p><br>[Term]
<br>id: MI:0665
<br>name: experiment attibute name
<br>def: "Attribute name of annotation associated to an experiment element." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "experiment att name" EXACT PSI-MI-short []
<br>is_a: MI:0590 ! attribute name
<p><br>[Term]
<br>id: MI:0666
<br>name: participant attribute name
<br>def: "Attribute name of annotation associated to a participant element." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "participant att name" EXACT PSI-MI-short []
<br>is_a: MI:0590 ! attribute name
<p><br>[Term]
<br>id: MI:0667
<br>name: controlled vocabulary attribute name
<br>def: "Attribute name of annotation associated to a CV term." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "cv att name" EXACT PSI-MI-short []
<br>is_a: MI:0590 ! attribute name
<p><br>[Term]
<br>id: MI:0668
<br>name: feature attribute name
<br>def: "Attribute name of annotation associated to a feature element." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "feature att name" EXACT PSI-MI-short []
<br>is_a: MI:0590 ! attribute name
<p><br>[Term]
<br>id: MI:0669
<br>name: organism attribute name
<br>def: "Attribute name of annotation associated to an organism element." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "organism att name" EXACT PSI-MI-short []
<br>is_a: MI:0590 ! attribute name
<p><br>[Term]
<br>id: MI:0670
<br>name: imex
<br>def: "International Molecular Interaction Exchange." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "IMEx" EXACT PSI-MI-alternate []
<br>is_a: MI:0461 ! interaction database
<p><br>[Term]
<br>id: MI:0671
<br>name: antibodies
<br>def: "This annotation topic should contain information about antibodies when specific antibodies (monoclonal or polyclonal raised against specific regions of the proteins or purified in a specific manner) have been used." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0665 ! experiment attibute name
<p><br>[Term]
<br>id: MI:0672
<br>name: library-used
<br>def: "This annotation topic will be used to store information about the cDNA library. If a name is available this should be reported along with a short description of the library." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0665 ! experiment attibute name
<p><br>[Term]
<br>id: MI:0673
<br>name: complex-synonym
<br>def: "Alternative names to describe a complex." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0300 ! alias type
<p><br>[Term]
<br>id: MI:0674
<br>name: peptide parent sequence reference
<br>def: "Describe a cross reference pointing to a peptide parent sequence." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "peptide-parent" EXACT PSI-MI-short []
<br>is_a: MI:0353 ! cross-reference type
<p><br>[Term]
<br>id: MI:0675
<br>name: international protein index
<br>def: "IPI provides a top level guide to the main databases that describe the proteomes of higher eukaryotic organisms. IPI effectively maintains a database of cross references between the primary data sources, provides minimally redundant yet maximally complete sets of proteins for featured species (one sequence per transcript) and maintains stable identifiers (with incremental versioning) to allow the tracking of sequences in IPI between IPI releases. IPI is updated monthly in accordance with the latest data released by the primary data sources." [PMID:15221759]
<br>subset: PSI-MI slim
<br>synonym: "ipi" EXACT PSI-MI-short []
<br>xref: id-validation-regexp:\"IPI[0-9\]+.[0-9\]+|IPI[0-9\]+\"
<br>is_a: MI:0683 ! sequence database
<p><br>[Term]
<br>id: MI:0676
<br>name: tandem affinity purification
<br>def: "Tandem affinity purification allows rapid purification under native conditions of complexes, even when expressed at their natural level. Prior knowledge of complex composition or function is not required. The TAP method requires fusion of the a multiple tag, either N- or C-terminally, to the target (or bait) protein of interest. The multiple tag allows two steps purification steps ensuring a highly selective complex purification." [PMID:11403571]
<br>subset: PSI-MI slim
<br>synonym: "TAP" EXACT PSI-MI-alternate []
<br>synonym: "tap" EXACT PSI-MI-short []
<br>is_a: MI:0004 ! affinity chromatography technology
<p><br>[Term]
<br>id: MI:0677
<br>name: tandem tag
<br>def: "A tandem tag consists of the concatenation of simple distinct tags that is engineered to be cloned as a unique element onto a sequence of interest. Note that when a protein is fused to many simple tags that are inserted individually and possibly in different sequence positions these should be reported as independent features." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0507 ! tag
<p><br>[Term]
<br>id: MI:0678
<br>name: antibody array
<br>def: "A microarray consisting of antibodies spotted on a solid support in appropriate orientation is incubated with a biological sample (or antigen). Some proteins are captured by the antibodies in the array. Protein of  forming complexes on the array are identified according to their prior labelling (tag, ELISA, biotin and others)." [PMID:12454649]
<br>subset: PSI-MI slim
<br>synonym: "antigen capture assay" EXACT PSI-MI-alternate []
<br>synonym: "sandwich immunoassay" EXACT PSI-MI-alternate []
<br>is_a: MI:0008 ! array technology
<p><br>[Term]
<br>id: MI:0679
<br>name: poly adenine
<br>def: "A sequence of adenine nucleotides that is added to the 3' end of some primary transcript messenger RNA molecules in eukaryotes during post-transcriptional processing. The added tail is believed to confer stability to the molecule." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "poly A" EXACT PSI-MI-alternate []
<br>synonym: "poly a" EXACT PSI-MI-short []
<br>is_a: MI:0320 ! ribonucleic acid
<p><br>[Term]
<br>id: MI:0680
<br>name: single stranded deoxyribonucleic acid
<br>def: "DNA that consists of only one chain of nucleotides rather than the two base pairing strands found in DNA in the double helix form." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "ss DNA" EXACT PSI-MI-alternate []
<br>synonym: "ss dna" EXACT PSI-MI-short []
<br>is_a: MI:0319 ! deoxyribonucleic acid
<p><br>[Term]
<br>id: MI:0681
<br>name: double stranded deoxyribonucleic acid
<br>def: "DNA that consists of two base pairing strands. The 2 nucleotide chains are held together by hydrogen bonds between base pairs of nucleotides.\n" [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "ds DNA" EXACT PSI-MI-alternate []
<br>synonym: "ds dna" EXACT PSI-MI-short []
<br>is_a: MI:0319 ! deoxyribonucleic acid
<p><br>[Term]
<br>id: MI:0682
<br>name: cofactor
<br>def: "A cofactor is a small molecule required for the catalysis of an enzyme." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "coenzyme" EXACT PSI-MI-alternate []
<br>is_a: MI:0500 ! biological role
<p><br>[Term]
<br>id: MI:0683
<br>name: sequence database
<br>def: "Database collecting nucleic or amino acid sequences mainly derived from genomic or mRNA sequencing." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0473 ! participant database
<p><br>[Term]
<br>id: MI:0684
<br>name: ancillary
<br>def: "Molecule required for an observed binary interaction to occur. This molecule may act as stabilizer of any of the interaction partners or may act as a bridge molecule between them but the method does not provide resolution or evidence to demonstrate its actual molecular function (i.e.Mudpit, tri hybrid etc)." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0495 ! experimental role
<p><br>[Term]
<br>id: MI:0685
<br>name: source reference
<br>def: "Publication or document describing the originating resource where an interaction, or other curated information, was first described." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0353 ! cross-reference type
<p><br>[Term]
<br>id: MI:0686
<br>name: unspecified method
<br>def: "Yet to be identified interaction detection method associated with interaction data imported from a third party database. This database may have potentially different standards of curation." [PMID:14755292]
<br>subset: Drugable
<br>subset: PSI-MI slim
<br>is_a: MI:0001 ! interaction detection method
<p><br>[Term]
<br>id: MI:0687
<br>name: fluorescent protein tag
<br>def: "Protein having well characterized fluorescence excitation and emission spectra used as fusion with a protein under study to facilitate its  localisation or identification." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "fluorescent prot tag" EXACT PSI-MI-short []
<br>is_a: MI:0240 ! fusion protein
<p><br>[Term]
<br>id: MI:0688
<br>name: dna binding domain tag
<br>def: "Part of a transcription factor responsible for the binding of gene regulatory region prior to their transcription. Such tags are generally used in two hybrid experiments where they are fused to a bait polypeptide tested for its ability to interact with a prey fused to an activation domain tag." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "dna binding domain" EXACT PSI-MI-short []
<br>is_a: MI:0240 ! fusion protein
<p><br>[Term]
<br>id: MI:0689
<br>name: activation domain tag
<br>def: "Part of a transcription factor responsible for the activation of DNA transcription. Such tags are generally used in two hybrid experiments where they are fused to a prey polypeptide tested for its ability to interact with a bait fused to a DNA binding domain tag." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "activation domain" EXACT PSI-MI-short []
<br>is_a: MI:0240 ! fusion protein
<p><br>[Term]
<br>id: MI:0690
<br>name: gal4 activation domain
<br>def: "Part of the yeast transcription factor GAL4 (amino acids 766-881) specifically responsible for DNA transcription activation." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "gal4 ad" EXACT PSI-MI-short []
<br>is_a: MI:0689 ! activation domain tag
<p><br>[Term]
<br>id: MI:0691
<br>name: vp16 activation domain
<br>def: "Full viral protein vp16 exclusively responsible for preferential transcriptional activation of viral genes. Activation requires the formation of a complex with the host cell transcription factor." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "vp16 ad" EXACT PSI-MI-short []
<br>is_a: MI:0689 ! activation domain tag
<p><br>[Term]
<br>id: MI:0692
<br>name: b42 activation domain
<br>def: "B42 is an acidic sequence of 89 residues  derived from Escherichia coli acting as weak transcription activation domain. When use in two hybrid experiments, the weakness of B42 as activator increases the sensitivity of the interaction detection." [PMID:14613974]
<br>subset: PSI-MI slim
<br>synonym: "b42 ad" EXACT PSI-MI-short []
<br>is_a: MI:0689 ! activation domain tag
<p><br>[Term]
<br>id: MI:0693
<br>name: gal4 dna binding domain
<br>def: "Part of the yeast transcription factor GAL4 (amino acids 11-38) specifically responsible for DNA binding of a 17 base-pair long sequence in the upstream activating sequence of galactose-induced genes.\n" [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "gal4 dna bd" EXACT PSI-MI-short []
<br>is_a: MI:0688 ! dna binding domain tag
<p><br>[Term]
<br>id: MI:0694
<br>name: lexa dna binding domain
<br>def: "Amino terminal (1-220) part of the Escherichia coli lexA repressor, binding to 16 base pair palindromic DNA sequences." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "lexa dna bd" EXACT PSI-MI-short []
<br>is_a: MI:0688 ! dna binding domain tag
<p><br>[Term]
<br>id: MI:0695
<br>name: sandwich immunoassay
<br>def: "Antibody array where proteins retained by the arrayed antibodies are identified using a detector antibody. The detector antibody is either modified with a directly detectable label (enzyme, fluorescent molecule, isotope, etc.), or it is biotinylated for detection after subsequent probing with labeled streptavidin." [PMID:12454649]
<br>subset: PSI-MI slim
<br>is_a: MI:0678 ! antibody array
<p><br>[Term]
<br>id: MI:0696
<br>name: polymerase assay
<br>def: "Measures the catalysis of the transfer of a free nucleotidyl group to a nucleic acid chain." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "nucleotidyltransferase assay" EXACT PSI-MI-alternate []
<br>is_a: MI:0415 ! enzymatic study
<p><br>[Term]
<br>id: MI:0697
<br>name: dna directed dna polymerase assay
<br>def: "Measures the catalysis of the reaction: deoxynucleoside triphosphate + DNA(n) = diphosphate + DNA(n+1); the synthesis of DNA from deoxyribonucleotide triphosphates in the presence of a DNA template or primer." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "dna dna pol assay" EXACT PSI-MI-short []
<br>is_a: MI:0696 ! polymerase assay
<p><br>[Term]
<br>id: MI:0698
<br>name: dna directed rna polymerase assay
<br>def: "Measures the catalysis of the reaction: nucleoside triphosphate + RNA(n) = diphosphate + RNA(n+1). Utilizes a DNA template, i.e. the catalysis of DNA-template-directed extension of the 3'-end of an RNA strand by one nucleotide at a time. Can initiate a chain 'de novo'." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "dna rna pol assay" EXACT PSI-MI-short []
<br>is_a: MI:0696 ! polymerase assay
<p><br>[Term]
<br>id: MI:0699
<br>name: rna directed dna polymerase assay
<br>def: "Measures the catalysis of the reaction: deoxynucleoside triphosphate + DNA(n) = diphosphate + DNA(n+1). Catalyzes RNA-template-directed extension of the 3'- end of a DNA strand by one deoxynucleotide at a time." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "rna dna pol assay" EXACT PSI-MI-short []
<br>is_a: MI:0696 ! polymerase assay
<p><br>[Term]
<br>id: MI:0700
<br>name: rna directed rna polymerase assay
<br>def: "Measures the catalysis of the reaction: nucleoside triphosphate + RNA (n) = diphosphate + RNA (n+1); uses an RNA template." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "rna rna pol assay" EXACT PSI-MI-short []
<br>is_a: MI:0696 ! polymerase assay
<p><br>[Term]
<br>id: MI:0701
<br>name: dna strand elongation
<br>def: "The process by which a DNA strand is synthesized from template DNA by the action of polymerases, which add nucleotides to the 3' end of the nascent DNA strand." [GO:0006271, PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "DNA replication elongation " EXACT PSI-MI-alternate []
<br>synonym: "elongation" EXACT PSI-MI-short []
<br>is_a: MI:0414 ! enzymatic reaction
<p><br>[Term]
<br>id: MI:0702
<br>name: panther
<br>def: "The PANTHER (Protein ANalysis THrough Evolutionary Relationships) Classification System is a unique resource that classifies genes by their functions using published scientific experimental evidence and evolutionary relationships to predict function even in the absence of direct experimental evidence.\nwww.pantherdb.org/" [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "PANTHER" EXACT PSI-MI-alternate []
<br>is_a: MI:0449 ! interpro
<p><br>[Term]
<br>id: MI:0703
<br>name: gene3d
<br>def: "The Gene3D database provides a combined structural, functional and evolutionary view of the protein world. It is focused on providing structural annotation for protein sequences without structural representatives--including the complete proteome sets of over 240 different species.\nhttp://cathwww.biochem.ucl.ac.uk:8080/Gene3D/" [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "Gene3D" EXACT PSI-MI-alternate []
<br>is_a: MI:0449 ! interpro
<p><br>[Term]
<br>id: MI:0704
<br>name: nucleic acid delivery
<br>def: "Method by which nucleic acids are delivered or engineered into a cell." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "nucl delivery" EXACT PSI-MI-short []
<br>is_a: MI:0307 ! delivery method
<p><br>[Term]
<br>id: MI:0705
<br>name: anti tag western blot
<br>def: "Western blot assay performed when specific antibodies for the protein of interest are not available. Therefore the protein is expressed as a hybrid protein fused to a tag for which efficient antibodies are available. The antibody may be either monoclonal or polyclonal." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "anti tag western" EXACT PSI-MI-short []
<br>is_a: MI:0113 ! western blot
<br>is_a: MI:0866 ! tag visualisation
<p><br>[Term]
<br>id: MI:0706
<br>name: nucleic acid transformation
<br>def: "Transformation is the genetic alteration of a cell resulting from the introduction, uptake and expression of foreign genetic material incorporated into the cell's genome (DNA or RNA)." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "nucl transformation" EXACT PSI-MI-short []
<br>is_a: MI:0704 ! nucleic acid delivery
<p><br>[Term]
<br>id: MI:0707
<br>name: anti tag immunostaining
<br>def: "Immunostaining assay performed when specific antibodies for the protein of interest are not available. Therefore the  protein is expressed as a hybrid protein fused to a tag peptide/protein for which efficient antibodies are available. The anti tag antibody may be either monoclonal or polyclonal." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "anti tag immunost" EXACT PSI-MI-short []
<br>is_a: MI:0422 ! immunostaining
<p><br>[Term]
<br>id: MI:0708
<br>name: monoclonal antibody immunostaining
<br>def: "A monospecific antibody for the protein of interest is available, this is used to detect a specific protein within a cell or tissue sample." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "monoclonal immunost" EXACT PSI-MI-short []
<br>is_a: MI:0422 ! immunostaining
<p><br>[Term]
<br>id: MI:0709
<br>name: polyclonal antibody immunostaining
<br>def: "Immunostaining assay carried out using a mixture of different antibodies that represent the immune response, normally in an experimental animal, to the protein of interest. These antibodies are used to detect the protein within a cell or tissue sample." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "polyclonal immunost" EXACT PSI-MI-short []
<br>is_a: MI:0422 ! immunostaining
<p><br>[Term]
<br>id: MI:0710
<br>name: nucleic acid transformation by treatment with divalent cation
<br>def: "Stable introduction and expression of nucleic acid carried out by treatment with divalent cation." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "n transformat cation" EXACT PSI-MI-short []
<br>is_a: MI:0706 ! nucleic acid transformation
<p><br>[Term]
<br>id: MI:0711
<br>name: nucleic acid electroporation
<br>def: "Electroporation, or electropermeabilization, is a significant increase in the electrical conductivity and permeability of the cell plasma membrane caused by externally applied electrical field. It is usually used in molecular biology as a way of introducing some substance inside the cell, such as loading it with a molecular probe, a drug that can change cell's function, or a piece of coding DNA." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "nucl electroporation" EXACT PSI-MI-short []
<br>is_a: MI:0308 ! electroporation
<br>is_a: MI:0704 ! nucleic acid delivery
<p><br>[Term]
<br>id: MI:0712
<br>name: nucleic acid microinjection
<br>def: "Microinjection refers to the process of using a micro needle to insert substances at a microscopic level into a single living cell. It is a simple mechanical process in which an extremely fine micro needle penetrates the cell membrane and sometimes the nuclear envelope and releases nucleic acids." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "nucl microinjection" EXACT PSI-MI-short []
<br>is_a: MI:0311 ! microinjection
<br>is_a: MI:0704 ! nucleic acid delivery
<p><br>[Term]
<br>id: MI:0713
<br>name: nucleic acid passive uptake
<br>def: "Nucleic acid entrance into cells that does not involved specific treatments but rely on natural cellular processes." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "nucl passive uptake" EXACT PSI-MI-short []
<br>is_a: MI:0704 ! nucleic acid delivery
<br>is_a: MI:0716 ! passive uptake
<p><br>[Term]
<br>id: MI:0714
<br>name: nucleic acid transduction
<br>def: "Transduction is the process by which bacterial DNA is moved from one bacterium to another by a virus." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "nucl transduction" EXACT PSI-MI-short []
<br>is_a: MI:0704 ! nucleic acid delivery
<p><br>[Term]
<br>id: MI:0715
<br>name: nucleic acid conjugation
<br>def: "Bacterial conjugation is the transfer of genetic material between bacteria through cell-to-cell contact. Bacterial conjugation is often incorrectly regarded as the bacterial equivalent of sexual reproduction or mating. It is not actually sexual, as it does not involve the fusing of gametes and the creation of a zygote. It is merely the transfer of  a conjugative plasmid from a donor cell to a recipient" [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "nucl conjugation" EXACT PSI-MI-short []
<br>is_a: MI:0704 ! nucleic acid delivery
<p><br>[Term]
<br>id: MI:0716
<br>name: passive uptake
<br>def: "Entrance of molecules into cells that does not involved specific treatments but relies on natural cellular processes." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0307 ! delivery method
<p><br>[Term]
<br>id: MI:0717
<br>name: nucleic acid transfection with liposome
<br>def: "Lipofection (or liposome transfection) is a technique used to inject genetic material into a cell by means of liposomes which are vesicles that can easily merge with the cell membrane since they are both made of a phospholipid bilayer." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "nucl lipotransfect" EXACT PSI-MI-short []
<br>is_a: MI:0312 ! nucleic acid transfection
<p><br>[Term]
<br>id: MI:0718
<br>name: nucleic acid transfection by treatment
<br>def: "Transient introduction and expression of nucleic acid carried out by treatment with chemicals." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "n transfection treat" EXACT PSI-MI-short []
<br>is_a: MI:0312 ! nucleic acid transfection
<p><br>[Term]
<br>id: MI:0719
<br>name: calcium phosphate nucleic acid transfection
<br>def: "Transient introduction and expression of nucleic acid carried out by treatment with calcium phosphate." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "ca po nuc transfect" EXACT PSI-MI-short []
<br>is_a: MI:0718 ! nucleic acid transfection by treatment
<p><br>[Term]
<br>id: MI:0720
<br>name: nucleic acid delivery by infection
<br>def: "Nucleic acid introduced into a cell via an external organism, usually a virus or bacteria.." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "nucl infection" EXACT PSI-MI-short []
<br>is_a: MI:0310 ! infection
<br>is_a: MI:0704 ! nucleic acid delivery
<p><br>[Term]
<br>id: MI:0721
<br>name: protein delivery
<br>def: "Method by which proteins are delivered into a cell." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0307 ! delivery method
<p><br>[Term]
<br>id: MI:0722
<br>name: protein electroporation
<br>def: "Method for temporarily permeabilising cell membranes in order to facilitate the entry of a protein." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "prot electroporation" EXACT PSI-MI-short []
<br>is_a: MI:0308 ! electroporation
<br>is_a: MI:0721 ! protein delivery
<p><br>[Term]
<br>id: MI:0723
<br>name: protein microinjection
<br>def: "Microinjection refers to the process of using a micro needle to insert substances at a microscopic level into a single living cell. It is a simple mechanical process in which an extremely fine micro needle penetrates the cell membrane and sometimes the nuclear envelope and releases proteins." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "prot microinjection" EXACT PSI-MI-short []
<br>is_a: MI:0311 ! microinjection
<br>is_a: MI:0721 ! protein delivery
<p><br>[Term]
<br>id: MI:0724
<br>name: protein delivery by cationic lipid treatment
<br>def: "Proteins are delivered into cells by treatment with cationic lipids." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "prot cationic lipid" EXACT PSI-MI-short []
<br>is_a: MI:0721 ! protein delivery
<p><br>[Term]
<br>id: MI:0725
<br>name: protein delivery by infection
<br>def: "Protein introduced into a cell via an external organism, usually a virus or bacteria." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "prot infection" EXACT PSI-MI-short []
<br>is_a: MI:0310 ! infection
<br>is_a: MI:0721 ! protein delivery
<p><br>[Term]
<br>id: MI:0726
<br>name: reverse two hybrid
<br>def: "Yeast strains are generated in which expression of DB-X/AD-Y or DBPX hybrid proteins is toxic under particular conditions (negative selection). Under these conditions, dissociation of an interaction should provide a selective advantage thereby facilitating detection: a few growing yeast colonies in which DB-X/AD-Y (or DBPX/binding site) fail to interact should be identified among many nongrowing colonies containing interacting DB-X/AD-Y or DBPX/binding site." [PMID:8816797]
<br>subset: PSI-MI slim
<br>is_a: MI:0232 ! transcriptional complementation assay
<p><br>[Term]
<br>id: MI:0727
<br>name: lexa b52 complementation
<br>def: "Yeast two-hybrid system using Escherichia coli LexA amino acids 1-202 as the DNA-binding domain (BD), E. coli B42 acidic sequence as the activation domain (AD), and two reporters, lacZ and LEU2, each containing upstream LexA binding elements." [PMID:14613974]
<br>subset: PSI-MI slim
<br>synonym: "lexa b52 complement" EXACT PSI-MI-short []
<br>synonym: "LexA B52 transcription complementation" EXACT PSI-MI-alternate []
<br>is_a: MI:0018 ! two hybrid
<p><br>[Term]
<br>id: MI:0728
<br>name: gal4 vp16 complementation
<br>def: "A chimeric protein consisting of the GAL4 DNA-binding domain (aa 1-147 of GAL4) and a transcriptional activation domain from the herpes simplex virus protein VP16 (either aa 411-490 or aa 411-455) can specifically activate transcription of a reporter gene located downstream ofGAL4 DNA binding sites and the E1B minimal promoter. Similarly, two chimeric proteins, one encoding a chimeric GAL4 protein and the other encoding a chimeric VP16 protein, can activate the reporter gene, if the domains fused to the GAL4 and VP16 sequences can complex with appropriate conformation. However, if the domains fused to the GAL4 and VP16 sequences do not interact specifically to form a + complex that reconstitutes GAL4 function, the reporter gene cannot be activated." [PMID:1387709]
<br>subset: PSI-MI slim
<br>synonym: "gal4 vp16 complement" EXACT PSI-MI-short []
<br>synonym: "karyoplasmic interaction ion strategy" EXACT PSI-MI-alternate []
<br>synonym: "KISS" EXACT PSI-MI-alternate []
<br>synonym: "mammalian two hybrid" EXACT PSI-MI-alternate []
<br>is_a: MI:0018 ! two hybrid
<p><br>[Term]
<br>id: MI:0729
<br>name: luminescence based mammalian interactome mapping
<br>def: "This strategy uses Renilla luciferase enzyme (RL) fused to proteins of interest, which are then coexpressed with individual Flag-tagged partners in mammalian cells. Their interactions are determined by performing an RL enzymatic assay on anti-Flag immunoprecipitates." [PMID:15761153]
<br>subset: PSI-MI slim
<br>synonym: "lumier" EXACT PSI-MI-short []
<br>is_a: MI:0004 ! affinity chromatography technology
<p><br>[Term]
<br>id: MI:0730
<br>name: pubchem
<br>def: "PubChem provides information on the biological activities of small molecules.\nhttp://pubchem.ncbi.nlm.nih.gov/" [PMID:16381840]
<br>subset: PSI-MI slim
<br>synonym: "PubChem" EXACT PSI-MI-alternate []
<br>is_a: MI:0473 ! participant database
<p><br>[Term]
<br>id: MI:0731
<br>name: 3d repertoire
<br>def: "The aim of 3D Repertoire is to determine the structures of all amenable complexes in a cell at medium or high resolution, which will later serve to integrate in toponomic and dynamic analyses of protein complexes in a cell. Complex models, EM pictures, expression and purification protocols obtained in the project will be collected in a database connected to the PDB repository." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "3D Repertoire" EXACT PSI-MI-alternate []
<br>is_a: MI:0489 ! source database
<p><br>[Term]
<br>id: MI:0732
<br>name: red fluorescent protein tag
<br>def: "The red fluorescent protein derived from Discosoma reef coral can be fused to individual proteins which then acquire fluorescence excitation and emission spectra virtually identical to those of the native." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "red fluorescent protein" EXACT PSI-MI-alternate []
<br>synonym: "RFP" EXACT PSI-MI-alternate []
<br>synonym: "rfp tag" EXACT PSI-MI-short []
<br>is_a: MI:0687 ! fluorescent protein tag
<p><br>[Term]
<br>id: MI:0733
<br>name: cyan fluorescent protein tag
<br>def: "The cyan fluorescent protein derived from Anthozoa reef coral can be fused to individual proteins which then acquire fluorescence excitation and emission spectra virtually identical to those of the native." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "CFP" EXACT PSI-MI-alternate []
<br>synonym: "cfp tag" EXACT PSI-MI-short []
<br>synonym: "cyan fluorescent protein" EXACT PSI-MI-alternate []
<br>is_a: MI:0687 ! fluorescent protein tag
<p><br>[Term]
<br>id: MI:0734
<br>name: enhanced green fluorescent protein tag
<br>def: "Variation of the green fluorescent protein derived from the bioluminescent jellyfish Aequorea victoria, can be fused to individual proteins which then acquire fluorescence excitation and emission spectra virtually identical to those of the native." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "EGFP" EXACT PSI-MI-alternate []
<br>synonym: "egfp tag" EXACT PSI-MI-short []
<br>synonym: "enhanced green fluorescent protein" EXACT PSI-MI-alternate []
<br>is_a: MI:0367 ! green fluorescent protein tag
<p><br>[Term]
<br>id: MI:0735
<br>name: transactivating tag
<br>def: "Tag derived from the transactivating (Tat) protein of human immunodeficiency virus 1 (HIV-1) that can enter cells efficiently when added exogenously in tissue culture. The Tat tag can carry other molecules into cells, by fusion of Tat peptides (residues 1-72 or 37-72,) to any molecule under study. Tat-mediated uptake may allow the delivery of macromolecules previously thought to be impermeable to living cells." [PMID:8290579]
<br>subset: PSI-MI slim
<br>synonym: "Tat tag" EXACT PSI-MI-alternate []
<br>synonym: "tat tag" EXACT PSI-MI-short []
<br>is_a: MI:0740 ! cell penetrating peptide tag
<p><br>[Term]
<br>id: MI:0736
<br>name: protein passive uptake
<br>def: "Proteins entrance into cells that does not involved specific treatments but rely on natural cellular processes." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "prot passive uptake" EXACT PSI-MI-short []
<br>is_a: MI:0716 ! passive uptake
<br>is_a: MI:0721 ! protein delivery
<p><br>[Term]
<br>id: MI:0737
<br>name: peptide sequence database
<br>def: "database storing sequences detected by peptide identification methods." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "pep seq db" EXACT PSI-MI-short []
<br>is_a: MI:0683 ! sequence database
<p><br>[Term]
<br>id: MI:0738
<br>name: pride
<br>def: "PRIDE is a public repository of protein and peptide identifications for the proteomics community.\nhttp://www.ebi.ac.uk/pride/" [PMID:16381953]
<br>subset: PSI-MI slim
<br>xref: search-url: "http://www.ebi.ac.uk/pride/searchSummary.do?experimentCvAccession=PRIDE:0000156&queryTypeSelected=ExperimentCV&experimentCvValue=${ac}"
<br>is_a: MI:0737 ! peptide sequence database
<p><br>[Term]
<br>id: MI:0739
<br>name: penetrating tag
<br>def: "Membrane shuttling peptide derived from the Drosophila homeobox protein Antennapedia: RQIKIWFQNRRMKWKK" [PMID:16574060]
<br>subset: PSI-MI slim
<br>is_a: MI:0740 ! cell penetrating peptide tag
<p><br>[Term]
<br>id: MI:0740
<br>name: cell penetrating peptide tag
<br>def: "The peptides named CPPs vary greatly in size, amino acid sequence, and charge, but share the common feature that they have the ability to rapidly translocate the plasma membrane and enable delivery to the cytoplasm or nucleus." [PMID:16574060]
<br>subset: PSI-MI slim
<br>synonym: "cell penetrating pep" EXACT PSI-MI-short []
<br>synonym: "cell-penetrating peptides" EXACT PSI-MI-alternate []
<br>synonym: "CPPs" EXACT PSI-MI-alternate []
<br>synonym: "membrane translocating sequences" EXACT PSI-MI-alternate []
<br>synonym: "MTS" EXACT PSI-MI-alternate []
<br>synonym: "protein transduction domains" EXACT PSI-MI-alternate []
<br>synonym: "PTDs" EXACT PSI-MI-alternate []
<br>synonym: "Trojan peptides" EXACT PSI-MI-alternate []
<br>is_a: MI:0507 ! tag
<p><br>[Term]
<br>id: MI:0741
<br>name: peptide atlas
<br>def: "PeptideAtlas addresses these needs by identifying peptides by tandem mass spectrometry (MS/MS), statistically validating those identifications and then mapping identified sequences to the genomes of eukaryotic organisms.\nhttp://www.peptideatlas.org/" [PMID:15642101, PMID:16381952]
<br>subset: PSI-MI slim
<br>synonym: "PeptideAtlas" EXACT PSI-MI-alternate []
<br>is_a: MI:0737 ! peptide sequence database
<p><br>[Term]
<br>id: MI:0742
<br>name: gpm
<br>def: "Global Proteome Machine aim to improve the quality of analysis, make the results portable and to provide a common platform for testing and validating proteomics results.\nhttp://www.thegpm.org" [PMID:15595733]
<br>subset: PSI-MI slim
<br>synonym: "Global Proteome Machine" EXACT PSI-MI-alternate []
<br>is_a: MI:0737 ! peptide sequence database
<p><br>[Term]
<br>id: MI:0787
<br>name: genetic experimental form
<br>def: "Descriptor of an experimental form involved in a genetic interaction" [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "genetic exp form" EXACT PSI-MI-short []
<br>is_a: MI:0346 ! experimental preparation
<p><br>[Term]
<br>id: MI:0788
<br>name: knock out
<br>def: "The gene has been completely removed e.g. by genetic engineering " [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "knock-out" EXACT PSI-MI-short []
<br>is_a: MI:0804 ! mutated gene
<p><br>[Term]
<br>id: MI:0789
<br>name: knock down
<br>def: "The gene expression has been significantly reduced compared to wild-type by introduction of an external substance, e.g. by RNA interference." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "knock-down" EXACT PSI-MI-short []
<br>is_a: MI:0804 ! mutated gene
<p><br>[Term]
<br>id: MI:0790
<br>name: hypermorph
<br>def: "The gene function has been partially improved compared to wild-type by altering its sequence." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0787 ! genetic experimental form
<p><br>[Term]
<br>id: MI:0791
<br>name: hypomorph
<br>def: "The gene function has been partially reduced compared to wild-type by altering its sequence e.g. a temperature sensitive mutant." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0787 ! genetic experimental form
<p><br>[Term]
<br>id: MI:0792
<br>name: antimorph
<br>def: "The gene function has been antagonized by a mutation in another copy of the gene." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0787 ! genetic experimental form
<p><br>[Term]
<br>id: MI:0793
<br>name: amorph
<br>def: "The gene function has been abolished by mutation, though the type of mutation is not known." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0787 ! genetic experimental form
<p><br>[Term]
<br>id: MI:0794
<br>name: synthetic genetic interaction defined by inequality
<br>def: "The phenotype resulting from genetic perturbation a and the phenotype resulting from genetic perturbation b have no effect on the WT background, but the combined genetic perturbation of A and B has a phenotypic effect. E. g., WT = a = b > ab." [PMID:15833125]
<br>subset: PSI-MI slim
<br>synonym: "synthetic" EXACT PSI-MI-short []
<br>synonym: "synthetic genetic interaction (sensu inequality)" EXACT PSI-MI-alternate []
<br>is_a: MI:0931 ! genetic interaction defined by inequality
<br>is_a: MI:0933 ! negative genetic interaction
<p><br>[Term]
<br>id: MI:0795
<br>name: asynthetic genetic interaction defined by inequality
<br>def: "The phenotype resulting from genetic perturbation of A alone,   B alone and AB combined have the same effect on the WT background. E. g., WT > a = b = ab." [PMID:15833125]
<br>subset: PSI-MI slim
<br>synonym: "asynthetic" EXACT PSI-MI-short []
<br>synonym: "asynthetic genetic interaction (sensu inequality)" EXACT PSI-MI-alternate []
<br>is_a: MI:0930 ! epistatic genetic interaction
<br>is_a: MI:0931 ! genetic interaction defined by inequality
<p><br>[Term]
<br>id: MI:0796
<br>name: suppressive genetic interaction defined by inequality
<br>def: "The phenotype resulting from genetic perturbation of A has an effect on WT, but that effect is abolished by adding the suppressor b, which itself shows no single-mutant effect. E. g., WT = b = ab > a." [PMID:15833125]
<br>subset: PSI-MI slim
<br>synonym: "suppression" EXACT PSI-MI-short []
<br>synonym: "suppressive genetic interaction (sensu inequality)" EXACT PSI-MI-alternate []
<br>is_a: MI:0930 ! epistatic genetic interaction
<br>is_a: MI:0931 ! genetic interaction defined by inequality
<p><br>[Term]
<br>id: MI:0797
<br>name: epistatic genetic interaction defined by inequality
<br>def: "The phenotype resulting from genetic perturbation of A and B have different effects (in terms of direction or magnitude) on the wild-type background and the double mutant has the same phenotype as either A or B (for example, A < WT < B = AB)." [PMID:15833125]
<br>subset: PSI-MI slim
<br>synonym: "epistatic" EXACT PSI-MI-short []
<br>synonym: "epistatic genetic interaction (sensu inequality)" EXACT PSI-MI-alternate []
<br>is_a: MI:0930 ! epistatic genetic interaction
<br>is_a: MI:0931 ! genetic interaction defined by inequality
<p><br>[Term]
<br>id: MI:0798
<br>name: conditional genetic interaction defined by inequality
<br>def: "The phenotype resulting from genetic perturbation of A has an effect only in the b background, or the b mutant has an effect only in the a background. a has an effect only in the b background, or the b mutant has an effect only in the a background. E. g., WT = a > ab > b or WT > a > b > ab." [PMID:15833125]
<br>subset: PSI-MI slim
<br>synonym: "conditional" EXACT PSI-MI-short []
<br>synonym: "conditional genetic interaction (sensu inequality)" EXACT PSI-MI-alternate []
<br>is_a: MI:0931 ! genetic interaction defined by inequality
<p><br>[Term]
<br>id: MI:0799
<br>name: additive genetic interaction defined by inequality
<br>def: "Single-mutant phenotype effects combine to give a double-mutant effect different from the wild type and different from single mutant effect. For instance, WT < a = b < ab, b < WT = ab < a, WT < a < b < ab, b < WT < ab < a, and all additional inequalities obtained by interchanging a and b, or reversing the effect of both a and b." [PMID:15833125]
<br>subset: PSI-MI slim
<br>synonym: "additive" EXACT PSI-MI-short []
<br>synonym: "additive genetic interaction (sensu inequality)" EXACT PSI-MI-alternate []
<br>is_a: MI:0931 ! genetic interaction defined by inequality
<p><br>[Term]
<br>id: MI:0800
<br>name: single nonmonotonic genetic interaction defined by inequality
<br>def: "The phenotype resulting from genetic perturbation of B shows opposing effects in the WT and a backgrounds (for example, b > WT and ab < a); or, a shows opposing effects in the WT and b backgrounds, but not both. E.g., WT > a > ab > b." [PMID:15833125]
<br>subset: PSI-MI slim
<br>synonym: "single nonmonotonic" EXACT PSI-MI-short []
<br>synonym: "single nonmonotonic genetic interaction (sensu inequality)" EXACT PSI-MI-alternate []
<br>is_a: MI:0931 ! genetic interaction defined by inequality
<p><br>[Term]
<br>id: MI:0801
<br>name: double nonmonotonic genetic interaction defined by inequality
<br>def: "The phenotype resulting from genetic perturbation of both A and B show opposing effects in the WT background and the background with the other mutant gene. E.g., WT >= ab > a >= b" [PMID:15833125]
<br>subset: PSI-MI slim
<br>synonym: "double nonmonotonic" EXACT PSI-MI-short []
<br>synonym: "double nonmonotonic genetic interaction (sensu inequality)" EXACT PSI-MI-alternate []
<br>is_a: MI:0931 ! genetic interaction defined by inequality
<br>is_a: MI:0935 ! positive genetic interaction
<p><br>[Term]
<br>id: MI:0802
<br>name: enhancement interaction
<br>def: "The A genetic perturbation enhances the phenotype of the B perturbation, or vice versa (e.g. WT = A < B < AB or WT = B < A < AB). This could be conditional or additive by the above scheme.\nOBSOLETE: remap to MI:0933 'negative genetic interaction'" [PMID:15833125]
<br>subset: PSI-MI slim
<br>synonym: "enhancement" EXACT PSI-MI-short []
<br>is_obsolete: true
<p><br>[Term]
<br>id: MI:0803
<br>name: expression level alteration
<br>def: "Synthesis rate of a molecule under investigation differs from its naturally occurring expression level in a cell." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "expression modif" EXACT PSI-MI-short []
<br>is_a: MI:0787 ! genetic experimental form
<p><br>[Term]
<br>id: MI:0804
<br>name: mutated gene
<br>def: "The gene is mutated in some unknown manner" [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0787 ! genetic experimental form
<p><br>[Term]
<br>id: MI:0805
<br>name: wwpdb
<br>def: "The Worldwide Protein Data Bank (wwPDB) consists of organizations that act as deposition, data processing and distribution centers for PDB data. The founding members are RCSB PDB (USA), MSD-EBI (Europe) and PDBj (Japan). The BMRB (USA) group joined the wwPDB in 2006. The mission of the wwPDB is to maintain a single Protein Data Bank Archive of macromolecular structural data that is freely and publicly available to the global community.\nhttp://www.wwpdb.org/" [PMID:14634627]
<br>subset: Drugable
<br>subset: PSI-MI slim
<br>synonym: "wwPDB" EXACT PSI-MI-alternate []
<br>xref: id-validation-regexp:\"[0-9\][a-zA-Z0-9\]\{3}\"
<br>is_a: MI:0447 ! feature database
<br>is_a: MI:0461 ! interaction database
<br>is_a: MI:0489 ! source database
<p><br>[Term]
<br>id: MI:0806
<br>name: pdbj
<br>def: "PDBj(Protein Data Bank Japan) maintains the database for the protein structures with financial assistance from the Institute for Bioinformatics Research and Development of Japan Science and Technology Corporation(BIRD-JST), collaborating with the Research Collaboration for Structural Bioinformatics(RCSB) and the MSD in the European Bioinformatics Institute(MSD-EBI) in EU. All three organizations serve as deposition, data processing and distribution sites.\nhttp://www.pdbj.org/" [PMID:12099029]
<br>subset: PSI-MI slim
<br>synonym: "PDBj" EXACT PSI-MI-alternate []
<br>xref: id-validation-regexp:\"[0-9\][a-zA-Z0-9\]\{3}\"
<br>xref: search-url: "http://pdbjs3.protein.osaka-u.ac.jp/xPSSS/DetailServlet?PDBID=${ac}&PAGEID=Summary"
<br>is_a: MI:0805 ! wwpdb
<p><br>[Term]
<br>id: MI:0807
<br>name: comigration in gel electrophoresis
<br>def: "The interaction of two molecules is determine by their very close proximity or the overlap of their relative bands in a gel." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "comigration in gel" EXACT PSI-MI-short []
<br>is_a: MI:0401 ! biochemical
<p><br>[Term]
<br>id: MI:0808
<br>name: comigration in sds page
<br>def: "Method allowing the detection of strong interactions between two molecules by their very close proximity or the overlap of their relative bands in a denaturing SDS gel." [PMID:14755292, PMID:16732283]
<br>subset: PSI-MI slim
<br>synonym: "comigration in sds" EXACT PSI-MI-short []
<br>is_a: MI:0807 ! comigration in gel electrophoresis
<p><br>[Term]
<br>id: MI:0809
<br>name: bimolecular fluorescence complementation
<br>def: "The bimolecular fluorescence complementation (BiFC) is an assay for determination of the locations of protein interactions in living cells. This approach is based on complementation between two non fluorescent fragments of the yellow fluorescent protein (YFP) when they are brought together by interactions between proteins fused to each fragment." [PMID:11983170]
<br>subset: PSI-MI slim
<br>synonym: "bifc" EXACT PSI-MI-short []
<br>is_a: MI:0051 ! fluorescence technology
<br>is_a: MI:0090 ! protein complementation assay
<p><br>[Term]
<br>id: MI:0810
<br>name: substitution analysis
<br>def: "In this approach, once a molecule is demonstrated to participate in an interaction, several substitution mutants are produced and tested in the binding assay to identify the residues which identity is crucial for the interaction." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "substitut analysis" EXACT PSI-MI-short []
<br>is_a: MI:0074 ! mutation analysis
<p><br>[Term]
<br>id: MI:0811
<br>name: insertion analysis
<br>def: "In this approach, once a molecule is demonstrated to participate in an interaction, several insertion derivatives are produced and tested in the binding assay to detect the regions that are important for the interaction." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0074 ! mutation analysis
<p><br>[Term]
<br>id: MI:0812
<br>name: calmodulin binding protein tag
<br>def: "The protein is expressed and purified as a fusion to the calmoduling-binding protein. The fusion protein can be purified by affinity chromatography using a calmodulin resin." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "cam  tag" EXACT PSI-MI-short []
<br>is_a: MI:0240 ! fusion protein
<p><br>[Term]
<br>id: MI:0813
<br>name: proximity enzyme linked immunosorbent assay
<br>def: "Method allowing efficient and precise interaction detection, along with extensive repertoires of specific binding reagents.  It is based on proximity a ligation mechanism that enables sensitive high-capacity protein measurements by converting the detection of specific proteins to the analysis of DNA sequences. Proximity probes containing oligonucleotide extensions are designed to bind pairwise to target proteins and to form amplifiable tag sequences by ligation when brought in proximity." [PMID:15155907]
<br>subset: PSI-MI slim
<br>synonym: "p elisa" EXACT PSI-MI-short []
<br>synonym: "pELISA" EXACT PSI-MI-alternate []
<br>is_a: MI:0400 ! affinity technology
<br>is_a: MI:0421 ! identification by antibody
<p><br>[Term]
<br>id: MI:0814
<br>name: protease accessibility laddering
<br>def: "In protease accessibility laddering (PAL) tagged proteins are purified on magnetic beads in their natively folded  state. While attached to the beads, proteins are probed with proteases. Proteolytic fragments are eluted and detected by immunoblotting with antibodies against the tag (e.g., Protein A, GFP, and 6xHis)." [PMID:16615907]
<br>subset: PSI-MI slim
<br>synonym: "pal" EXACT PSI-MI-alternate []
<br>synonym: "protease access" EXACT PSI-MI-short []
<br>is_a: MI:0605 ! enzymatic footprinting
<p><br>[Term]
<br>id: MI:0815
<br>name: confirmation by molecular weight
<br>def: "Molecule whose sequence identity is derived from their molecular weight" [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "weight identificat" EXACT PSI-MI-short []
<br>is_a: MI:0396 ! predetermined participant
<p><br>[Term]
<br>id: MI:0816
<br>name: molecular weight estimation by staining
<br>def: "Molecule whose sequence identity is derived from their molecular weight after a comigration in a gel with molecular weight marker." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "weight by staining" EXACT PSI-MI-short []
<br>is_a: MI:0815 ! confirmation by molecular weight
<p><br>[Term]
<br>id: MI:0817
<br>name: molecular weight estimation by silver staining
<br>def: "Molecule whose sequence identity is derived from their molecular weight after a comigration in a gel with molecular weight marker and staining of the molecules with silver." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "weight silver stain" EXACT PSI-MI-short []
<br>is_a: MI:0816 ! molecular weight estimation by staining
<p><br>[Term]
<br>id: MI:0818
<br>name: molecular weight estimation by coomasie staining
<br>def: "Molecule whose sequence identity is derived from their molecular weight after a comigration in a gel with molecular weight marker and staining of the molecules with comassie dye." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "weight by comassie" EXACT PSI-MI-short []
<br>is_a: MI:0816 ! molecular weight estimation by staining
<p><br>[Term]
<br>id: MI:0819
<br>name: molecular weight estimation by bromide staining
<br>def: "Molecule whose sequence identity is derived from their molecular weight after a comigration in a gel with molecular weight marker and staining of the molecules with bromide dyes." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "weight by bromide" EXACT PSI-MI-short []
<br>is_a: MI:0816 ! molecular weight estimation by staining
<p><br>[Term]
<br>id: MI:0820
<br>name: molecular weight estimation by sybr staining
<br>def: "Molecule whose sequence identity is derived from their molecular weight after a comigration in a gel with molecular weight marker and staining of the molecules with sybr dyes." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "safe DNA gel stain" EXACT PSI-MI-alternate []
<br>synonym: "weight by sybr" EXACT PSI-MI-short []
<br>is_a: MI:0816 ! molecular weight estimation by staining
<p><br>[Term]
<br>id: MI:0821
<br>name: molecular weight estimation by autoradiography
<br>def: "Molecule whose sequence identity is derived from their molecular weight after a comigration in a gel with molecular weight marker and staining by autoradiography." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "weight autoradiogra" EXACT PSI-MI-short []
<br>is_a: MI:0815 ! confirmation by molecular weight
<p><br>[Term]
<br>id: MI:0822
<br>name: molecular weight estimation by hoechst staining
<br>def: "Molecule whose sequence identity is derived from their molecular weight after a comigration in a gel with molecular weight marker and staining of the molecules with Hoechst dyes." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "weight by hoechst" EXACT PSI-MI-short []
<br>is_a: MI:0816 ! molecular weight estimation by staining
<p><br>[Term]
<br>id: MI:0823
<br>name: predetermined feature
<br>def: "Feature detection not verified in the context of an experiment but assumed from external or previous experimental evidence(s)." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "predetermined featur" EXACT PSI-MI-short []
<br>is_a: MI:0093 ! protein sequence identification
<br>is_a: MI:0427 ! Identification by mass spectrometry
<br>is_a: MI:0659 ! experimental feature detection
<p><br>[Term]
<br>id: MI:0824
<br>name: x-ray powder diffraction
<br>def: "Analysis of a diffraction pattern generated by an isotropic sample composed of many randomly oriented crystals." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "X-ray" EXACT PSI-MI-alternate []
<br>synonym: "x-ray powder diffrac" EXACT PSI-MI-short []
<br>is_a: MI:0114 ! x-ray crystallography
<p><br>[Term]
<br>id: MI:0825
<br>name: x-ray fiber diffraction
<br>def: "Analysis of the diffraction pattern of a partially ordered sample composed of fibers oriented parallel to each other using X-ray." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "X-ray" EXACT PSI-MI-alternate []
<br>synonym: "x-ray fiber diffrac" EXACT PSI-MI-short []
<br>is_a: MI:0114 ! x-ray crystallography
<p><br>[Term]
<br>id: MI:0826
<br>name: x ray scattering
<br>def: "Method where the internal structure of a sample is derived from the intensity distribution of the scattered monochromatic X-ray beam at very low scattering angles." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "saxs" EXACT PSI-MI-short []
<br>is_a: MI:0067 ! light scattering
<p><br>[Term]
<br>id: MI:0827
<br>name: x-ray tomography
<br>def: "X-ray Tomography is a branch of X-ray microscopy. A series of projection images are used to calculate a three dimensional reconstruction of an object. The technique has found many applications in materials science and later in biology and biomedical research. In terms of the latter, the National Center for X-ray Tomography (NCXT) is one of the principle developers of this technology, in particular for imaging whole, hydrated cells." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0428 ! imaging technique
<p><br>[Term]
<br>id: MI:0828
<br>name: polyprotein fragment
<br>def: "Subpart of a polyprotein that is naturally cleaved in vivo." [PMID:14577292]
<br>subset: PSI-MI slim
<br>synonym: "chain" RELATED []
<br>synonym: "polyprotein frag" EXACT PSI-MI-short []
<br>is_a: MI:0252 ! biological feature
<p><br>[Term]
<br>id: MI:0829
<br>name: multiple parent reference
<br>def: "This qualifier is used  for hybrid or composite molecules with more than one cross-reference to parent molecules." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "multiple parent" EXACT PSI-MI-short []
<br>is_a: MI:0353 ! cross-reference type
<p><br>[Term]
<br>id: MI:0830
<br>name: tissue list
<br>def: "List of tissue used as  topic in UniProt RC line.\nhttp://www.expasy.org/cgi-bin/lists?tisslist.txt" [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0473 ! participant database
<p><br>[Term]
<br>id: MI:0831
<br>name: cell ontology
<br>def: "Ontology of cell types.\nhttp://obo.sourceforge.net/cgi-bin/detail.cgi?cell" [PMID:16381901]
<br>subset: PSI-MI slim
<br>is_a: MI:0473 ! participant database
<p><br>[Term]
<br>id: MI:0832
<br>name: half cystine
<br>def: "A protein modification that is effectively either one half of a cystine cross-link, or a cysteine residue with one hydrogen atom or proton removed" [MOD:00798, RESID:AA0025]
<br>subset: PSI-MI slim
<br>synonym: "Half of a disulfide bridge" EXACT PSI-MI-alternate []
<br>is_a: MI:0179 ! other modification
<p><br>[Term]
<br>id: MI:0833
<br>name: autoradiography
<br>def: "Experimental method by which radiolabel is detected by exposure to a photographic emulsion forming a pattern on the film." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0659 ! experimental feature detection
<br>is_a: MI:0866 ! tag visualisation
<p><br>[Term]
<br>id: MI:0834
<br>name: ka
<br>def: "Association rate constant or rate of complex formation. Unit MOLE per SECOND (M-1 s-1)" [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "kon" EXACT PSI-MI-short []
<br>is_a: MI:0640 ! parameter type
<p><br>[Term]
<br>id: MI:0835
<br>name: kd
<br>def: "Dissociation rate constant measuring the stability of a complex. Unit SECOND (s-1)" [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "koff" EXACT PSI-MI-short []
<br>is_a: MI:0640 ! parameter type
<p><br>[Term]
<br>id: MI:0836
<br>name: temperature of interaction
<br>def: "Temperature at which interaction was determined. Unit KELVIN (K)" [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "T interaction" EXACT PSI-MI-alternate []
<br>synonym: "temp interaction" EXACT PSI-MI-short []
<br>synonym: "Tint" EXACT PSI-MI-alternate []
<br>is_a: MI:0640 ! parameter type
<p><br>[Term]
<br>id: MI:0837
<br>name: pH of interaction
<br>def: "pH at which interaction was determined." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "pHint" EXACT PSI-MI-alternate []
<br>is_a: MI:0640 ! parameter type
<p><br>[Term]
<br>id: MI:0838
<br>name: kelvin
<br>def: "A scale that measures an object's temperature over absolute zero, the theoretical coolest temperature where all molecular and atomic motion ceases. On the Kelvin scale, the freezing point of water is 273 (273 K = 0 o C = 32 o F)." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "K" EXACT PSI-MI-alternate []
<br>is_a: MI:0647 ! parameter unit
<p><br>[Term]
<br>id: MI:0839
<br>name: mole per second
<br>def: "Per mole per second, unit for association rate constant" [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "M-1s-1" EXACT PSI-MI-alternate []
<br>is_a: MI:0647 ! parameter unit
<p><br>[Term]
<br>id: MI:0840
<br>name: stimulator
<br>def: "Molecule stimulating an interaction by interacting with one or more of the participants." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0500 ! biological role
<p><br>[Term]
<br>id: MI:0841
<br>name: phosphotransfer assay
<br>def: "Measures the rate of a phosphate transfer between two proteins" [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "phosphotransfer assa" EXACT PSI-MI-short []
<br>is_a: MI:0415 ! enzymatic study
<p><br>[Term]
<br>id: MI:0842
<br>name: phosphate donor
<br>def: "Any molecule that is able to transfer an phosphate to another chemical species." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0918 ! donor
<p><br>[Term]
<br>id: MI:0843
<br>name: phosphate acceptor
<br>def: "Molecule to which a phosphate may be transferred from a phosphate donor." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0919 ! acceptor
<p><br>[Term]
<br>id: MI:0844
<br>name: phosphotransfer reaction
<br>def: "Reaction where a phosphate is transferred between two proteins of a phosphorelay system. " [PMID:14755292, PMID:16712436]
<br>subset: PSI-MI slim
<br>synonym: "phosphotransfer" EXACT PSI-MI-short []
<br>is_a: MI:0414 ! enzymatic reaction
<p><br>[Term]
<br>id: MI:0845
<br>name: spin label
<br>def: "Paramagnetic fragment, most often a cyclic nitroxide derivative, covalently attached to a molecule of interest." [PMID:10966640]
<br>subset: PSI-MI slim
<br>is_a: MI:0505 ! experimental feature
<p><br>[Term]
<br>id: MI:0846
<br>name: r1 spin label
<br>def: "Paramagnetic molecule (1-oxyl-2,2,5,5-tetramethylpyrroline-\n3-methyl)-methanethiosulfonate. that can be covalently attached to any cysteine aminoacid producing a nitroxide\nside chain designated R1." [PMID:10966640]
<br>subset: PSI-MI slim
<br>is_a: MI:0845 ! spin label
<p><br>[Term]
<br>id: MI:0847
<br>name: dansyl tag
<br>def: "Dansyl is fluorescent tag. Dansyl is the acronyme of 5-dimethylaminonaphthalene-1-sulfonyl radical group reacting with any NH2 groups." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "5-dimethylaminonaphthalene-1-sulfonyl tag" EXACT PSI-MI-alternate []
<br>synonym: "dansyl label" EXACT PSI-MI-alternate []
<br>is_a: MI:0857 ! fluorescent dye label
<p><br>[Term]
<br>id: MI:0848
<br>name: 125i radiolabel
<br>def: "Molecule labelled with 125 radio isotope of iodine atoms." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "125I" EXACT PSI-MI-alternate []
<br>synonym: "I125" EXACT PSI-MI-alternate []
<br>is_a: MI:0517 ! radiolabel
<p><br>[Term]
<br>id: MI:0849
<br>name: ncbi taxonomy
<br>def: "The NCBI taxonomy database indexes over 55 000 organisms that are represented in the sequence databases with at least one nucleotide or protein sequence. The Taxonomy Browser can be used to view the taxonomic position or retrieve sequence and structural data for a particular organism or group of organisms. Searches of the NCBI taxonomy may be made on the basis of whole or partial organism names, and direct links to organisms commonly used in biological research are also provided. The Taxonomy Browser can also be used to display the number of nucleic acid sequences, protein sequences, and protein structures available for organisms included in the branch. From the data display for a particular organism, one can retrieve and download the sequence data for that organism, or protein 3D structure data if available.\nhttp://www.ncbi.nlm.nih.gov/Taxonomy/" [PMID:10592169]
<br>subset: PSI-MI slim
<br>xref: id-validation-regexp:\"[0-9\]+\"
<br>xref: search-url: "http://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?mode=Info&id=${ac}&lvl=3&lin=f&keep=1&srchmode=1&unlock"
<br>is_a: MI:0473 ! participant database
<p><br>[Term]
<br>id: MI:0850
<br>name: encode
<br>def: "ENCODE (the Encyclopedia Of DNA Elements) seeks to identify all protein-coding genes. The current ENCODE data set is derived from 1% of the human genome and has been selected for analysis in the pilot phase of the project.\nhttp://www.genome.gov/10005107" [PMID:17372197]
<br>subset: PSI-MI slim
<br>synonym: "ENCODE" EXACT PSI-MI-alternate []
<br>synonym: "Encyclopedia Of DNA Elements" EXACT PSI-MI-alternate []
<br>is_a: MI:0683 ! sequence database
<p><br>[Term]
<br>id: MI:0851
<br>name: protein genbank identifier
<br>def: "GenBank Identifier or GI numbers for proteins." [PMID:17170002]
<br>subset: PSI-MI slim
<br>synonym: "GenBank Protein GI" EXACT PSI-MI-alternate []
<br>synonym: "genbank protein gi" EXACT PSI-MI-alternate []
<br>synonym: "genbank_protein_gi" EXACT PSI-MI-short []
<br>xref: id-validation-regexp:\"[0-9\]+\"
<br>xref: search-url: "http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=protein&cmd=Retrieve&dopt=Graphics&list_uids=${ac}"
<br>is_a: MI:0860 ! genbank indentifier
<p><br>[Term]
<br>id: MI:0852
<br>name: nucleotide genbank identifier
<br>def: "GenBank Identifier or GI numbers for nucleotide." [PMID:17170002]
<br>subset: PSI-MI slim
<br>synonym: "genbank nucleotide" EXACT PSI-MI-alternate []
<br>synonym: "GenBank Nucleotide" EXACT PSI-MI-alternate []
<br>synonym: "genbank_nucl_gi" EXACT PSI-MI-short []
<br>xref: id-validation-regexp:\"[0-9\]+\"
<br>xref: search-url: "http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=nucleotide&cmd=Retrieve&dopt=Graphics&list_uids=$"
<br>is_a: MI:0860 ! genbank indentifier
<p><br>[Term]
<br>id: MI:0853
<br>name: dna overhang
<br>def: "An overhang is a stretch of unpaired nucleotides in the end of a DNA molecule. These unpaired nucleotides can be in either strand, creating either 3' or 5' overhangs. Longer overhangs are called cohessive ends or sticky ends. They are most often created by restriction endonucleases when they cut DNA. Very often they cut the two DNA strands four base pairs from each other, creating a four-base 3' overhang in the other molecule and a complementary 5' overhang in the other. These ends are called cohessive since they are easily joined back together by a ligase" [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "cohessive ends" EXACT PSI-MI-alternate []
<br>synonym: "sticky ends" EXACT PSI-MI-alternate []
<br>is_a: MI:0505 ! experimental feature
<p><br>[Term]
<br>id: MI:0854
<br>name: 3 prime overhang
<br>def: "An overhang is a stretch of unpaired nucleotides in the end of a 3' strand of a DNA molecule." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "3 prime sticky end" EXACT PSI-MI-alternate []
<br>is_a: MI:0853 ! dna overhang
<p><br>[Term]
<br>id: MI:0855
<br>name: 5 prime overhang
<br>def: "An overhang is a stretch of unpaired nucleotides in the end of a 5' strand of a DNA molecule." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "5 prime sticky end" EXACT PSI-MI-alternate []
<br>is_a: MI:0853 ! dna overhang
<p><br>[Term]
<br>id: MI:0856
<br>name: fluorophore
<br>def: "A fluorophore is a component of a molecule which causes a molecule to be fluorescent. It is a functional group in a molecule which will absorb energy of a specific wavelength and re-emit energy at a different (but equally specific) wavelength. The amount and wavelength of the emitted energy depend on both the fluorophore and the chemical environment of the fluorophore." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0505 ! experimental feature
<p><br>[Term]
<br>id: MI:0857
<br>name: fluorescent dye label
<br>def: "Dye label containing a  fluorophore  which  absorb energy of a specific wavelength and re-emit energy at a different (but equally specific) wavelength." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "fluorescent dye" EXACT PSI-MI-short []
<br>is_a: MI:0856 ! fluorophore
<p><br>[Term]
<br>id: MI:0858
<br>name: immunodepleted coimmunoprecipitation
<br>def: "Method involving consecutive coimmunoimmunoprecipitations on the same sample, a control  where an interaction is detected, and other CoIPs where the sample is previously treated with a specific antibody that precipitates a candidate interactor and leads to the suppression of an interaction or a change in composition of a complex." [PMID:17081976]
<br>subset: PSI-MI slim
<br>synonym: "immunodepleted coip" EXACT PSI-MI-alternate []
<br>synonym: "immunodepletion" EXACT PSI-MI-short []
<br>is_a: MI:0019 ! coimmunoprecipitation
<p><br>[Term]
<br>id: MI:0859
<br>name: intermolecular force
<br>def: "An optical tweezer is a scientific instrument that uses a focused laser beam to provide an attractive or repulsive force, depending on the index mismatch (typically on the order of piconewtons) to physically hold and move microscopic dielectric objects (i.e.highly resistants to the flow of an electric current). This instrument allow to measure the forces generated between interacting molecules - either at the level of just single interacting pair of molecules or at the level of larger molecular assemblies. " [PMID:17023539, PMID:17081984]
<br>subset: PSI-MI slim
<br>synonym: "force measurement" EXACT PSI-MI-alternate []
<br>synonym: "molecular force measurement" EXACT PSI-MI-alternate []
<br>synonym: "optical tweezer" EXACT PSI-MI-alternate []
<br>synonym: "single molecule force measurement" EXACT PSI-MI-alternate []
<br>synonym: "surface adhesion force measurement" EXACT PSI-MI-alternate []
<br>is_a: MI:0013 ! biophysical
<p><br>[Term]
<br>id: MI:0860
<br>name: genbank indentifier
<br>def: " edit" [PMID:15078858]
<br>subset: PSI-MI slim
<br>xref: id-validation-regexp:\"ENS[A-Z\]+[0-9\]\{11}\"
<br>is_a: MI:0683 ! sequence database
<p><br>[Term]
<br>id: MI:0861
<br>name: protein a tag
<br>def: "The protein A is a bacterial cell wall  isolated from Staphylococcus aureus that  binds to mammalian IgGs mainly through Fc regions. The protein A can be use to retaint antibodies or as fusion tag of a protein under analysis." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "protein A" EXACT PSI-MI-alternate []
<br>is_a: MI:0240 ! fusion protein
<p><br>[Term]
<br>id: MI:0862
<br>name: zz tag
<br>def: "The ZZ tag is a tag made out of two tandem repeats of the Protein A IgG binding domain. " [PMID:11694505]
<br>subset: PSI-MI slim
<br>synonym: "ZZ tag" EXACT PSI-MI-alternate []
<br>is_a: MI:0861 ! protein a tag
<p><br>[Term]
<br>id: MI:0863
<br>name: thiol reactive lanthanide label
<br>def: "Thiol-reactive lanthanide complexes have been synthesized that are luminescent when bound to terbium and/or europium. The Tb3+-DTPA-cs124-EMCH complexes consist of a diethylenetriaminepentaacetate (DTPA) chelate covalently joined through one amide bond to a chromophore, carbostyril 124, and via a second amide bond to a maleimide, bromoacetamide, or pyridyldithio moiety. This label can be Site-specific attachted to both proteins and DNA." [PMID:10077482]
<br>subset: PSI-MI slim
<br>synonym: "Tb3+-DTPA-cs124-EMCH" EXACT PSI-MI-alternate []
<br>synonym: "thiol lanthanide" EXACT PSI-MI-short []
<br>is_a: MI:0505 ! experimental feature
<p><br>[Term]
<br>id: MI:0864
<br>name: brenda
<br>def: "A structured controlled vocabulary for the source of an enzyme. It comprises terms for tissues, cell lines, cell types and cell cultures from uni- and multicellular organisms.\nhttp://www.brenda-enzymes.info" [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0473 ! participant database
<p><br>[Term]
<br>id: MI:0865
<br>name: fluorescence acceptor donor pair
<br>def: "Pair of fluorophores attached to the same molecule used in a fret  experiment to observe the details of conformational  changes ." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "fret pair" EXACT PSI-MI-short []
<br>is_a: MI:0495 ! experimental role
<p><br>[Term]
<br>id: MI:0866
<br>name: tag visualisation
<br>def: "Molecule whose sequence identity is not checked after the interaction but its presence is detected through its tag" [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0396 ! predetermined participant
<p><br>[Term]
<br>id: MI:0867
<br>name: tag visualisation by fluorescence
<br>def: "The protein is either expressed as a hybrid protein fused to a tag containing a fluorophore, such as green fluorescent protein, or a fluorophore has been chemically attached to the protein. Subsequence observation or measurement of fluorescence is used to identify the presence of the molecule in an interaction." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "tag fluorescence" EXACT PSI-MI-short []
<br>is_a: MI:0866 ! tag visualisation
<p><br>[Term]
<br>id: MI:0868
<br>name: author identifier
<br>def: "Author published identifier." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0353 ! cross-reference type
<p><br>[Term]
<br>id: MI:0869
<br>name: originally assigned identifier
<br>def: "Identifier assigned when the record was created by a source database." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "original identifier" EXACT PSI-MI-short []
<br>is_a: MI:0353 ! cross-reference type
<p><br>[Term]
<br>id: MI:0870
<br>name: demethylase assay
<br>def: "Measures the catalysis of the hydrolysis of an methyl group from a substrate molecule." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0415 ! enzymatic study
<p><br>[Term]
<br>id: MI:0871
<br>name: demethylation reaction
<br>def: "The cleavage of a methyl group from a polypeptide. Methylation is generally an irreversible reaction except in mamalian." [GO:0006482, PMID:17277772]
<br>subset: PSI-MI slim
<br>synonym: "demethylation" EXACT PSI-MI-short []
<br>is_a: MI:0414 ! enzymatic reaction
<p><br>[Term]
<br>id: MI:0872
<br>name: atomic force microscopy
<br>def: "The atomic force microscope (AFM) is a very high-resolution type of scanning probe microscope, with demonstrated resolution of fractions of a nanometer, more than 1000 times better than the optical diffraction limit. The AFM was invented by Binnig, Quate and Gerber in 1986, and is one of the foremost tools for imaging, measuring and manipulating matter at the nanoscale.The term 'microscope' in the name is actually a misnomer because it implies looking, while in fact the information is gathered by feeling out the surface with a mechanical feeler.\nhttp://en.wikipedia.org/wiki/Atomic_force_microscope" [PMID:17502105]
<br>subset: PSI-MI slim
<br>synonym: "AFM" EXACT PSI-MI-alternate []
<br>synonym: "atomic force microsc" EXACT PSI-MI-short []
<br>is_a: MI:0428 ! imaging technique
<p><br>[Term]
<br>id: MI:0873
<br>name: curation request
<br>def: "Annotation of a published paper which has been externally requested." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0665 ! experiment attibute name
<p><br>[Term]
<br>id: MI:0875
<br>name: dataset
<br>def: "Targeted curation dataset grouping experiments by topic or dataset origin." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0665 ! experiment attibute name
<p><br>[Term]
<br>id: MI:0878
<br>name: author submitted
<br>def: "Data directly submitted by the authors to a database prior to publication." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0665 ! experiment attibute name
<p><br>[Term]
<br>id: MI:0879
<br>name: nucleoside triphosphatase assay
<br>def: "Measures the catalysis of the hydrolisis of a  nucleoside triphosphate into a nucleoside diphosphate plus phosphate." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "triphosphatase ass" EXACT PSI-MI-short []
<br>is_a: MI:0415 ! enzymatic study
<p><br>[Term]
<br>id: MI:0880
<br>name: atpase assay
<br>def: "Measures the catalysis of the hydrolisis of ATP+ H2O = ADP + phosphate." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0879 ! nucleoside triphosphatase assay
<p><br>[Term]
<br>id: MI:0881
<br>name: nucleoside triphosphatase reaction
<br>def: "Catalysis of the hydrolisis of a  nucleoside triphosphate into a nucleoside diphosphate plus phosphate." [GO:0017111, PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "triphosphatase react" EXACT PSI-MI-short []
<br>is_a: MI:0414 ! enzymatic reaction
<p><br>[Term]
<br>id: MI:0882
<br>name: atpase reaction
<br>def: "Catalysis of the hydrolisis of ATP+ H2O = ADP + phosphate." [GO:0016887, PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0881 ! nucleoside triphosphatase reaction
<p><br>[Term]
<br>id: MI:0883
<br>name: gtpase reaction
<br>def: "Catalysis of the hydrolisis of GTP+ H2O = GDP + phosphate." [GO:0003924, PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0881 ! nucleoside triphosphatase reaction
<p><br>[Term]
<br>id: MI:0884
<br>name: vsv tag
<br>def: "Epitope tag derived from vesicular stomatitis virus (VSV) glycoprotein. The tag sequence is YTDIEMNRLGK and many antibodies against it are commercially available." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "vesicular stomatitis virus tag" EXACT PSI-MI-alternate []
<br>is_a: MI:0507 ! tag
<p><br>[Term]
<br>id: MI:0885
<br>name: journal
<br>def: "Name and details of a journal from which paper has been taken." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0665 ! experiment attibute name
<p><br>[Term]
<br>id: MI:0886
<br>name: publication year
<br>def: "Year of publication of a paper." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0665 ! experiment attibute name
<p><br>[Term]
<br>id: MI:0887
<br>name: histone acetylation assay
<br>def: "In histone acetylation the histones are acetylated on lysine residues in the N-terminal tail as part of gene regulation. Typically, these reactions are catalyzed by enzymes with \"histone acetyltransferase\" (HAt)" [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "HAt" EXACT PSI-MI-alternate []
<br>synonym: "hat" EXACT PSI-MI-short []
<br>is_a: MI:0889 ! acetylation assay
<p><br>[Term]
<br>id: MI:0888
<br>name: small angle neutron scattering
<br>def: "During a SANS experiment a beam of neutrons is directed at a sample. The neutrons are elastically scattered by a sample and the resulting scattering pattern is analyzed to provide information about the size, shape and orientation of some component of the sample." [PMID:11578931]
<br>subset: PSI-MI slim
<br>synonym: "SANS" EXACT PSI-MI-alternate []
<br>synonym: "sans" EXACT PSI-MI-short []
<br>is_a: MI:0013 ! biophysical
<p><br>[Term]
<br>id: MI:0889
<br>name: acetylation assay
<br>def: "Measures the catalysis of the addition of an acetyl group to a target molecule." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0415 ! enzymatic study
<p><br>[Term]
<br>id: MI:0890
<br>name: qdot
<br>def: "Qdot are nanocrystals fluorophores . Nanocrystals a are extremely efficient materials for generating and they have a highly customizable surface for directing their bioactivity, producing a fluorescent probe that outperforms traditional dyes in many fluorescence applications." [PMID:17569782]
<br>subset: PSI-MI slim
<br>is_a: MI:0857 ! fluorescent dye label
<p><br>[Term]
<br>id: MI:0891
<br>name: neutron fiber diffraction
<br>def: "Analysis of diffraction pattern of a partially ordered sample composed of fibers oriented parallel to each other using neutron beam." [PMID:10771422, PMID:15272083, PMID:15546977, PMID:15914673, PMID:16041074, PMID:16707576]
<br>subset: PSI-MI slim
<br>synonym: "neutron fiber diff" EXACT PSI-MI-short []
<br>is_a: MI:0013 ! biophysical
<p><br>[Term]
<br>id: MI:0892
<br>name: solid phase assay
<br>def: "Assay where at least one molecule under analysis is bound to a solid surface, such as a microplate wall or the sides of a tube, the other reactants being free in solution." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0400 ! affinity technology
<p><br>[Term]
<br>id: MI:0893
<br>name: neutron diffraction
<br>def: "Analysis of diffraction pattern using neutron beam" [PMID:10771422, PMID:15272083, PMID:15546977, PMID:15914673, PMID:16041074, PMID:16707576]
<br>subset: PSI-MI slim
<br>is_a: MI:0013 ! biophysical
<p><br>[Term]
<br>id: MI:0894
<br>name: electron diffraction
<br>def: "Analysis of diffraction pattern using electron beam." [PMID:10949309, PMID:11034202, PMID:11171962, PMID:11532455, PMID:11700061, PMID:15141214, PMID:16325200]
<br>subset: PSI-MI slim
<br>is_a: MI:0013 ! biophysical
<p><br>[Term]
<br>id: MI:0895
<br>name: protein kinase A complementation
<br>def: "This method uses Renilla luciferase (Rluc)-based protein fragment complementation assay (PCA) that is designed specifically to investigate dynamic protein complexes (association and dissociation). It is chose to generate a PCA based on the Rluc, which is, because of its simplicity and sensitivity, a widely used bioluminescence reporter. The general scheme for construction and detection of the Rluc-PCA PKA sensor consists of fusing complementary fragments of Rluc to the regulatory (Reg) and catalytic (Cat) PKA subunits of PKA." [PMID:17942691]
<br>subset: PSI-MI slim
<br>synonym: "pka complementation" EXACT PSI-MI-short []
<br>is_a: MI:0228 ! cytoplasmic complementation assay
<p><br>[Term]
<br>id: MI:0896
<br>name: renilla luciferase protein tag
<br>def: "Renilla luciferase, is an enzyme of the sea pansy catalyzing the oxidation of the lucifering pigment (reaction with ATP or oxigen)  that produces light. This protein is often used as a vizualisation tag because compare to many fluorescent tag it has a better signal to noise ratio." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "renilla lucirefase" EXACT PSI-MI-short []
<br>is_a: MI:0240 ! fusion protein
<p><br>[Term]
<br>id: MI:0897
<br>name: protein modification ontology
<br>def: "Catalogue of covalent modification of, or a change resulting in an alteration of the measured molecular mass of, a peptide or protein amino acid residue." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "psi-mod" EXACT PSI-MI-short []
<br>xref: id-validation-regexp:\"MOD\:[0-9\]\{5}\"
<br>is_a: MI:0447 ! feature database
<p><br>[Term]
<br>id: MI:0898
<br>name: putative self
<br>def: "Molecule that is reported to self-interact but the experimental condition does not allow to resolve whether the interaction is intramolecular (true self interaction) or intermolecular (homodimer)." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0495 ! experimental role
<br>is_a: MI:0500 ! biological role
<p><br>[Term]
<br>id: MI:0899
<br>name: p3 filamentous phage display
<br>def: "pIII is one of the minor coat proteins that decorates in five copies the emerging tip of filamentous phage. Similarly to pVIII pIII also tolerates peptide insertions at the amino-terminus. The sequences to be displayed can either be encoded in the phage copy of the coat gene or in an extra pIII gene copy carried on a phagemid. In the first case 5 copies o the hybrid proteins are displayed while in the latter only a few capsid display one copy and the majority display none.  Because of the low copy number pIII display is often referred to as low valency display." [PMID:1696028]
<br>subset: PSI-MI slim
<br>synonym: "low valency display" EXACT PSI-MI-alternate []
<br>synonym: "p3 filamentous phage" EXACT PSI-MI-short []
<br>is_a: MI:0048 ! filamentous phage display
<p><br>[Term]
<br>id: MI:0900
<br>name: p8 filamentous phage display
<br>def: "pVIII is the major coat protein of filamentous phage. Its amino-terminus is exposed to solvent and tolerates the insertion of relatively large peptide fragments. By inserting the peptide coding sequence into the phage copy of the pVIII gene up to 3000 copies of the hybrid proteins can be displayed along the phage capsid. Alternatively the hybrid protein can be encoded on a phagemid that is incorporated in a virus like particle by infection with a helper phage. In this latter case one obtains a chimeric capsid where hybrid proteins are interspersed at different density in an otherwise wild type coat. Because of the high copy number pVIII display is also referred to as high valency display." [PMID:1720463]
<br>subset: PSI-MI slim
<br>synonym: "high valency display" EXACT PSI-MI-alternate []
<br>synonym: "p8 filamentous phage" EXACT PSI-MI-short []
<br>is_a: MI:0048 ! filamentous phage display
<p><br>[Term]
<br>id: MI:0901
<br>name: isotope label footprinting
<br>def: "Exposed amino acid residues undergo a rapid exchange of a specific radio-isotope e.g. hydrogen/deuterium. Residues involved in a molecular interaction are protected from this exchange and exhibit a much slower rate of exchange. This method of binding range identification must be coupled to NMR or mass spectrometry  technologies in order to detect the radio-isotope exchange." [PMID:18184591]
<br>subset: PSI-MI slim
<br>synonym: "isotope footprinting" EXACT PSI-MI-short []
<br>is_a: MI:0605 ! enzymatic footprinting
<p><br>[Term]
<br>id: MI:0902
<br>name: rna cleavage
<br>def: "Any process by which an RNA molecule is cleaved at specific sites or in a regulated manner." [GO:0006396, PMID:14681407]
<br>subset: PSI-MI slim
<br>is_a: MI:0910 ! nucleic acid cleavage
<p><br>[Term]
<br>id: MI:0903
<br>name: mpidb
<br>def: "The microbial protein-protein interactions database (MPDIB) aims to collect and provide all known physical prokaryotic interactions." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "MPDIB" EXACT PSI-MI-alternate []
<br>is_a: MI:0461 ! interaction database
<p><br>[Term]
<br>id: MI:0904
<br>name: polysaccharide
<br>def: "A polysaccharide is a complex polymer of carbohydrate monormers. They are polymers made up of many monosaccharides joined together by glycosidic bonds. They are therefore very large, often branched, macromolecules." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0383 ! biopolymer
<p><br>[Term]
<br>id: MI:0905
<br>name: amplified luminescent proximity homogeneous assay
<br>def: "AlphaScreen relies on the use of Donor and Acceptor beads that are coated with a layer of hydrogel providing functional groups for bioconjugation. When a biological interaction between molecules brings the beads into proximity, a cascade of chemical reactions is initiated to produce a greatly amplified signal. Upon laser excitation, a photosensitizer in the Donor bead converts ambient oxygen to a more excited singlet state. The singlet state oxygen molecules diffuse across to react with a chemiluminescer in the Acceptor bead that further activates fluorophores contained within the same bead. The fluorophores subsequently emit light at 520-620 nm. In the absence of a specific biological interaction, the singlet state oxygen molecules produced by the Donor bead go undetected without the close proximity of the Acceptor bead. AlphaScreen has successfully been developed for enzyme assays (kinase, helicase, protease, ...), interaction assays (ligand/receptor, protein/protein, protein/DNA), immunoassays, and GPCR functional assays (cAMP, IP3)." [PMID:17092917]
<br>subset: PSI-MI slim
<br>synonym: "alfa-screen" EXACT PSI-MI-short []
<br>synonym: "AlphaScreen" EXACT PSI-MI-alternate []
<br>is_a: MI:0013 ! biophysical
<p><br>[Term]
<br>id: MI:0906
<br>name: au1 tag
<br>def: "The protein of interest is expressed as a fusion to the peptide DTYRYI for which antibodies are commercially available." [PMID:18216269]
<br>subset: PSI-MI slim
<br>synonym: "DTYRYI epitope tag" EXACT PSI-MI-alternate []
<br>is_a: MI:0507 ! tag
<p><br>[Term]
<br>id: MI:0907
<br>name: conformational status
<br>def: "Statement about the native/denatured conformation of the protein." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "conformation" EXACT PSI-MI-short []
<br>is_a: MI:0346 ! experimental preparation
<p><br>[Term]
<br>id: MI:0908
<br>name: denatured
<br>def: "Altered conformation state of the protein as a result of heat or chemical modification resulting in a changed structure of the protein." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0907 ! conformational status
<p><br>[Term]
<br>id: MI:0909
<br>name: native
<br>def: "State of the protein without interference i.e. the natural form." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0907 ! conformational status
<p><br>[Term]
<br>id: MI:0910
<br>name: nucleic acid cleavage
<br>def: "Covalent bond breakage of a nucleic acid molecule leading to the formation of smaller fragments." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "ncl acid cleavage" EXACT PSI-MI-short []
<br>is_a: MI:0194 ! cleavage reaction
<p><br>[Term]
<br>id: MI:0911
<br>name: cross linker
<br>def: "A variety of crosslinkers are used to analyze subunit structure of proteins, protein interactions and various parameters of protein function. Subunit structure is deduced since crosslinkers only bind surface amino residues in relatively close proximity in the native state. Protein interactions are often too weak or transient to be easily detected, but by crosslinking, the interactions can be captured and analyzed." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "crosslinker" EXACT PSI-MI-alternate []
<br>is_a: MI:0505 ! experimental feature
<p><br>[Term]
<br>id: MI:0912
<br>name: spdp cross linker
<br>def: "N -Succinimidyl 3-(2-pyridyldithio)-propionate, is heterobifunctional, thiol-cleavable \nand membrane permeable crosslinkers. It contains an amine-reactive N-hydroxysuccinimide (NHS) ester \nthat will react with lysine residues to form a stable amide bond. The other end of the spacer arm is terminated in the pyridyl disulfide group that will react with sulfhydryls to form a reversible disulfide bond." [PMID:17360572]
<br>subset: PSI-MI slim
<br>synonym: "N -Succinimidyl 3-(2-pyridyldithio)-propionate" EXACT PSI-MI-alternate []
<br>is_a: MI:0911 ! cross linker
<p><br>[Term]
<br>id: MI:0913
<br>name: lc-spdp cross linker
<br>def: "Succinimidyl 6-(3-[2-pyridyldithio]-propionamido)hexanoate, is an heterobifunctional, thiol-cleavable \nand membrane permeable crosslinkers. It contains an amine-reactive N-hydroxysuccinimide (NHS) ester \nthat will react with lysine residues to form a stable amide bond. The other end of the spacer arm is terminated in the pyridyl disulfide group that will react with sulfhydryls to form a reversible disulfide bond. LC-SPDP is a derivative of the classical SPDP with a longer spacer arm." [PMID:17360572]
<br>subset: PSI-MI slim
<br>synonym: "Succinimidyl 6-(3-[2-pyridyldithio]-propionamido)hexanoate" EXACT PSI-MI-alternate []
<br>is_a: MI:0912 ! spdp cross linker
<p><br>[Term]
<br>id: MI:0914
<br>name: association
<br>def: "Molecules that are experimentally shown to be associated potentially by sharing just one interactor. Often associated molecules are co-purified by a pull-down or coimmunoprecipitation and share the same bait molecule." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0190 ! interaction type
<p><br>[Term]
<br>id: MI:0915
<br>name: physical association
<br>def: "Molecules that are experimentally shown to belong to the same functional or structural complex." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0914 ! association
<p><br>[Term]
<br>id: MI:0916
<br>name: lexa vp16 complementation
<br>def: "Yeast two-hybrid system using Escherichia coli LexA amino acids 1-202 as the DNA-binding domain (BD), and a transcriptional activation domain from the herpes simplex virus protein VP16 (either aa 411-490 or aa 411-455) that can specifically activate transcription of a reporter gene located downstream." [PMID:9371806]
<br>subset: PSI-MI slim
<br>synonym: "lexa vp16 complement" EXACT PSI-MI-short []
<br>synonym: "LexA VP16 transcription complementation" EXACT PSI-MI-alternate []
<br>is_a: MI:0018 ! two hybrid
<p><br>[Term]
<br>id: MI:0917
<br>name: matrixdb
<br>def: "Knowledgebase of the extracellular matrix storing experimentally determined interactions involving extracellular biomolecules. It includes protein-protein, protein-polysaccharide, and protein-lipid interactions." [PMID:19147664]
<br>subset: PSI-MI slim
<br>is_a: MI:0461 ! interaction database
<br>is_a: MI:0489 ! source database
<p><br>[Term]
<br>id: MI:0918
<br>name: donor
<br>def: "Molecule which emits electrons or ions that are transfered to an acceptor molecule." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0500 ! biological role
<p><br>[Term]
<br>id: MI:0919
<br>name: acceptor
<br>def: "Molecule able to receive electrons or ions from a donor molecule." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0500 ! biological role
<p><br>[Term]
<br>id: MI:0920
<br>name: ribonuclease assay
<br>def: "Measures the catalysis of the hydrolysis of phosphodiester bonds in chains of RNA." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0415 ! enzymatic study
<p><br>[Term]
<br>id: MI:0921
<br>name: surface plasmon resonance array
<br>def: "This array technology allows the screening of binding abilities of hundreds or thousands of biomolecules (small molecule, peptide, protein, sugar, lipid, nucleic acid, and their fragments) printed onto the gold-coated chip by using an instrument (micro-arrayer) that is capable of spotting liquid samples in a reproducible manner onto a planar support. The ordered protein array can then be probed with a non-labelled sample (small molecule, peptide, protein, sugar, lipid, nucleic acid, and their fragments) to identify the baits that can bind to it.  This is done in real time, allowing direct measurement of both the on-rate and the off-rate and of the affinity constant of complex formation on each spot." [PMID:16510109, PMID:16837183, PMID:17889820]
<br>subset: PSI-MI slim
<br>synonym: "Biacore Flexchip(r)" EXACT PSI-MI-alternate []
<br>synonym: "spr array" EXACT PSI-MI-short []
<br>synonym: "SPRi-Plex " EXACT PSI-MI-alternate []
<br>synonym: "SPRImager" EXACT PSI-MI-alternate []
<br>is_a: MI:0008 ! array technology
<br>is_a: MI:0107 ! surface plasmon resonance
<p><br>[Term]
<br>id: MI:0922
<br>name: imex evidence
<br>def: "Experimental evidence supporting an interaction curated and released under the IMEx agreement." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0353 ! cross-reference type
<p><br>[Term]
<br>id: MI:0923
<br>name: irefindex
<br>def: "iRefIndex provides an index of protein interactions available in a number of primary interaction databases including BIND, BioGRID, DIP, HPRD, IntAct, MINT, MPact, MPPI and OPHID.  This index allows the user to search for a protein and retrieve a non-redundant list of interactors for that protein. iRefIndex uses the Sequence Global Unique Identifier (SEGUID) to group proteins and interactions into redundant groups.  This method allows users to integrate their own data with the iRefIndex in a way that ensures proteins with the exact same sequence will be represented only once.\nhttp://irefindex.uio.no/" [PMID:18823568]
<br>subset: PSI-MI slim
<br>synonym: "iRefIndex" EXACT PSI-MI-alternate []
<br>is_a: MI:0461 ! interaction database
<br>is_a: MI:0489 ! source database
<p><br>[Term]
<br>id: MI:0924
<br>name: camjedb
<br>def: "Camjedb is a comprehensive database for information on the genome of Campylobacter jejuni.\nhttp://www.sanger.ac.uk/Projects/C_jejuni/" [PMID:106882042]
<br>subset: PSI-MI slim
<br>is_a: MI:0683 ! sequence database
<p><br>[Term]
<br>id: MI:0925
<br>name: observed ptm
<br>def: "Post translational modification observed on a protein in the context of an interaction." [PMID:14755292]
<br>subset: PSI-MI slim
<br>is_a: MI:0668 ! feature attribute name
<p><br>[Term]
<br>id: MI:0926
<br>name: fiash label
<br>def: "This fluorescent label is a small ligand  membrane-permeant and nonfluorescent until it binds with high affinity and specificity to the tetracysteine domain. Such in situ labeling adds much less mass than does green fluorescent protein and offers greater versatility in attachment sites as well as potential spectroscopic and chemical properties." [PMID:9657724]
<br>subset: PSI-MI slim
<br>synonym: "4',5'-bis(1,3,2-dithioarsolan-2-yl)fluorescein " EXACT PSI-MI-alternate []
<br>synonym: "FIAsH label" EXACT PSI-MI-alternate []
<br>is_a: MI:0857 ! fluorescent dye label
<p><br>[Term]
<br>id: MI:0927
<br>name: iaedans label
<br>def: "IAEDANS is fluorescent tag that bind to cysteines. IAEDANS is the acronyme of (5((((2-iodoacetyl)amino)ethyl)amino)-naphthalene-1-sulfonic acid)  with free SH groups." [PMID:11052891]
<br>subset: PSI-MI slim
<br>synonym: "(5((((2-iodoacetyl)amino)ethyl)amino)-naphthalene-1-sulfonic acid) " EXACT PSI-MI-alternate []
<br>synonym: "1,5-IAEDANS" EXACT PSI-MI-alternate []
<br>synonym: "IAEDANS" EXACT PSI-MI-alternate []
<br>is_a: MI:0857 ! fluorescent dye label
<p><br>[Term]
<br>id: MI:0928
<br>name: filter trap assay
<br>def: "Biomolecules are mixed in a buffer and the resulting mixture is\npassed through a filter. Large\naggregates are retained on the filter and the pariticipants may then be identified." [PMID:10859365]
<br>subset: PSI-MI slim
<br>synonym: "<new synonym>" RELATED []
<br>synonym: "filter binding" EXACT []
<br>synonym: "filter retardation assay" RELATED []
<br>synonym: "Filter retention assay" EXACT PSI-MI-alternate []
<br>is_a: MI:0013 ! biophysical
<br>is_a: MI:0091 ! chromatography technology
<p><br>[Term]
<br>id: MI:0929
<br>name: nothern blot
<br>def: "A standard procedure to identify RNA fragments containing specific\nsequences. In this procedure RNA fragments are separated by\nelectrophoresis, the fragments are transferred to a membrane and the\nmembrane is incubated with a radio labelled probe that hybridises any\ncomplementary subsequence." [PMID:414220]
<br>subset: PSI-MI slim
<br>synonym: "northen" EXACT PSI-MI-alternate []
<br>is_a: MI:0078 ! nucleotide sequence identification
<p><br>[Term]
<br>id: MI:0930
<br>name: epistatic genetic interaction
<br>def: "The genetic interaction between a group of genes, in which the effects of an allele of one member gene of the group hide the effects of alleles of other genes in the group." [PMID:11988766]
<br>subset: PSI-MI slim
<br>synonym: "epistatis" EXACT PSI-MI-short []
<br>is_a: MI:0935 ! positive genetic interaction
<p><br>[Term]
<br>id: MI:0931
<br>name: genetic interaction defined by inequality
<br>def: "Two genes A and B present an genetic interaction defined by inequality  if the phenotypes of the two single mutants a and b, the double mutant  ab and the wild-type WT can be measured quantitatively and described  relative to each other by an inequality relationship." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "genetic inequality" EXACT PSI-MI-short []
<br>is_a: MI:0208 ! genetic interaction
<p><br>[Term]
<br>id: MI:0932
<br>name: noninteractive genetic interaction defined by inequality
<br>def: "Mutation that has no effect in the WT and b backgrounds, or b has no effect in the a and WT backgrounds, or both hold true. E. g., WT = a > b = ab." [PMID:15833125]
<br>subset: PSI-MI slim
<br>synonym: "noninteractive" EXACT PSI-MI-short []
<br>synonym: "noninteractive genetic interaction (sensu inequality)" EXACT PSI-MI-alternate []
<br>is_a: MI:0931 ! genetic interaction defined by inequality
<br>is_a: MI:0934 ! neutral genetic interaction
<p><br>[Term]
<br>id: MI:0933
<br>name: negative genetic interaction
<br>def: "The phenotype of the double mutant ab is stronger than expected from the combination of the individual effects of a and b." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "negative gen int" EXACT PSI-MI-short []
<br>is_a: MI:0208 ! genetic interaction
<p><br>[Term]
<br>id: MI:0934
<br>name: neutral genetic interaction
<br>def: "The phenotype of the double mutant ab is equal to the phenotype expected from the combination of the individual effects of a and b." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "neutral gent int" EXACT PSI-MI-short []
<br>is_a: MI:0208 ! genetic interaction
<p><br>[Term]
<br>id: MI:0935
<br>name: positive genetic interaction
<br>def: "The phenotype of the double mutant ab is weaker than expected from the combination of the individual effects of a and b." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "positive gent int" EXACT PSI-MI-short []
<br>is_a: MI:0208 ! genetic interaction
<p><br>[Term]
<br>id: MI:0936
<br>name: emdb
<br>def: "The Electron Microscopy Data Bank (EMDB) contains experimentally determined three-dimensional maps and associated experimental data and files." [PMID:14643225]
<br>subset: PSI-MI slim
<br>synonym: "Electron Microscopy Data Bank" EXACT PSI-MI-alternate []
<br>synonym: "eMDB" EXACT PSI-MI-short []
<br>is_a: MI:0472 ! pdbe
<p><br>[Term]
<br>id: MI:0937
<br>name: glu tag
<br>def: "This peptide is a 314 to 319 amino acids fragment of the middle T antigen of mouse polymavirus. Glu-Glu epitope peptide." [PMID:8077219]
<br>subset: PSI-MI slim
<br>synonym: "glu tag" EXACT PSI-MI-short []
<br>is_a: MI:0507 ! tag
<p><br>[Term]
<br>id: MI:0938
<br>name: rheology measurement
<br>def: "Characterization of viscoelastic properties of biomolecule solution is used to infer interactions between molecules." [PMID:18445655]
<br>subset: PSI-MI slim
<br>synonym: "rheology" EXACT PSI-MI-short []
<br>is_a: MI:0013 ! biophysical
<p><br>[Term]
<br>id: MI:0939
<br>name: fluorescein label
<br>def: "Fluorescence label used to monitor the presence of a protein." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "fluorescein" EXACT PSI-MI-short []
<br>is_a: MI:0857 ! fluorescent dye label
<p><br>[Term]
<br>id: MI:0940
<br>name: fluorescein-5-maleimide label
<br>def: "Fluorescein-5-maleimide is one of the most popular fluorescent dyes for thiol modifications of proteins at cysteine residues that either are intrinsically present or result from reduction of cystines. Unlike iodoacetamides, maleimides do not react with histidines and methionines under physiological conditions." [PMID:18066077]
<br>subset: PSI-MI slim
<br>synonym: "fluor-5-maleimide" EXACT PSI-MI-short []
<br>is_a: MI:0939 ! fluorescein label
<p><br>[Term]
<br>id: MI:0941
<br>name: competitor
<br>def: "Binds to the bait molecule in competition with other prey molecules, for example at a shared binding site." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "competitor" EXACT PSI-MI-short []
<br>is_a: MI:0500 ! biological role
<p><br>[Term]
<br>id: MI:0942
<br>name: uniprot taxonomy
<br>def: "Based on NCBO Taxonomy but adapted for UniProt\nhttp://www.uniprot.org/taxonomy/" [PMID:18836194]
<br>subset: PSI-MI slim
<br>synonym: "uniprot taxon" EXACT PSI-MI-alternate []
<br>xref: id-validation-regexp:\"[0-9\]+\"
<br>xref: search-url: "http://www.uniprot.org/taxonomy/${ac}"
<br>is_a: MI:0473 ! participant database
<p><br>[Term]
<br>id: MI:0943
<br>name: detection by mass spectrometry
<br>def: "'Study of interactions by an analytical technique based on measurements of mass-to-charge ratio of charged particles in a mass spectrometer." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "ms interact detect" EXACT PSI-MI-short []
<br>is_a: MI:0013 ! biophysical
<p><br>[Term]
<br>id: MI:0944
<br>name: mass spectrometry study of hydrogen/deuterium exchange
<br>def: "Mass spectroscopy based measurement of the rate and/or extent of the hydrogen/deuterium exchange." [PMID:18948593]
<br>subset: PSI-MI slim
<br>synonym: "h1-h2 ms" EXACT PSI-MI-short []
<br>is_a: MI:0943 ! detection by mass spectrometry
<p><br>[Term]
<br>id: MI:0945
<br>name: oxidoreductase activity electron transfer assay
<br>def: "An oxidation-reduction (redox) reaction, a reversible chemical reaction in which the oxidation state of an atom or atoms within a molecule is altered. One substrate acts as a hydrogen or electron donor and becomes oxidized, while the other acts as hydrogen or electron acceptor and becomes reduced." [GO:GO\:0016491, PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "redox reaction" EXACT PSI-MI-short []
<br>is_a: MI:0414 ! enzymatic reaction
<p><br>[Term]
<br>id: MI:0946
<br>name: ping
<br>def: "Combines on-chip in-vitro protein synthesis with an in situ microfluidic affinity assay." [PMID:19098921]
<br>subset: PSI-MI slim
<br>is_a: MI:0092 ! protein in situ array
<p><br>[Term]
<br>id: MI:0947
<br>name: bead aggregation assay
<br>def: "Binding of proteins bound to beads leads to a measurable aggregation of the beads." [PMID:19114658]
<br>subset: PSI-MI slim
<br>synonym: "bead aggregation" EXACT PSI-MI-short []
<br>is_a: MI:0892 ! solid phase assay
<p><br>[Term]
<br>id: MI:0948
<br>name: kinetic conditions
<br>def: "A brief description (such as temp, pH) of the conditions under which a kinetic measurment has been performed." [PMID:14755292]
<br>subset: PSI-MI slim
<br>synonym: "kinetic_conditions" EXACT PSI-MI-short []
<br>is_a: MI:0664 ! interaction attribute name
<p><br>[Term]
<br>id: MI:0949
<br>name: gdp/gtp exchange assay
<br>def: "Experiments monitoring\ninteractions of GTP-GDP exchange factors with their cognate GTPases." [PMID:17925023]
<br>subset: PSI-MI slim
<br>synonym: "gdp_gtp exchange" EXACT PSI-MI-short []
<br>synonym: "gtp/gdp exchange" EXACT PSI-MI-alternate []
<br>synonym: "guanine nucleotide exchange assay" EXACT PSI-MI-alternate []
<br>is_a: MI:0401 ! biochemical
<p><br>[Term]
<br>id: MI:0950
<br>name: trapping mutant
<br>def: "Permits the identification of substrates of enzymes by mutating residues, usually in the active site such that the enzyme will bind but not act on its substrate." [PMID:9050838]
<br>subset: PSI-MI slim
<br>synonym: "trap-mutant" EXACT PSI-MI-short []
<br>is_a: MI:0505 ! experimental feature
<p><br>[Term]
<br>id: MI:0951
<br>name: chain parent sequence reference
<br>def: "Reference to the master sequence from which this chain has been derived." [PMID:17925023]
<br>subset: PSI-MI slim
<br>synonym: "chain-parent" EXACT PSI-MI-short []
<br>is_a: MI:0353 ! cross-reference type
<p><br>[Term]
<br>id: MI:0952
<br>name: imex secondary
<br>def: "Deprecated IMEx identifiers should be exchanged in IMEx records and stored as cross reference with this qualifier." [PMID:17925023]
<br>subset: PSI-MI slim
<br>is_a: MI:0353 ! cross-reference type
<p><br>[Term]
<br>id: MI:0953
<br>name: polymerization
<br>def: "Interaction inferred by monitoring polymerization/depolymerization of an interactor" [PMID:19081060]
<br>subset: PSI-MI slim
<br>synonym: "polymerization" EXACT []
<br>is_a: MI:0401 ! biochemical
<p><br>[Term]
<br>id: MI:0954
<br>name: curation quality
<br>def: "An assessment of the depth and extent to which a paper has been curated" [PMID:17893861]
<br>subset: PSI-MI slim
<br>relationship: part_of MI:0000 ! molecular interaction
<p><br>[Term]
<br>id: MI:0955
<br>name: curation depth
<br>def: "Assessment of the depth to which a paper has been curated" [PMID:17893861]
<br>relationship: part_of MI:0954 ! curation quality
<p><br>[Term]
<br>id: MI:0956
<br>name: curation coverage
<br>def: "Assessment to the extent of interactions captured in this paper" [PMID:17893861]
<br>subset: PSI-MI slim
<br>relationship: part_of MI:0954 ! curation quality
<p><br>[Term]
<br>id: MI:0957
<br>name: full coverage
<br>def: "All interactions which can be ascribed to an unambiguous identified in this paper have been captured." [PMID:17893861]
<br>subset: PSI-MI slim
<br>is_a: MI:0956 ! curation coverage
<p><br>[Term]
<br>id: MI:0958
<br>name: partial coverage
<br>def: "Not all interactions which can be ascribed to an unambiguous identified in this paper have been captured." [PMID:17893861]
<br>subset: PSI-MI slim
<br>is_a: MI:0956 ! curation coverage
<p><br>[Term]
<br>id: MI:0959
<br>name: imex curation
<br>def: "Paper has been curated to full IMEx specifications" [PMID:17893861]
<br>subset: PSI-MI slim
<br>is_a: MI:0955 ! curation depth
<p><br>[Term]
<br>id: MI:0960
<br>name: mimix curation
<br>def: "Paper has been curated to meet MIMIx specifications" [PMID:17687370]
<br>subset: PSI-MI slim
<br>is_a: MI:0955 ! curation depth
<p><br>[Term]
<br>id: MI:0961
<br>name: rapid curation
<br>def: "Minimal interaction data has been extracted from the paper" [PMID:17687370]
<br>subset: PSI-MI slim
<br>is_a: MI:0955 ! curation depth
<p><br>[Term]
<br>id: MI:0962
<br>name: strep ii tag
<br>def: "The protein of interest is expressed with a StrepII fusion peptide (WSHPQFEK)." [PMID:17571060]
<br>subset: PSI-MI slim
<br>synonym: "Strep (II)" EXACT PSI-MI-alternate []
<br>synonym: "Strep II" EXACT PSI-MI-alternate []
<br>is_a: MI:0507 ! tag
<p><br>[Term]
<br>id: MI:0963
<br>name: interactome parallel affinity capture
<br>def: "A specific pull down method where the protein of interest (bait) is endogenously expressed with at least two affinity tags (GFP, FLAG or others). The bait is purified in parallel using different purification protocols in contrast to tandem affinity purification (TAP) (publication currently in press)." [PMID:14681455]
<br>synonym: "ipac" EXACT PSI-MI-short []
<br>is_a: MI:0096 ! pull down
<br>created_by: orchard
<br>creation_date: 2009-05-14T10:56:21Z
<p><br>[Term]
<br>id: MI:2002
<br>name: drugbank
<br>def: "DrugBank Accession number consisting of the 4 letter prefix and a 5 number suffix. Each Accession number is unique to the drug's generic name. The 4 letter suffix (APRD, EXPT, BIOD, NUTR) indicates the type of drug (APRD=approved small molecule drug, EXPT=experimental drug, BIOD=biotech drug, NUTR=nutraceutical or natural product). Biotech drugs consist of FDA approved peptide, protein or nucleic acid drugs, approved small molecule drugs are FDA approved non-biotech drugs, nutraceuticals are natural products (amino acids, vitamins, other metabolites) and experimental drugs include drugs under trial, pre-clinical drugs, unapproved drugs, well known inhibitors and possible toxins." [PMID:14755292]
<br>subset: Drugable
<br>is_a: MI:2054 ! bioactive entity reference
<p><br>[Term]
<br>id: MI:2003
<br>name: commercial name
<br>def: "Standard name of drug or any reagent as provided by its manufacturer." [PMID:14755292]
<br>subset: Drugable
<br>synonym: "generic name" EXACT PSI-MI-alternate []
<br>is_a: MI:0300 ! alias type
<p><br>[Term]
<br>id: MI:2004
<br>name: drug brand name
<br>def: "Alternate names of the drug, brand names from different manufacturers." [PMID:14755292]
<br>subset: Drugable
<br>is_a: MI:0300 ! alias type
<p><br>[Term]
<br>id: MI:2005
<br>name: drug mixture brand name
<br>def: "Brand names and composition of mixtures that include the drug described in this DrugCard file." [PMID:14755292]
<br>subset: Drugable
<br>synonym: "mix brand name" EXACT PSI-MI-short []
<br>is_a: MI:0300 ! alias type
<p><br>[Term]
<br>id: MI:2006
<br>name: biotech product preparation
<br>def: "Description of the drug (for biotech drugs) describing its composition and/or preparation." [PMID:14755292]
<br>subset: Drugable
<br>synonym: "biotech prep" EXACT PSI-MI-short []
<br>is_a: MI:2089 ! bioactive entity attribute name
<p><br>[Term]
<br>id: MI:2007
<br>name: iupac name
<br>def: "IUPAC or standard chemical name for a drug, or a chemical." [PMID:14755292]
<br>subset: Drugable
<br>is_a: MI:0300 ! alias type
<p><br>[Term]
<br>id: MI:2008
<br>name: chemical formula
<br>def: "Chemical formula describing atomic or elemental composition" [PMID:14755292]
<br>subset: Drugable
<br>is_a: MI:2086 ! physicochemical attribute name
<p><br>[Term]
<br>id: MI:2009
<br>name: chemical structure
<br>def: "Image of the drug structure (if small molecule) or its sequence (if biotech drug)" [PMID:14755292]
<br>subset: Drugable
<br>is_a: MI:2086 ! physicochemical attribute name
<p><br>[Term]
<br>id: MI:2010
<br>name: inchi id
<br>def: "IUPAC approved InCHI identifier" [PMID:14755292]
<br>subset: Drugable
<br>is_a: MI:2091 ! structure representation attribute name
<p><br>[Term]
<br>id: MI:2011
<br>name: cas registry number
<br>def: "Chemical Abstract Service identification number" [PMID:14755292]
<br>subset: Drugable
<br>is_a: MI:2054 ! bioactive entity reference
<p><br>[Term]
<br>id: MI:2012
<br>name: kegg compound
<br>def: "Kyoto Encyclopedia of Genes and Genomes compound identification number (if molecule is in KEGG)" [PMID:14755292]
<br>subset: Drugable
<br>synonym: "KEGG Compound ID" EXACT PSI-MI-alternate []
<br>is_a: MI:0470 ! kegg
<p><br>[Term]
<br>id: MI:2013
<br>name: pubchem
<br>def: "NCBI's PubChem database identification number (if molecule is in PubChem).\nOBSOLETE as redudant with MI:0730" [PMID:14755292]
<br>subset: Drugable
<br>synonym: "PubChem ID" EXACT PSI-MI-alternate []
<br>is_obsolete: true
<p><br>[Term]
<br>id: MI:2015
<br>name: pharmgkb
<br>def: "Pharmacogenomics Knowledge Base identification number (if molecule is in PharmGKB)" [PMID:14755292]
<br>subset: Drugable
<br>is_a: MI:2054 ! bioactive entity reference
<p><br>[Term]
<br>id: MI:2016
<br>name: bind smid
<br>def: "BIND database Small Molecule Identification number (if molecule is in BIND)" [PMID:14755292]
<br>comment: May not be publicly available any more since now owned by Thompson Scientific.
<br>subset: Drugable
<br>is_a: MI:2054 ! bioactive entity reference
<p><br>[Term]
<br>id: MI:2017
<br>name: heterogen
<br>def: "The HET records are used to describe non-standard residues, such as prosthetic groups, inhibitors, solvent molecules, and ions for\nwhich coordinates are supplied. Groups are considered HET if they are: \n- not one of the standard amino acids, and \n- not one of the nucleic acids (C, G, A, T, U, and I), and \n- not one of the modified versions of nucleic acids (+C, +G, +A,\n+T, +U, and +I), and \n- not an unknown amino acid or nucleic acid where UNK is used to\nindicate the unknown residue name. \nHet records also describe heterogens for which the chemical identity is unknown, in which case the group is assigned the hetID UNK." [PMID:14755292]
<br>subset: Drugable
<br>synonym: "het" EXACT PSI-MI-short []
<br>is_a: MI:0460 ! rcsb pdb
<br>is_a: MI:0472 ! pdbe
<br>is_a: MI:0806 ! pdbj
<p><br>[Term]
<br>id: MI:2020
<br>name: canadian drug identification number
<br>def: "Drug Identification Number (Canadian Drug ID system)" [PMID:14755292]
<br>subset: Drugable
<br>synonym: "din" EXACT PSI-MI-short []
<br>is_a: MI:2054 ! bioactive entity reference
<p><br>[Term]
<br>id: MI:2021
<br>name: rxlist link
<br>def: "Hyperlink to RxList entry for the given drug (if it exists)" [PMID:14755292]
<br>subset: Drugable
<br>is_a: MI:2054 ! bioactive entity reference
<p><br>[Term]
<br>id: MI:2023
<br>name: material safety data sheet
<br>def: "Material Safety Data Sheet (if it exists). A Material Safety Data Sheet (MSDS) is designed to provide both workers and emergency personnel with the proper procedures for handling or working with a particular substance. MSDS's include information such as physical data (melting point, boiling point, flash point etc.), toxicity, health effects, first aid, reactivity, storage, disposal, protective equipment, andspill/leak procedures. These are of particular use if a spill or other accident occurs." [PMID:14755292]
<br>subset: Drugable
<br>synonym: "msds" EXACT PSI-MI-short []
<br>synonym: "MSDS Material Safety Sheet" EXACT PSI-MI-alternate []
<br>is_a: MI:2086 ! physicochemical attribute name
<p><br>[Term]
<br>id: MI:2024
<br>name: patent number
<br>def: "number of the patent describing a drug's synthesis or use." [PMID:14755292]
<br>subset: Drugable
<br>is_a: MI:0353 ! cross-reference type
<p><br>[Term]
<br>id: MI:2025
<br>name: molecular weight
<br>def: "Molecular weight in g/mol, determined from molecular formula or sequence." [PMID:14755292]
<br>subset: Drugable
<br>is_a: MI:0640 ! parameter type
<p><br>[Term]
<br>id: MI:2026
<br>name: melting point
<br>def: "The melting point of a solid is the temperature range at which it changes state from solid to liquid." [PMID:14755292]
<br>subset: Drugable
<br>is_a: MI:0640 ! parameter type
<p><br>[Term]
<br>id: MI:2027
<br>name: water solubility
<br>def: "Water solubility in mg/mL or g/L" [PMID:14755292]
<br>subset: Drugable
<br>synonym: "logSw" EXACT PSI-MI-alternate []
<br>is_a: MI:2160 ! logs
<p><br>[Term]
<br>id: MI:2029
<br>name: logp
<br>def: "Water/octanol partition coefficient  of a small molecule." [PMID:14755292]
<br>subset: Drugable
<br>is_a: MI:0640 ! parameter type
<p><br>[Term]
<br>id: MI:2030
<br>name: isoelectric point
<br>def: "The isoelectric point (pI) is the pH at which a particular molecule or surface carries no net electrical charge. For an amino acid with only one amine and one carboxyl group, the pI can be calculated from the pKas of this molecule." [PMID:14755292]
<br>subset: Drugable
<br>is_a: MI:0640 ! parameter type
<p><br>[Term]
<br>id: MI:2033
<br>name: hydrophobicity
<br>def: "Physical property of a molecule (known as a hydrophobe) that is repelled from a mass of water. Gravy score." [PMID:14755292]
<br>subset: Drugable
<br>is_a: MI:0640 ! parameter type
<p><br>[Term]
<br>id: MI:2036
<br>name: boiling point
<br>def: "The boiling point of a liquid is the temperature at which the vapor pressure of the liquid equals the environmental pressure surrounding the liquid. A liquid in a vacuum environment has a lower boiling point than when the liquid is at atmospheric pressure. A liquid in a high pressure environment has a higher boiling point than when the liquid is at atmospheric pressure. In other words, the boiling point of liquids varies with and depends upon the surrounding environmental pressure." [PMID:14755292]
<br>subset: Drugable
<br>is_a: MI:0640 ! parameter type
<p><br>[Term]
<br>id: MI:2039
<br>name: smiles string
<br>def: "SMILES string corresponding to drug structure" [PMID:14755292]
<br>subset: Drugable
<br>is_a: MI:2091 ! structure representation attribute name
<p><br>[Term]
<br>id: MI:2040
<br>name: drug type
<br>def: "Type of drug (approved, experimental, biotech, nutraceutical)" [PMID:14755292]
<br>subset: Drugable
<br>is_a: MI:2089 ! bioactive entity attribute name
<p><br>[Term]
<br>id: MI:2041
<br>name: drug category
<br>def: "Therapeutic category or general category of drug (anti-convulsant, antibacterial, etc.)." [PMID:14755292]
<br>subset: Drugable
<br>is_a: MI:2089 ! bioactive entity attribute name
<p><br>[Term]
<br>id: MI:2042
<br>name: disease indication
<br>def: "Description or common names of diseases that the drug is used to treat." [PMID:14755292]
<br>comment: Source of further terms could be MeSH term or SNOWMAN.
<br>subset: Drugable
<br>is_a: MI:2089 ! bioactive entity attribute name
<p><br>[Term]
<br>id: MI:2043
<br>name: pharmacology
<br>def: "Text description of how the drug works at a clinical or physiological level." [PMID:14755292]
<br>subset: Drugable
<br>is_a: MI:2089 ! bioactive entity attribute name
<p><br>[Term]
<br>id: MI:2044
<br>name: mechanism of action
<br>def: "Description of how the drug works or what it binds to at a molecular level." [PMID:14755292]
<br>subset: Drugable
<br>is_a: MI:2089 ! bioactive entity attribute name
<p><br>[Term]
<br>id: MI:2045
<br>name: drug absorption
<br>def: "Determination of how quickly and how much of a drug reaches its intended target (site) of action." [PMID:14755292]
<br>subset: Drugable
<br>is_a: MI:0640 ! parameter type
<p><br>[Term]
<br>id: MI:2046
<br>name: lethal dose 50
<br>def: "The LD50 is the dose that kills half (50%) of the animals tested" [PMID:14755292]
<br>subset: Drugable
<br>synonym: "ld50" EXACT PSI-MI-alternate []
<br>synonym: "lethal dose 50 %" EXACT PSI-MI-alternate []
<br>is_a: MI:0640 ! parameter type
<p><br>[Term]
<br>id: MI:2047
<br>name: percentage of plasma protein binding
<br>def: "Percentage of the drug that is bound in plasma proteins" [PMID:14755292]
<br>subset: Drugable
<br>synonym: "plasma prot binding" EXACT PSI-MI-short []
<br>synonym: "protein binding %" EXACT PSI-MI-alternate []
<br>is_a: MI:0640 ! parameter type
<p><br>[Term]
<br>id: MI:2048
<br>name: drug biotransformation
<br>def: "The chemical conversion of drugs to other compounds in the body, excluding degradation due to any inherent chemical instability of drugs in biological media." [PMID:14755292]
<br>subset: Drugable
<br>synonym: "drug metabolism" EXACT PSI-MI-short []
<br>is_a: MI:2115 ! pharmacokinetics attribute name
<p><br>[Term]
<br>id: MI:2049
<br>name: elimination half life
<br>def: "Rate The time it takes for the body to eliminate or breakdown half of a dose of a pharmacologic agent, in practice the time taken for plasma concentration to reduce by 50%." [PMID:14755292]
<br>subset: Drugable
<br>synonym: "distribution halflife" EXACT PSI-MI-alternate []
<br>synonym: "elimin half life" EXACT PSI-MI-short []
<br>synonym: "t1/2" EXACT PSI-MI-alternate []
<br>is_a: MI:0640 ! parameter type
<p><br>[Term]
<br>id: MI:2050
<br>name: dosage form
<br>def: "How the drug is dispensed (tablets, capsules, solutions), packing material." [PMID:14755292]
<br>subset: Drugable
<br>is_a: MI:2089 ! bioactive entity attribute name
<p><br>[Term]
<br>id: MI:2051
<br>name: patient information
<br>def: "Information on the disease indications and treatment regime for the drug. May also include contra-indications." [PMID:14755292]
<br>subset: Drugable
<br>is_a: MI:2089 ! bioactive entity attribute name
<p><br>[Term]
<br>id: MI:2053
<br>name: contraindications
<br>def: "Cautions or conditions indicating why or when the drug should not be taken or prescribed." [PMID:14755292]
<br>subset: Drugable
<br>is_a: MI:2089 ! bioactive entity attribute name
<p><br>[Term]
<br>id: MI:2054
<br>name: bioactive entity reference
<br>def: "General on-line reference to other details about a drug or other bioactive entity." [PMID:14755292]
<br>subset: Drugable
<br>subset: PSI-MI slim
<br>synonym: "bioactive entity ref" EXACT PSI-MI-short []
<br>is_a: MI:0473 ! participant database
<p><br>[Term]
<br>id: MI:2055
<br>name: chemical stability
<br>def: "chemical stability occurs when a substance is in a (dynamic) chemical equilibrium with its environment. In this well-defined state, the substance is expected to persist indefinitely (assuming that the environment does not change).  A substance which is not chemically stable (yet exists) is metastable or kinetically persistent." [PMID:14755292]
<br>subset: Drugable
<br>synonym: "thermodynamic stability" EXACT PSI-MI-alternate []
<br>is_a: MI:2086 ! physicochemical attribute name
<p><br>[Term]
<br>id: MI:2064
<br>name: solubility
<br>def: "Potential ability of a substance to dissolve in a liquid." [PMID:14755292]
<br>subset: Drugable
<br>synonym: "dt theoretical pi" EXACT PSI-MI-short []
<br>is_a: MI:0640 ! parameter type
<p><br>[Term]
<br>id: MI:2084
<br>name: organisms affected
<br>def: "Names of organisms which are affected, positively or negatively, by the drug." [PMID:14755292]
<br>subset: Drugable
<br>is_a: MI:2089 ! bioactive entity attribute name
<p><br>[Term]
<br>id: MI:2086
<br>name: physicochemical attribute name
<br>def: "Chemical and physical properties of a molecule." [PMID:14755292]
<br>subset: Drugable
<br>synonym: "physicochemical att" EXACT PSI-MI-short []
<br>is_a: MI:0590 ! attribute name
<p><br>[Term]
<br>id: MI:2089
<br>name: bioactive entity attribute name
<br>def: "Properties of a chemical tested or used as a drug, herbicide, insecticide etc." [PMID:14755292]
<br>subset: Drugable
<br>synonym: "bioactive entity att" EXACT PSI-MI-short []
<br>is_a: MI:0590 ! attribute name
<p><br>[Term]
<br>id: MI:2091
<br>name: structure representation attribute name
<br>def: "Human artefact to describe and report the structure of a molecule." [PMID:14755292]
<br>subset: Drugable
<br>synonym: "struc representation" EXACT PSI-MI-short []
<br>is_a: MI:0590 ! attribute name
<p><br>[Term]
<br>id: MI:2097
<br>name: anti-convulsant
<br>def: "Therapeutic category or general category of drug -anti-convulsant" [PMID:14755292]
<br>subset: Drugable
<br>is_a: MI:2041 ! drug category
<p><br>[Term]
<br>id: MI:2098
<br>name: anti-bacterial
<br>def: "Therapeutic category or general category of drug -anti-bacterial" [PMID:14755292]
<br>subset: Drugable
<br>is_a: MI:2041 ! drug category
<p><br>[Term]
<br>id: MI:2099
<br>name: fda approved drug
<br>def: "A drug licensed for sale in the USA by the FDA." [PMID:14755292]
<br>subset: Drugable
<br>is_a: MI:2040 ! drug type
<p><br>[Term]
<br>id: MI:2100
<br>name: experimental drug
<br>def: "A drug which has yet to be formally approved for the indication which it is currently being used to treat." [PMID:14755292]
<br>subset: Drugable
<br>is_a: MI:2040 ! drug type
<p><br>[Term]
<br>id: MI:2101
<br>name: biotech drug
<br>def: "A natural product, such as a protein or peptide, which is produced used biotechnology as a drug." [PMID:14755292]
<br>subset: Drugable
<br>is_a: MI:2040 ! drug type
<p><br>[Term]
<br>id: MI:2102
<br>name: nutraceutical drug
<br>def: "A drug which may also be regarded as a foodstuff." [PMID:14755292]
<br>subset: Drugable
<br>is_a: MI:2040 ! drug type
<p><br>[Term]
<br>id: MI:2105
<br>name: pka
<br>def: "Negative decimal logarithm of Ka, acid dissociation equilibrium constant for the dissociation of a weak acid." [PMID:14755292]
<br>comment: Quantitative prediction of this parameter is possible.
<br>subset: Drugable
<br>is_a: MI:0640 ! parameter type
<p><br>[Term]
<br>id: MI:2106
<br>name: degree of ionisation ph 7.4
<br>def: "The degree of ionization refers to the proportion of neutral particles such as those in a gas or aqueous solution, that are ionized into charged particles. A low degree of ionization is sometimes called partially ionized, and a very high degree of ionization as fully ionized. This measurment is performed at pH 7.4" [PMID:14755292]
<br>comment: Quantitative prediction of this parameter is possible.
<br>subset: Drugable
<br>synonym: "ionisation ph 7.4" EXACT PSI-MI-short []
<br>is_a: MI:0640 ! parameter type
<p><br>[Term]
<br>id: MI:2107
<br>name: logd
<br>def: "The LogD is the ratio of the equilibrium concentrations of all species (unionized and ionized) of a molecule in octanol to same species in the water phase at a given temperature, normally 25 C. It differs from LogP in that ionized species are considered as well as the neutral form of the molecule.pH 7.4" [PMID:14755292]
<br>comment: Quantitative prediction of this parameter is possible.
<br>subset: Drugable
<br>is_a: MI:0640 ! parameter type
<p><br>[Term]
<br>id: MI:2108
<br>name: solubility ph 7.4
<br>def: "Solubility pH 7.4" [PMID:14755292]
<br>comment: Quantitative prediction of this parameter is possible.
<br>subset: Drugable
<br>is_a: MI:2064 ! solubility
<p><br>[Term]
<br>id: MI:2109
<br>name: solubility in dmso
<br>def: "Solubility in DMSO" [PMID:14755292]
<br>subset: Drugable
<br>is_a: MI:2064 ! solubility
<p><br>[Term]
<br>id: MI:2111
<br>name: diffusion coefficient
<br>def: "Diffusion coefficient D is proportional to the velocity of the diffusing particles, which depends on the temperature, viscosity of the fluid and the size of the particles according to the Stokes-Einstein relation. In dilute aqueous solutions the diffusion coefficients of most ions are similar and have values that at room temperature are in the range of 0.6x10-9 to 2x10-9 m2/s. For biological molecules the diffusion coefficients normally range from 10-11 to 10-10 m2/s." [PMID:14755292]
<br>comment: Quantitative prediction of this parameter is possible.
<br>subset: Drugable
<br>synonym: "diffusion coeff" EXACT PSI-MI-short []
<br>is_a: MI:0640 ! parameter type
<p><br>[Term]
<br>id: MI:2112
<br>name: chemical stability at pH 2
<br>def: "Chemical stability at pH 2" [PMID:14755292]
<br>comment: Qualitative prediction of this parameter is possible.
<br>subset: Drugable
<br>synonym: "chem stab ph 2" EXACT PSI-MI-short []
<br>is_a: MI:2055 ! chemical stability
<p><br>[Term]
<br>id: MI:2113
<br>name: dissolution profile
<br>def: "The rate of dissolution is a key target for controlling the duration of a drug's effect, and as such, several dosage forms that contain the same active ingredient may be available, differing only in the rate of dissolution. If a drug is supplied in a form that is not readily dissolved, the drug may be released more gradually over time with a longer duration of action. Having a longer duration of action may improve compliance since the medication will not have to be taken as often. Additionally, slow-release dosage forms may maintain concentrations within an acceptable therapeutic range over a long period of time, as opposed to quick-release dosage forms which may result in sharper peaks and troughs in serum concentrations." [PMID:14755292]
<br>comment: The prediction of the value for this paramter is currently not possible.
<br>subset: Drugable
<br>is_a: MI:0640 ! parameter type
<p><br>[Term]
<br>id: MI:2115
<br>name: pharmacokinetics attribute name
<br>def: "Determination of the fate of substances administered externally to a living organism i.e. the study of what the body does to a drug." [PMID:14755292]
<br>subset: Drugable
<br>is_a: MI:2086 ! physicochemical attribute name
<p><br>[Term]
<br>id: MI:2116
<br>name: cell permeability
<br>def: "The permitting or activating of the passage of substances into, out of, or through cells." [PMID:14755292]
<br>comment: Quantitative prediction of this parameter is possible.
<br>subset: Drugable
<br>is_a: MI:0640 ! parameter type
<p><br>[Term]
<br>id: MI:2118
<br>name: volume of distribution
<br>def: "The Volume of Distribution is the amount of drug in the body divided by the concentration in the blood. Drugs that are highly lipid soluble have a very high volume of distribution (500 litres). Drugs which are lipid insoluble remain in the blood, and have a low Vd." [PMID:14755292]
<br>comment: Quantitative prediction of this parameter is possible.
<br>subset: Drugable
<br>synonym: "distribution volume" EXACT PSI-MI-alternate []
<br>synonym: "vd" EXACT PSI-MI-alternate []
<br>synonym: "vol of distribution" EXACT PSI-MI-short []
<br>is_a: MI:0640 ! parameter type
<p><br>[Term]
<br>id: MI:2120
<br>name: tissue distribution
<br>def: "Accumulation of a drug or chemical substance in various organs (including those not relevant to its pharmacologic or therapeutic action). This distribution depends on the blood flow or perfusion rate of the organ, the ability of the drug to penetrate organ membranes, tissue specificity, protein binding. The distribution is usually expressed as tissue to plasma ratios." [PMID:14755292]
<br>comment: Algorithms have been published to predict the value of this parameter but the quality of the prediction is unknown.
<br>subset: Drugable
<br>is_a: MI:0640 ! parameter type
<p><br>[Term]
<br>id: MI:2121
<br>name: transporter binding
<br>def: "Substrate of a carrier system allowing the intake of an agent into an organ or part of body." [PMID:14755292]
<br>comment: The prediction of the value for this parameter is currently not possible.
<br>subset: Drugable
<br>is_a: MI:2115 ! pharmacokinetics attribute name
<p><br>[Term]
<br>id: MI:2122
<br>name: clearance
<br>def: "The ratio of excretion is or measure of the speed at which a constituent is lost from the body." [PMID:14755292]
<br>subset: Drugable
<br>is_a: MI:0640 ! parameter type
<p><br>[Term]
<br>id: MI:2123
<br>name: renal clearance
<br>def: "The renal clearance ratio or fractional excretion is a measure of the speed at which a constituent of urine passes through the kidneys, in this context the rate at which a pharmacological agent is lost from the body via urine." [PMID:14755292]
<br>comment: Algorithms have been published to predict the value of this parameter but the quality of the prediction is unknown.
<br>subset: Drugable
<br>is_a: MI:0640 ! parameter type
<p><br>[Term]
<br>id: MI:2124
<br>name: total clearance
<br>def: "The clearance of a drug is the volume of plasma from which the drug is completely removed per unit time. The amount eliminated is proportional to the concentration of the drug in the blood." [PMID:14755292]
<br>comment: The prediction of the value for this paramter is currently not possible.
<br>subset: Drugable
<br>synonym: "cl" EXACT PSI-MI-alternate []
<br>synonym: "clearance" EXACT PSI-MI-alternate []
<br>is_a: MI:0640 ! parameter type
<p><br>[Term]
<br>id: MI:2125
<br>name: maximum absorbable dose
<br>def: "The Maximum Absorbable Dose (MAD) represents the amount of drug that can permeate across a barrier." [PMID:8987073]
<br>comment: Quantitative prediction of this parameter is possible.
<br>subset: Drugable
<br>synonym: "mad" EXACT PSI-MI-short []
<br>is_a: MI:0640 ! parameter type
<p><br>[Term]
<br>id: MI:2126
<br>name: paracellular absorption
<br>def: "Water soluble compounds are absorbed in the small intestine mainly via two pathways, the transcellular and the paracellular pathways. The paracellular absorption involves movement of solutes through a restrictive aqueous channel in the tight junctions of adjoining cells  by diffusion." [PMID:14755292]
<br>comment: Quantitative prediction of this parameter is possible.
<br>subset: Drugable
<br>synonym: "paracellular absorp" EXACT PSI-MI-short []
<br>is_a: MI:0640 ! parameter type
<p><br>[Term]
<br>id: MI:2127
<br>name: tmax/cmax
<br>def: "Ratio between the time value at Cmax (maximum concentration) in a dose response curve, and the Cmax value itself." [PMID:14755292]
<br>comment: The prediction of the value for this paramter is currently not possible.
<br>subset: Drugable
<br>is_a: MI:0640 ! parameter type
<p><br>[Term]
<br>id: MI:2128
<br>name: ABCB1 transporter substrate
<br>def: "Substrate for the representitive member of the ABC transprorter family ABCB1 (MDR1, pgy1, P08183). ABC transporters preventing uptake or facilitating clearance of toxic substances, playing an important role in drug excretion through the bile." [PMID:14755292]
<br>comment: Algorithms have been published to predict the value of this parameter but the quality of the prediction is unknown.
<br>subset: Drugable
<br>synonym: "abcb1 substrate" EXACT PSI-MI-short []
<br>synonym: "pgp(mdr1) substrate " EXACT PSI-MI-alternate []
<br>is_a: MI:2121 ! transporter binding
<p><br>[Term]
<br>id: MI:2129
<br>name: bile transporter substrate
<br>def: "Substrate of the bile acid carrier system in both the intestinal tract and the liver. System catalyses of the transfer of bile acid from one side of the membrane to the other. Bile acids are any of a group of steroid carboxylic acids occurring in bile, where they are present as the sodium salts of their amides with glycine or taurine." [PMID:14755292]
<br>comment: The prediction of the value for this paramater is currently not possible.
<br>subset: Drugable
<br>synonym: "bile trans substrate" EXACT PSI-MI-short []
<br>is_a: MI:2121 ! transporter binding
<p><br>[Term]
<br>id: MI:2130
<br>name: cyp-450 inhibition
<br>def: "Inhibitor of one or more of the family of cytochrome p450 enzymes, probably the most important elements of oxidative metabolism of exogenous compounds." [PMID:14755292]
<br>comment: Algorithms have been published to predict the value of this parameter but the quality of the prediction is unknown.
<br>subset: Drugable
<br>synonym: "cyp-450 inhibition" EXACT PSI-MI-alternate []
<br>synonym: "Cytochrome P450 inhibition" EXACT PSI-MI-alternate []
<br>is_a: MI:2135 ! toxicity attribute name
<p><br>[Term]
<br>id: MI:2131
<br>name: metabolite identification
<br>def: "Identification of the breakdown products of a substance, either through chemical instability or the actions of enzymes within the body" [PMID:14755292]
<br>comment: Algorithms have been published to predict the value of this parameter but the quality of the prediction is unknown.
<br>subset: Drugable
<br>synonym: "metabolite identific" EXACT PSI-MI-short []
<br>is_a: MI:2115 ! pharmacokinetics attribute name
<p><br>[Term]
<br>id: MI:2132
<br>name: gsh adducts
<br>def: "Derivative molecule which has formed from a reaction with glutathione." [PMID:14755292]
<br>comment: Algorithms have been published to predict the value of this parameter but the quality of the prediction is unknown.
<br>subset: Drugable
<br>is_a: MI:2115 ! pharmacokinetics attribute name
<p><br>[Term]
<br>id: MI:2133
<br>name: neutralization by glucuronidation or sulfatation
<br>def: "Neutralization of a compound occuring via its glucuronidation or sulfatation." [PMID:14755292]
<br>comment: Quantitative prediction of this parameter is possible.
<br>subset: Drugable
<br>synonym: "neutraliz gluc/sulf" EXACT PSI-MI-short []
<br>is_a: MI:2048 ! drug biotransformation
<p><br>[Term]
<br>id: MI:2135
<br>name: toxicity attribute name
<br>def: "The mechanism by which a substance can harm humans or animals." [PMID:14755292]
<br>subset: Drugable
<br>is_a: MI:2089 ! bioactive entity attribute name
<p><br>[Term]
<br>id: MI:2136
<br>name: herg binding
<br>def: "Binds to the hERG (human Ether-a-go-go Related Gene) (Q12809) which encodes the Kv11.1 potassium ion channel responsible for the repolarizing IKr current in the cardiac action potential." [PMID:14755292]
<br>comment: Algorithms have been published to predict the value of this parameter but the quality of the prediction is unknown.
<br>subset: Drugable
<br>is_a: MI:2135 ! toxicity attribute name
<p><br>[Term]
<br>id: MI:2137
<br>name: genotoxicity
<br>def: "Tendency of a bioactive entity to induce damage at the level of the gene." [PMID:14755292]
<br>comment: Algorithms have been published to predict the value of this parameter but the quality of the prediction is unknown.
<br>subset: Drugable
<br>is_a: MI:2135 ! toxicity attribute name
<p><br>[Term]
<br>id: MI:2138
<br>name: mutagenicity
<br>def: "Tendency of a bioactive entity to induce genetic mutations at the nucleotide level e.g. substitution of nucleotide base-pairs and insertions and deletions of one or more nucleotides in DNA sequences." [PMID:14755292]
<br>comment: Algorithms have been published to predict the value of this parameter but the quality of the prediction is unknown.
<br>subset: Drugable
<br>is_a: MI:2135 ! toxicity attribute name
<p><br>[Term]
<br>id: MI:2139
<br>name: carcinogenicity
<br>def: "Tendency of a bioactive entity to induce a cancer." [PMID:14755292]
<br>comment: Algorithms have been published to predict the value of this parameter but the quality of the prediction is unknown.
<br>subset: Drugable
<br>synonym: "cancerogenicity" EXACT PSI-MI-alternate []
<br>is_a: MI:2135 ! toxicity attribute name
<p><br>[Term]
<br>id: MI:2140
<br>name: chromosome damage
<br>def: "Tendency of a bioactive entity to induce damage at the level of the chromosome e.g. induce a change in chromosome structure and number." [PMID:14755292]
<br>comment: Algorithms have been published to predict the value of this parameter but the quality of the prediction is unknown.
<br>subset: Drugable
<br>is_a: MI:2135 ! toxicity attribute name
<p><br>[Term]
<br>id: MI:2141
<br>name: hepatotoxicity
<br>def: "Tendency of a bioactive entity to affect liver function." [PMID:14755292]
<br>comment: Algorithms have been published to predict the value of this parameter but the quality of the prediction is unknown.
<br>subset: Drugable
<br>is_a: MI:2135 ! toxicity attribute name
<p><br>[Term]
<br>id: MI:2142
<br>name: phospholipidosis
<br>def: "Causes excess phospholipids to accumulate within cells." [PMID:14755292]
<br>comment: Algorithms have been published to predict the value of this parameter but the quality of the prediction is unknown.
<br>subset: Drugable
<br>is_a: MI:2135 ! toxicity attribute name
<p><br>[Term]
<br>id: MI:2145
<br>name: solubility ph 6.5
<br>def: "Solubility pH  6.5." [PMID:14755292]
<br>comment: Quantitative prediction of this parameter is possible.
<br>subset: Drugable
<br>is_a: MI:2064 ! solubility
<p><br>[Term]
<br>id: MI:2146
<br>name: solubility ph 2.0
<br>def: "Solubility pH 2.0" [PMID:14755292]
<br>comment: Quantitative prediction of this parameter is possible.
<br>subset: Drugable
<br>synonym: "solubility ph2.0" EXACT PSI-MI-alternate []
<br>is_a: MI:2064 ! solubility
<p><br>[Term]
<br>id: MI:2147
<br>name: chemical stability at pH 7.4
<br>def: "Chemical stabilityat at pH 7.4" [PMID:14755292]
<br>comment: Qualitative prediction of this parameter is possible.
<br>subset: Drugable
<br>synonym: "chem stab ph 7.4" EXACT PSI-MI-short []
<br>is_a: MI:2055 ! chemical stability
<p><br>[Term]
<br>id: MI:2148
<br>name: investigational drug
<br>def: "A drug currently under clinical development." [PMID:14755292]
<br>comment: Intact.
<br>subset: Drugable
<br>is_a: MI:2040 ! drug type
<p><br>[Term]
<br>id: MI:2149
<br>name: withdrawn drug
<br>def: "A drug for which the licencing for prescriptive use has been withdrawn." [PMID:14755292]
<br>comment: Intact.
<br>subset: Drugable
<br>is_a: MI:2040 ! drug type
<p><br>[Term]
<br>id: MI:2150
<br>name: illicit drug
<br>def: "A drug which has not been approved for sale, a drug taken for recreational purposes or a licensed drug sold without a prescription." [PMID:14755292]
<br>subset: Drugable
<br>is_a: MI:2040 ! drug type
<p><br>[Term]
<br>id: MI:2151
<br>name: other drug interaction
<br>def: "Effect of additional drug treatments on a given drug action." [PMID:14755292]
<br>subset: Drugable
<br>synonym: "drug interaction" EXACT PSI-MI-short []
<br>is_a: MI:2089 ! bioactive entity attribute name
<p><br>[Term]
<br>id: MI:2152
<br>name: food interaction
<br>def: "Effect of food ingestion on a given drug treatment." [PMID:14755292]
<br>comment: IntAct MeSH term or SNOWMAN.
<br>subset: Drugable
<br>is_a: MI:2089 ! bioactive entity attribute name
<p><br>[Term]
<br>id: MI:2153
<br>name: pdr health
<br>def: "Hyperlink to PDRhealth entry for the given drug (if it exists)" [PMID:14755292]
<br>subset: Drugable
<br>synonym: "PDRhealth" EXACT PSI-MI-alternate []
<br>is_a: MI:2054 ! bioactive entity reference
<p><br>[Term]
<br>id: MI:2154
<br>name: wikipedia
<br>def: "Hyperlink to wikipedia entry for the given drug (if it exists)" [PMID:14755292]
<br>subset: Drugable
<br>is_a: MI:2054 ! bioactive entity reference
<p><br>[Term]
<br>id: MI:2155
<br>name: average molecular weight
<br>def: "Molecular weight in g/mol, determined from molecular formula or sequence." [PMID:14755292]
<br>subset: Drugable
<br>synonym: "avrg mol weight" EXACT PSI-MI-short []
<br>is_a: MI:2025 ! molecular weight
<p><br>[Term]
<br>id: MI:2156
<br>name: monoisotopic molecular weight
<br>def: "Molecular weight in g/mol, determined from molecular formula or sequence." [PMID:14755292]
<br>subset: Drugable
<br>synonym: "monoisotopic mol wgt" EXACT PSI-MI-short []
<br>is_a: MI:2025 ! molecular weight
<p><br>[Term]
<br>id: MI:2157
<br>name: experimental water solubility
<br>def: "Water solubility in mg/mL or g/L." [PMID:14755292]
<br>subset: Drugable
<br>synonym: "exp h2o solubilty" EXACT PSI-MI-short []
<br>synonym: "experimental h2o solubility" EXACT PSI-MI-alternate []
<br>is_a: MI:2027 ! water solubility
<br>is_a: MI:2161 ! experimental logs
<p><br>[Term]
<br>id: MI:2158
<br>name: predicted water solubility
<br>def: "Water solubility in mg/mL or g/L" [PMID:14755292]
<br>subset: Drugable
<br>synonym: "predicted h2o solub" EXACT PSI-MI-short []
<br>synonym: "predicted h2o solubility" EXACT PSI-MI-alternate []
<br>is_a: MI:2027 ! water solubility
<p><br>[Term]
<br>id: MI:2160
<br>name: logs
<br>def: "Solubility of a molecule in a given solvant." [PMID:14755292]
<br>subset: Drugable
<br>synonym: "logS" EXACT PSI-MI-alternate []
<br>is_a: MI:0640 ! parameter type
<p><br>[Term]
<br>id: MI:2161
<br>name: experimental logs
<br>def: "Experimental derived value for the solubility of a molecule in a given solvant." [PMID:14755292]
<br>comment: Quantitative prediction of this parameter is possible.
<br>subset: Drugable
<br>synonym: "experimental logS" EXACT PSI-MI-alternate []
<br>is_a: MI:2160 ! logs
<p><br>[Term]
<br>id: MI:2162
<br>name: experimental CaCO2 permeability
<br>def: "Experimentally derived value for ability of a  compound to cross epithelial and endothelial cell barriers Using the CaCo2 cell line derived from a human colorectal adenocarcinoma. Used as an in vitro permeability models to predict human intestinal absorption" [PMID:14755292]
<br>comment: Quantitative prediction of this parameter is possible.
<br>subset: Drugable
<br>synonym: "caco2 permeability" EXACT PSI-MI-short []
<br>is_a: MI:0640 ! parameter type
<p><br>[Term]
<br>id: MI:2163
<br>name: by homology
<br>def: "Reference assigned to a molecule by sequence homology with another similar sequence." [PMID:14755292]
<br>subset: Drugable
<br>is_a: MI:0353 ! cross-reference type
<p><br>[Typedef]
<br>id: part_of
<br>name: part of
<br>is_transitive: true


<ul>
	<a href="./index.html">Home</a> | <a href="./about.html">About</a>
</ul>